<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="" xml:lang="">
<head>
<title>4b78c8d89b22cb315d90fd1274945de5-html.html</title>

<meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
 <br/>
<style type="text/css">
<!--
	p {margin: 0; padding: 0;}	.ft10{font-size:13px;font-family:Times;color:#cedde4;}
	.ft11{font-size:10px;font-family:Times;color:#000000;}
	.ft12{font-size:10px;font-family:Times;color:#000000;}
	.ft13{font-size:10px;font-family:Times;color:#000000;}
	.ft14{font-size:13px;font-family:Times;color:#000000;}
	.ft15{font-size:37px;font-family:Times;color:#000000;}
	.ft16{font-size:13px;font-family:Times;color:#000000;}
	.ft17{font-size:7px;font-family:Times;color:#000000;}
	.ft18{font-size:11px;font-family:Times;color:#000000;}
	.ft19{font-size:11px;font-family:Times;color:#000000;}
	.ft110{font-size:6px;font-family:Times;color:#0000ff;}
	.ft111{font-size:6px;font-family:Times;color:#000000;}
	.ft112{font-size:10px;font-family:Times;color:#0000ff;}
	.ft113{font-size:25px;font-family:Times;color:#f58220;}
	.ft114{font-size:37px;line-height:55px;font-family:Times;color:#000000;}
	.ft115{font-size:11px;line-height:16px;font-family:Times;color:#000000;}
-->
</style>
</head>
<body bgcolor="#A0A0A0" vlink="blue" link="blue">
<div id="page1-div" style="position:relative;width:892px;height:1173px;">
<p style="position:absolute;top:1132px;left:825px;white-space:nowrap" class="ft10">1</p>
<p style="position:absolute;top:1135px;left:61px;white-space:nowrap" class="ft11"><b>Scientific</b>&#160;<b>RepoRtS</b>&#160;|&#160;</p>
<p style="position:absolute;top:1135px;left:192px;white-space:nowrap" class="ft13"><i>&#160; &#160; &#160; &#160; (2019) 9:18612&#160;</i>&#160;|&#160;<a href="https://doi.org/10.1038/s41598-019-54859-8">https://doi.org/10.1038/s41598-019-54859-8</a></p>
<p style="position:absolute;top:19px;left:585px;white-space:nowrap" class="ft14">www.nature.com/scientificreports</p>
<p style="position:absolute;top:220px;left:233px;white-space:nowrap" class="ft15"><b>Modeling&#160;somatic computation&#160;</b></p>
<p style="position:absolute;top:265px;left:233px;white-space:nowrap" class="ft15"><b>with non-neural bioelectric&#160;</b></p>
<p style="position:absolute;top:310px;left:233px;white-space:nowrap" class="ft114"><b>networks<br/>Santosh&#160;Manicka&#160;&amp; Michael&#160;Levin<a href="http://orcid.org/0000-0001-7292-8084">&#160;&#160;&#160;</a></b><b>*</b></p>
<p style="position:absolute;top:397px;left:233px;white-space:nowrap" class="ft115"><b>The&#160;field&#160;of&#160;basal&#160;cognition&#160;seeks&#160;to&#160;understand&#160;how&#160;adaptive,&#160;context-specific&#160;behavior&#160;occurs&#160;in&#160;non-<br/>neural&#160;biological systems.&#160;embryogenesis&#160;and&#160;regeneration&#160;require&#160;plasticity in many&#160;tissue types to&#160;<br/>achieve&#160;structural&#160;and&#160;functional&#160;goals&#160;in&#160;diverse&#160;circumstances.&#160;Thus,&#160;advances&#160;in&#160;both&#160;evolutionary&#160;<br/>cell&#160;biology&#160;and&#160;regenerative&#160;medicine&#160;require&#160;an&#160;understanding&#160;of&#160;how&#160;non-neural&#160;tissues&#160;could&#160;<br/>process&#160;information.&#160;Neurons&#160;evolved&#160;from&#160;ancient&#160;cell&#160;types&#160;that&#160;used&#160;bioelectric&#160;signaling&#160;to&#160;perform&#160;<br/>computation.&#160;However,&#160;it&#160;has&#160;not&#160;been&#160;shown&#160;whether&#160;or&#160;how&#160;non-neural&#160;bioelectric&#160;cell&#160;networks&#160;can&#160;<br/>support&#160;computation.&#160;We&#160;generalize&#160;connectionist&#160;methods&#160;to&#160;non-neural&#160;tissue&#160;architectures,&#160;showing&#160;<br/>that&#160;a&#160;minimal&#160;non-neural&#160;Bio-Electric&#160;Network&#160;(BEN)&#160;model&#160;that&#160;utilizes&#160;the&#160;general&#160;principles&#160;of&#160;<br/>bioelectricity&#160;(electrodiffusion&#160;and&#160;gating)&#160;can&#160;compute.&#160;We&#160;characterize&#160;BEN&#160;behaviors&#160;ranging&#160;from&#160;<br/>elementary&#160;logic&#160;gates&#160;to&#160;pattern&#160;detectors,&#160;using&#160;both&#160;fixed&#160;and&#160;transient&#160;inputs&#160;to&#160;recapitulate&#160;<br/>various&#160;biological&#160;scenarios.&#160;We&#160;characterize&#160;the&#160;mechanisms&#160;of&#160;such&#160;networks&#160;using&#160;dynamical-<br/>systems&#160;and&#160;information-theory&#160;tools,&#160;demonstrating&#160;that&#160;logic&#160;can&#160;manifest&#160;in&#160;bidirectional,&#160;<br/>continuous,&#160;and&#160;relatively&#160;slow&#160;bioelectrical&#160;systems,&#160;complementing&#160;conventional&#160;neural-centric&#160;<br/>architectures.&#160;Our&#160;results&#160;reveal&#160;a&#160;variety&#160;of&#160;non-neural&#160;decision-making&#160;processes&#160;as&#160;manifestations&#160;<br/>of&#160;general&#160;cellular&#160;biophysical&#160;mechanisms&#160;and&#160;suggest&#160;novel&#160;bioengineering&#160;approaches&#160;to&#160;construct&#160;<br/>functional&#160;tissues&#160;for&#160;regenerative&#160;medicine&#160;and&#160;synthetic&#160;biology&#160;as&#160;well&#160;as&#160;new&#160;machine&#160;learning&#160;<br/>architectures.</b></p>
<p style="position:absolute;top:708px;left:233px;white-space:nowrap" class="ft19">Biological systems have long served as an inspiration and a design challenge for the engineering of artificial&#160;</p>
<p style="position:absolute;top:723px;left:233px;white-space:nowrap" class="ft19">intelligence and machine learning<a href="4b78c8d89b22cb315d90fd1274945de5s.html#13">1,2</a>, with a special focus on the brain<a href="4b78c8d89b22cb315d90fd1274945de5s.html#13">3</a>. However, many biological phenomena,&#160;</p>
<p style="position:absolute;top:738px;left:233px;white-space:nowrap" class="ft19">ranging from maze solving by cel&#160;s and slime molds to complex regulative morphogenesis and regeneration, can&#160;</p>
<p style="position:absolute;top:753px;left:233px;white-space:nowrap" class="ft19">be viewed as processes involving information-processing and decision-makin<a href="4b78c8d89b22cb315d90fd1274945de5s.html#13">g4, in t</a>he absence of a bra<a href="4b78c8d89b22cb315d90fd1274945de5s.html#13">in4–12.&#160;</a></p>
<p style="position:absolute;top:768px;left:233px;white-space:nowrap" class="ft19">Memory, anticipation, and problem solving have been demonstrated in sperm, amoebae, yeast, and plan<a href="4b78c8d89b22cb315d90fd1274945de5s.html#13">ts8–12,&#160;</a></p>
<p style="position:absolute;top:783px;left:233px;white-space:nowrap" class="ft19">and&#160;these&#160;capabilities&#160;scaled with&#160;the&#160;emergence&#160;of&#160;the&#160;Metazoa.&#160;For&#160;example,&#160;when&#160;craniofacial&#160;anatomical&#160;</p>
<p style="position:absolute;top:798px;left:233px;white-space:nowrap" class="ft19">features&#160;of&#160;tadpoles&#160;are&#160;scrambled&#160;into abnormal&#160;configurations,&#160;they&#160;normalize their&#160;aberrant positions&#160;over&#160;</p>
<p style="position:absolute;top:813px;left:233px;white-space:nowrap" class="ft19">time to regain a correct frog face morphology and then cease remodeling<a href="4b78c8d89b22cb315d90fd1274945de5s.html#14">13</a>. Thus, cel&#160;s and tissues functional&#160;y&#160;</p>
<p style="position:absolute;top:828px;left:233px;white-space:nowrap" class="ft19">ascertain the difference between the current, incorrect craniofacial morphology and the frog’s native craniofacial&#160;</p>
<p style="position:absolute;top:843px;left:233px;white-space:nowrap" class="ft19">morphology and undertake corrective movements to reduce the error. Similarly, the regeneration of entire limbs&#160;</p>
<p style="position:absolute;top:858px;left:233px;white-space:nowrap" class="ft19">in animals such as salamanders can be understood as being driven by a ‘test-operate-text-exist’ mode<a href="4b78c8d89b22cb315d90fd1274945de5s.html#13">l7</a>,&#160;in which&#160;</p>
<p style="position:absolute;top:873px;left:233px;white-space:nowrap" class="ft19">cel&#160;s act to implement an invariant anatomical outcome from diverse starting conditions, and stop once this target&#160;</p>
<p style="position:absolute;top:888px;left:233px;white-space:nowrap" class="ft19">morphology has been achieved<a href="4b78c8d89b22cb315d90fd1274945de5s.html#13">7</a><a href="4b78c8d89b22cb315d90fd1274945de5s.html#14">,14. Thi</a>s theme is reinforced by classical observations such as the fact that tails&#160;</p>
<p style="position:absolute;top:903px;left:233px;white-space:nowrap" class="ft19">grafted onto the side of salamanders slowly remodel into lim<a href="4b78c8d89b22cb315d90fd1274945de5s.html#14">bs15, dem</a>onstrating the ability of tissue to ascertain&#160;</p>
<p style="position:absolute;top:918px;left:233px;white-space:nowrap" class="ft19">its&#160;position&#160;within&#160;the&#160;whole,&#160;compare&#160;its&#160;organ-level&#160;anatomy&#160;with&#160;that&#160;dictated&#160;by&#160;the&#160;target&#160;morphology,&#160;and&#160;</p>
<p style="position:absolute;top:933px;left:233px;white-space:nowrap" class="ft19">remodel toward that correct anatomical setpoint<a href="4b78c8d89b22cb315d90fd1274945de5s.html#14">16</a>.</p>
<p style="position:absolute;top:948px;left:251px;white-space:nowrap" class="ft19">Importantly, unicel&#160;ular life&#160;forms and&#160;somatic cel&#160;s&#160;of multicel&#160;ular organisms were making flexible&#160;decisions&#160;</p>
<p style="position:absolute;top:963px;left:233px;white-space:nowrap" class="ft19">based on inputs in their microenvironment long before neurons appeare<a href="4b78c8d89b22cb315d90fd1274945de5s.html#13">d11,</a><a href="4b78c8d89b22cb315d90fd1274945de5s.html#14">17</a>. Nerves may have speed-optimized&#160;</p>
<p style="position:absolute;top:978px;left:233px;white-space:nowrap" class="ft19">ancient bioelectric processes that, since the time of bacterial biofilm<a href="4b78c8d89b22cb315d90fd1274945de5s.html#14">s18, w</a>ere already exploited by evolution to&#160;</p>
<p style="position:absolute;top:993px;left:233px;white-space:nowrap" class="ft19">implement memory, long-range coordination, and decision-making utilized for maintenance and construction&#160;</p>
<p style="position:absolute;top:1008px;left:233px;white-space:nowrap" class="ft19">of&#160;anatomical&#160;structures<a href="4b78c8d89b22cb315d90fd1274945de5s.html#14">19</a>. In multicel&#160;ular organisms, these same functions are used to control large-scale pat-</p>
<p style="position:absolute;top:1023px;left:233px;white-space:nowrap" class="ft19">terning<a href="4b78c8d89b22cb315d90fd1274945de5s.html#14">20</a>–<a href="4b78c8d89b22cb315d90fd1274945de5s.html#14">23</a>. For example, bioelectric signals mediate important aspects of the long-range coordination that keep&#160;</p>
<p style="position:absolute;top:1038px;left:233px;white-space:nowrap" class="ft19">cel&#160;s harnessed towards maintenance of a body-plan and away from tumorigenesis<a href="4b78c8d89b22cb315d90fd1274945de5s.html#13">5</a>. Stable bioelectric circuits&#160;</p>
<p style="position:absolute;top:1053px;left:233px;white-space:nowrap" class="ft19">also maintain the information needed for fragments of planaria to regenerate the correct number and distribution&#160;</p>
<p style="position:absolute;top:1093px;left:233px;white-space:nowrap" class="ft12">Allen&#160;Discovery&#160;Center,&#160;200&#160;College&#160;Ave.,&#160;Tufts&#160;University,&#160;Medford,&#160;MA,&#160;02155,&#160;USA.&#160;*e<a href="mailto:michael.levin@tufts.edu">mail:&#160;michael.levin@tufts.edu</a></p>
<p style="position:absolute;top:221px;left:141px;white-space:nowrap" class="ft113"><b>open</b></p>
</div>
</body>
</html>
<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="" xml:lang="">
<head>
<title>4b78c8d89b22cb315d90fd1274945de5-html.html</title>

<meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
 <br/>
<style type="text/css">
<!--
	p {margin: 0; padding: 0;}	.ft20{font-size:13px;font-family:Times;color:#cedde4;}
	.ft21{font-size:10px;font-family:Times;color:#000000;}
	.ft22{font-size:10px;font-family:Times;color:#000000;}
	.ft23{font-size:10px;font-family:Times;color:#000000;}
	.ft24{font-size:13px;font-family:Times;color:#000000;}
	.ft25{font-size:11px;font-family:Times;color:#000000;}
	.ft26{font-size:6px;font-family:Times;color:#0000ff;}
	.ft27{font-size:6px;font-family:Times;color:#000000;}
	.ft28{font-size:14px;font-family:Times;color:#000000;}
	.ft29{font-size:11px;font-family:Times;color:#000000;}
	.ft210{font-size:11px;font-family:Times;color:#0000ff;}
-->
</style>
</head>
<body bgcolor="#A0A0A0" vlink="blue" link="blue">
<div id="page2-div" style="position:relative;width:892px;height:1173px;">
<p style="position:absolute;top:1132px;left:824px;white-space:nowrap" class="ft20">2</p>
<p style="position:absolute;top:1135px;left:61px;white-space:nowrap" class="ft21"><b>Scientific</b>&#160;<b>RepoRtS</b>&#160;|&#160;</p>
<p style="position:absolute;top:1135px;left:192px;white-space:nowrap" class="ft23"><i>&#160; &#160; &#160; &#160; (2019) 9:18612&#160;</i>&#160;|&#160;<a href="https://doi.org/10.1038/s41598-019-54859-8">https://doi.org/10.1038/s41598-019-54859-8</a></p>
<p style="position:absolute;top:19px;left:585px;white-space:nowrap" class="ft24">www.nature.com/scientificreports</p>
<p style="position:absolute;top:19px;left:61px;white-space:nowrap" class="ft24">www.nature.com/scientificreports/</p>
<p style="position:absolute;top:71px;left:233px;white-space:nowrap" class="ft25">of&#160;heads&#160;and&#160;tail<a href="4b78c8d89b22cb315d90fd1274945de5s.html#14">s24,25</a>,&#160;and&#160;spatial&#160;distributions&#160;of&#160;resting&#160;potentials&#160;that&#160;are&#160;critical&#160;for&#160;the&#160;correct&#160;formation&#160;of&#160;</p>
<p style="position:absolute;top:86px;left:233px;white-space:nowrap" class="ft25">hearts<a href="4b78c8d89b22cb315d90fd1274945de5s.html#14">26,27</a>, ey<a href="4b78c8d89b22cb315d90fd1274945de5s.html#14">es28, a</a>nd brain<a href="4b78c8d89b22cb315d90fd1274945de5s.html#14">s29. B</a>ioelectric signaling is involved at many scales, from decisions of stem cel&#160;s as to&#160;</p>
<p style="position:absolute;top:101px;left:233px;white-space:nowrap" class="ft25">when and how to differentiat<a href="4b78c8d89b22cb315d90fd1274945de5s.html#14">e30, t</a>o the control of size<a href="4b78c8d89b22cb315d90fd1274945de5s.html#14">31</a>&#160;and regeneration of entire organ<a href="4b78c8d89b22cb315d90fd1274945de5s.html#14">s32,33. T</a>ransient inputs&#160;</p>
<p style="position:absolute;top:116px;left:233px;white-space:nowrap" class="ft25">that shift the endogenous bioelectric network states into alternate stable modes can permanently change regen-</p>
<p style="position:absolute;top:131px;left:233px;white-space:nowrap" class="ft25">erative&#160;and developmental&#160;morphology<a href="4b78c8d89b22cb315d90fd1274945de5s.html#14">34</a>; because of this, mutations in ion&#160;channel genes are an important cause&#160;</p>
<p style="position:absolute;top:146px;left:233px;white-space:nowrap" class="ft25">of human birth defects<a href="4b78c8d89b22cb315d90fd1274945de5s.html#14">35,36, w</a>hile modulation of bioelectric circuit state has been used to repair brain structur<a href="4b78c8d89b22cb315d90fd1274945de5s.html#14">e37.&#160;</a></p>
<p style="position:absolute;top:161px;left:233px;white-space:nowrap" class="ft25">Thus, a&#160;number of&#160;important&#160;fields converge on the&#160;need to&#160;understand how tissues process information&#160;via bio-</p>
<p style="position:absolute;top:176px;left:233px;white-space:nowrap" class="ft25">electric processes: evolutionary developmental biology (plasticity in cell behavior contributes to evolvability of&#160;</p>
<p style="position:absolute;top:191px;left:233px;white-space:nowrap" class="ft25">body plans), regenerative medicine (induction of complex repair by modulating the patterns to which cel&#160;s build),&#160;</p>
<p style="position:absolute;top:206px;left:233px;white-space:nowrap" class="ft25">and synthetic bioengineering (the pursuit of novel synthetic ‘living machines’).</p>
<p style="position:absolute;top:221px;left:251px;white-space:nowrap" class="ft25">Decision-making often exploits some form of logic. The need to understand tissue functions as computation&#160;</p>
<p style="position:absolute;top:236px;left:233px;white-space:nowrap" class="ft25">mediated&#160;by&#160;ancient&#160;biophysical&#160;mechanisms&#160;led&#160;us&#160;to&#160;seek&#160;a&#160;quantitative&#160;model&#160;of&#160;how&#160;non-neural&#160;networks&#160;</p>
<p style="position:absolute;top:251px;left:233px;white-space:nowrap" class="ft25">could&#160;possess&#160;logical&#160;abilities&#160;like&#160;those&#160;commonly&#160;studied&#160;in&#160;brains.&#160;The&#160;simplest&#160;realizations&#160;of&#160;logic&#160;are&#160;known&#160;</p>
<p style="position:absolute;top:266px;left:233px;white-space:nowrap" class="ft25">as “logic gates”,&#160;which&#160;are&#160;tiny circuits&#160;that&#160;perform&#160;elementary logic&#160;operations like AND, OR,&#160;NOT,&#160;etc.&#160;that&#160;</p>
<p style="position:absolute;top:281px;left:233px;white-space:nowrap" class="ft25">comprise any modern digital computer. A unique feature of logic gates is that, although they are typical&#160;y real-</p>
<p style="position:absolute;top:296px;left:233px;white-space:nowrap" class="ft25">ized as&#160;electronic circuits,&#160;the underlying mathematical&#160;formulations have&#160;been&#160;used&#160;as&#160;coarse-grained models&#160;</p>
<p style="position:absolute;top:311px;left:233px;white-space:nowrap" class="ft25">of neurons<a href="4b78c8d89b22cb315d90fd1274945de5s.html#14">38,39, g</a>en<a href="4b78c8d89b22cb315d90fd1274945de5s.html#14">es40,41</a>, and even physiolog<a href="4b78c8d89b22cb315d90fd1274945de5s.html#13">y12. L</a>ogic networks can be assembled to compute any computable&#160;</p>
<p style="position:absolute;top:326px;left:233px;white-space:nowrap" class="ft25">function<a href="4b78c8d89b22cb315d90fd1274945de5s.html#14">42</a>, and thus can be valuable building blocks of intelligent information-processing systems.</p>
<p style="position:absolute;top:341px;left:251px;white-space:nowrap" class="ft25">Here, we show that non-neural bioelectric networks can compute logic functions, suggesting one way in which&#160;</p>
<p style="position:absolute;top:356px;left:233px;white-space:nowrap" class="ft25">evolution can exploit biophysics for decision-making in cel&#160;ular systems. This provides a new connection between&#160;</p>
<p style="position:absolute;top:371px;left:233px;white-space:nowrap" class="ft25">non-neural physiology and a common kind of computational task, thus expanding the known capabilities of&#160;</p>
<p style="position:absolute;top:386px;left:233px;white-space:nowrap" class="ft25">developmental bioelectricity. While defining necessary and sufficient conditions for “computation” is a profound&#160;</p>
<p style="position:absolute;top:401px;left:233px;white-space:nowrap" class="ft25">question debated elsewhere<a href="4b78c8d89b22cb315d90fd1274945de5s.html#14">43–46</a>, logic gates are widely accepted as a non-controversial example of computation.&#160;</p>
<p style="position:absolute;top:416px;left:233px;white-space:nowrap" class="ft25">Thus, we extend established connectionist approaches to a more general physiological setting and analyze them to&#160;</p>
<p style="position:absolute;top:431px;left:233px;white-space:nowrap" class="ft25">reveal how ionic dynamics in non-neuronal cel&#160;s could implement both simple and complex logic gates.</p>
<p style="position:absolute;top:462px;left:233px;white-space:nowrap" class="ft28"><b>A&#160;new Bioelectric&#160;network Model</b></p>
<p style="position:absolute;top:479px;left:233px;white-space:nowrap" class="ft25">To facilitate the analysis of dynamic, context-dependent biological processes (e.g., morphogenesis and remod-</p>
<p style="position:absolute;top:494px;left:233px;white-space:nowrap" class="ft25">eling) as cognitive tasks<a href="4b78c8d89b22cb315d90fd1274945de5s.html#13">4, w</a>here networks of non-neural cells collectively make decisions, we constructed a&#160;</p>
<p style="position:absolute;top:509px;left:233px;white-space:nowrap" class="ft25">minimal BioElectric Network (BEN) that comprises the simplest components and processes of bioelectrical sig-</p>
<p style="position:absolute;top:524px;left:233px;white-space:nowrap" class="ft25">naling. BEN is inspired by a sophisticated and realistic model known as the&#160;<i>Bioelectric&#160;Tissue&#160;Simulation&#160;Engine</i><a href="4b78c8d89b22cb315d90fd1274945de5s.html#14">47&#160;</a></p>
<p style="position:absolute;top:539px;left:233px;white-space:nowrap" class="ft25">(BETSE), which has been used to show how bioelectric patterns can be created and sustaine<a href="4b78c8d89b22cb315d90fd1274945de5s.html#14">d21,48</a>–<a href="4b78c8d89b22cb315d90fd1274945de5s.html#14">50</a>&#160;and how such&#160;</p>
<p style="position:absolute;top:554px;left:233px;white-space:nowrap" class="ft25">a system might interact with genetic networks to give rise to morphological patterns<a href="4b78c8d89b22cb315d90fd1274945de5s.html#14">51</a>. Our goal was to define a&#160;</p>
<p style="position:absolute;top:569px;left:233px;white-space:nowrap" class="ft25">generic, minimal&#160;biophysical&#160;system&#160;using&#160;realistic yet&#160;simple&#160;signaling&#160;dynamics,&#160;and&#160;study&#160;its&#160;computational&#160;</p>
<p style="position:absolute;top:584px;left:233px;white-space:nowrap" class="ft25">properties in the absence of the specialized, highly-derived specifications that define neural networks. Thus, BEN&#160;</p>
<p style="position:absolute;top:599px;left:233px;white-space:nowrap" class="ft25">is a simplified version of BETSE that is minimal enough to aid the investigation of the computational capabilities&#160;</p>
<p style="position:absolute;top:614px;left:233px;white-space:nowrap" class="ft25">of&#160;a&#160;bioelectric&#160;system&#160;that&#160;have&#160;not&#160;yet&#160;been&#160;understood.&#160;The&#160;inputs&#160;and&#160;outputs&#160;of&#160;this&#160;network,&#160;representing&#160;</p>
<p style="position:absolute;top:629px;left:233px;white-space:nowrap" class="ft25">the logic values, are represented by the bioelectric states of the cel&#160;s (resting potential or Vmem).</p>
<p style="position:absolute;top:644px;left:251px;white-space:nowrap" class="ft25">BEN is defined by a set of bioelectric components and processes that dictate their production, transport, and&#160;</p>
<p style="position:absolute;top:659px;left:233px;white-space:nowrap" class="ft25">decay. BEN consists of two types of regions: an environment and a network of cel&#160;s. A single cell in a BEN net-</p>
<p style="position:absolute;top:674px;left:233px;white-space:nowrap" class="ft25">work consists of different types of proteins: ion channels and ion pumps. There are two type of molecules: ions&#160;</p>
<p style="position:absolute;top:689px;left:233px;white-space:nowrap" class="ft25">(charged) and signaling molecules (generic uncharged molecules that may represent a ligand, a neurotransmitter,&#160;</p>
<p style="position:absolute;top:704px;left:233px;white-space:nowrap" class="ft25">a secondary messenger, etc.). BEN contains three types of ions: sodium (Na+), potassium (K+), and chloride&#160;</p>
<p style="position:absolute;top:719px;left:233px;white-space:nowrap" class="ft25">(Cl−). Furthermore, there are two types of ion channels (IC) that each selectively allow the passage of Na+&#160;and&#160;</p>
<p style="position:absolute;top:734px;left:233px;white-space:nowrap" class="ft25">K+&#160;ions respectively between the cell and the environment; no such channel exists for Cl−. There is a single&#160;</p>
<p style="position:absolute;top:749px;left:233px;white-space:nowrap" class="ft25">sodium-potassium ion pump that actively strives to maintain a non-zero membrane potential (Vmem). Two cel&#160;s&#160;</p>
<p style="position:absolute;top:764px;left:233px;white-space:nowrap" class="ft25">in&#160;a BEN network&#160;may&#160;be connected&#160;by an&#160;electrical&#160;synapse&#160;known&#160;as a “gap junction”<a href="4b78c8d89b22cb315d90fd1274945de5s.html#14">52</a>&#160;(GJ), represented&#160;</p>
<p style="position:absolute;top:779px;left:233px;white-space:nowrap" class="ft25">as an undirected edge in the network, which allows the passage of molecules between the cells. The process&#160;</p>
<p style="position:absolute;top:794px;left:233px;white-space:nowrap" class="ft25">that&#160;governs&#160;the&#160;passage&#160;of&#160;ions&#160;through&#160;either&#160;the&#160;GJ&#160;or&#160;IC&#160;is&#160;cal&#160;ed&#160;“electrodiffusion”,&#160;a&#160;combination&#160;of&#160;elec-</p>
<p style="position:absolute;top:809px;left:233px;white-space:nowrap" class="ft25">trophoresis&#160;and regular&#160;diffusion,&#160;where&#160;the&#160;flux&#160;is&#160;driven&#160;by voltage&#160;gradients&#160;in&#160;the&#160;former and&#160;concentra-</p>
<p style="position:absolute;top:824px;left:233px;white-space:nowrap" class="ft25">tion gradients in the latter. Following the convention adopted in BETSE, we modeled electrodiffusion through&#160;</p>
<p style="position:absolute;top:839px;left:233px;white-space:nowrap" class="ft25">GJs using the Nernst-Planck equation, and the transmembrane flux of ions through ion channels using the&#160;</p>
<p style="position:absolute;top:854px;left:233px;white-space:nowrap" class="ft25">Goldman-Hodgkin-Katz flux equation<a href="4b78c8d89b22cb315d90fd1274945de5s.html#14">47</a>. However, since BEN is a networked system, where the processes occur&#160;</p>
<p style="position:absolute;top:869px;left:233px;white-space:nowrap" class="ft25">at&#160;discrete&#160;points&#160;in&#160;space&#160;(the&#160;cel&#160;s&#160;in&#160;Fig<a href="4b78c8d89b22cb315d90fd1274945de5s.html#3">. 1a),&#160;</a>we&#160;used&#160;simple&#160;discretized&#160;versions&#160;of the&#160;same&#160;equations.&#160;The&#160;flux&#160;</p>
<p style="position:absolute;top:884px;left:233px;white-space:nowrap" class="ft25">of the signaling molecules in BEN is governed by a generic nonlinear reaction-diffusion process consisting of two&#160;</p>
<p style="position:absolute;top:899px;left:233px;white-space:nowrap" class="ft25">layers of sigmoidal transformations. In the first transformation layer, the flux across each GJ is calculated using a&#160;</p>
<p style="position:absolute;top:914px;left:233px;white-space:nowrap" class="ft25">sigmoidal transformation of the concentration difference of the signaling molecule. In the second layer, the trans-</p>
<p style="position:absolute;top:929px;left:233px;white-space:nowrap" class="ft25">formed flux across each GJ is collectively transformed using a second sigmoid, constituting the net concentration&#160;</p>
<p style="position:absolute;top:944px;left:233px;white-space:nowrap" class="ft25">change of the signaling molecule. These two layers minimal&#160;y represent the multi-layered complexity prevalent in&#160;</p>
<p style="position:absolute;top:959px;left:233px;white-space:nowrap" class="ft25">cell signaling networks<a href="4b78c8d89b22cb315d90fd1274945de5s.html#14">53,54</a>. For example, calcium (Ca2+) transport within a cell is mediated by clusters of inositol&#160;</p>
<p style="position:absolute;top:974px;left:233px;white-space:nowrap" class="ft25">1,4,5-triphosphate (IP3) receptors distributed inside the cell in a multi-layered fashio<a href="4b78c8d89b22cb315d90fd1274945de5s.html#15">n55–57</a>. Moreover, such a&#160;</p>
<p style="position:absolute;top:989px;left:233px;white-space:nowrap" class="ft25">two-layer&#160;transformation is&#160;also found&#160;in neuronal&#160;cel&#160;s,&#160;where&#160;the first&#160;stage involves&#160;nonlinear&#160;transformations&#160;</p>
<p style="position:absolute;top:1004px;left:233px;white-space:nowrap" class="ft25">of local dendritic potentials, and a second stage involving a linear summation of the outputs from the first stage to&#160;</p>
<p style="position:absolute;top:1019px;left:233px;white-space:nowrap" class="ft25">determine&#160;the&#160;overal&#160;&#160;cel&#160;ular&#160;response<a href="4b78c8d89b22cb315d90fd1274945de5s.html#15">58</a>–<a href="4b78c8d89b22cb315d90fd1274945de5s.html#15">60</a>.&#160;Thus,&#160;the&#160;two-layer&#160;signal&#160;transformation&#160;architecture&#160;of&#160;BEN&#160;is&#160;rep-</p>
<p style="position:absolute;top:1034px;left:233px;white-space:nowrap" class="ft25">resentative of the multi-layered signaling complexity characteristic of multiple cell types. Moreover, the dynamics&#160;</p>
<p style="position:absolute;top:1049px;left:233px;white-space:nowrap" class="ft25">of the signal may be viewed as a reaction-diffusion system; see the ‘Methods’ section for more details. Final&#160;y, the&#160;</p>
<p style="position:absolute;top:1064px;left:233px;white-space:nowrap" class="ft25">extracel&#160;ular environment simulated in&#160;BEN&#160;includes the&#160;same&#160;three&#160;types&#160;of ions&#160;as the&#160;cel&#160;s&#160;(Na+,&#160;K+, and&#160;Cl−),&#160;</p>
</div>
</body>
</html>
<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="" xml:lang="">
<head>
<title>4b78c8d89b22cb315d90fd1274945de5-html.html</title>

<meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
 <br/>
<style type="text/css">
<!--
	p {margin: 0; padding: 0;}	.ft30{font-size:13px;font-family:Times;color:#cedde4;}
	.ft31{font-size:10px;font-family:Times;color:#000000;}
	.ft32{font-size:10px;font-family:Times;color:#000000;}
	.ft33{font-size:10px;font-family:Times;color:#000000;}
	.ft34{font-size:13px;font-family:Times;color:#000000;}
	.ft35{font-size:11px;font-family:Times;color:#000000;}
	.ft36{font-size:6px;font-family:Times;color:#0000ff;}
	.ft37{font-size:6px;font-family:Times;color:#000000;}
	.ft38{font-size:11px;font-family:Times;color:#000000;}
	.ft39{font-size:11px;font-family:Times;color:#0000ff;}
	.ft310{font-size:11px;font-family:Times;color:#000000;}
-->
</style>
</head>
<body bgcolor="#A0A0A0" vlink="blue" link="blue">
<div id="page3-div" style="position:relative;width:892px;height:1173px;">
<p style="position:absolute;top:1132px;left:825px;white-space:nowrap" class="ft30">3</p>
<p style="position:absolute;top:1135px;left:61px;white-space:nowrap" class="ft31"><b>Scientific</b>&#160;<b>RepoRtS</b>&#160;|&#160;</p>
<p style="position:absolute;top:1135px;left:192px;white-space:nowrap" class="ft33"><i>&#160; &#160; &#160; &#160; (2019) 9:18612&#160;</i>&#160;|&#160;<a href="https://doi.org/10.1038/s41598-019-54859-8">https://doi.org/10.1038/s41598-019-54859-8</a></p>
<p style="position:absolute;top:19px;left:585px;white-space:nowrap" class="ft34">www.nature.com/scientificreports</p>
<p style="position:absolute;top:19px;left:61px;white-space:nowrap" class="ft34">www.nature.com/scientificreports/</p>
<p style="position:absolute;top:701px;left:233px;white-space:nowrap" class="ft35">but initialized with different concentrations. We assume that the environment is effectively infinite in size, hence&#160;</p>
<p style="position:absolute;top:716px;left:233px;white-space:nowrap" class="ft35">the ion concentrations there remain constant.</p>
<p style="position:absolute;top:731px;left:251px;white-space:nowrap" class="ft35">There are&#160;two types&#160;of&#160;gating mechanisms&#160;in BEN: chemical-gating&#160;of&#160;ion&#160;channels&#160;and&#160;voltage-gating of&#160;</p>
<p style="position:absolute;top:746px;left:233px;white-space:nowrap" class="ft35">gap junctions. Chemical-gating of IC is a mechanism by which the permeability of an IC is modulated by the&#160;</p>
<p style="position:absolute;top:761px;left:233px;white-space:nowrap" class="ft35">binding&#160;of&#160;a&#160;gating&#160;molecule&#160;(typical&#160;y&#160;a&#160;‘ligand’)&#160;to&#160;the&#160;channel,&#160;as&#160;a&#160;function&#160;of&#160;the&#160;concentration&#160;of&#160;the&#160;mole-</p>
<p style="position:absolute;top:776px;left:233px;white-space:nowrap" class="ft35">cule<a href="4b78c8d89b22cb315d90fd1274945de5s.html#14">51</a><a href="4b78c8d89b22cb315d90fd1274945de5s.html#15">,61</a>(Ch.31). Rather than the more conventional Hill function to model it<a href="4b78c8d89b22cb315d90fd1274945de5s.html#14">51</a>, we used a simpler sigmoid function&#160;</p>
<p style="position:absolute;top:791px;left:233px;white-space:nowrap" class="ft35">that maps the concentration of the signaling molecule to the proportion of the maximum permeability of the&#160;</p>
<p style="position:absolute;top:806px;left:233px;white-space:nowrap" class="ft35">IC. In BEN, a higher concentration of the signal tends to depolarize the cel&#160;, and lower concentrations hyperpo-</p>
<p style="position:absolute;top:821px;left:233px;white-space:nowrap" class="ft35">larize it. Voltage-gating of&#160;GJ&#160;is&#160;a mechanism&#160;by&#160;which the permeability&#160;of&#160;a&#160;GJ&#160;is&#160;modulated&#160;by the Vmem&#160;of&#160;the&#160;</p>
<p style="position:absolute;top:836px;left:233px;white-space:nowrap" class="ft35">connected cel&#160;s and also the difference between them (known as the “transjunctional voltage<a href="4b78c8d89b22cb315d90fd1274945de5s.html#14">”)51,</a><a href="4b78c8d89b22cb315d90fd1274945de5s.html#15">62,63</a>. We adopted&#160;</p>
<p style="position:absolute;top:851px;left:233px;white-space:nowrap" class="ft35">a sigmoidal transformation that maps the individual Vmem&#160;of the connected cel&#160;s that are then averaged over to&#160;</p>
<p style="position:absolute;top:866px;left:233px;white-space:nowrap" class="ft35">compute the net GJ permeability, in contrast to the more complex but similar Hill function based model adopted&#160;</p>
<p style="position:absolute;top:881px;left:233px;white-space:nowrap" class="ft35">in&#160;BETSE<a href="4b78c8d89b22cb315d90fd1274945de5s.html#14">51</a>. In BEN, positive Vmem&#160;levels tend to make the GJ more permeable, and negative Vmem&#160;make them less&#160;</p>
<p style="position:absolute;top:896px;left:233px;white-space:nowrap" class="ft35">permeable. The mathematical details of BEN are described in the ‘Methods’ section.</p>
<p style="position:absolute;top:911px;left:251px;white-space:nowrap" class="ft35">There are two types of&#160;<i>learnable</i>&#160;parameters in BEN: weight and bias, following the convention of artificial&#160;</p>
<p style="position:absolute;top:926px;left:233px;white-space:nowrap" class="ft35">neural&#160;networ<a href="4b78c8d89b22cb315d90fd1274945de5s.html#15">ks64&#160;</a>(ANN).&#160;The&#160;weight&#160;is&#160;associated&#160;with&#160;a&#160;GJ,&#160;and&#160;it&#160;helps&#160;modulate&#160;the&#160;flux&#160;of&#160;the&#160;signal&#160;as&#160;wel&#160;&#160;</p>
<p style="position:absolute;top:941px;left:233px;white-space:nowrap" class="ft35">as the voltage-gating of GJ. Negative weights tend to lower the signal concentration in a cel&#160;, and positive weights&#160;</p>
<p style="position:absolute;top:956px;left:233px;white-space:nowrap" class="ft35">tend to increase it. With&#160;regards to voltage-gating, smaller&#160;weights&#160;diminish&#160;the effect of Vmem, and&#160;larger weights&#160;</p>
<p style="position:absolute;top:971px;left:233px;white-space:nowrap" class="ft35">tend&#160;to&#160;magnify&#160;it.&#160;The&#160;bias&#160;is&#160;associated&#160;with&#160;a&#160;single&#160;cel&#160;&#160;that&#160;helps&#160;modulate&#160;signal&#160;flux:&#160;it&#160;specifies&#160;the&#160;threshold&#160;</p>
<p style="position:absolute;top:986px;left:233px;white-space:nowrap" class="ft35">at which the signal concentration switches direction of change. By linking these parameters to the core features of&#160;</p>
<p style="position:absolute;top:1001px;left:233px;white-space:nowrap" class="ft35">BEN, we have assumed that they are not special to ANNs.</p>
<p style="position:absolute;top:1016px;left:251px;white-space:nowrap" class="ft35">BEN networks have a layered architecture (Fig. <a href="4b78c8d89b22cb315d90fd1274945de5s.html#3">1a</a>), a scheme that is widely adopted for ANN<a href="4b78c8d89b22cb315d90fd1274945de5s.html#15">s64</a>&#160;but which&#160;</p>
<p style="position:absolute;top:1031px;left:233px;white-space:nowrap" class="ft35">also reflects the tissue layering of many different types of body structures. In this scheme, a network consists of&#160;</p>
<p style="position:absolute;top:1046px;left:233px;white-space:nowrap" class="ft35">multiple layers of cel&#160;s, with different choices for the inter-layer and intra-layer connectivity. Some layers have&#160;</p>
<p style="position:absolute;top:1061px;left:233px;white-space:nowrap" class="ft35">clearly designated roles—for example, the first layer is the “input” and the last layer is the “output”. This follows&#160;</p>
<p style="position:absolute;top:1076px;left:233px;white-space:nowrap" class="ft35">the&#160;scheme&#160;of&#160;various&#160;biological&#160;systems&#160;with&#160;specific&#160;roles&#160;assigned&#160;to&#160;the&#160;layers.&#160;For&#160;example,&#160;the&#160;retina&#160;is&#160;the&#160;</p>
<p style="position:absolute;top:1091px;left:233px;white-space:nowrap" class="ft35">“sensory” layer&#160;that&#160;directly&#160;receives&#160;signals&#160;from the environment&#160;and&#160;sends&#160;it&#160;to&#160;the&#160;upstream&#160;inter-neuron&#160;</p>
<p style="position:absolute;top:565px;left:233px;white-space:nowrap" class="ft310"><b>Figure 1.</b>&#160;&#160;Schematics of the BEN architecture and its constituents. (<b>a</b>) The general architecture of a BEN&#160;</p>
<p style="position:absolute;top:581px;left:233px;white-space:nowrap" class="ft35">network; (<b>b</b>) A BEN network consisting of two cel&#160;s; (<b>c</b>) The network of components (black) related by&#160;</p>
<p style="position:absolute;top:596px;left:233px;white-space:nowrap" class="ft35">processes (blue, red and green) within a single cel&#160;, some of which receive inputs from adjacent cel&#160;s. For&#160;</p>
<p style="position:absolute;top:611px;left:233px;white-space:nowrap" class="ft35">example, the Vmem&#160;of a cell together with that of the adjacent cell gates the gap junction which in turn affects the&#160;</p>
<p style="position:absolute;top:626px;left:233px;white-space:nowrap" class="ft35">concentrations of the ion and the signaling molecule, and the ion concentration ultimately determines the Vmem,&#160;</p>
<p style="position:absolute;top:641px;left:233px;white-space:nowrap" class="ft35">closing the loop. This highlights the recurrent nature of the dynamics in BEN.</p>
</div>
</body>
</html>
<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="" xml:lang="">
<head>
<title>4b78c8d89b22cb315d90fd1274945de5-html.html</title>

<meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
 <br/>
<style type="text/css">
<!--
	p {margin: 0; padding: 0;}	.ft40{font-size:13px;font-family:Times;color:#cedde4;}
	.ft41{font-size:10px;font-family:Times;color:#000000;}
	.ft42{font-size:10px;font-family:Times;color:#000000;}
	.ft43{font-size:10px;font-family:Times;color:#000000;}
	.ft44{font-size:13px;font-family:Times;color:#000000;}
	.ft45{font-size:11px;font-family:Times;color:#000000;}
	.ft46{font-size:6px;font-family:Times;color:#0000ff;}
	.ft47{font-size:6px;font-family:Times;color:#000000;}
	.ft48{font-size:11px;font-family:Times;color:#000000;}
	.ft49{font-size:14px;font-family:Times;color:#000000;}
	.ft410{font-size:13px;font-family:Times;color:#000000;}
	.ft411{font-size:11px;font-family:Times;color:#0000ff;}
-->
</style>
</head>
<body bgcolor="#A0A0A0" vlink="blue" link="blue">
<div id="page4-div" style="position:relative;width:892px;height:1173px;">
<p style="position:absolute;top:1132px;left:824px;white-space:nowrap" class="ft40">4</p>
<p style="position:absolute;top:1135px;left:61px;white-space:nowrap" class="ft41"><b>Scientific</b>&#160;<b>RepoRtS</b>&#160;|&#160;</p>
<p style="position:absolute;top:1135px;left:192px;white-space:nowrap" class="ft43"><i>&#160; &#160; &#160; &#160; (2019) 9:18612&#160;</i>&#160;|&#160;<a href="https://doi.org/10.1038/s41598-019-54859-8">https://doi.org/10.1038/s41598-019-54859-8</a></p>
<p style="position:absolute;top:19px;left:585px;white-space:nowrap" class="ft44">www.nature.com/scientificreports</p>
<p style="position:absolute;top:19px;left:61px;white-space:nowrap" class="ft44">www.nature.com/scientificreports/</p>
<p style="position:absolute;top:71px;left:233px;white-space:nowrap" class="ft45">layers that integrates the inpu<a href="4b78c8d89b22cb315d90fd1274945de5s.html#15">t65. Th</a>e layered architecture is inspired by biological counterparts as diverse as the&#160;</p>
<p style="position:absolute;top:86px;left:233px;white-space:nowrap" class="ft45">mammalian visual cortex<a href="4b78c8d89b22cb315d90fd1274945de5s.html#13">1,2,</a><a href="4b78c8d89b22cb315d90fd1274945de5s.html#15">65</a>&#160;and cel&#160;ular signaling networ<a href="4b78c8d89b22cb315d90fd1274945de5s.html#15">ks66, w</a>hose benefits include better organization and&#160;</p>
<p style="position:absolute;top:101px;left:233px;white-space:nowrap" class="ft45">efficiency of information processin<a href="4b78c8d89b22cb315d90fd1274945de5s.html#15">g64</a>&#160;and predictio<a href="4b78c8d89b22cb315d90fd1274945de5s.html#15">n67.</a></p>
<p style="position:absolute;top:116px;left:251px;white-space:nowrap" class="ft45">A simulation of BEN proceeds as follows. The input layer cel&#160;s are set to specific Vmem&#160;levels at the beginning,&#160;</p>
<p style="position:absolute;top:131px;left:233px;white-space:nowrap" class="ft45">while all other cel&#160;s are set at Vmem&#160;=&#160;0. From that point onward, the Vmem&#160;levels of the network will dynamical&#160;y&#160;</p>
<p style="position:absolute;top:146px;left:233px;white-space:nowrap" class="ft45">change according to the network parameters, until the output settles at some Vmem. The input to the network is&#160;</p>
<p style="position:absolute;top:161px;left:233px;white-space:nowrap" class="ft45">either&#160;<i>fixed</i>&#160;(clamped throughout a simulation) or&#160;<i>transient</i>&#160;(set at only the beginning of the simulation and then&#160;</p>
<p style="position:absolute;top:176px;left:233px;white-space:nowrap" class="ft45">allowed&#160;to&#160;change).&#160;These&#160;two&#160;conditions&#160;represent&#160;the&#160;nature&#160;of biological&#160;information&#160;processing&#160;where the&#160;</p>
<p style="position:absolute;top:191px;left:233px;white-space:nowrap" class="ft45">outcome may depend on the duration of the stimulus. For example, the induction of certain proteins in cel&#160;ular&#160;</p>
<p style="position:absolute;top:206px;left:233px;white-space:nowrap" class="ft45">signaling networks require long-duration input signal<a href="4b78c8d89b22cb315d90fd1274945de5s.html#15">s68,69</a>; while a brief stimulus (progesterone) is sufficient to&#160;</p>
<p style="position:absolute;top:221px;left:233px;white-space:nowrap" class="ft45">induce long-term regenerative response in&#160;<i>Xenopus</i>&#160;limb<a href="4b78c8d89b22cb315d90fd1274945de5s.html#15">70</a>, a brief exposure to a gap-junction blocker is sufficient&#160;</p>
<p style="position:absolute;top:236px;left:233px;white-space:nowrap" class="ft45">to cause stochastic phenotypic shifts in planaria<a href="4b78c8d89b22cb315d90fd1274945de5s.html#14">24</a>, a transient invasion into a bacterial ecosystem can dramatical&#160;y&#160;</p>
<p style="position:absolute;top:251px;left:233px;white-space:nowrap" class="ft45">switch its charact<a href="4b78c8d89b22cb315d90fd1274945de5s.html#15">er71&#160;a</a>nd transient&#160;alterations&#160;to&#160;Vmem&#160;cause permanent alterations&#160;to&#160;gene&#160;expression and&#160;phe-</p>
<p style="position:absolute;top:266px;left:233px;white-space:nowrap" class="ft45">notypic patterning in planaria<a href="4b78c8d89b22cb315d90fd1274945de5s.html#15">72</a>.</p>
<p style="position:absolute;top:297px;left:233px;white-space:nowrap" class="ft49"><b>Results</b></p>
<p style="position:absolute;top:314px;left:233px;white-space:nowrap" class="ft45">The ability of somatic tissues outside the brain to compute is still controversial, in part because there is not a quan-</p>
<p style="position:absolute;top:329px;left:233px;white-space:nowrap" class="ft45">titative model available that demonstrates how basic operations of cognition could be implemented by generic&#160;</p>
<p style="position:absolute;top:344px;left:233px;white-space:nowrap" class="ft45">cel&#160;s. We demonstrate the logic-capabilities of BEN networks by constructing (1) small elementary logic gates; (2)&#160;</p>
<p style="position:absolute;top:359px;left:233px;white-space:nowrap" class="ft45">larger “tissue-level” elementary logic gates; (3) compound logic gates composed from elementary gates; and (4) a&#160;</p>
<p style="position:absolute;top:374px;left:233px;white-space:nowrap" class="ft45">pattern detector (showing how a complex regenerative response function can be implemented with these compo-</p>
<p style="position:absolute;top:389px;left:233px;white-space:nowrap" class="ft45">nents)&#160;using&#160;standard&#160;machine&#160;learning&#160;methods.&#160;Below&#160;we&#160;show&#160;that&#160;BEN&#160;networks&#160;can&#160;indeed&#160;function&#160;as&#160;logic&#160;</p>
<p style="position:absolute;top:404px;left:233px;white-space:nowrap" class="ft45">gates. We then analyze the behavior of a successful&#160;y trained BEN logic gate using the tools of information theory.</p>
<p style="position:absolute;top:436px;left:233px;white-space:nowrap" class="ft410"><b>BEN&#160;networks&#160;can&#160;implement&#160;small&#160;elementary&#160;logic&#160;gates:&#160;AND,&#160;XOR.&#160;&#160;</b>A logic gate&#160;is a circuit&#160;</p>
<p style="position:absolute;top:452px;left:233px;white-space:nowrap" class="ft45">that computes a binary-valued output from binary-valued inputs, according to a set of rules. For example, the&#160;</p>
<p style="position:absolute;top:467px;left:233px;white-space:nowrap" class="ft45">AND gate outputs a “HIGH” signal only if both inputs are “HIGH”, otherwise it outputs a “LOW”. The OR gate,&#160;</p>
<p style="position:absolute;top:482px;left:233px;white-space:nowrap" class="ft45">on the other hand, outputs a “HIGH” signal if at least one of the inputs has a “HIGH” signal. Thus, a logic gate is&#160;</p>
<p style="position:absolute;top:497px;left:233px;white-space:nowrap" class="ft45">defined by a “rule table” that maps an input configuration to a unique output. Many biological systems appear to&#160;</p>
<p style="position:absolute;top:512px;left:233px;white-space:nowrap" class="ft45">implement logic functions to facilitate, for example, integration of the various inputs they receive from the envi-</p>
<p style="position:absolute;top:527px;left:233px;white-space:nowrap" class="ft45">ronment with the required specificity<a href="4b78c8d89b22cb315d90fd1274945de5s.html#14">54,</a><a href="4b78c8d89b22cb315d90fd1274945de5s.html#15">73</a>.</p>
<p style="position:absolute;top:542px;left:251px;white-space:nowrap" class="ft45">Here&#160;we&#160;demonstrate&#160;that&#160;BEN networks&#160;can&#160;function&#160;as&#160;logic&#160;gates.&#160;We&#160;used standard&#160;machine&#160;learning&#160;</p>
<p style="position:absolute;top:557px;left:233px;white-space:nowrap" class="ft45">methods to identify the parameters of BEN networks that can perform a desired logical function. This discovery&#160;</p>
<p style="position:absolute;top:572px;left:233px;white-space:nowrap" class="ft45">phase in the biological world can be implemented either by evolution (which tweaks the parameters on a phy-</p>
<p style="position:absolute;top:587px;left:233px;white-space:nowrap" class="ft45">logenetic&#160;time scale) or within the lifetime of a single animal by the&#160;dynamic adjustment of ion channel and gap&#160;</p>
<p style="position:absolute;top:602px;left:233px;white-space:nowrap" class="ft45">junctional open states as a function of experience (plasticity)<a href="4b78c8d89b22cb315d90fd1274945de5s.html#15">74</a>.</p>
<p style="position:absolute;top:617px;left:251px;white-space:nowrap" class="ft45">To provide a proof-of-principle that somatic bioelectric networks can perform logical operations, we sought&#160;</p>
<p style="position:absolute;top:632px;left:233px;white-space:nowrap" class="ft45">to identify BEN networks with specific behaviors. Since it is difficult to find the requisite parameters manual&#160;y, we&#160;</p>
<p style="position:absolute;top:647px;left:233px;white-space:nowrap" class="ft45">used machine learning to automatical&#160;y discover suitable parameter values. We used “backpropagation” (BP) for&#160;</p>
<p style="position:absolute;top:662px;left:233px;white-space:nowrap" class="ft45">that purpose. BP is a training method, often used in machine learning, that starts with a random network (defined&#160;</p>
<p style="position:absolute;top:677px;left:233px;white-space:nowrap" class="ft45">with&#160;a&#160;random&#160;set&#160;of&#160;parameters)&#160;and&#160;gradual&#160;y&#160;tweaks&#160;the&#160;parameters&#160;using&#160;gradient&#160;descent&#160;until&#160;the&#160;network&#160;</p>
<p style="position:absolute;top:692px;left:233px;white-space:nowrap" class="ft45">performs the desired function. Since BEN networks are recurrent by definition (edges are bidirectional), we used&#160;</p>
<p style="position:absolute;top:707px;left:233px;white-space:nowrap" class="ft45">“backpropagation through time” (BPTT) to train them; see ‘Methods’ section for more details. We do not claim&#160;</p>
<p style="position:absolute;top:722px;left:233px;white-space:nowrap" class="ft45">that BP or BPTT is the method that organisms use to learn in the real world; they are just convenient tools that we&#160;</p>
<p style="position:absolute;top:737px;left:233px;white-space:nowrap" class="ft45">used&#160;to&#160;discover parameters that&#160;il&#160;ustrate&#160;the&#160;power of&#160;BEN networks&#160;(see Supplementary 3&#160;for&#160;alternative train-</p>
<p style="position:absolute;top:752px;left:233px;white-space:nowrap" class="ft45">ing methods). Below, we show examples of BEN networks implementing the AND gate, and the more difficult&#160;</p>
<p style="position:absolute;top:767px;left:233px;white-space:nowrap" class="ft45">XOR gate.&#160;First, we show&#160;the results&#160;of training&#160;for&#160;a set of&#160;100 training&#160;runs for each gate. Then,&#160;we describe the&#160;</p>
<p style="position:absolute;top:782px;left:233px;white-space:nowrap" class="ft45">behavior of the best gate in each set using time series and dynamical phase space il&#160;ustrations.</p>
<p style="position:absolute;top:797px;left:251px;white-space:nowrap" class="ft45">We&#160;first&#160;sought&#160;to&#160;determine&#160;if&#160;bioelectric&#160;networks&#160;can&#160;implement&#160;an&#160;AND&#160;gate,&#160;by&#160;training&#160;a&#160;small&#160;five-cell&#160;</p>
<p style="position:absolute;top:812px;left:233px;white-space:nowrap" class="ft45">BEN network (Fig. <a href="4b78c8d89b22cb315d90fd1274945de5s.html#5">2a</a>) to follow the AND rules that specifies the expected output for a given pair of inputs&#160;</p>
<p style="position:absolute;top:827px;left:233px;white-space:nowrap" class="ft45">(Fig<a href="4b78c8d89b22cb315d90fd1274945de5s.html#5">. 2b). W</a>e trained 100 instances of&#160;this network,&#160;each&#160;starting from different initial conditions of the&#160;param-</p>
<p style="position:absolute;top:842px;left:233px;white-space:nowrap" class="ft45">eters, discovering that some were indeed able to attain good performance, il&#160;ustrating that BEN networks can&#160;</p>
<p style="position:absolute;top:857px;left:233px;white-space:nowrap" class="ft45">perform the AND function (Fig<a href="4b78c8d89b22cb315d90fd1274945de5s.html#5">. 2c</a>). Out of the hundred networks we trained, seven achieved an error of 0.3&#160;mV&#160;</p>
<p style="position:absolute;top:872px;left:233px;white-space:nowrap" class="ft45">or less, while a total of 33 networks achieved an error of 1&#160;mV or less. The behavior of the best AND gate we&#160;</p>
<p style="position:absolute;top:887px;left:233px;white-space:nowrap" class="ft45">identified&#160;is&#160;il&#160;ustrated below by way of time-series plots and phase space diagrams&#160;(F<a href="4b78c8d89b22cb315d90fd1274945de5s.html#5">igs. 2d</a>, <a href="4b78c8d89b22cb315d90fd1274945de5s.html#5">2e</a>)&#160;by considering&#160;</p>
<p style="position:absolute;top:902px;left:233px;white-space:nowrap" class="ft45">various input-output conditions.</p>
<p style="position:absolute;top:917px;left:251px;white-space:nowrap" class="ft45">We&#160;next&#160;discovered&#160;a&#160;small&#160;five-cell&#160;BEN&#160;network&#160;that&#160;can&#160;function&#160;as&#160;an&#160;XOR&#160;gate&#160;by&#160;following&#160;the&#160;same&#160;</p>
<p style="position:absolute;top:932px;left:233px;white-space:nowrap" class="ft45">training&#160;procedure&#160;as&#160;for&#160;the&#160;AND&#160;gate&#160;above&#160;(we&#160;report&#160;the&#160;parameter&#160;details&#160;in&#160;Supplementary 1).&#160;The&#160;rule&#160;</p>
<p style="position:absolute;top:947px;left:233px;white-space:nowrap" class="ft45">table, training performances, behavior of the best XOR gate and the phase space diagram are shown below&#160;</p>
<p style="position:absolute;top:962px;left:233px;white-space:nowrap" class="ft45">(Fig. <a href="4b78c8d89b22cb315d90fd1274945de5s.html#7">3). Of t</a>he fifty networks we trained, only one network achieved an error of 1&#160;mV. Compared to the AND&#160;</p>
<p style="position:absolute;top:977px;left:233px;white-space:nowrap" class="ft45">gate, clearly training the XOR gate is more difficult. This is consistent with the known difficulty of training any&#160;</p>
<p style="position:absolute;top:992px;left:233px;white-space:nowrap" class="ft45">dynamical network to behave as an XOR function<a href="4b78c8d89b22cb315d90fd1274945de5s.html#15">64</a>, since it is a nonlinear function compared to AND, which is&#160;</p>
<p style="position:absolute;top:1007px;left:233px;white-space:nowrap" class="ft45">linear.</p>
<p style="position:absolute;top:1039px;left:233px;white-space:nowrap" class="ft410"><b>Ben&#160;networks can implement larger “tissue-level” elementary logic gates.&#160;&#160;</b>We next asked&#160;</p>
<p style="position:absolute;top:1055px;left:233px;white-space:nowrap" class="ft45">whether larger networks, which contain more cel&#160;s than strictly necessary, could still perform the logic functions.&#160;</p>
<p style="position:absolute;top:1070px;left:233px;white-space:nowrap" class="ft45">These larger networks are like tissues and organs, which can contain a range of diverse cel&#160;s that communicate.&#160;</p>
<p style="position:absolute;top:1085px;left:233px;white-space:nowrap" class="ft45">Thus, we next describe a “tissue-level” AND gate. First, we&#160;show the results of training for a set of 100&#160;training&#160;</p>
</div>
</body>
</html>
<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="" xml:lang="">
<head>
<title>4b78c8d89b22cb315d90fd1274945de5-html.html</title>

<meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
 <br/>
<style type="text/css">
<!--
	p {margin: 0; padding: 0;}	.ft50{font-size:13px;font-family:Times;color:#cedde4;}
	.ft51{font-size:10px;font-family:Times;color:#000000;}
	.ft52{font-size:10px;font-family:Times;color:#000000;}
	.ft53{font-size:10px;font-family:Times;color:#000000;}
	.ft54{font-size:13px;font-family:Times;color:#000000;}
	.ft55{font-size:11px;font-family:Times;color:#000000;}
	.ft56{font-size:11px;font-family:Times;color:#000000;}
	.ft57{font-size:6px;font-family:Times;color:#000000;}
-->
</style>
</head>
<body bgcolor="#A0A0A0" vlink="blue" link="blue">
<div id="page5-div" style="position:relative;width:892px;height:1173px;">
<p style="position:absolute;top:1132px;left:825px;white-space:nowrap" class="ft50">5</p>
<p style="position:absolute;top:1135px;left:61px;white-space:nowrap" class="ft51"><b>Scientific</b>&#160;<b>RepoRtS</b>&#160;|&#160;</p>
<p style="position:absolute;top:1135px;left:192px;white-space:nowrap" class="ft53"><i>&#160; &#160; &#160; &#160; (2019) 9:18612&#160;</i>&#160;|&#160;<a href="https://doi.org/10.1038/s41598-019-54859-8">https://doi.org/10.1038/s41598-019-54859-8</a></p>
<p style="position:absolute;top:19px;left:585px;white-space:nowrap" class="ft54">www.nature.com/scientificreports</p>
<p style="position:absolute;top:19px;left:61px;white-space:nowrap" class="ft54">www.nature.com/scientificreports/</p>
<p style="position:absolute;top:953px;left:233px;white-space:nowrap" class="ft55">runs&#160;for&#160;each&#160;gate.&#160;Then,&#160;we&#160;describe&#160;the&#160;behavior&#160;of&#160;the&#160;best&#160;gate&#160;in&#160;each&#160;set,&#160;using&#160;time&#160;series&#160;and&#160;dynamical&#160;</p>
<p style="position:absolute;top:968px;left:233px;white-space:nowrap" class="ft55">phase space il&#160;ustrations.</p>
<p style="position:absolute;top:983px;left:251px;white-space:nowrap" class="ft55">We used a combination of genetic algorithm (GA) and BP for identifying this kind of circuit; the GA discovers&#160;</p>
<p style="position:absolute;top:998px;left:233px;white-space:nowrap" class="ft55">the&#160;network&#160;structure&#160;(example&#160;structure&#160;in Fig.&#160;4 inset)&#160;and&#160;BP&#160;finds&#160;the&#160;parameters&#160;of&#160;the&#160;network&#160;we&#160;sought.&#160;</p>
<p style="position:absolute;top:1013px;left:233px;white-space:nowrap" class="ft55">The best network was obtained at the end of 233 generations, which achieved an average error of about 0.6&#160;mV.&#160;</p>
<p style="position:absolute;top:1028px;left:233px;white-space:nowrap" class="ft55">More details of the training are described in the ‘Methods’ section.</p>
<p style="position:absolute;top:1043px;left:251px;white-space:nowrap" class="ft55">We conclude from the above results that it is indeed possible for large BEN circuits to compute simple logic&#160;</p>
<p style="position:absolute;top:1058px;left:233px;white-space:nowrap" class="ft55">functions, suggesting that this type of signaling can be carried out by developmental compartments with varying&#160;</p>
<p style="position:absolute;top:1073px;left:233px;white-space:nowrap" class="ft55">cell numbers.</p>
<p style="position:absolute;top:641px;left:233px;white-space:nowrap" class="ft56"><b>Figure 2.</b>&#160;&#160;The AND gate. (<b>a</b>) A schematic of the AND logic gate and its rule table. The orange and purple cel&#160;s&#160;</p>
<p style="position:absolute;top:656px;left:233px;white-space:nowrap" class="ft55">are the inputs, and the green cell is the output; the grey cel&#160;s are intermediate cel&#160;s offering computational&#160;</p>
<p style="position:absolute;top:671px;left:233px;white-space:nowrap" class="ft55">support.&#160;(<b>b</b>) Specification of the output Vmem&#160;for various combinations of input Vmem&#160;levels. This is known as&#160;</p>
<p style="position:absolute;top:686px;left:233px;white-space:nowrap" class="ft55">the “truth table” in the Boolean logic literature, where the “hyperpolarized” state (around&#160;−80 mV) is indicated&#160;</p>
<p style="position:absolute;top:701px;left:233px;white-space:nowrap" class="ft55">as “0”, “False” or “OFF” and the “depolarized” state (around&#160;+80&#160;mV) as “1”, “True” or “ON”. This rule table&#160;</p>
<p style="position:absolute;top:716px;left:233px;white-space:nowrap" class="ft55">essential&#160;y summarizes the mechanism of the BEN-based AND gate: the output is depolarized only when&#160;</p>
<p style="position:absolute;top:731px;left:233px;white-space:nowrap" class="ft55">both of its inputs are depolarized; in Boolean logic terms the output is ON only when both inputs are ON. (<b>c</b>)&#160;</p>
<p style="position:absolute;top:746px;left:233px;white-space:nowrap" class="ft55">Pareto front of training errors over time (one unit is equal to a single training epoch) for the AND gate. This&#160;</p>
<p style="position:absolute;top:761px;left:233px;white-space:nowrap" class="ft55">plot depicts the “front” with the best errors achieved over time. This figure demonstrates that the training does&#160;</p>
<p style="position:absolute;top:776px;left:233px;white-space:nowrap" class="ft55">indeed result in learning. (<b>d</b>) The behavior of the best AND gate shown in the form of time series of the input&#160;</p>
<p style="position:absolute;top:791px;left:233px;white-space:nowrap" class="ft55">and output nodes of the gate, shown for all four input-output conditions generated in a random sequence.&#160;</p>
<p style="position:absolute;top:806px;left:233px;white-space:nowrap" class="ft55">The red and blue lines represent the states of the two input nodes, and green represents the output. The grey&#160;</p>
<p style="position:absolute;top:821px;left:233px;white-space:nowrap" class="ft55">triangles mark the time points at which the inputs are switched to a different state. For example, both inputs are&#160;</p>
<p style="position:absolute;top:836px;left:233px;white-space:nowrap" class="ft55">hyperpolarized at time point (1), while at time point (2) the red input is depolarized. (<b>e</b>) The dynamical phase of&#160;</p>
<p style="position:absolute;top:851px;left:233px;white-space:nowrap" class="ft55">the logic gate:a depiction of a set of trajectories in the input-output space, il&#160;ustrated in a time-lapse style. This&#160;</p>
<p style="position:absolute;top:866px;left:233px;white-space:nowrap" class="ft55">dynamical system has two attractors in the output space, highlighted in filled red (depolarized state) and blue&#160;</p>
<p style="position:absolute;top:881px;left:233px;white-space:nowrap" class="ft55">(hyperpolarized state) circles. The trajectories look straight because the inputs are fixed, and only the output&#160;</p>
<p style="position:absolute;top:896px;left:233px;white-space:nowrap" class="ft55">changes.</p>
</div>
</body>
</html>
<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="" xml:lang="">
<head>
<title>4b78c8d89b22cb315d90fd1274945de5-html.html</title>

<meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
 <br/>
<style type="text/css">
<!--
	p {margin: 0; padding: 0;}	.ft60{font-size:13px;font-family:Times;color:#cedde4;}
	.ft61{font-size:10px;font-family:Times;color:#000000;}
	.ft62{font-size:10px;font-family:Times;color:#000000;}
	.ft63{font-size:10px;font-family:Times;color:#000000;}
	.ft64{font-size:13px;font-family:Times;color:#000000;}
	.ft65{font-size:13px;font-family:Times;color:#000000;}
	.ft66{font-size:11px;font-family:Times;color:#000000;}
	.ft67{font-size:6px;font-family:Times;color:#0000ff;}
	.ft68{font-size:6px;font-family:Times;color:#000000;}
	.ft69{font-size:11px;font-family:Times;color:#000000;}
-->
</style>
</head>
<body bgcolor="#A0A0A0" vlink="blue" link="blue">
<div id="page6-div" style="position:relative;width:892px;height:1173px;">
<p style="position:absolute;top:1132px;left:824px;white-space:nowrap" class="ft60">6</p>
<p style="position:absolute;top:1135px;left:61px;white-space:nowrap" class="ft61"><b>Scientific</b>&#160;<b>RepoRtS</b>&#160;|&#160;</p>
<p style="position:absolute;top:1135px;left:192px;white-space:nowrap" class="ft63"><i>&#160; &#160; &#160; &#160; (2019) 9:18612&#160;</i>&#160;|&#160;<a href="https://doi.org/10.1038/s41598-019-54859-8">https://doi.org/10.1038/s41598-019-54859-8</a></p>
<p style="position:absolute;top:19px;left:585px;white-space:nowrap" class="ft64">www.nature.com/scientificreports</p>
<p style="position:absolute;top:19px;left:61px;white-space:nowrap" class="ft64">www.nature.com/scientificreports/</p>
<p style="position:absolute;top:984px;left:233px;white-space:nowrap" class="ft65"><b>BEN&#160;can&#160;implement&#160;a&#160;complex&#160;“tissue-level”&#160;logic&#160;gate:&#160;a&#160;pattern&#160;detector.&#160;&#160;</b>One&#160;of&#160;the most&#160;</p>
<p style="position:absolute;top:1000px;left:233px;white-space:nowrap" class="ft66">impressive computational tasks that tissues undertake is the set of decisions that enable large-scale regeneration&#160;</p>
<p style="position:absolute;top:1015px;left:233px;white-space:nowrap" class="ft66">of organs and appendages in some spe<a href="4b78c8d89b22cb315d90fd1274945de5s.html#14">cies19,</a><a href="4b78c8d89b22cb315d90fd1274945de5s.html#15">75</a>. Specifical&#160;y, to repair after (unanticipated) injury and stop growth&#160;</p>
<p style="position:absolute;top:1030px;left:233px;white-space:nowrap" class="ft66">and remodeling when the anatomy has been corrected, cel&#160;s need to make decisions about the physiological and&#160;</p>
<p style="position:absolute;top:1045px;left:233px;white-space:nowrap" class="ft66">geometric state of other tissues. Cel&#160;s need to ascertain whether a large-scale morphology is correct or not, in&#160;</p>
<p style="position:absolute;top:1060px;left:233px;white-space:nowrap" class="ft66">order to regulate regenerative pathways or cease further change (errors in achieving morphostasis can manifest as&#160;</p>
<p style="position:absolute;top:1075px;left:233px;white-space:nowrap" class="ft66">cancer).&#160;Much&#160;work&#160;has&#160;gone&#160;into&#160;characterizing&#160;the&#160;physiological&#160;events&#160;that&#160;signal&#160;the&#160;binary&#160;event&#160;of&#160;“injury”,&#160;</p>
<p style="position:absolute;top:1090px;left:233px;white-space:nowrap" class="ft66">but it is now clear that even in the absence of trauma, some complex organ systems&#160;such as the craniofacial&#160;</p>
<p style="position:absolute;top:711px;left:233px;white-space:nowrap" class="ft69"><b>Figure 3.</b>&#160;&#160;The XOR gate. (<b>a</b>) Schematic of the XOR logic gate; the orange and purple cel&#160;s are the inputs, and&#160;</p>
<p style="position:absolute;top:727px;left:233px;white-space:nowrap" class="ft66">the green cell is the output; the grey cel&#160;s are intermediate cel&#160;s offering computational support. (<b>b</b>) The XOR&#160;</p>
<p style="position:absolute;top:742px;left:233px;white-space:nowrap" class="ft66">rule table: output is depolarized only when one of its inputs, but not both, is depolarized. Table specifies the&#160;</p>
<p style="position:absolute;top:757px;left:233px;white-space:nowrap" class="ft66">output&#160;Vmem&#160;for various combinations of input Vmem&#160;levels; this is known as the “truth table” in the Boolean&#160;</p>
<p style="position:absolute;top:772px;left:233px;white-space:nowrap" class="ft66">logic literature, where the “hyperpolarized” state (around&#160;−80 mV) is indicated as “0”, “False” or “OFF” and&#160;</p>
<p style="position:absolute;top:787px;left:233px;white-space:nowrap" class="ft66">the “depolarized” state (around&#160;+80&#160;mV) as “1”, “True” or “ON”. (<b>c</b>) Pareto front of training errors over time&#160;</p>
<p style="position:absolute;top:802px;left:233px;white-space:nowrap" class="ft66">(one unit is equal to a single training epoch) for the XOR gate. This plot depicts the “front” with the best errors&#160;</p>
<p style="position:absolute;top:817px;left:233px;white-space:nowrap" class="ft66">achieved over time. This figure demonstrates that the training does indeed result in learning, but fewer networks&#160;</p>
<p style="position:absolute;top:832px;left:233px;white-space:nowrap" class="ft66">are successful compared to AND (Fig. 2). (<b>d</b>) Behavior of the best XOR gate. Shown here are the time series&#160;</p>
<p style="position:absolute;top:847px;left:233px;white-space:nowrap" class="ft66">of the input and output nodes of the gate, shown for all four input-output conditions generated in a random&#160;</p>
<p style="position:absolute;top:862px;left:233px;white-space:nowrap" class="ft66">sequence. The red and blue lines represent the states of the two input nodes, and green represents the output.&#160;</p>
<p style="position:absolute;top:877px;left:233px;white-space:nowrap" class="ft66">The grey triangles mark the time points at which the inputs are switched to a different state. (<b>e</b>) The dynamical&#160;</p>
<p style="position:absolute;top:892px;left:233px;white-space:nowrap" class="ft66">phase space of the gate: a depiction of a set of trajectories in the input-output space, il&#160;ustrated in a time-lapse&#160;</p>
<p style="position:absolute;top:907px;left:233px;white-space:nowrap" class="ft66">style. This dynamical system has two attractors in the output space, highlighted in filled red (depolarized state)&#160;</p>
<p style="position:absolute;top:922px;left:233px;white-space:nowrap" class="ft66">and blue (hyperpolarized state) circles. The trajectories look straight because the inputs are fixed, and only the&#160;</p>
<p style="position:absolute;top:937px;left:233px;white-space:nowrap" class="ft66">output changes.</p>
</div>
</body>
</html>
<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="" xml:lang="">
<head>
<title>4b78c8d89b22cb315d90fd1274945de5-html.html</title>

<meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
 <br/>
<style type="text/css">
<!--
	p {margin: 0; padding: 0;}	.ft70{font-size:13px;font-family:Times;color:#cedde4;}
	.ft71{font-size:10px;font-family:Times;color:#000000;}
	.ft72{font-size:10px;font-family:Times;color:#000000;}
	.ft73{font-size:10px;font-family:Times;color:#000000;}
	.ft74{font-size:13px;font-family:Times;color:#000000;}
	.ft75{font-size:11px;font-family:Times;color:#000000;}
	.ft76{font-size:6px;font-family:Times;color:#0000ff;}
	.ft77{font-size:6px;font-family:Times;color:#000000;}
	.ft78{font-size:11px;font-family:Times;color:#000000;}
	.ft79{font-size:11px;font-family:Times;color:#000000;}
	.ft710{font-size:11px;font-family:Times;color:#0000ff;}
-->
</style>
</head>
<body bgcolor="#A0A0A0" vlink="blue" link="blue">
<div id="page7-div" style="position:relative;width:892px;height:1173px;">
<p style="position:absolute;top:1132px;left:826px;white-space:nowrap" class="ft70">7</p>
<p style="position:absolute;top:1135px;left:61px;white-space:nowrap" class="ft71"><b>Scientific</b>&#160;<b>RepoRtS</b>&#160;|&#160;</p>
<p style="position:absolute;top:1135px;left:192px;white-space:nowrap" class="ft73"><i>&#160; &#160; &#160; &#160; (2019) 9:18612&#160;</i>&#160;|&#160;<a href="https://doi.org/10.1038/s41598-019-54859-8">https://doi.org/10.1038/s41598-019-54859-8</a></p>
<p style="position:absolute;top:19px;left:585px;white-space:nowrap" class="ft74">www.nature.com/scientificreports</p>
<p style="position:absolute;top:19px;left:61px;white-space:nowrap" class="ft74">www.nature.com/scientificreports/</p>
<p style="position:absolute;top:890px;left:233px;white-space:nowrap" class="ft75">structures<a href="4b78c8d89b22cb315d90fd1274945de5s.html#14">13,</a><a href="4b78c8d89b22cb315d90fd1274945de5s.html#15">76</a>&#160;can begin drastic remodeling when the configuration is incorrect. Despite the advances in molec-</p>
<p style="position:absolute;top:905px;left:233px;white-space:nowrap" class="ft75">ular biology that have identified genes&#160;<i>necessary</i>&#160;for this to occur, it is entirely unclear how cel&#160;s recognize correct&#160;</p>
<p style="position:absolute;top:920px;left:233px;white-space:nowrap" class="ft75">vs.&#160;incorrect&#160;patterns&#160;on&#160;a&#160;spatial&#160;scale&#160;much&#160;larger&#160;than&#160;themselves.&#160;Thus,&#160;we&#160;next&#160;sought&#160;to&#160;demonstrate&#160;that&#160;</p>
<p style="position:absolute;top:935px;left:233px;white-space:nowrap" class="ft75">realistic biophysical mechanisms can implement computations which are&#160;<i>sufficient</i>&#160;to enable this crucial capabil-</p>
<p style="position:absolute;top:950px;left:233px;white-space:nowrap" class="ft75">ity of living systems.</p>
<p style="position:absolute;top:965px;left:251px;white-space:nowrap" class="ft75">A pattern detector can be thought of as a complex (more than two inputs, thus non-elementary) AND gate,&#160;</p>
<p style="position:absolute;top:980px;left:233px;white-space:nowrap" class="ft75">since&#160;its&#160;output is&#160;ON&#160;only&#160;when&#160;each&#160;of&#160;the&#160;inputs&#160;is&#160;at&#160;a&#160;specific&#160;desired&#160;state—in&#160;other&#160;words,&#160;only&#160;when&#160;<i>all</i>&#160;</p>
<p style="position:absolute;top:995px;left:233px;white-space:nowrap" class="ft75">input conditions&#160;are&#160;satisfied.&#160;This&#160;is&#160;a&#160;more&#160;realistic&#160;setting&#160;that&#160;the&#160;tissue-level&#160;AND&#160;described&#160;in&#160;the&#160;previous&#160;</p>
<p style="position:absolute;top:1010px;left:233px;white-space:nowrap" class="ft75">section, since it has many more than two inputs (a tissue typical&#160;y consists of many more than two cel&#160;s). We iden-</p>
<p style="position:absolute;top:1025px;left:233px;white-space:nowrap" class="ft75">tified, as proof-of-principle, a relatively large (44 cel&#160;s) non-elementary logic gate that recognizes French-flag-like&#160;</p>
<p style="position:absolute;top:1040px;left:233px;white-space:nowrap" class="ft75">patterns; we&#160;used the&#160;French-flag as the&#160;pattern of&#160;interest due to its&#160;prominence in developmental biology as&#160;</p>
<p style="position:absolute;top:1055px;left:233px;white-space:nowrap" class="ft75">a simple morphogen gradient<a href="4b78c8d89b22cb315d90fd1274945de5s.html#15">77. W</a>e define a French-flag pattern as a particular configuration of Vmem&#160;levels in&#160;</p>
<p style="position:absolute;top:1070px;left:233px;white-space:nowrap" class="ft75">the input layer (3&#160;×&#160;6): the leftmost two columns (band) are blue (hyperpolarized Vmem), the middle band is grey&#160;</p>
<p style="position:absolute;top:1085px;left:233px;white-space:nowrap" class="ft75">(intermediate Vmem), and the rightmost band is red (depolarized Vmem). This network recognizes the French-flag&#160;</p>
<p style="position:absolute;top:342px;left:233px;white-space:nowrap" class="ft79"><b>Figure 4.</b>&#160;&#160;The structure and behavior of the best evolved tissue-level AND gate. Following the convention of&#160;</p>
<p style="position:absolute;top:358px;left:233px;white-space:nowrap" class="ft75">the color-coding, orange and purple represent input nodes, while green represents the output node. Behavior&#160;</p>
<p style="position:absolute;top:373px;left:233px;white-space:nowrap" class="ft75">correctly follows the AND rule (Fig<a href="4b78c8d89b22cb315d90fd1274945de5s.html#5">. 2b</a>). Inputs are transient: they are set (external&#160;y) to their respective states at&#160;</p>
<p style="position:absolute;top:388px;left:233px;white-space:nowrap" class="ft75">the time steps marked by the grey triangles and then removed. As can be seen, they slightly oscil&#160;ate for a small&#160;</p>
<p style="position:absolute;top:403px;left:233px;white-space:nowrap" class="ft75">period of time after they are set initial&#160;y, before dynamical&#160;y fixing themselves (marked by grey dashed arrows)&#160;</p>
<p style="position:absolute;top:418px;left:233px;white-space:nowrap" class="ft75">during every simulation. Thus, the network has evolved the capacity for memory: it remembers the inputs (at&#160;</p>
<p style="position:absolute;top:433px;left:233px;white-space:nowrap" class="ft75">least their qualitative levels) even after they are removed. Inset shows the structure of the network.</p>
<p style="position:absolute;top:721px;left:233px;white-space:nowrap" class="ft79"><b>Figure 5.</b>&#160;&#160;The dynamical phase space of the best tissue-level AND gate. A depiction of a set of trajectories&#160;</p>
<p style="position:absolute;top:736px;left:233px;white-space:nowrap" class="ft75">in the input-output space, il&#160;ustrated in a time-lapse style. This dynamical system has two attractors in the&#160;</p>
<p style="position:absolute;top:751px;left:233px;white-space:nowrap" class="ft75">output space, highlighted in filled blue (hyperpolarized state) and red (depolarized state) circles. As can be&#160;</p>
<p style="position:absolute;top:766px;left:233px;white-space:nowrap" class="ft75">seen, there are three standard hyperpolarized attractors (fixed-point-like), and a fourth cyclic hyperpolarized&#160;</p>
<p style="position:absolute;top:781px;left:233px;white-space:nowrap" class="ft75">attractor (marked with a blue arrow). Even though the cyclic attractor is in the appropriate region of the phase&#160;</p>
<p style="position:absolute;top:796px;left:233px;white-space:nowrap" class="ft75">space, it was not actual&#160;y required for during the training—it accidental&#160;y emerged. Furthermore, as opposed&#160;</p>
<p style="position:absolute;top:811px;left:233px;white-space:nowrap" class="ft75">to the straight trajectories of the previous phase space diagrams, the trajectories maneuver freely here, and&#160;</p>
<p style="position:absolute;top:826px;left:233px;white-space:nowrap" class="ft75">occasional&#160;y spill over the usual Vmem&#160;range which is about [−100,100] mV. This is because the inputs are&#160;</p>
<p style="position:absolute;top:841px;left:233px;white-space:nowrap" class="ft75">transient and not fixed, hence they change states as well over time.</p>
</div>
</body>
</html>
<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="" xml:lang="">
<head>
<title>4b78c8d89b22cb315d90fd1274945de5-html.html</title>

<meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
 <br/>
<style type="text/css">
<!--
	p {margin: 0; padding: 0;}	.ft80{font-size:13px;font-family:Times;color:#cedde4;}
	.ft81{font-size:10px;font-family:Times;color:#000000;}
	.ft82{font-size:10px;font-family:Times;color:#000000;}
	.ft83{font-size:10px;font-family:Times;color:#000000;}
	.ft84{font-size:13px;font-family:Times;color:#000000;}
	.ft85{font-size:11px;font-family:Times;color:#000000;}
	.ft86{font-size:6px;font-family:Times;color:#000000;}
	.ft87{font-size:11px;font-family:Times;color:#0000ff;}
	.ft88{font-size:11px;font-family:Times;color:#000000;}
-->
</style>
</head>
<body bgcolor="#A0A0A0" vlink="blue" link="blue">
<div id="page8-div" style="position:relative;width:892px;height:1173px;">
<p style="position:absolute;top:1132px;left:824px;white-space:nowrap" class="ft80">8</p>
<p style="position:absolute;top:1135px;left:61px;white-space:nowrap" class="ft81"><b>Scientific</b>&#160;<b>RepoRtS</b>&#160;|&#160;</p>
<p style="position:absolute;top:1135px;left:192px;white-space:nowrap" class="ft83"><i>&#160; &#160; &#160; &#160; (2019) 9:18612&#160;</i>&#160;|&#160;<a href="https://doi.org/10.1038/s41598-019-54859-8">https://doi.org/10.1038/s41598-019-54859-8</a></p>
<p style="position:absolute;top:19px;left:585px;white-space:nowrap" class="ft84">www.nature.com/scientificreports</p>
<p style="position:absolute;top:19px;left:61px;white-space:nowrap" class="ft84">www.nature.com/scientificreports/</p>
<p style="position:absolute;top:843px;left:233px;white-space:nowrap" class="ft85">pattern and slightly noisy versions of it by expressing a depolarized Vmem&#160;in the output cel&#160;, while expressing a&#160;</p>
<p style="position:absolute;top:858px;left:233px;white-space:nowrap" class="ft85">hyperpolarized Vmem&#160;in the output as a response to all other patterns.</p>
<p style="position:absolute;top:873px;left:251px;white-space:nowrap" class="ft85">The pattern detector is set up similar to the tissue-level AND gate. The main differences between the two are:&#160;</p>
<p style="position:absolute;top:888px;left:233px;white-space:nowrap" class="ft85">(1) inputs are fixed in the pattern detector, as was the case for the small logic gates described in Section 4.1; (2)&#160;</p>
<p style="position:absolute;top:903px;left:233px;white-space:nowrap" class="ft85">inputs are set after an initial time period during which the network is allowed to settle at its intrinsic “baseline”&#160;</p>
<p style="position:absolute;top:918px;left:233px;white-space:nowrap" class="ft85">state (this captures the essence of&#160;biological systems that&#160;have intrinsic activity even in the&#160;absence&#160;of exter-</p>
<p style="position:absolute;top:933px;left:233px;white-space:nowrap" class="ft85">nal&#160;stimuli);&#160;and&#160;(3)&#160;input&#160;patterns&#160;are&#160;generated&#160;randomly&#160;where&#160;two&#160;sets&#160;are generated,&#160;one&#160;consisting&#160;of&#160;the&#160;</p>
<p style="position:absolute;top:948px;left:233px;white-space:nowrap" class="ft85">French-flag and&#160;noisy&#160;variants&#160;of&#160;it,&#160;and&#160;the&#160;other&#160;consisting&#160;ful&#160;y&#160;randomized versions&#160;of&#160;the&#160;French-flag&#160;(details&#160;</p>
<p style="position:absolute;top:963px;left:233px;white-space:nowrap" class="ft85">in the ‘Methods’ section). We used the combined GA-BP search-train method, as before, to find this network;&#160;</p>
<p style="position:absolute;top:978px;left:233px;white-space:nowrap" class="ft85">see&#160;more&#160;details in the ‘Methods’&#160;section. The best evolved pattern detector&#160;is the individual with the&#160;highest&#160;</p>
<p style="position:absolute;top:993px;left:233px;white-space:nowrap" class="ft85">fitness score in the last generation of the GA, which was discovered relatively quickly at the end of 11 generations&#160;</p>
<p style="position:absolute;top:1008px;left:233px;white-space:nowrap" class="ft85">(three other networks achieved similar errors). Even though the pattern detector is larger than the tissue-level&#160;</p>
<p style="position:absolute;top:1023px;left:233px;white-space:nowrap" class="ft85">AND described above, it was easier to train than the latter. This is due to the advantage of the inputs being fixed&#160;</p>
<p style="position:absolute;top:1038px;left:233px;white-space:nowrap" class="ft85">in&#160;the&#160;case&#160;of&#160;the&#160;pattern&#160;detector&#160;but&#160;transient&#160;in&#160;the&#160;case&#160;of&#160;the&#160;tissue-level&#160;AND gate:&#160;the&#160;fixed&#160;inputs&#160;provide&#160;</p>
<p style="position:absolute;top:1053px;left:233px;white-space:nowrap" class="ft85">a constant supply of information that the network does not have to remember unlike the tissue-level AND gate.</p>
<p style="position:absolute;top:1068px;left:251px;white-space:nowrap" class="ft85">The above search successful&#160;y discovered a successful French-flag detector. The behavior of this detector when&#160;</p>
<p style="position:absolute;top:1083px;left:233px;white-space:nowrap" class="ft85">it sees a French-flag pattern and a random pattern in its input layer is depicted in Fig. <a href="4b78c8d89b22cb315d90fd1274945de5s.html#8">6a</a>: in the first case the&#160;</p>
<p style="position:absolute;top:525px;left:233px;white-space:nowrap" class="ft88"><b>Figure 6.</b>&#160;&#160;The French-flag detector. (<b>a</b>) Snapshots of the equilibrium (final) network states (Vmem&#160;levels) in two&#160;</p>
<p style="position:absolute;top:540px;left:233px;white-space:nowrap" class="ft85">detection scenarios. Left: input pattern is the French flag, and the output is depolarized (representing “correct”&#160;</p>
<p style="position:absolute;top:555px;left:233px;white-space:nowrap" class="ft85">pattern). Right: input pattern is a highly distorted version of the French flag, and the output is hyperpolarized&#160;</p>
<p style="position:absolute;top:570px;left:233px;white-space:nowrap" class="ft85">(representing “incorrect” pattern). Colors represent polarity levels of the cel&#160;s in units of mV. Notice that the&#160;</p>
<p style="position:absolute;top:585px;left:233px;white-space:nowrap" class="ft85">only three intermediate cel&#160;s that differ in their states between the two cases are those marked with a black&#160;</p>
<p style="position:absolute;top:600px;left:233px;white-space:nowrap" class="ft85">star, suggesting that the detector makes minimal use of information to distinguish between patterns. Also&#160;</p>
<p style="position:absolute;top:615px;left:233px;white-space:nowrap" class="ft85">notice that the edges are thicker in the right than in the left, suggesting that the voltage-gating dynamics of the&#160;</p>
<p style="position:absolute;top:630px;left:233px;white-space:nowrap" class="ft85">edges (representing gap junctions) play a crucial role. (<b>b</b>) The overall performance of the best evolved pattern&#160;</p>
<p style="position:absolute;top:645px;left:233px;white-space:nowrap" class="ft85">detector, with data collected from a set of 1000 simulations: 100 parallel sets each with a random sequence of 10&#160;</p>
<p style="position:absolute;top:660px;left:233px;white-space:nowrap" class="ft85">simulations. As expected, the pattern detector classifies patterns similar to the French flag (a total of 500 sample&#160;</p>
<p style="position:absolute;top:675px;left:233px;white-space:nowrap" class="ft85">inputs) as “French flag”, and those that are dissimilar (500 inputs) as “not French flag”. This suggests that the&#160;</p>
<p style="position:absolute;top:690px;left:233px;white-space:nowrap" class="ft85">pattern detector is robust to noise to a sensible extent. The width of the classification boundary was set implicitly&#160;</p>
<p style="position:absolute;top:705px;left:233px;white-space:nowrap" class="ft85">due to the way sample input patterns from the two classes were generated (details in the ‘Methods’ section). (<b>c</b>)&#160;</p>
<p style="position:absolute;top:720px;left:233px;white-space:nowrap" class="ft85">The behavior of the best French-flag detector shown for a set of four representative cel&#160;s (three inputs out of a&#160;</p>
<p style="position:absolute;top:735px;left:233px;white-space:nowrap" class="ft85">total of eighteen and one output) for a random sequence of five patterns. Inset highlights the four nodes whose&#160;</p>
<p style="position:absolute;top:750px;left:233px;white-space:nowrap" class="ft85">colors correspond with those in the time series. Highlighted in grey squares are the cases where a French-flag-</p>
<p style="position:absolute;top:765px;left:233px;white-space:nowrap" class="ft85">like pattern is input for which the output is depolarized, as expected; for all other cases where a random pattern&#160;</p>
<p style="position:absolute;top:780px;left:233px;white-space:nowrap" class="ft85">is shown, the output is hyperpolarized.</p>
</div>
</body>
</html>
<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="" xml:lang="">
<head>
<title>4b78c8d89b22cb315d90fd1274945de5-html.html</title>

<meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
 <br/>
<style type="text/css">
<!--
	p {margin: 0; padding: 0;}	.ft90{font-size:13px;font-family:Times;color:#cedde4;}
	.ft91{font-size:10px;font-family:Times;color:#000000;}
	.ft92{font-size:10px;font-family:Times;color:#000000;}
	.ft93{font-size:10px;font-family:Times;color:#000000;}
	.ft94{font-size:13px;font-family:Times;color:#000000;}
	.ft95{font-size:11px;font-family:Times;color:#000000;}
	.ft96{font-size:11px;font-family:Times;color:#0000ff;}
	.ft97{font-size:11px;font-family:Times;color:#000000;}
	.ft98{font-size:13px;font-family:Times;color:#000000;}
	.ft99{font-size:6px;font-family:Times;color:#0000ff;}
	.ft910{font-size:6px;font-family:Times;color:#000000;}
	.ft911{font-size:14px;font-family:Times;color:#000000;}
	.ft912{font-size:11px;font-family:Times;color:#000000;}
-->
</style>
</head>
<body bgcolor="#A0A0A0" vlink="blue" link="blue">
<div id="page9-div" style="position:relative;width:892px;height:1173px;">
<p style="position:absolute;top:1132px;left:824px;white-space:nowrap" class="ft90">9</p>
<p style="position:absolute;top:1135px;left:61px;white-space:nowrap" class="ft91"><b>Scientific</b>&#160;<b>RepoRtS</b>&#160;|&#160;</p>
<p style="position:absolute;top:1135px;left:192px;white-space:nowrap" class="ft93"><i>&#160; &#160; &#160; &#160; (2019) 9:18612&#160;</i>&#160;|&#160;<a href="https://doi.org/10.1038/s41598-019-54859-8">https://doi.org/10.1038/s41598-019-54859-8</a></p>
<p style="position:absolute;top:19px;left:585px;white-space:nowrap" class="ft94">www.nature.com/scientificreports</p>
<p style="position:absolute;top:19px;left:61px;white-space:nowrap" class="ft94">www.nature.com/scientificreports/</p>
<p style="position:absolute;top:402px;left:233px;white-space:nowrap" class="ft95">output is depolarized (red), while in the second case it is hyperpolarized (blue), both as expected. Furthermore,&#160;</p>
<p style="position:absolute;top:417px;left:233px;white-space:nowrap" class="ft95">this&#160;detector recognizes slightly distorted variants of the&#160;French-flag where the distance of a pattern&#160;from the&#160;</p>
<p style="position:absolute;top:432px;left:233px;white-space:nowrap" class="ft95">French-flag is the Euclidean distance (Fig<a href="4b78c8d89b22cb315d90fd1274945de5s.html#8">. 6b). W</a>e observe that input patterns up to a distance of about 150&#160;mV&#160;</p>
<p style="position:absolute;top:447px;left:233px;white-space:nowrap" class="ft95">from&#160;the French-flag are&#160;recognized&#160;as French-flag, while those at a distance&#160;of about 350&#160;mV and above are&#160;</p>
<p style="position:absolute;top:462px;left:233px;white-space:nowrap" class="ft95">classed as not French-flag. Final&#160;y, the detector also responds correctly to a&#160;<i>sequence</i>&#160;of randomly chosen input&#160;</p>
<p style="position:absolute;top:477px;left:233px;white-space:nowrap" class="ft95">patterns (Fig<a href="4b78c8d89b22cb315d90fd1274945de5s.html#8">. 6c), s</a>howing that the behavior of the network only depends on the state of the inputs and not on&#160;</p>
<p style="position:absolute;top:492px;left:233px;white-space:nowrap" class="ft95">the rest of the network.</p>
<p style="position:absolute;top:507px;left:251px;white-space:nowrap" class="ft95">We&#160;conclude&#160;that&#160;BEN&#160;has&#160;in&#160;principle&#160;the&#160;ability&#160;to&#160;distinguish&#160;between&#160;patterns.&#160;The&#160;biological&#160;implication&#160;</p>
<p style="position:absolute;top:522px;left:233px;white-space:nowrap" class="ft95">is that somatic tissues can in principle have&#160;the same ability, and thus be a part of much larger pattern regulation&#160;</p>
<p style="position:absolute;top:537px;left:233px;white-space:nowrap" class="ft95">mechanisms.</p>
<p style="position:absolute;top:566px;left:233px;white-space:nowrap" class="ft98"><b>Ben&#160;can implement&#160;compound logic gates.&#160;&#160;</b>Biological&#160;processes&#160;are complex by nature. The&#160;rampant&#160;</p>
<p style="position:absolute;top:582px;left:233px;white-space:nowrap" class="ft95">complexity&#160;is&#160;partly&#160;managed&#160;by&#160;nature&#160;by&#160;way&#160;of&#160;modularity<a href="4b78c8d89b22cb315d90fd1274945de5s.html#15">78</a>,&#160;in&#160;gene&#160;regulatio<a href="4b78c8d89b22cb315d90fd1274945de5s.html#15">n79&#160;</a>and&#160;the&#160;brain<a href="4b78c8d89b22cb315d90fd1274945de5s.html#15">80,81,&#160;</a>for&#160;exam-</p>
<p style="position:absolute;top:597px;left:233px;white-space:nowrap" class="ft95">ple, at multiple scales. Modularity is beneficial because the modules can be independently tinkered with for the&#160;</p>
<p style="position:absolute;top:612px;left:233px;white-space:nowrap" class="ft95">purpose of large-scale outcomes<a href="4b78c8d89b22cb315d90fd1274945de5s.html#15">79</a>. Thus, it is important to understand whether bioelectric circuits can implement&#160;</p>
<p style="position:absolute;top:627px;left:233px;white-space:nowrap" class="ft95">logic functions that are too complex to be described by a single gate but are actual&#160;y combinations of elementary&#160;</p>
<p style="position:absolute;top:642px;left:233px;white-space:nowrap" class="ft95">logic primitives.</p>
<p style="position:absolute;top:657px;left:251px;white-space:nowrap" class="ft95">Compound logic gates are compositions of modules of logic gates, and their biological equivalents underlie&#160;</p>
<p style="position:absolute;top:672px;left:233px;white-space:nowrap" class="ft95">several processes including sporulation in&#160;<i>B.&#160;subtilis</i>&#160;and the neuronal dynamics of&#160;<i>C.&#160;elegans</i><a href="4b78c8d89b22cb315d90fd1274945de5s.html#15">82,83. Th</a>ey can also&#160;</p>
<p style="position:absolute;top:687px;left:233px;white-space:nowrap" class="ft95">have&#160;pharmaceutical&#160;applications—for&#160;example,&#160;the&#160;control&#160;of&#160;bacterial&#160;invasion&#160;of&#160;tumor&#160;cel&#160;s&#160;and&#160;diagnosis&#160;of&#160;</p>
<p style="position:absolute;top:702px;left:233px;white-space:nowrap" class="ft95">cel&#160;ular environments and automatic release of drugs<a href="4b78c8d89b22cb315d90fd1274945de5s.html#15">82</a>.</p>
<p style="position:absolute;top:717px;left:251px;white-space:nowrap" class="ft95">One way to design compound networks, in general, is to compose them from appropriate pre-designed mod-</p>
<p style="position:absolute;top:732px;left:233px;white-space:nowrap" class="ft95">ules. For example, a “NAND” gate can be constructed by combining the AND and NOT gates. Compound genetic&#160;</p>
<p style="position:absolute;top:747px;left:233px;white-space:nowrap" class="ft95">circuits&#160;have&#160;been&#160;constructed&#160;in&#160;this&#160;fashio<a href="4b78c8d89b22cb315d90fd1274945de5s.html#15">n84,85</a>,&#160;as&#160;well&#160;as&#160;designed&#160;<i>de&#160;novo</i>&#160;from&#160;scratch<a href="4b78c8d89b22cb315d90fd1274945de5s.html#15">82</a>.&#160;Here,&#160;we&#160;show&#160;how&#160;</p>
<p style="position:absolute;top:762px;left:233px;white-space:nowrap" class="ft95">to construct compound logic gates in physiological networks by composing pre-trained BEN logic gate modules.&#160;</p>
<p style="position:absolute;top:777px;left:233px;white-space:nowrap" class="ft95">Unlike genetic networks and neural networks, which are directed, BEN networks are bidirectional, due to the&#160;</p>
<p style="position:absolute;top:792px;left:233px;white-space:nowrap" class="ft95">symmetric nature of gap junctions. Thus, unlike genetic networks, compound logic gates in BEN cannot be con-</p>
<p style="position:absolute;top:807px;left:233px;white-space:nowrap" class="ft95">structed simply by connecting the modules. Put more precisely, if the modules were simply connected together,&#160;</p>
<p style="position:absolute;top:822px;left:233px;white-space:nowrap" class="ft95">then&#160;the&#160;output&#160;of&#160;one&#160;module&#160;will&#160;also receive&#160;signals&#160;from&#160;the&#160;input&#160;of&#160;the&#160;connected&#160;module.&#160;This&#160;could&#160;poten-</p>
<p style="position:absolute;top:837px;left:233px;white-space:nowrap" class="ft95">tial&#160;y result in the upstream module outputting the wrong state, and thus the downstream module receiving and&#160;</p>
<p style="position:absolute;top:852px;left:233px;white-space:nowrap" class="ft95">outputting the wrong states as wel&#160;. To mitigate this, we included a “bridge” layer that interfaces the modules and&#160;</p>
<p style="position:absolute;top:867px;left:233px;white-space:nowrap" class="ft95">trained it so that the compound gate as a whole behaves as desired: the downstream module outputs the correct&#160;</p>
<p style="position:absolute;top:882px;left:233px;white-space:nowrap" class="ft95">state for every set of inputs received by the upstream module. For more details on the architecture and training of&#160;</p>
<p style="position:absolute;top:897px;left:233px;white-space:nowrap" class="ft95">the bridge, see Supplementary 4. Figur<a href="4b78c8d89b22cb315d90fd1274945de5s.html#9">e 7&#160;s</a>hows an example of a successful&#160;y trained NAND gate by composing&#160;</p>
<p style="position:absolute;top:912px;left:233px;white-space:nowrap" class="ft95">pre-trained AND and NOT gates.</p>
<p style="position:absolute;top:927px;left:251px;white-space:nowrap" class="ft95">We conclude that it is possible to construct compound logic gates by composing pre-trained modules and&#160;</p>
<p style="position:absolute;top:942px;left:233px;white-space:nowrap" class="ft95">training only an interface bridge connecting the modules. This method can in principle be extended to more&#160;</p>
<p style="position:absolute;top:957px;left:233px;white-space:nowrap" class="ft95">complex circuits.</p>
<p style="position:absolute;top:984px;left:233px;white-space:nowrap" class="ft911"><b>Discussion</b></p>
<p style="position:absolute;top:1002px;left:233px;white-space:nowrap" class="ft95">Various forms of natural and artificial computing systems with unconventional mechanisms have been pro-</p>
<p style="position:absolute;top:1017px;left:233px;white-space:nowrap" class="ft95">posed<a href="4b78c8d89b22cb315d90fd1274945de5s.html#15">86. E</a>ven the modern von Neumann computer architecture was inspired by the mechanisms of DNA&#160;</p>
<p style="position:absolute;top:1032px;left:233px;white-space:nowrap" class="ft95">transcription and translation<a href="4b78c8d89b22cb315d90fd1274945de5s.html#15">87. O</a>ther examples include “liquid computers” that involve interactions of fluid&#160;</p>
<p style="position:absolute;top:1047px;left:233px;white-space:nowrap" class="ft95">streams and droplet flows to perform logic operations<a href="4b78c8d89b22cb315d90fd1274945de5s.html#15">88</a>; “chemical computers” that utilize the dynamics and&#160;</p>
<p style="position:absolute;top:1062px;left:233px;white-space:nowrap" class="ft95">equilibrium-behavior of chemical reaction<a href="4b78c8d89b22cb315d90fd1274945de5s.html#15">s89–93</a>&#160;for logic operations and learning<a href="4b78c8d89b22cb315d90fd1274945de5s.html#15">94,95</a>; DNA-based computers&#160;</p>
<p style="position:absolute;top:1077px;left:233px;white-space:nowrap" class="ft95">that utilize various forms of catalytic reactions such as ‘seesawing’ and ‘strand-displacement’ for pattern recog-</p>
<p style="position:absolute;top:1092px;left:233px;white-space:nowrap" class="ft95">nition<a href="4b78c8d89b22cb315d90fd1274945de5s.html#15">96</a>,&#160;protein-DNA&#160;interactions&#160;to&#160;implement&#160;logic&#160;functions<a href="4b78c8d89b22cb315d90fd1274945de5s.html#15">97&#160;</a>and&#160;even&#160;for&#160;general-purpose&#160;computin<a href="4b78c8d89b22cb315d90fd1274945de5s.html#15">g98;&#160;</a></p>
<p style="position:absolute;top:289px;left:233px;white-space:nowrap" class="ft912"><b>Figure 7.</b>&#160;&#160;The behavior of a successful&#160;y trained NAND gate for a random sequence of four inputs. Orange and&#160;</p>
<p style="position:absolute;top:304px;left:233px;white-space:nowrap" class="ft95">purple lines represent the activities of the inputs, while green represents the output. The vertical dashed lines&#160;</p>
<p style="position:absolute;top:319px;left:233px;white-space:nowrap" class="ft95">mark the beginning of each trial in the sequence, however the inputs are applied, after an initial transient, at the&#160;</p>
<p style="position:absolute;top:334px;left:233px;white-space:nowrap" class="ft95">time points marked by the grey triangles. Inset is a schematic of the architecture of the gate: pretrained AND&#160;</p>
<p style="position:absolute;top:349px;left:233px;white-space:nowrap" class="ft95">and NOT gates are connected by a “bridge”, a single layer of cel&#160;s, that is alone trained.</p>
</div>
</body>
</html>
<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="" xml:lang="">
<head>
<title>4b78c8d89b22cb315d90fd1274945de5-html.html</title>

<meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
 <br/>
<style type="text/css">
<!--
	p {margin: 0; padding: 0;}	.ft100{font-size:13px;font-family:Times;color:#cedde4;}
	.ft101{font-size:10px;font-family:Times;color:#000000;}
	.ft102{font-size:10px;font-family:Times;color:#000000;}
	.ft103{font-size:10px;font-family:Times;color:#000000;}
	.ft104{font-size:13px;font-family:Times;color:#000000;}
	.ft105{font-size:11px;font-family:Times;color:#000000;}
	.ft106{font-size:6px;font-family:Times;color:#0000ff;}
	.ft107{font-size:6px;font-family:Times;color:#000000;}
	.ft108{font-size:11px;font-family:Times;color:#000000;}
-->
</style>
</head>
<body bgcolor="#A0A0A0" vlink="blue" link="blue">
<div id="page10-div" style="position:relative;width:892px;height:1173px;">
<p style="position:absolute;top:1132px;left:816px;white-space:nowrap" class="ft100">10</p>
<p style="position:absolute;top:1135px;left:61px;white-space:nowrap" class="ft101"><b>Scientific</b>&#160;<b>RepoRtS</b>&#160;|&#160;</p>
<p style="position:absolute;top:1135px;left:192px;white-space:nowrap" class="ft103"><i>&#160; &#160; &#160; &#160; (2019) 9:18612&#160;</i>&#160;|&#160;<a href="https://doi.org/10.1038/s41598-019-54859-8">https://doi.org/10.1038/s41598-019-54859-8</a></p>
<p style="position:absolute;top:19px;left:585px;white-space:nowrap" class="ft104">www.nature.com/scientificreports</p>
<p style="position:absolute;top:19px;left:61px;white-space:nowrap" class="ft104">www.nature.com/scientificreports/</p>
<p style="position:absolute;top:71px;left:233px;white-space:nowrap" class="ft105">gene regulatory networks that take advantage of the multistability of gene activity for learning<a href="4b78c8d89b22cb315d90fd1274945de5s.html#15">99</a>,&#160;memory<a href="4b78c8d89b22cb315d90fd1274945de5s.html#15">100–</a><a href="4b78c8d89b22cb315d90fd1274945de5s.html#16">102&#160;</a></p>
<p style="position:absolute;top:86px;left:233px;white-space:nowrap" class="ft105">and logic<a href="4b78c8d89b22cb315d90fd1274945de5s.html#15">84,85,101</a><a href="4b78c8d89b22cb315d90fd1274945de5s.html#16">,102; “</a>bacterial computers” that use enzyme-based computing to implement logic gates<a href="4b78c8d89b22cb315d90fd1274945de5s.html#16">103; e</a>lectric&#160;</p>
<p style="position:absolute;top:101px;left:233px;white-space:nowrap" class="ft105">current flow making use of Kirchhoff’s law for solving mazes<a href="4b78c8d89b22cb315d90fd1274945de5s.html#16">104; a</a>nd even “sandstone computers” that utilize&#160;</p>
<p style="position:absolute;top:116px;left:233px;white-space:nowrap" class="ft105">natural erosion processes to implement logic gates<a href="4b78c8d89b22cb315d90fd1274945de5s.html#16">105. B</a>EN is an unconventional computing system, since it is a&#160;</p>
<p style="position:absolute;top:131px;left:233px;white-space:nowrap" class="ft105">non-neural system like the examples above.</p>
<p style="position:absolute;top:146px;left:251px;white-space:nowrap" class="ft105">Our work does not require a commitment to the view that every biological question must be dealt with via&#160;</p>
<p style="position:absolute;top:161px;left:233px;white-space:nowrap" class="ft105">a computational perspective, despite the fact that there is a robust body of new experimental work driven by&#160;</p>
<p style="position:absolute;top:176px;left:233px;white-space:nowrap" class="ft105">the hypothesis that some tissues are indeed implementing processes we would recognize as memory, compari-</p>
<p style="position:absolute;top:191px;left:233px;white-space:nowrap" class="ft105">son, or measurement<a href="4b78c8d89b22cb315d90fd1274945de5s.html#14">53</a><a href="4b78c8d89b22cb315d90fd1274945de5s.html#16">,106–108</a>. The key question we answer here is: are pre-neural physiological networks able, in&#160;</p>
<p style="position:absolute;top:206px;left:233px;white-space:nowrap" class="ft105">principle, to perform simple computations such as logical operations? Prior to these results, it was not known,&#160;</p>
<p style="position:absolute;top:221px;left:233px;white-space:nowrap" class="ft105">and many developmental biologists’ intuitions are that such networks cannot perform such functions - they are&#160;</p>
<p style="position:absolute;top:236px;left:233px;white-space:nowrap" class="ft105">almost never considered for model-building. Our data provide the first proof-of-existence showing that the class&#160;</p>
<p style="position:absolute;top:251px;left:233px;white-space:nowrap" class="ft105">of biological systems where a computational approach&#160;<i>could</i>&#160;be useful, such as evolved or synthetic regulative&#160;</p>
<p style="position:absolute;top:266px;left:233px;white-space:nowrap" class="ft105">morphogenesis, needs to be expanded to include somatic bioelectric networks.</p>
<p style="position:absolute;top:281px;left:251px;white-space:nowrap" class="ft105">BEN is also a type of patterning system. Patterning is the ability of a system to generate, sustain, and recog-</p>
<p style="position:absolute;top:296px;left:233px;white-space:nowrap" class="ft105">nize patterns in the states, whether physical, chemical, or biological, of the system. It is a widespread phenome-</p>
<p style="position:absolute;top:311px;left:233px;white-space:nowrap" class="ft105">non&#160;reported&#160;in biological&#160;systems,&#160;including&#160;pattern recognition&#160;of pathogen molecules&#160;by cel&#160;ular&#160;receptor<a href="4b78c8d89b22cb315d90fd1274945de5s.html#16">s109,&#160;</a></p>
<p style="position:absolute;top:326px;left:233px;white-space:nowrap" class="ft105">axial patterning&#160;in&#160;planari<a href="4b78c8d89b22cb315d90fd1274945de5s.html#16">a110</a>, spatial&#160;patterning&#160;during neural&#160;patterning<a href="4b78c8d89b22cb315d90fd1274945de5s.html#16">111</a>, cell polarity<a href="4b78c8d89b22cb315d90fd1274945de5s.html#16">112</a>, genetic patterning&#160;</p>
<p style="position:absolute;top:341px;left:233px;white-space:nowrap" class="ft105">in bacterial population<a href="4b78c8d89b22cb315d90fd1274945de5s.html#16">s113</a>&#160;and voltage patterning in neural tubes<a href="4b78c8d89b22cb315d90fd1274945de5s.html#14">49. H</a>ere we focus on bioelectric non-neural&#160;</p>
<p style="position:absolute;top:356px;left:233px;white-space:nowrap" class="ft105">networks&#160;of patterning.&#160;Various&#160;non-neural network models of&#160;patterning&#160;have&#160;been&#160;proposed. Some&#160;examples&#160;</p>
<p style="position:absolute;top:371px;left:233px;white-space:nowrap" class="ft105">include the “packet-routing” system that uses an internet-like intercel&#160;ular messaging system<a href="4b78c8d89b22cb315d90fd1274945de5s.html#16">114</a>, auto-associative&#160;</p>
<p style="position:absolute;top:386px;left:233px;white-space:nowrap" class="ft105">dynamical&#160;systems<a href="4b78c8d89b22cb315d90fd1274945de5s.html#16">115</a>,&#160;and cel&#160;ular automat<a href="4b78c8d89b22cb315d90fd1274945de5s.html#16">a116&#160;b</a>ased&#160;systems&#160;that&#160;combine local and global communication to&#160;</p>
<p style="position:absolute;top:401px;left:233px;white-space:nowrap" class="ft105">model pattern regeneration. Our laboratory has produced sophisticated non-neural bioelectric network models&#160;</p>
<p style="position:absolute;top:416px;left:233px;white-space:nowrap" class="ft105">of tissue patterning, like the&#160;<i>Bioelectric&#160;Tissue&#160;Simulation&#160;Engine</i>&#160;(BETSE) that incorporates detailed biophysical&#160;</p>
<p style="position:absolute;top:431px;left:233px;white-space:nowrap" class="ft105">mechanisms of ion channels and transport processes<a href="4b78c8d89b22cb315d90fd1274945de5s.html#14">47,51,</a><a href="4b78c8d89b22cb315d90fd1274945de5s.html#16">117</a>, as well as relatively simpler models that incorporate&#160;</p>
<p style="position:absolute;top:446px;left:233px;white-space:nowrap" class="ft105">only the high-level&#160;functionality&#160;of&#160;the&#160;bioelectric components<a href="4b78c8d89b22cb315d90fd1274945de5s.html#14">48</a>,<a href="4b78c8d89b22cb315d90fd1274945de5s.html#16">118</a>,<a href="4b78c8d89b22cb315d90fd1274945de5s.html#16">119</a>.&#160;These&#160;models&#160;have&#160;been&#160;used to&#160;show&#160;</p>
<p style="position:absolute;top:461px;left:233px;white-space:nowrap" class="ft105">that networks of ordinary tissues can give rise to gradient-pattern<a href="4b78c8d89b22cb315d90fd1274945de5s.html#14">s47,51</a>, Turing pattern<a href="4b78c8d89b22cb315d90fd1274945de5s.html#14">s21</a><a href="4b78c8d89b22cb315d90fd1274945de5s.html#16">,119</a>, as well as oscil&#160;atory&#160;</p>
<p style="position:absolute;top:476px;left:233px;white-space:nowrap" class="ft105">patterns<a href="4b78c8d89b22cb315d90fd1274945de5s.html#16">118,120</a>&#160;that are thought to be essential for long-distance communication. BEN differs from these models&#160;</p>
<p style="position:absolute;top:491px;left:233px;white-space:nowrap" class="ft105">on two fronts: (1) it is biophysical&#160;y simpler than BETSE, serving as a minimal model of dynamics sufficient for&#160;</p>
<p style="position:absolute;top:506px;left:233px;white-space:nowrap" class="ft105">computation; and (2) it is more realistic than the model of Reference<a href="4b78c8d89b22cb315d90fd1274945de5s.html#16">118,</a>&#160;which uses “equivalent circuits” where&#160;</p>
<p style="position:absolute;top:521px;left:233px;white-space:nowrap" class="ft105">the&#160;equilibrium&#160;Vmem&#160;is&#160;explicitly&#160;specified&#160;in the equations,&#160;whereas&#160;in&#160;BEN&#160;the&#160;equilibrium&#160;Vmem&#160;emerges from&#160;</p>
<p style="position:absolute;top:536px;left:233px;white-space:nowrap" class="ft105">the ion channel parameters.</p>
<p style="position:absolute;top:551px;left:251px;white-space:nowrap" class="ft105">We have shown here that BEN networks can implement elementary logic gates, more complex tissue-level&#160;</p>
<p style="position:absolute;top:566px;left:233px;white-space:nowrap" class="ft105">logic gates, pattern detectors and compound logic gates. Furthermore, we have demonstrated that logic can be&#160;</p>
<p style="position:absolute;top:581px;left:233px;white-space:nowrap" class="ft105">implemented in circuits with bidirectional connections that is typical of non-neural tissues, in contrast to the&#160;</p>
<p style="position:absolute;top:596px;left:233px;white-space:nowrap" class="ft105">conventional&#160;directed&#160;circuits&#160;like&#160;neural&#160;networks&#160;and&#160;digital&#160;electronic&#160;circuits.&#160;This&#160;implies&#160;that&#160;even&#160;though&#160;</p>
<p style="position:absolute;top:611px;left:233px;white-space:nowrap" class="ft105">non-neural tissues may find it harder to implement formal logic (due to lower control over information flow; see&#160;</p>
<p style="position:absolute;top:626px;left:233px;white-space:nowrap" class="ft105">Supplementary Fig. S2b, for example, where information flow is recurrent even in a small network, making it&#160;</p>
<p style="position:absolute;top:641px;left:233px;white-space:nowrap" class="ft105">hard to control), they can achieve it. Not only can BEN networks compute, but they can also be robust to damage.&#160;</p>
<p style="position:absolute;top:656px;left:233px;white-space:nowrap" class="ft105">One of the most important, and heretofore unexplained, properties of biological control circuits is that they con-</p>
<p style="position:absolute;top:671px;left:233px;white-space:nowrap" class="ft105">tinue to function computational&#160;y under significant perturbation (as seen by work on metamorphosis, stability of&#160;</p>
<p style="position:absolute;top:686px;left:233px;white-space:nowrap" class="ft105">memory and behavioral programs during drastic tissue remodeling, and invariance of morphogenesis to changes&#160;</p>
<p style="position:absolute;top:701px;left:233px;white-space:nowrap" class="ft105">in cell number<a href="4b78c8d89b22cb315d90fd1274945de5s.html#15">76</a><a href="4b78c8d89b22cb315d90fd1274945de5s.html#16">,121,122</a>). BEN networks can exhibit a similar behavior, retaining function even after the removal of&#160;</p>
<p style="position:absolute;top:716px;left:233px;white-space:nowrap" class="ft105">cel&#160;s. We found that BEN achieves this by distributing information in the network (Supplementary 2).</p>
<p style="position:absolute;top:731px;left:251px;white-space:nowrap" class="ft105">The particular methods by which BEN networks were trained is not of central importance in this work and&#160;</p>
<p style="position:absolute;top:746px;left:233px;white-space:nowrap" class="ft105">do not impact the main result – that BENs can be readily found which implement logic functions. We used BPTT&#160;</p>
<p style="position:absolute;top:761px;left:233px;white-space:nowrap" class="ft105">simply as a way to discover the parameters of (train) BEN networks that perform specific logic functions or rec-</p>
<p style="position:absolute;top:776px;left:233px;white-space:nowrap" class="ft105">ognize patterns. In the biological world, training (learning) may happen in other ways that may be broadly clas-</p>
<p style="position:absolute;top:791px;left:233px;white-space:nowrap" class="ft105">sified&#160;as&#160;offline&#160;and&#160;online.&#160;Offline&#160;learning&#160;happens&#160;outside&#160;of&#160;the&#160;lifetime&#160;of&#160;an&#160;individual&#160;and&#160;at&#160;a&#160;population&#160;</p>
<p style="position:absolute;top:806px;left:233px;white-space:nowrap" class="ft105">level, while online learning happens during the lifetime of an individual. Evolution is an example of the former,&#160;</p>
<p style="position:absolute;top:821px;left:233px;white-space:nowrap" class="ft105">while Hebbian learning is an example of the latter. BPTT may be classified as a form of online learning, but it&#160;</p>
<p style="position:absolute;top:836px;left:233px;white-space:nowrap" class="ft105">has been&#160;claimed&#160;to be&#160;not biological&#160;y&#160;realistic<a href="4b78c8d89b22cb315d90fd1274945de5s.html#16">123,124. M</a>ore&#160;biological&#160;y&#160;realistic&#160;forms of&#160;online learning have&#160;</p>
<p style="position:absolute;top:851px;left:233px;white-space:nowrap" class="ft105">been proposed, for example, forward propagation of eligibility traces<a href="4b78c8d89b22cb315d90fd1274945de5s.html#16">123</a>, extended Hebbian learnin<a href="4b78c8d89b22cb315d90fd1274945de5s.html#16">g125</a>, dynamic&#160;</p>
<p style="position:absolute;top:866px;left:233px;white-space:nowrap" class="ft105">self-organizing&#160;ma<a href="4b78c8d89b22cb315d90fd1274945de5s.html#16">ps126&#160;</a>and&#160;instantaneously&#160;trained&#160;neural&#160;network<a href="4b78c8d89b22cb315d90fd1274945de5s.html#16">s127</a>.&#160;Such&#160;methods&#160;could&#160;potential&#160;y&#160;advance&#160;</p>
<p style="position:absolute;top:881px;left:233px;white-space:nowrap" class="ft105">biological&#160;y&#160;realistic&#160;training&#160;of&#160;BEN&#160;networks,&#160;which we&#160;wil&#160;&#160;develop&#160;in&#160;future&#160;research,&#160;but&#160;they&#160;are&#160;not neces-</p>
<p style="position:absolute;top:896px;left:233px;white-space:nowrap" class="ft105">sary to find examples demonstrating that BEN networks can compute.</p>
<p style="position:absolute;top:911px;left:251px;white-space:nowrap" class="ft105">BEN models the bioelectric&#160;<i>principles</i>&#160;of a generic bioelectric network, and not&#160;<i>all</i>&#160;the physiological details,&#160;</p>
<p style="position:absolute;top:926px;left:233px;white-space:nowrap" class="ft105">as&#160;these&#160;can&#160;vary&#160;widely&#160;across&#160;tissues.&#160;Individual&#160;cell&#160;types&#160;can&#160;be&#160;modeled&#160;in&#160;future&#160;effort&#160;simply&#160;by&#160;alter-</p>
<p style="position:absolute;top:941px;left:233px;white-space:nowrap" class="ft105">ing&#160;the&#160;ion&#160;channel&#160;parameters&#160;while&#160;retaining&#160;the&#160;basic&#160;form&#160;of&#160;the&#160;equations.&#160;Although&#160;BEN&#160;is&#160;a&#160;model&#160;of&#160;</p>
<p style="position:absolute;top:956px;left:233px;white-space:nowrap" class="ft105">a&#160;non-neural&#160;bioelectric&#160;network,&#160;it&#160;has&#160;certain features that&#160;resemble&#160;those of&#160;a neural network, as&#160;described&#160;</p>
<p style="position:absolute;top:971px;left:233px;white-space:nowrap" class="ft105">above (the weight and bias parameters, and the two-level nonlinear signal transformation). This may suggest&#160;</p>
<p style="position:absolute;top:986px;left:233px;white-space:nowrap" class="ft105">that neural-like features are necessary for computation even in a non-neural network. On the other hand, it also&#160;</p>
<p style="position:absolute;top:1001px;left:233px;white-space:nowrap" class="ft105">suggests that non-neural features may support neural computation. Indeed, this is the subject matter of ongoing&#160;</p>
<p style="position:absolute;top:1016px;left:233px;white-space:nowrap" class="ft105">debates in neuroscience about how much glia contribute to neural computatio<a href="4b78c8d89b22cb315d90fd1274945de5s.html#16">n128,129.</a></p>
<p style="position:absolute;top:1031px;left:251px;white-space:nowrap" class="ft105">Our&#160;work&#160;contributes&#160;to&#160;the&#160;field&#160;of&#160;basal&#160;cognition&#160;by&#160;describing&#160;how&#160;a&#160;simple&#160;network&#160;of&#160;aneural&#160;cel&#160;s&#160;may&#160;</p>
<p style="position:absolute;top:1046px;left:233px;white-space:nowrap" class="ft105">perform both basic and complex logic operations. However, our goal is not to validate basal cognition as a field&#160;</p>
<p style="position:absolute;top:1061px;left:233px;white-space:nowrap" class="ft105">– it&#160;is just&#160;one context&#160;(besides&#160;others,&#160;such&#160;as developmental&#160;neurobiology, evolutionary cognitive science, and&#160;</p>
<p style="position:absolute;top:1076px;left:233px;white-space:nowrap" class="ft105">synthetic bioengineering) which our findings impact. Even if some computational views of specific biological&#160;</p>
<p style="position:absolute;top:1091px;left:233px;white-space:nowrap" class="ft105">systems&#160;turn&#160;out&#160;to&#160;be&#160;wrong,&#160;our&#160;results&#160;provide&#160;value&#160;in&#160;showing&#160;how&#160;physiological&#160;circuits&#160;can&#160;be&#160;made&#160;with&#160;</p>
</div>
</body>
</html>
<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="" xml:lang="">
<head>
<title>4b78c8d89b22cb315d90fd1274945de5-html.html</title>

<meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
 <br/>
<style type="text/css">
<!--
	p {margin: 0; padding: 0;}	.ft110{font-size:13px;font-family:Times;color:#cedde4;}
	.ft111{font-size:10px;font-family:Times;color:#000000;}
	.ft112{font-size:10px;font-family:Times;color:#000000;}
	.ft113{font-size:10px;font-family:Times;color:#000000;}
	.ft114{font-size:13px;font-family:Times;color:#000000;}
	.ft115{font-size:11px;font-family:Times;color:#000000;}
	.ft116{font-size:6px;font-family:Times;color:#0000ff;}
	.ft117{font-size:6px;font-family:Times;color:#000000;}
	.ft118{font-size:14px;font-family:Times;color:#000000;}
	.ft119{font-size:13px;font-family:Times;color:#000000;}
	.ft1110{font-size:11px;font-family:Times;color:#0000ff;}
	.ft1111{font-size:11px;font-family:Times;color:#000000;}
	.ft1112{font-size:7px;font-family:Times;color:#000000;}
	.ft1113{font-size:7px;font-family:Times;color:#000000;}
-->
</style>
</head>
<body bgcolor="#A0A0A0" vlink="blue" link="blue">
<div id="page11-div" style="position:relative;width:892px;height:1173px;">
<p style="position:absolute;top:1132px;left:817px;white-space:nowrap" class="ft110">11</p>
<p style="position:absolute;top:1135px;left:61px;white-space:nowrap" class="ft111"><b>Scientific</b>&#160;<b>RepoRtS</b>&#160;|&#160;</p>
<p style="position:absolute;top:1135px;left:192px;white-space:nowrap" class="ft113"><i>&#160; &#160; &#160; &#160; (2019) 9:18612&#160;</i>&#160;|&#160;<a href="https://doi.org/10.1038/s41598-019-54859-8">https://doi.org/10.1038/s41598-019-54859-8</a></p>
<p style="position:absolute;top:19px;left:585px;white-space:nowrap" class="ft114">www.nature.com/scientificreports</p>
<p style="position:absolute;top:19px;left:61px;white-space:nowrap" class="ft114">www.nature.com/scientificreports/</p>
<p style="position:absolute;top:71px;left:233px;white-space:nowrap" class="ft115">specific (and useful) behaviors, both for synthetic biology applications and to understand functional aspects of&#160;</p>
<p style="position:absolute;top:86px;left:233px;white-space:nowrap" class="ft115">developmental physiology, the evolution of brain circuits, etc. In prior work, a computational perspective of cel&#160;&#160;</p>
<p style="position:absolute;top:101px;left:233px;white-space:nowrap" class="ft115">behavior<a href="4b78c8d89b22cb315d90fd1274945de5s.html#14">14,16,</a><a href="4b78c8d89b22cb315d90fd1274945de5s.html#16">130</a>&#160;allowed our lab and other groups to formulate (1) ways of experimental&#160;y controlling large-scale&#160;</p>
<p style="position:absolute;top:116px;left:233px;white-space:nowrap" class="ft115">anatomical outcom<a href="4b78c8d89b22cb315d90fd1274945de5s.html#14">es13, a</a>nd (2) formulate and solve the “inverse problem<a href="4b78c8d89b22cb315d90fd1274945de5s.html#14">”14</a>&#160;of how the cells might actually&#160;</p>
<p style="position:absolute;top:131px;left:233px;white-space:nowrap" class="ft115">achieve specific anatomical configurations, given cel&#160;ular constraints and a range of starting conditions<a href="4b78c8d89b22cb315d90fd1274945de5s.html#15">76</a><a href="4b78c8d89b22cb315d90fd1274945de5s.html#16">,131. O</a>ur&#160;</p>
<p style="position:absolute;top:146px;left:233px;white-space:nowrap" class="ft115">current work addresses (2), by demonstrating a possible approach that cel&#160;s could use to implement logic func-</p>
<p style="position:absolute;top:161px;left:233px;white-space:nowrap" class="ft115">tions under realistic cel&#160;ular constraints.</p>
<p style="position:absolute;top:176px;left:251px;white-space:nowrap" class="ft115">Overall, BEN has the potential to shed light on as-yet-unexplained non-neural bioelectric&#160;</p>
<p style="position:absolute;top:191px;left:233px;white-space:nowrap" class="ft115">information-processing. For example, when certain planarian species (highly regenerative animals) are cut they&#160;</p>
<p style="position:absolute;top:206px;left:233px;white-space:nowrap" class="ft115">sometimes develop two heads, instead of a head and a tail, as a response to a temporary exposure to 1-octanol, a&#160;</p>
<p style="position:absolute;top:221px;left:233px;white-space:nowrap" class="ft115">gap junction blocker. Without the gap junction blocker, they always develop a wild type morphology consisting&#160;</p>
<p style="position:absolute;top:236px;left:233px;white-space:nowrap" class="ft115">of a single head and a tail. It has been shown that this decision to switch morphologies is bioelectrical&#160;y con-</p>
<p style="position:absolute;top:251px;left:233px;white-space:nowrap" class="ft115">trolled&#160;and&#160;non-neural&#160;in&#160;natur<a href="4b78c8d89b22cb315d90fd1274945de5s.html#14">e24,</a><a href="4b78c8d89b22cb315d90fd1274945de5s.html#16">132</a>.&#160;We&#160;hypothesize&#160;that&#160;the&#160;planarian&#160;bioelectric&#160;system&#160;encodes&#160;a&#160;pattern&#160;that&#160;</p>
<p style="position:absolute;top:266px;left:233px;white-space:nowrap" class="ft115">either consists of a record of the gap junction blocker or not, and then maps it into different morphological deci-</p>
<p style="position:absolute;top:281px;left:233px;white-space:nowrap" class="ft115">sions (wild-type or double-head) in a systematic way. We currently have a prototype BEN model that reproduces&#160;</p>
<p style="position:absolute;top:296px;left:233px;white-space:nowrap" class="ft115">this phenomenon in a qualitative way, which we plan to report in the future. Final&#160;y, we have demonstrated in&#160;</p>
<p style="position:absolute;top:311px;left:233px;white-space:nowrap" class="ft115">this&#160;work&#160;that&#160;the&#160;slower&#160;(reason&#160;described&#160;in&#160;‘Methods’),&#160;continuous&#160;mode&#160;of&#160;non-neural&#160;bioelectric&#160;signaling,&#160;</p>
<p style="position:absolute;top:326px;left:233px;white-space:nowrap" class="ft115">compared to the&#160;faster, pulsating&#160;mode of neural signaling, is&#160;sufficient to perform&#160;logical computations usual&#160;y&#160;</p>
<p style="position:absolute;top:341px;left:233px;white-space:nowrap" class="ft115">associated with the brain. This firmly supports the possibility of somatic computation in real biological systems.&#160;</p>
<p style="position:absolute;top:356px;left:233px;white-space:nowrap" class="ft115">How it compares to the timescales of genetic signaling and how it could facilitate bioelectricity to be upstream of&#160;</p>
<p style="position:absolute;top:371px;left:233px;white-space:nowrap" class="ft115">genetics<a href="4b78c8d89b22cb315d90fd1274945de5s.html#14">53</a><a href="4b78c8d89b22cb315d90fd1274945de5s.html#16">,103&#160;i</a>s an open question that our future investigations will answer.</p>
<p style="position:absolute;top:386px;left:251px;white-space:nowrap" class="ft115">Our work paves the foundation for future research in training actual biological tissues for somatic compu-</p>
<p style="position:absolute;top:401px;left:233px;white-space:nowrap" class="ft115">tation. Ongoing research in our laboratory has identified ways by which gap junctions can be manipulated and&#160;</p>
<p style="position:absolute;top:416px;left:233px;white-space:nowrap" class="ft115">controlled, further opening up the possibilities on this front. Overal&#160;, our research provides the conceptual and&#160;</p>
<p style="position:absolute;top:431px;left:233px;white-space:nowrap" class="ft115">modeling&#160;foundations&#160;to understand and manipulate&#160;development and regeneration, and&#160;to construct computa-</p>
<p style="position:absolute;top:446px;left:233px;white-space:nowrap" class="ft115">tional synthetic living machines.</p>
<p style="position:absolute;top:475px;left:233px;white-space:nowrap" class="ft118"><b>Methods</b></p>
<p style="position:absolute;top:492px;left:233px;white-space:nowrap" class="ft119"><b>Mathematical&#160;details&#160;of&#160;BEN.&#160;&#160;</b>The equations that define a BEN model are shown in Fig<a href="4b78c8d89b22cb315d90fd1274945de5s.html#12">. 8&#160;u</a>sing an exam-</p>
<p style="position:absolute;top:508px;left:233px;white-space:nowrap" class="ft115">ple 2-cell system. The equations are shown for cell-2 only; the same equations with appropriate change in sub-</p>
<p style="position:absolute;top:523px;left:233px;white-space:nowrap" class="ft115">scripts apply to cell-1 and likewise to any other cell in larger networks. The processes that the equations represent,&#160;</p>
<p style="position:absolute;top:538px;left:233px;white-space:nowrap" class="ft115">like electrodiffusion, reaction-diffusion and gating, are marked appropriately in the figure, thus self-explanatory.&#160;</p>
<p style="position:absolute;top:553px;left:233px;white-space:nowrap" class="ft115">The following remarks serve to better il&#160;ustrate the model. As described in the introductory section, electrodiffu-</p>
<p style="position:absolute;top:568px;left:233px;white-space:nowrap" class="ft115">sion utilizes two types of gradients, namely voltage and concentration, to drive the flux. Those gradients are rep-</p>
<p style="position:absolute;top:584px;left:233px;white-space:nowrap" class="ft115">resented by the terms&#160;</p>
<p style="position:absolute;top:585px;left:377px;white-space:nowrap" class="ft115">−</p>
<p style="position:absolute;top:584px;left:350px;white-space:nowrap" class="ft1111"><i>v&#160;t</i></p>
<p style="position:absolute;top:584px;left:392px;white-space:nowrap" class="ft1111"><i>v&#160;t</i></p>
<p style="position:absolute;top:584px;left:359px;white-space:nowrap" class="ft115">(&#160;)</p>
<p style="position:absolute;top:584px;left:402px;white-space:nowrap" class="ft115">(&#160;)</p>
<p style="position:absolute;top:591px;left:355px;white-space:nowrap" class="ft1112">1</p>
<p style="position:absolute;top:591px;left:398px;white-space:nowrap" class="ft1112">2</p>
<p style="position:absolute;top:584px;left:415px;white-space:nowrap" class="ft115">&#160;and&#160;</p>
<p style="position:absolute;top:585px;left:468px;white-space:nowrap" class="ft115">−</p>
<p style="position:absolute;top:584px;left:440px;white-space:nowrap" class="ft1111"><i>c&#160;t</i></p>
<p style="position:absolute;top:584px;left:482px;white-space:nowrap" class="ft1111"><i>c&#160;t</i></p>
<p style="position:absolute;top:584px;left:449px;white-space:nowrap" class="ft115">(&#160;)</p>
<p style="position:absolute;top:584px;left:492px;white-space:nowrap" class="ft115">(&#160;)</p>
<p style="position:absolute;top:591px;left:446px;white-space:nowrap" class="ft1112">1</p>
<p style="position:absolute;top:591px;left:488px;white-space:nowrap" class="ft1112">2</p>
<p style="position:absolute;top:584px;left:506px;white-space:nowrap" class="ft115">&#160;respectively in the electrodiffusion equations. The ion-pump&#160;</p>
<p style="position:absolute;top:599px;left:233px;white-space:nowrap" class="ft115">equations are approximate versions of the full mechanism modeled in equations (23–29) of BETSE<a href="4b78c8d89b22cb315d90fd1274945de5s.html#14">51. B</a>EN makes&#160;</p>
<p style="position:absolute;top:614px;left:233px;white-space:nowrap" class="ft115">use of just the terms involving the ion concentrations in equation (27) of BETSE<a href="4b78c8d89b22cb315d90fd1274945de5s.html#14">51</a>, as it lacks the other biochem-</p>
<p style="position:absolute;top:629px;left:233px;white-space:nowrap" class="ft115">ical components like ATP, and by choosing appropriate term coefficients to compensate for the approximation it&#160;</p>
<p style="position:absolute;top:644px;left:233px;white-space:nowrap" class="ft115">effectively preserves the relationship between ion concentrations and the pump flux.</p>
<p style="position:absolute;top:659px;left:251px;white-space:nowrap" class="ft115">The dynamics of the signaling molecule is viewed as a nonlinear reaction-diffusion process as it can be&#160;</p>
<p style="position:absolute;top:675px;left:233px;white-space:nowrap" class="ft115">abstractly represented as:&#160;</p>
<p style="position:absolute;top:676px;left:399px;white-space:nowrap" class="ft115">∇</p>
<p style="position:absolute;top:675px;left:371px;white-space:nowrap" class="ft1111"><i>R&#160;D&#160;c</i></p>
<p style="position:absolute;top:675px;left:380px;white-space:nowrap" class="ft115">(&#160;(&#160;)), where&#160;∇<i>c</i>&#160;is the concentration gradient,&#160;&#160;.</p>
<p style="position:absolute;top:675px;left:646px;white-space:nowrap" class="ft1111"><i>D</i>(&#160;)&#160;represents the 1st&#160;layer of sig-</p>
<p style="position:absolute;top:691px;left:233px;white-space:nowrap" class="ft115">moid approximating&#160;a nonlinear&#160;diffusion, and&#160;&#160;.</p>
<p style="position:absolute;top:691px;left:487px;white-space:nowrap" class="ft1111"><i>R</i>(&#160;)&#160;represents the&#160;2nd&#160;layer of sigmoid approximating a nonlin-</p>
<p style="position:absolute;top:707px;left:233px;white-space:nowrap" class="ft115">ear&#160;reaction.&#160;Moreover,&#160;the&#160;nested&#160;form of&#160;</p>
<p style="position:absolute;top:708px;left:497px;white-space:nowrap" class="ft115">.</p>
<p style="position:absolute;top:708px;left:469px;white-space:nowrap" class="ft1111"><i>R&#160;D</i></p>
<p style="position:absolute;top:708px;left:477px;white-space:nowrap" class="ft115">(&#160;(&#160;))&#160;departs from&#160;the conventional&#160;form of&#160;reaction-diffusion&#160;</p>
<p style="position:absolute;top:724px;left:233px;white-space:nowrap" class="ft115">which is&#160;&#160;=</p>
<p style="position:absolute;top:725px;left:341px;white-space:nowrap" class="ft115">+</p>
<p style="position:absolute;top:725px;left:312px;white-space:nowrap" class="ft1111"><i>D&#160;c</i></p>
<p style="position:absolute;top:725px;left:356px;white-space:nowrap" class="ft1111"><i>R&#160;c</i></p>
<p style="position:absolute;top:725px;left:322px;white-space:nowrap" class="ft115">(&#160;)</p>
<p style="position:absolute;top:725px;left:364px;white-space:nowrap" class="ft115">(&#160;)</p>
<p style="position:absolute;top:722px;left:282px;white-space:nowrap" class="ft1113"><i>dc</i></p>
<p style="position:absolute;top:735px;left:283px;white-space:nowrap" class="ft1113"><i>dt</i></p>
<p style="position:absolute;top:724px;left:378px;white-space:nowrap" class="ft115">, for example,&#160;&#160;=&#160;∇&#160;+</p>
<p style="position:absolute;top:725px;left:497px;white-space:nowrap" class="ft1111"><i>c</i></p>
<p style="position:absolute;top:725px;left:522px;white-space:nowrap" class="ft1111"><i>c</i></p>
<p style="position:absolute;top:722px;left:456px;white-space:nowrap" class="ft1113"><i>dc</i></p>
<p style="position:absolute;top:735px;left:457px;white-space:nowrap" class="ft1113"><i>dt</i></p>
<p style="position:absolute;top:723px;left:528px;white-space:nowrap" class="ft1112">2. Nevertheless, it qualifies as reaction-diffusion since it&#160;</p>
<p style="position:absolute;top:741px;left:233px;white-space:nowrap" class="ft115">clearly involves diffusion (∇<i>c</i>)and a nonlinear component (the sigmoid) that alters the net mass of&#160;<i>c</i>, representing&#160;</p>
<p style="position:absolute;top:757px;left:233px;white-space:nowrap" class="ft115">production or decay depending on the sign of&#160;<i>dc</i></p>
<p style="position:absolute;top:769px;left:485px;white-space:nowrap" class="ft1113"><i>dt</i></p>
<p style="position:absolute;top:757px;left:495px;white-space:nowrap" class="ft115">. Another important point to note is that even though sigmoid&#160;</p>
<p style="position:absolute;top:774px;left:233px;white-space:nowrap" class="ft115">functions, which define the signal flux, also constitute the activation functions of neurons in neural network&#160;</p>
<p style="position:absolute;top:789px;left:233px;white-space:nowrap" class="ft115">models<a href="4b78c8d89b22cb315d90fd1274945de5s.html#14">38</a>,<a href="4b78c8d89b22cb315d90fd1274945de5s.html#15">64</a>, there is a crucial difference: while neuron models are input-activated since neuronal synaptic junc-</p>
<p style="position:absolute;top:804px;left:233px;white-space:nowrap" class="ft115">tions&#160;are&#160;directional,&#160;a&#160;cel&#160;&#160;in&#160;BEN&#160;is&#160;gradient-activated&#160;since&#160;gap-junctions&#160;are&#160;bidirectional.&#160;This&#160;is&#160;also&#160;partly&#160;</p>
<p style="position:absolute;top:819px;left:233px;white-space:nowrap" class="ft115">the reason why non-neuronal cel&#160;s tend to be slower than neural cel&#160;s—diffusion is a slower process than directed&#160;</p>
<p style="position:absolute;top:834px;left:233px;white-space:nowrap" class="ft115">currents.</p>
<p style="position:absolute;top:849px;left:251px;white-space:nowrap" class="ft115">There’s&#160;no&#160;set&#160;range&#160;of&#160;Vmem&#160;levels&#160;that&#160;a&#160;given&#160;BEN&#160;network&#160;wil&#160;&#160;not&#160;exceed&#160;(as&#160;there&#160;are&#160;no&#160;explicit&#160;bounds&#160;on&#160;</p>
<p style="position:absolute;top:864px;left:233px;white-space:nowrap" class="ft115">the ion concentration levels), but there are typical&#160;y set to lie within [−80&#160;mV,&#160;+80&#160;mV]; see for examp<a href="4b78c8d89b22cb315d90fd1274945de5s.html#5">le Figs. 2d,&#160;</a></p>
<p style="position:absolute;top:879px;left:233px;white-space:nowrap" class="ft1110"><a href="4b78c8d89b22cb315d90fd1274945de5s.html#6">3d</a><a href="4b78c8d89b22cb315d90fd1274945de5s.html#7">,&#160;4,&#160;</a><a href="4b78c8d89b22cb315d90fd1274945de5s.html#8">6</a>&#160;and<a href="4b78c8d89b22cb315d90fd1274945de5s.html#9">&#160;7. Th</a>e concentration of the signaling molecule, on the other hand, is forced to lie with [0, 1], that is,&#160;</p>
<p style="position:absolute;top:894px;left:233px;white-space:nowrap" class="ft115">within the generic minimum and maximum possible values of the actual concentration which is not explicitly&#160;</p>
<p style="position:absolute;top:909px;left:233px;white-space:nowrap" class="ft115">considered in BEN.</p>
<p style="position:absolute;top:924px;left:251px;white-space:nowrap" class="ft115">The software to simulate, train and evolve BEN networks is available at:&#160;<a href="https://gitlab.com/smanicka/BEN">https://gitlab.com/smanicka/BEN.</a></p>
<p style="position:absolute;top:955px;left:233px;white-space:nowrap" class="ft119"><b>Details&#160;of&#160;the&#160;simulation.&#160;&#160;</b>We used the standard Euler integration method for integration the equations.&#160;</p>
<p style="position:absolute;top:971px;left:233px;white-space:nowrap" class="ft115">We used a time step of 0.01 for the bioelectric dynamics&#160;and 0.02 for the signaling molecule dynamics. Each simu-</p>
<p style="position:absolute;top:986px;left:233px;white-space:nowrap" class="ft115">lation was&#160;set&#160;up&#160;to&#160;run&#160;for&#160;a&#160;number&#160;of&#160;steps&#160;ranging&#160;from&#160;300 to&#160;600;&#160;smal&#160;er&#160;networks were&#160;run&#160;for&#160;fewer&#160;steps.&#160;</p>
<p style="position:absolute;top:1001px;left:233px;white-space:nowrap" class="ft115">Thus, the total simulation ranged between 300&#160;</p>
<p style="position:absolute;top:1000px;left:480px;white-space:nowrap" class="ft115">*&#160;0.01&#160;=&#160;3 simulation seconds to 600&#160;*&#160;0.01&#160;=&#160;6 simulation seconds.</p>
<p style="position:absolute;top:1031px;left:233px;white-space:nowrap" class="ft119"><b>The&#160;backpropagation&#160;method&#160;for&#160;training&#160;elementary&#160;logic&#160;gates.&#160;&#160;</b>Regardless of&#160;the&#160;gate,&#160;a&#160;train-</p>
<p style="position:absolute;top:1047px;left:233px;white-space:nowrap" class="ft115">ing run was set up as follows. A single BEN network was instantiated with randomized parameters. The initial&#160;</p>
<p style="position:absolute;top:1062px;left:233px;white-space:nowrap" class="ft115">weight parameters were chosen in the range [−1, 1], and the biases in the range [0, 1]. It was then fed with a batch&#160;</p>
<p style="position:absolute;top:1077px;left:233px;white-space:nowrap" class="ft115">of&#160;four&#160;inputs&#160;and was&#160;simulated&#160;for about&#160;300 time-steps, with&#160;the inputs being&#160;held&#160;<i>fixed</i>&#160;throughout&#160;the&#160;sim-</p>
<p style="position:absolute;top:1092px;left:233px;white-space:nowrap" class="ft115">ulation. An “input”&#160;constitutes&#160;a specific&#160;Vmem&#160;pattern of the&#160;input cel&#160;s. For example, input cell-1 may be set to&#160;a&#160;</p>
</div>
</body>
</html>
<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="" xml:lang="">
<head>
<title>4b78c8d89b22cb315d90fd1274945de5-html.html</title>

<meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
 <br/>
<style type="text/css">
<!--
	p {margin: 0; padding: 0;}	.ft120{font-size:13px;font-family:Times;color:#cedde4;}
	.ft121{font-size:10px;font-family:Times;color:#000000;}
	.ft122{font-size:10px;font-family:Times;color:#000000;}
	.ft123{font-size:10px;font-family:Times;color:#000000;}
	.ft124{font-size:13px;font-family:Times;color:#000000;}
	.ft125{font-size:11px;font-family:Times;color:#000000;}
	.ft126{font-size:6px;font-family:Times;color:#000000;}
	.ft127{font-size:6px;font-family:Times;color:#0000ff;}
	.ft128{font-size:11px;font-family:Times;color:#0000ff;}
	.ft129{font-size:11px;font-family:Times;color:#000000;}
-->
</style>
</head>
<body bgcolor="#A0A0A0" vlink="blue" link="blue">
<div id="page12-div" style="position:relative;width:892px;height:1173px;">
<p style="position:absolute;top:1132px;left:816px;white-space:nowrap" class="ft120">12</p>
<p style="position:absolute;top:1135px;left:61px;white-space:nowrap" class="ft121"><b>Scientific</b>&#160;<b>RepoRtS</b>&#160;|&#160;</p>
<p style="position:absolute;top:1135px;left:192px;white-space:nowrap" class="ft123"><i>&#160; &#160; &#160; &#160; (2019) 9:18612&#160;</i>&#160;|&#160;<a href="https://doi.org/10.1038/s41598-019-54859-8">https://doi.org/10.1038/s41598-019-54859-8</a></p>
<p style="position:absolute;top:19px;left:585px;white-space:nowrap" class="ft124">www.nature.com/scientificreports</p>
<p style="position:absolute;top:19px;left:61px;white-space:nowrap" class="ft124">www.nature.com/scientificreports/</p>
<p style="position:absolute;top:764px;left:233px;white-space:nowrap" class="ft125">hyperpolarized Vmem&#160;and cell-2 to a depolarized Vmem. The input cel&#160;s were then randomly set to a different state,&#160;</p>
<p style="position:absolute;top:779px;left:233px;white-space:nowrap" class="ft125">drawn from the training batch, without disturbing the state of the rest of the network (a method referred to as&#160;</p>
<p style="position:absolute;top:794px;left:233px;white-space:nowrap" class="ft125">“continuous computation”<a href="4b78c8d89b22cb315d90fd1274945de5s.html#15">101</a>) and the simulation was continued for another 300 time-steps; this was repeated for&#160;</p>
<p style="position:absolute;top:809px;left:233px;white-space:nowrap" class="ft125">ten times. At the end of each of 10 simulations, the observed states of the output cell during the last 10 time-steps&#160;</p>
<p style="position:absolute;top:824px;left:233px;white-space:nowrap" class="ft125">were&#160;matched&#160;against&#160;the&#160;desired&#160;output&#160;corresponding&#160;to&#160;the&#160;input&#160;in&#160;the&#160;training&#160;batch&#160;(see&#160;Figs. <a href="4b78c8d89b22cb315d90fd1274945de5s.html#5">2b&#160;a</a>nd&#160;<a href="4b78c8d89b22cb315d90fd1274945de5s.html#6">3b&#160;</a></p>
<p style="position:absolute;top:839px;left:233px;white-space:nowrap" class="ft125">for examples). An error was then calculated based on the difference for each of the 10 simulations for each input&#160;</p>
<p style="position:absolute;top:854px;left:233px;white-space:nowrap" class="ft125">sequence&#160;in&#160;the&#160;batch&#160;and&#160;averaged&#160;over.&#160;The&#160;derivatives&#160;of&#160;the&#160;final&#160;average&#160;error&#160;(gradient)&#160;with&#160;respect&#160;to&#160;al&#160;&#160;the&#160;</p>
<p style="position:absolute;top:869px;left:233px;white-space:nowrap" class="ft125">parameters were then calculated. Due to the recurrent nature of the dynamics (Fig<a href="4b78c8d89b22cb315d90fd1274945de5s.html#3">. 1), t</a>he simulation takes many&#160;</p>
<p style="position:absolute;top:884px;left:233px;white-space:nowrap" class="ft125">more steps than the number of layers in the network before the output reaches an equilibrium. Thus, the gradients&#160;</p>
<p style="position:absolute;top:899px;left:233px;white-space:nowrap" class="ft125">would in principle have to computed over all those steps—a method known as “backpropagation through time”&#160;</p>
<p style="position:absolute;top:914px;left:233px;white-space:nowrap" class="ft125">(BPTT). However, due to computational constraints and known problems like “vanishing gradient”, the gradients&#160;</p>
<p style="position:absolute;top:929px;left:233px;white-space:nowrap" class="ft125">are typical&#160;y computed over fewer time steps—this version of&#160;backpropagation is known as “truncated” BPTT. In&#160;</p>
<p style="position:absolute;top:944px;left:233px;white-space:nowrap" class="ft125">this work, the gradients were computed over only the last 100 steps of a simulation. The quantum of changes for&#160;</p>
<p style="position:absolute;top:959px;left:233px;white-space:nowrap" class="ft125">all the parameters were then calculated from the gradients after scaling it with a “learning rate” that set the pace of&#160;</p>
<p style="position:absolute;top:974px;left:233px;white-space:nowrap" class="ft125">learning. The weights and bias were assigned separate “dynamic” learning rates, where the rates were fixed in such&#160;</p>
<p style="position:absolute;top:989px;left:233px;white-space:nowrap" class="ft125">a way that the maximum change of a weight parameter was 0.1 and the maximum change of a bias parameter was&#160;</p>
<p style="position:absolute;top:1004px;left:233px;white-space:nowrap" class="ft125">0.01. We adopted this strategy mainly to mitigate the vanishing gradient problem, and the assumption that small&#160;</p>
<p style="position:absolute;top:1019px;left:233px;white-space:nowrap" class="ft125">changes in the parameters should cause a smooth change&#160;in behavior. We also employed a&#160;“momentum” factor,&#160;</p>
<p style="position:absolute;top:1034px;left:233px;white-space:nowrap" class="ft125">a method&#160;often used&#160;in backpropagation,&#160;that determines the extent&#160;to which the previous parameter-change&#160;</p>
<p style="position:absolute;top:1049px;left:233px;white-space:nowrap" class="ft125">vector contributed to the following vector. We used a&#160;momentum of 0.5 for the first 100 training iterations,&#160;and&#160;</p>
<p style="position:absolute;top:1064px;left:233px;white-space:nowrap" class="ft125">0.9 for the rest; the allowed range is [0, 1]. Final&#160;y, the parameter-updates were applied (thus error was backprop-</p>
<p style="position:absolute;top:1079px;left:233px;white-space:nowrap" class="ft125">agated),&#160;and&#160;the&#160;whole&#160;process&#160;was&#160;repeated&#160;for&#160;about&#160;600,000&#160;“epochs”.&#160;The&#160;weights&#160;were&#160;allowed&#160;to&#160;change&#160;</p>
<p style="position:absolute;top:675px;left:233px;white-space:nowrap" class="ft129"><b>Figure 8.</b>&#160;&#160;Mathematical details of the BEN model. (Top) The equations that describe the dynamics in a 2-cell&#160;</p>
<p style="position:absolute;top:690px;left:233px;white-space:nowrap" class="ft125">system il&#160;ustrated in the figure on the left. (Bottom) A glossary of variables used in the equations, the constants&#160;</p>
<p style="position:absolute;top:705px;left:233px;white-space:nowrap" class="ft125">and initial conditions.</p>
</div>
</body>
</html>
<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="" xml:lang="">
<head>
<title>4b78c8d89b22cb315d90fd1274945de5-html.html</title>

<meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
 <br/>
<style type="text/css">
<!--
	p {margin: 0; padding: 0;}	.ft130{font-size:13px;font-family:Times;color:#cedde4;}
	.ft131{font-size:10px;font-family:Times;color:#000000;}
	.ft132{font-size:10px;font-family:Times;color:#000000;}
	.ft133{font-size:10px;font-family:Times;color:#000000;}
	.ft134{font-size:13px;font-family:Times;color:#000000;}
	.ft135{font-size:11px;font-family:Times;color:#000000;}
	.ft136{font-size:11px;font-family:Times;color:#000000;}
	.ft137{font-size:13px;font-family:Times;color:#000000;}
	.ft138{font-size:6px;font-family:Times;color:#0000ff;}
	.ft139{font-size:14px;font-family:Times;color:#000000;}
	.ft1310{font-size:11px;font-family:Times;color:#0000ff;}
	.ft1311{font-size:9px;font-family:Times;color:#000000;}
	.ft1312{font-size:9px;font-family:Times;color:#000000;}
	.ft1313{font-size:9px;font-family:Times;color:#000000;}
	.ft1314{font-size:9px;font-family:Times;color:#0000ff;}
-->
</style>
</head>
<body bgcolor="#A0A0A0" vlink="blue" link="blue">
<div id="page13-div" style="position:relative;width:892px;height:1173px;">
<p style="position:absolute;top:1132px;left:817px;white-space:nowrap" class="ft130">13</p>
<p style="position:absolute;top:1135px;left:61px;white-space:nowrap" class="ft131"><b>Scientific</b>&#160;<b>RepoRtS</b>&#160;|&#160;</p>
<p style="position:absolute;top:1135px;left:192px;white-space:nowrap" class="ft133"><i>&#160; &#160; &#160; &#160; (2019) 9:18612&#160;</i>&#160;|&#160;<a href="https://doi.org/10.1038/s41598-019-54859-8">https://doi.org/10.1038/s41598-019-54859-8</a></p>
<p style="position:absolute;top:19px;left:585px;white-space:nowrap" class="ft134">www.nature.com/scientificreports</p>
<p style="position:absolute;top:19px;left:61px;white-space:nowrap" class="ft134">www.nature.com/scientificreports/</p>
<p style="position:absolute;top:71px;left:233px;white-space:nowrap" class="ft135">infinitely in both directions, whereas the biases were limited within the range [0, 1]. We deemed a network as suc-</p>
<p style="position:absolute;top:86px;left:233px;white-space:nowrap" class="ft135">cessful&#160;y trained if it achieved an average performance error of 0.0002 or below at some point during the training.</p>
<p style="position:absolute;top:101px;left:251px;white-space:nowrap" class="ft135">We used the Python software package called&#160;<i>Pytorch</i>&#160;for computing derivatives during backpropagation.&#160;</p>
<p style="position:absolute;top:116px;left:233px;white-space:nowrap" class="ft135">This package uses a method called “automatic differentiation” to compute derivatives based on “computational&#160;</p>
<p style="position:absolute;top:131px;left:233px;white-space:nowrap" class="ft135">graphs”—a graph that keeps track of every computation performed during a simulation, which is then swept&#160;</p>
<p style="position:absolute;top:146px;left:233px;white-space:nowrap" class="ft135">backwards for computing the derivatives, essential&#160;y constituting an algorithm for the “chain rule” of differential&#160;</p>
<p style="position:absolute;top:161px;left:233px;white-space:nowrap" class="ft135">calculus.</p>
<p style="position:absolute;top:191px;left:233px;white-space:nowrap" class="ft137"><b>The&#160;combined&#160;backpropagation-genetic-algorithm&#160;method&#160;for&#160;training&#160;tissue-level&#160;logic&#160;gate&#160;</b></p>
<p style="position:absolute;top:206px;left:233px;white-space:nowrap" class="ft137"><b>and the pattern detector.&#160;&#160;</b>A population of 50 BEN “genomes” was instantiated. A genome consists of a&#160;</p>
<p style="position:absolute;top:222px;left:233px;white-space:nowrap" class="ft135">combination&#160;of&#160;the&#160;network’s&#160;parameters,&#160;namely&#160;the&#160;weight&#160;matrix&#160;and&#160;the&#160;bias&#160;vector,&#160;that&#160;define&#160;a&#160;network.&#160;</p>
<p style="position:absolute;top:237px;left:233px;white-space:nowrap" class="ft135">Henceforth, by “genome” we refer to a BEN network, for clarity. Each network consists of the following structure:&#160;</p>
<p style="position:absolute;top:252px;left:233px;white-space:nowrap" class="ft135">an input layer consisting of 2 nodes, an intermediate “tissue” layer consisting of 25 nodes, and the output layer&#160;</p>
<p style="position:absolute;top:267px;left:233px;white-space:nowrap" class="ft135">consisting&#160;of&#160;a&#160;single&#160;node.&#160;Initial&#160;y&#160;the&#160;tissue&#160;layer&#160;was&#160;designed&#160;to&#160;be&#160;a&#160;two-dimensional&#160;lattice.&#160;Next,&#160;the&#160;tissue&#160;</p>
<p style="position:absolute;top:282px;left:233px;white-space:nowrap" class="ft135">layer of each individual in the population was randomly rewired with a rewiring probability chosen uniformly&#160;</p>
<p style="position:absolute;top:298px;left:233px;white-space:nowrap" class="ft135">from the range&#160;[0,&#160;1]. Thus, the population consisted of the full spectrum of tissue layer structures ranging from&#160;</p>
<p style="position:absolute;top:313px;left:233px;white-space:nowrap" class="ft135">lattice to random. Next, the input layer and the output layers of each individual in the population were respec-</p>
<p style="position:absolute;top:328px;left:233px;white-space:nowrap" class="ft135">tively connected to approximately 50% of the nodes in the tissue layer; this gave the GA a chance to explore both&#160;</p>
<p style="position:absolute;top:343px;left:233px;white-space:nowrap" class="ft135">denser&#160;and sparser&#160;inter-layer connectivity.&#160;Every&#160;individual&#160;in&#160;the&#160;population was&#160;then assigned randomized&#160;</p>
<p style="position:absolute;top:358px;left:233px;white-space:nowrap" class="ft135">weight and bias parameters chosen in the same range as backpropagation. They were each then backpropagated&#160;</p>
<p style="position:absolute;top:373px;left:233px;white-space:nowrap" class="ft135">for 80 iterations in the same fashion as described above for the elementary gates, with an important difference: the&#160;</p>
<p style="position:absolute;top:388px;left:233px;white-space:nowrap" class="ft135">inputs for these gates were&#160;<i>transient</i>, as they are set for the first time-step only. The final average error of each&#160;</p>
<p style="position:absolute;top:403px;left:233px;white-space:nowrap" class="ft135">individual became its fitness score. The GA then goes through the conventional steps of selection, mutation, cre-</p>
<p style="position:absolute;top:418px;left:233px;white-space:nowrap" class="ft135">ating a new generation of individuals.</p>
<p style="position:absolute;top:433px;left:251px;white-space:nowrap" class="ft135">We used a simple flavor of GA known as the “microbial genetic algorithm<a href="4b78c8d89b22cb315d90fd1274945de5s.html#16">”133&#160;t</a>hat capitalizes on the notion of&#160;</p>
<p style="position:absolute;top:448px;left:233px;white-space:nowrap" class="ft135">horizontal&#160;gene&#160;transfer&#160;observed&#160;in&#160;microbes.&#160;It is&#160;essential&#160;y&#160;a&#160;form&#160;of&#160;tournament&#160;selection&#160;where&#160;two&#160;individ-</p>
<p style="position:absolute;top:463px;left:233px;white-space:nowrap" class="ft135">uals&#160;in a population&#160;are picked&#160;and pitted&#160;against each&#160;other (their&#160;fitness&#160;scores are&#160;compared).&#160;We used&#160;a “geo-</p>
<p style="position:absolute;top:478px;left:233px;white-space:nowrap" class="ft135">graphical selection” method where individuals are placed on a 1D ring, and only geographical&#160;y close individuals&#160;</p>
<p style="position:absolute;top:493px;left:233px;white-space:nowrap" class="ft135">are picked for the tournament. The size of selection-neighborhood is known as the “deme size”, which was set to&#160;</p>
<p style="position:absolute;top:508px;left:233px;white-space:nowrap" class="ft135">10% in this work. The genome of the winner is transmitted to the loser at a certain “recombination rate”, which&#160;</p>
<p style="position:absolute;top:523px;left:233px;white-space:nowrap" class="ft135">was set to 0.1. The genome of the loser is then mutated at a certain “mutation rate” (set to 0.05) and reinserted into&#160;</p>
<p style="position:absolute;top:538px;left:233px;white-space:nowrap" class="ft135">the population. This process is repeated for 1000 generations. The individual with the best fitness score in the final&#160;</p>
<p style="position:absolute;top:553px;left:233px;white-space:nowrap" class="ft135">generation was deemed as the best individual—the most successful&#160;y trained tissue-level logic gate.</p>
<p style="position:absolute;top:583px;left:233px;white-space:nowrap" class="ft137"><b>Generation&#160;of&#160;the&#160;input&#160;patterns&#160;during&#160;the&#160;training&#160;of&#160;the&#160;pattern&#160;detector.&#160;&#160;</b>The set of input pat-</p>
<p style="position:absolute;top:599px;left:233px;white-space:nowrap" class="ft135">terns&#160;used&#160;for&#160;training&#160;the&#160;pattern&#160;detector&#160;consists&#160;of&#160;two&#160;categories:&#160;(1) French-flag&#160;(FF)&#160;and&#160;(2)&#160;Not-French-flag&#160;</p>
<p style="position:absolute;top:614px;left:233px;white-space:nowrap" class="ft135">(NFF). The subset FF consists of the French-flag pattern itself and its slightly noisy variants. The latter is generated&#160;</p>
<p style="position:absolute;top:629px;left:233px;white-space:nowrap" class="ft135">by perturbing each pixel of the French-flag approximately within the range [−24&#160;mV,&#160;+24&#160;mV]. Any perturbation&#160;</p>
<p style="position:absolute;top:644px;left:233px;white-space:nowrap" class="ft135">that takes a pixel below the bounds of [−80&#160;mV,&#160;+80&#160;mV] are reset to the closest boundary values. The subset&#160;</p>
<p style="position:absolute;top:659px;left:233px;white-space:nowrap" class="ft135">NFF&#160;consists of the&#160;more distorted&#160;variants of&#160;the French-flag. It is generated&#160;in&#160;the same&#160;way&#160;as FF except the&#160;</p>
<p style="position:absolute;top:674px;left:233px;white-space:nowrap" class="ft135">perturbation&#160;range&#160;is&#160;[−40&#160;mV,&#160;+40&#160;mV].&#160;A&#160;total&#160;of&#160;20&#160;samples&#160;in&#160;each&#160;set&#160;were&#160;generated&#160;during&#160;each&#160;iteration&#160;</p>
<p style="position:absolute;top:689px;left:233px;white-space:nowrap" class="ft135">of the GA-BP search.</p>
<p style="position:absolute;top:718px;left:233px;white-space:nowrap" class="ft139"><b>Data availability</b></p>
<p style="position:absolute;top:735px;left:233px;white-space:nowrap" class="ft135">All parameters used for the networks in this manuscript, as well as the source code, can be downloaded a<a href="https://gitlab.com/smanicka/BEN">t&#160;https://</a></p>
<p style="position:absolute;top:750px;left:233px;white-space:nowrap" class="ft1310"><a href="https://gitlab.com/smanicka/BEN">gitlab.com/smanicka/BEN</a>.</p>
<p style="position:absolute;top:779px;left:233px;white-space:nowrap" class="ft132">Received:&#160;21 June 2019;&#160;Accepted:&#160;13 November 2019;</p>
<p style="position:absolute;top:794px;left:233px;white-space:nowrap" class="ft132">Published: xx xx xxxx</p>
<p style="position:absolute;top:822px;left:233px;white-space:nowrap" class="ft139"><b>References</b></p>
<p style="position:absolute;top:839px;left:233px;white-space:nowrap" class="ft1311">&#160;&#160;1.&#160;&#160;Wiener, N. &amp; Schadé, J. P.&#160;<i>Cybernetics of the nervous system</i>. (Elsevier Pub. C., 1965).</p>
<p style="position:absolute;top:852px;left:233px;white-space:nowrap" class="ft1311">&#160;&#160;2.&#160;&#160;Wiener, N. &amp; Schadé, J. P.&#160;<i>Nerve, brain, and memory models</i>. (Elsevier Pub. Co., 1963).</p>
<p style="position:absolute;top:864px;left:233px;white-space:nowrap" class="ft1311">&#160;&#160;3.&#160;&#160;Simon, H. A. Studying human intelligence by creating artificial intelligence.&#160;<i>Am Sci</i>&#160;<b>69</b>, 300–309 (1981).</p>
<p style="position:absolute;top:877px;left:233px;white-space:nowrap" class="ft1311">&#160;&#160;4.&#160;&#160;Manicka, S. &amp; Levin, M. The Cognitive Lens: a primer on conceptual tools for analysing information processing in developmental&#160;</p>
<p style="position:absolute;top:890px;left:257px;white-space:nowrap" class="ft1311">and regenerative morphogenesis.&#160;<i>Philos Trans R Soc Lond B Biol Sci</i>&#160;<b>374</b>, 20180369,&#160;<a href="https://doi.org/10.1098/rstb.2018.0369">https://doi.org/10.1098/rstb.2018.0369&#160;(2019).</a></p>
<p style="position:absolute;top:903px;left:233px;white-space:nowrap" class="ft1311">&#160;&#160;5.&#160;&#160;Moore, D., Walker, S. I. &amp; Levin, M. Cancer as a disorder of patterning information: computational and biophysical perspectives&#160;</p>
<p style="position:absolute;top:915px;left:257px;white-space:nowrap" class="ft1311">on the cancer problem.&#160;<i>Convergent Science Physical</i>.&#160;<i>Oncology</i>&#160;<b>3</b>, 043001 (2017).</p>
<p style="position:absolute;top:928px;left:233px;white-space:nowrap" class="ft1311">&#160;&#160;6.&#160;&#160;Levin, M. &amp;&#160;Martyniuk, C. J. The bioelectric code: An ancient computational&#160;medium for dynamic control of growth and form.&#160;</p>
<p style="position:absolute;top:941px;left:257px;white-space:nowrap" class="ft1312"><i>Biosystems</i>,&#160;<a href="https://doi.org/10.1016/j.biosystems.2017.08.009">https://doi.org/10.1016/j.biosystems.2017.08.009</a>&#160;(2017).</p>
<p style="position:absolute;top:954px;left:233px;white-space:nowrap" class="ft1311">&#160;&#160;7.&#160;&#160;Pezzulo, G. &amp; Levin, M. Top-down models in biology: explanation and control of complex living systems above the molecular level.&#160;</p>
<p style="position:absolute;top:966px;left:257px;white-space:nowrap" class="ft1312"><i>J R Soc Interface</i>&#160;<b>13</b><a href="https://doi.org/10.1098/rsif.2016.0555">,&#160;https://doi.org/10.1098/rsif.2016.0555</a>&#160;(2016).</p>
<p style="position:absolute;top:979px;left:233px;white-space:nowrap" class="ft1311">&#160;&#160;8.&#160;&#160;Baluška, F. &amp; Levin, M. On Having No Head: Cognition throughout Biological Systems.&#160;<i>Front&#160;Psychol</i>&#160;<b>7</b>, 902,&#160;<a href="https://doi.org/10.3389/fpsyg.2016.00902">https://doi.</a></p>
<p style="position:absolute;top:992px;left:257px;white-space:nowrap" class="ft1314"><a href="https://doi.org/10.3389/fpsyg.2016.00902">org/10.3389/fpsyg.2016.00902</a>&#160;(2016).</p>
<p style="position:absolute;top:1005px;left:233px;white-space:nowrap" class="ft1311">&#160;&#160;9.&#160;&#160;Tang, S. K. Y. &amp; Marshal&#160;, W. F. Cell learning.&#160;<i>Curr&#160;Biol</i>&#160;<b>28</b>, R1180–R1184,&#160;<a href="https://doi.org/10.1016/j.cub.2018.09.015">https://doi.org/10.1016/j.cub.2018.09.015&#160;(2018).</a></p>
<p style="position:absolute;top:1017px;left:233px;white-space:nowrap" class="ft1311">&#160;&#160;10.&#160;&#160;Mitchell, A. &amp; Lim, W. Cellular perception and misperception: Internal models for decision-making shaped by evolutionary&#160;</p>
<p style="position:absolute;top:1030px;left:257px;white-space:nowrap" class="ft1311">experience.&#160;<i>Bioessays</i>&#160;<b>38</b>, 845–849,&#160;<a href="https://doi.org/10.1002/bies.201600090">https://doi.org/10.1002/bies.201600090</a>&#160;(2016).</p>
<p style="position:absolute;top:1043px;left:233px;white-space:nowrap" class="ft1311">&#160;&#160;11.&#160;&#160;Lyon, P. The cognitive cell: bacterial behavior reconsidered.&#160;<i>Front&#160;Microbiol</i>&#160;<b>6</b><a href="https://doi.org/10.3389/fmicb.2015.00264">, 264,&#160;https://doi.org/10.3389/fmicb.2015.00264&#160;</a></p>
<p style="position:absolute;top:1056px;left:257px;white-space:nowrap" class="ft1311">(2015).</p>
<p style="position:absolute;top:1068px;left:233px;white-space:nowrap" class="ft1311">&#160;&#160;12.&#160;&#160;Silva-Rocha, R., Tamames, J., dos Santos, V. M. &amp; de Lorenzo, V. The logicome of environmental bacteria: merging catabolic and&#160;</p>
<p style="position:absolute;top:1081px;left:257px;white-space:nowrap" class="ft1311">regulatory events with Boolean formalisms.&#160;<i>Environ&#160;Microbiol</i>&#160;<b>13</b><a href="https://doi.org/10.1111/j.1462-2920.2011.02455.x">, 2389–2402,&#160;https://doi.org/10.1111/j.1462-2920.2011.02455.x&#160;</a></p>
<p style="position:absolute;top:1094px;left:257px;white-space:nowrap" class="ft1311">(2011).</p>
</div>
</body>
</html>
<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="" xml:lang="">
<head>
<title>4b78c8d89b22cb315d90fd1274945de5-html.html</title>

<meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
 <br/>
<style type="text/css">
<!--
	p {margin: 0; padding: 0;}	.ft140{font-size:13px;font-family:Times;color:#cedde4;}
	.ft141{font-size:10px;font-family:Times;color:#000000;}
	.ft142{font-size:10px;font-family:Times;color:#000000;}
	.ft143{font-size:10px;font-family:Times;color:#000000;}
	.ft144{font-size:13px;font-family:Times;color:#000000;}
	.ft145{font-size:9px;font-family:Times;color:#000000;}
	.ft146{font-size:9px;font-family:Times;color:#000000;}
	.ft147{font-size:9px;font-family:Times;color:#000000;}
	.ft148{font-size:9px;font-family:Times;color:#0000ff;}
-->
</style>
</head>
<body bgcolor="#A0A0A0" vlink="blue" link="blue">
<div id="page14-div" style="position:relative;width:892px;height:1173px;">
<p style="position:absolute;top:1132px;left:816px;white-space:nowrap" class="ft140">14</p>
<p style="position:absolute;top:1135px;left:61px;white-space:nowrap" class="ft141"><b>Scientific</b>&#160;<b>RepoRtS</b>&#160;|&#160;</p>
<p style="position:absolute;top:1135px;left:192px;white-space:nowrap" class="ft143"><i>&#160; &#160; &#160; &#160; (2019) 9:18612&#160;</i>&#160;|&#160;<a href="https://doi.org/10.1038/s41598-019-54859-8">https://doi.org/10.1038/s41598-019-54859-8</a></p>
<p style="position:absolute;top:19px;left:585px;white-space:nowrap" class="ft144">www.nature.com/scientificreports</p>
<p style="position:absolute;top:19px;left:61px;white-space:nowrap" class="ft144">www.nature.com/scientificreports/</p>
<p style="position:absolute;top:71px;left:233px;white-space:nowrap" class="ft145">&#160;&#160;13.&#160;&#160;Vandenberg, L. N., Adams, D. S. &amp; Levin, M. Normalized shape and location of perturbed craniofacial structures in the Xenopus&#160;</p>
<p style="position:absolute;top:84px;left:257px;white-space:nowrap" class="ft145">tadpole reveal an innate ability to achieve correct morphology.&#160;<i>Dev&#160;Dyn</i>&#160;<b>241</b>, 863–878,&#160;<a href="https://doi.org/10.1002/dvdy.23770">https://doi.org/10.1002/dvdy.23770&#160;(2012).</a></p>
<p style="position:absolute;top:97px;left:233px;white-space:nowrap" class="ft145">&#160;&#160;14.&#160;&#160;Pezzulo, G. &amp; Levin, M. Re-membering the body: applications of computational neuroscience to the top-down control of&#160;</p>
<p style="position:absolute;top:110px;left:257px;white-space:nowrap" class="ft145">regeneration of limbs and other complex organs.&#160;<i>Integr Biol (Camb)</i>&#160;<b>7</b><a href="https://doi.org/10.1039/c5ib00221d">, 1487–1517,&#160;https://doi.org/10.1039/c5ib00221d&#160;(2015).</a></p>
<p style="position:absolute;top:122px;left:233px;white-space:nowrap" class="ft145">&#160;&#160;15.&#160;&#160;Farinel&#160;a-Ferruzza, N. The transformation of a tail into a limb after xenoplastic transformation.&#160;<i>Experientia</i>&#160;<b>15</b>, 304–305 (1956).</p>
<p style="position:absolute;top:135px;left:233px;white-space:nowrap" class="ft145">&#160;&#160;16.&#160;&#160;Friston, K., Levin, M., Sengupta, B. &amp; Pezzulo, G. Knowing one’s place: a free-energy approach to pattern regulation.&#160;<i>J R Soc&#160;</i></p>
<p style="position:absolute;top:148px;left:257px;white-space:nowrap" class="ft146"><i>Interface</i>&#160;<b>12</b><a href="https://doi.org/10.1098/rsif.2014.1383">,&#160;https://doi.org/10.1098/rsif.2014.1383&#160;(2015).</a></p>
<p style="position:absolute;top:161px;left:233px;white-space:nowrap" class="ft145">&#160;&#160;17.&#160;&#160;Lyon, P. The biogenic approach to cognition.&#160;<i>Cogn Process</i>&#160;<b>7</b><a href="https://doi.org/10.1007/s10339-005-0016-8">, 11–29,&#160;https://doi.org/10.1007/s10339-005-0016-8&#160;(2006).</a></p>
<p style="position:absolute;top:173px;left:233px;white-space:nowrap" class="ft145">&#160;&#160;18.&#160;&#160;Prindle, A.&#160;<i>et&#160;al</i>. Ion channels enable electrical communication in bacterial communities.&#160;<i>Nature</i>,&#160;<a href="https://doi.org/10.1038/nature15709">https://doi.org/10.1038/</a></p>
<p style="position:absolute;top:186px;left:257px;white-space:nowrap" class="ft148"><a href="https://doi.org/10.1038/nature15709">nature15709</a>&#160;(2015).</p>
<p style="position:absolute;top:199px;left:233px;white-space:nowrap" class="ft145">&#160;&#160;19.&#160;&#160;Levin, M. Morphogenetic fields in embryogenesis, regeneration, and cancer: non-local control of complex patterning.&#160;<i>Bio Systems</i>&#160;</p>
<p style="position:absolute;top:211px;left:257px;white-space:nowrap" class="ft147"><b>109</b><a href="https://doi.org/10.1016/j.biosystems.2012.04.005">, 243–261,&#160;https://doi.org/10.1016/j.biosystems.2012.04.005&#160;(2012).</a></p>
<p style="position:absolute;top:224px;left:233px;white-space:nowrap" class="ft145">&#160;&#160;20.&#160;&#160;Bates, E. Ion Channels in Development and Cancer.&#160;<i>Annu Rev Cell Dev Biol</i>&#160;<b>31</b>, 231–247,&#160;<a href="https://doi.org/10.1146/annurev-cellbio-100814-125338">https://doi.org/10.1146/annurev-</a></p>
<p style="position:absolute;top:237px;left:257px;white-space:nowrap" class="ft148"><a href="https://doi.org/10.1146/annurev-cellbio-100814-125338">cel&#160;bio-100814-125338&#160;(2015).</a></p>
<p style="position:absolute;top:250px;left:233px;white-space:nowrap" class="ft145">&#160;&#160;21.&#160;&#160;Pietak,&#160;A.&#160;&amp;&#160;Levin,&#160;M.&#160;Bioelectrical&#160;control&#160;of&#160;positional&#160;information&#160;in&#160;development&#160;and&#160;regeneration:&#160;A&#160;review&#160;of&#160;conceptual&#160;and&#160;</p>
<p style="position:absolute;top:263px;left:257px;white-space:nowrap" class="ft145">computational advances.&#160;<i>Prog Biophys Mol Biol</i>&#160;<b>137</b>, 52–68,&#160;<a href="https://doi.org/10.1016/j.pbiomolbio.2018.03.008">https://doi.org/10.1016/j.pbiomolbio.2018.03.008</a>&#160;(2018).</p>
<p style="position:absolute;top:275px;left:233px;white-space:nowrap" class="ft145">&#160;&#160;22.&#160;&#160;McLaughlin, K. A. &amp; Levin, M. Bioelectric signaling in regeneration: Mechanisms of ionic controls of growth and form.&#160;<i>Dev&#160;Biol</i>&#160;</p>
<p style="position:absolute;top:288px;left:257px;white-space:nowrap" class="ft147"><b>433</b><a href="https://doi.org/10.1016/j.ydbio.2017.08.032">, 177–189,&#160;https://doi.org/10.1016/j.ydbio.2017.08.032&#160;(2018).</a></p>
<p style="position:absolute;top:301px;left:233px;white-space:nowrap" class="ft145">&#160;&#160;23.&#160;&#160;Mathews, J. &amp; Levin, M. The body electric 2.0: recent advances in developmental bioelectricity for regenerative and synthetic&#160;</p>
<p style="position:absolute;top:314px;left:257px;white-space:nowrap" class="ft145">bioengineering.&#160;<i>Curr&#160;Opin&#160;Biotechnol</i>&#160;<b>52</b>, 134–144,&#160;<a href="https://doi.org/10.1016/j.copbio.2018.03.008">https://doi.org/10.1016/j.copbio.2018.03.008&#160;(2018).</a></p>
<p style="position:absolute;top:326px;left:233px;white-space:nowrap" class="ft145">&#160;&#160;24.&#160;&#160;Durant, F.&#160;<i>et&#160;al</i>. Long-Term, Stochastic Editing of Regenerative Anatomy via Targeting Endogenous Bioelectric Gradients.&#160;<i>Biophys&#160;</i></p>
<p style="position:absolute;top:339px;left:257px;white-space:nowrap" class="ft146"><i>J</i>&#160;<b>112</b>, 2231–2243,&#160;<a href="https://doi.org/10.1016/j.bpj.2017.04.011">https://doi.org/10.1016/j.bpj.2017.04.011&#160;(2017).</a></p>
<p style="position:absolute;top:352px;left:233px;white-space:nowrap" class="ft145">&#160;&#160;25.&#160;&#160;Beane, W.&#160;S.,&#160;Morokuma, J., Adams, D. S.&#160;&amp; Levin, M. A Chemical&#160;genetics&#160;approach&#160;reveals H,K-ATPase-mediated membrane&#160;</p>
<p style="position:absolute;top:365px;left:257px;white-space:nowrap" class="ft145">voltage is required for planarian head regeneration.&#160;<i>Chemistry &amp;</i>.&#160;<i>Biology</i>&#160;<b>18</b>, 77–89 (2011).</p>
<p style="position:absolute;top:377px;left:233px;white-space:nowrap" class="ft145">&#160;&#160;26.&#160;&#160;Pitcairn, E.&#160;<i>et&#160;al</i>. Coordinating heart morphogenesis: A novel role for Hyperpolarization-activated cyclic nucleotide-gated (HCN)&#160;</p>
<p style="position:absolute;top:390px;left:257px;white-space:nowrap" class="ft145">channels during cardiogenesis in Xenopus laevis.&#160;<i>Communicative&#160;&amp;&#160;Integrative&#160;Biology</i>&#160;<b>10</b>, e1309488,&#160;<a href="https://doi.org/10.1080/19420889.2017.1309488">https://doi.org/10.1080/194</a></p>
<p style="position:absolute;top:403px;left:257px;white-space:nowrap" class="ft148"><a href="https://doi.org/10.1080/19420889.2017.1309488">20889.2017.1309488&#160;(2017).</a></p>
<p style="position:absolute;top:416px;left:233px;white-space:nowrap" class="ft145">&#160;&#160;27.&#160;&#160;Pai, V. P.&#160;<i>et&#160;al</i>. HCN4 ion channel function is required for early events that regulate anatomical left-right patterning in a nodal and&#160;</p>
<p style="position:absolute;top:428px;left:257px;white-space:nowrap" class="ft145">lefty asymmetric gene expression-independent manner.&#160;<i>Biology&#160;Open</i>&#160;<b>6</b><a href="https://doi.org/10.1242/bio.025957">, 1445–1457,&#160;https://doi.org/10.1242/bio.025957&#160;(2017).</a></p>
<p style="position:absolute;top:441px;left:233px;white-space:nowrap" class="ft145">&#160;&#160;28.&#160;&#160;Pai, V. P., Aw, S., Shomrat, T., Lemire, J. M. &amp; Levin, M. Transmembrane voltage potential controls embryonic eye patterning in&#160;</p>
<p style="position:absolute;top:454px;left:257px;white-space:nowrap" class="ft145">Xenopus laevis.&#160;<i>Development</i>&#160;<b>139</b>, 313–323,&#160;<a href="https://doi.org/10.1242/dev.073759">https://doi.org/10.1242/dev.073759&#160;(2012).</a></p>
<p style="position:absolute;top:467px;left:233px;white-space:nowrap" class="ft145">&#160;&#160;29.&#160;&#160;Pai, V. P.&#160;<i>et&#160;al</i>. Endogenous Gradients of Resting Potential Instructively Pattern Embryonic Neural Tissue via Notch Signaling and&#160;</p>
<p style="position:absolute;top:479px;left:257px;white-space:nowrap" class="ft145">Regulation of Proliferation.&#160;<i>The Journal of Neuroscience</i>&#160;<b>35</b><a href="https://doi.org/10.1523/JNEUROSCI.1877-14.2015">, 4366–4385,&#160;https://doi.org/10.1523/JNEUROSCI.1877-14.2015&#160;(2015).</a></p>
<p style="position:absolute;top:492px;left:233px;white-space:nowrap" class="ft145">&#160;&#160;30.&#160;&#160;Sundelacruz, S., Levin, M. &amp; Kaplan, D. L. Membrane potential controls adipogenic and osteogenic differentiation of mesenchymal&#160;</p>
<p style="position:absolute;top:505px;left:257px;white-space:nowrap" class="ft145">stem cel&#160;s.&#160;<i>PLoS One</i>&#160;<b>3</b>, e3737 (2008).</p>
<p style="position:absolute;top:517px;left:233px;white-space:nowrap" class="ft145">&#160;&#160;31.&#160;&#160;Perathoner, S.&#160;<i>et&#160;al</i>. Bioelectric signaling regulates size in zebrafish fins.&#160;<i>PLoS genetics</i>&#160;<b>10</b><a href="https://doi.org/10.1371/journal.pgen.1004080">, e1004080,&#160;https://doi.org/10.1371/</a></p>
<p style="position:absolute;top:530px;left:257px;white-space:nowrap" class="ft148"><a href="https://doi.org/10.1371/journal.pgen.1004080">journal.pgen.1004080&#160;(2014).</a></p>
<p style="position:absolute;top:543px;left:233px;white-space:nowrap" class="ft145">&#160;&#160;32.&#160;&#160;Tseng, A. S., Beane, W. S., Lemire, J. M., Masi, A. &amp; Levin, M. Induction of vertebrate regeneration by a transient sodium current.&#160;</p>
<p style="position:absolute;top:556px;left:257px;white-space:nowrap" class="ft146"><i>J Neurosci</i>&#160;<b>30</b>, 13192–13200, 30/39/13192 [pii],&#160;<a href="https://doi.org/10.1523/JNEUROSCI.3315-10.2010">https://doi.org/10.1523/JNEUROSCI.3315-10.2010</a>&#160;(2010).</p>
<p style="position:absolute;top:568px;left:233px;white-space:nowrap" class="ft145">&#160;&#160;33.&#160;&#160;Adams, D. S., Masi, A. &amp; Levin, M. H+&#160;pump-dependent changes in membrane voltage are an early mechanism necessary and&#160;</p>
<p style="position:absolute;top:581px;left:257px;white-space:nowrap" class="ft145">sufficient to induce Xenopus tail regeneration.&#160;<i>Development</i>&#160;<b>134</b>, 1323–1335 (2007).</p>
<p style="position:absolute;top:594px;left:233px;white-space:nowrap" class="ft145">&#160;&#160;34.&#160;&#160;Oviedo, N. J.&#160;<i>et&#160;al</i>. Long-range neural and gap junction protein-mediated cues control polarity during planarian regeneration.&#160;<i>Dev&#160;</i></p>
<p style="position:absolute;top:607px;left:257px;white-space:nowrap" class="ft146"><i>Biol</i>&#160;<b>339</b>, 188–199, S0012-1606(09)01402-X [pii],&#160;<a href="https://doi.org/10.1016/j.ydbio.2009.12.012">https://doi.org/10.1016/j.ydbio.2009.12.012&#160;(2010).</a></p>
<p style="position:absolute;top:619px;left:233px;white-space:nowrap" class="ft145">&#160;&#160;35.&#160;&#160;Adams, D. S.&#160;<i>et&#160;al</i>. Bioelectric signalling via potassium channels: a mechanism for craniofacial dysmorphogenesis in KCNJ2-</p>
<p style="position:absolute;top:632px;left:257px;white-space:nowrap" class="ft145">associated Andersen-Tawil Syndrome.&#160;<i>J&#160;Physiol</i>&#160;<b>594</b>, 3245–3270,&#160;<a href="https://doi.org/10.1113/JP271930">https://doi.org/10.1113/JP271930</a>&#160;(2016).</p>
<p style="position:absolute;top:645px;left:233px;white-space:nowrap" class="ft145">&#160;&#160;36.&#160;&#160;Masotti,&#160;A.&#160;<i>et&#160;al</i>. Keppen-Lubinsky Syndrome Is Caused by Mutations in the Inwardly Rectifying K(+) Channel Encoded by&#160;</p>
<p style="position:absolute;top:658px;left:257px;white-space:nowrap" class="ft145">KCNJ6.&#160;<i>Am J Hum Genet</i>&#160;<b>96</b>, 295–300,&#160;<a href="https://doi.org/10.1016/j.ajhg.2014.12.011">https://doi.org/10.1016/j.ajhg.2014.12.011</a>&#160;(2015).</p>
<p style="position:absolute;top:670px;left:233px;white-space:nowrap" class="ft145">&#160;&#160;37.&#160;&#160;Pai, V. P.&#160;<i>et&#160;al</i>.&#160;HCN2&#160;Rescues&#160;brain&#160;defects&#160;by&#160;enforcing&#160;endogenous&#160;voltage&#160;pre-patterns.&#160;<i>Nature</i>&#160;<i>Communications</i>&#160;<b>9</b>,<a href="https://doi.org/10.1038/s41467-018-03334-5">&#160;https://doi.</a></p>
<p style="position:absolute;top:683px;left:257px;white-space:nowrap" class="ft148"><a href="https://doi.org/10.1038/s41467-018-03334-5">org/10.1038/s41467-018-03334-5&#160;(2018).</a></p>
<p style="position:absolute;top:696px;left:233px;white-space:nowrap" class="ft145">&#160;&#160;38.&#160;&#160;McCulloch, W. S. &amp; Pitts, W. A logical calculus of the ideas immanent in nervous activity.&#160;<i>Bul&#160;etin&#160;of&#160;Mathematical&#160;Biophysics</i>&#160;<b>5</b>&#160;</p>
<p style="position:absolute;top:709px;left:257px;white-space:nowrap" class="ft145">(1943).</p>
<p style="position:absolute;top:721px;left:233px;white-space:nowrap" class="ft145">&#160;&#160;39.&#160;&#160;Goldental, A.,&#160;Guberman,&#160;S.,&#160;Vardi,&#160;R.&#160;&amp; Kanter,&#160;I.&#160;A&#160;computational&#160;paradigm&#160;for&#160;dynamic&#160;logic-gates&#160;in&#160;neuronal&#160;activity.&#160;<i>Front&#160;</i></p>
<p style="position:absolute;top:734px;left:257px;white-space:nowrap" class="ft146"><i>Comput Neurosci</i>&#160;<b>8</b><a href="https://doi.org/10.3389/fncom.2014.00052">, 52,&#160;https://doi.org/10.3389/fncom.2014.00052&#160;(2014).</a></p>
<p style="position:absolute;top:747px;left:233px;white-space:nowrap" class="ft145">&#160;&#160;40.&#160;&#160;Albert, R. E. In&#160;<i>Complex Networks. Lecture Notes in Physics</i>&#160;Vol. 650 (ed Frauenfelder H. Ben-Naim E., Toroczkai Z.) (Springer,&#160;</p>
<p style="position:absolute;top:760px;left:257px;white-space:nowrap" class="ft145">2004).</p>
<p style="position:absolute;top:772px;left:233px;white-space:nowrap" class="ft145">&#160;&#160;41.&#160;&#160;Albert, R. &amp; Thakar, J. Boolean modeling:&#160;a logic-based&#160;dynamic approach&#160;for&#160;understanding&#160;signaling&#160;and regulatory networks&#160;</p>
<p style="position:absolute;top:785px;left:257px;white-space:nowrap" class="ft145">and for making useful predictions.&#160;<i>Wiley&#160;Interdisciplinary Reviews: Systems Biology and Medicine</i>&#160;<b>6</b>, 353–369,&#160;<a href="https://doi.org/10.1002/wsbm.1273">https://doi.</a></p>
<p style="position:absolute;top:798px;left:257px;white-space:nowrap" class="ft148"><a href="https://doi.org/10.1002/wsbm.1273">org/10.1002/wsbm.1273&#160;(2014).</a></p>
<p style="position:absolute;top:811px;left:233px;white-space:nowrap" class="ft145">&#160;&#160;42.&#160;&#160;Crama, Y. &amp; Peter L. Hammer.&#160;<i>Boolean functions: Theory, algorithms, and applications</i>. (Cambridge University Press, 2011).</p>
<p style="position:absolute;top:823px;left:233px;white-space:nowrap" class="ft145">&#160;&#160;43.&#160;&#160;Copeland, B. J. What is computation.&#160;<i>Synthese</i>&#160;<b>108</b>, 335–359 (1996).</p>
<p style="position:absolute;top:836px;left:233px;white-space:nowrap" class="ft145">&#160;&#160;44.&#160;&#160;Horsman, C., Stepney, S., Wagner, R. C. &amp; Kendon, V. When does a physical system compute?&#160;<i>Proc Math Phys Eng Sci</i>&#160;<b>470</b>,&#160;</p>
<p style="position:absolute;top:849px;left:257px;white-space:nowrap" class="ft145">20140182,&#160;<a href="https://doi.org/10.1098/rspa.2014.0182">https://doi.org/10.1098/rspa.2014.0182&#160;(2014).</a></p>
<p style="position:absolute;top:862px;left:233px;white-space:nowrap" class="ft145">&#160;&#160;45.&#160;&#160;Nicholson, D. J. Is the cell real&#160;y a machine?&#160;<i>J&#160;Theor&#160;Biol</i>&#160;<b>477</b>, 108–126,&#160;<a href="https://doi.org/10.1016/j.jtbi.2019.06.002">https://doi.org/10.1016/j.jtbi.2019.06.002</a>&#160;(2019).</p>
<p style="position:absolute;top:874px;left:233px;white-space:nowrap" class="ft145">&#160;&#160;46.&#160;&#160;Harvey, I. Cognition is Not Computation; Evolution is Not Optimisation (1996).</p>
<p style="position:absolute;top:887px;left:233px;white-space:nowrap" class="ft145">&#160;&#160;47.&#160;&#160;Pietak, A. &amp; Levin, M. Exploring Instructive Physiological Signaling with the Bioelectric Tissue Simulation Engine.&#160;<i>Front&#160;Bioeng&#160;</i></p>
<p style="position:absolute;top:900px;left:257px;white-space:nowrap" class="ft146"><i>Biotechnol</i>&#160;<b>4</b>, 55,&#160;<a href="https://doi.org/10.3389/fbioe.2016.00055">https://doi.org/10.3389/fbioe.2016.00055&#160;(2016).</a></p>
<p style="position:absolute;top:913px;left:233px;white-space:nowrap" class="ft145">&#160;&#160;48.&#160;&#160;Javier Cervera, A. P. &amp; Levin, M. Salvador Mafe. Bioelectrical coupling in multicel&#160;ular domains regulated by gap junctions - A&#160;</p>
<p style="position:absolute;top:925px;left:257px;white-space:nowrap" class="ft145">conceptual approach.&#160;<i>Bioelectrochemistry</i>, 45–61,&#160;<a href="https://doi.org/10.1016/j.bioelechem.2018.04.013">https://doi.org/10.1016/j.bioelechem.2018.04.013&#160;(2018).</a></p>
<p style="position:absolute;top:938px;left:233px;white-space:nowrap" class="ft145">&#160;&#160;49.&#160;&#160;Pai,&#160;V.&#160;P.&#160;<i>et&#160;al</i>. HCN2 Rescues brain defects by enforcing endogenous voltage pre-patterns.&#160;<i>Nat&#160;Commun</i>&#160;<b>9</b><a href="https://doi.org/10.1038/s41467-018-03334-5">, 998,&#160;https://doi.</a></p>
<p style="position:absolute;top:951px;left:257px;white-space:nowrap" class="ft148"><a href="https://doi.org/10.1038/s41467-018-03334-5">org/10.1038/s41467-018-03334-5&#160;(2018).</a></p>
<p style="position:absolute;top:964px;left:233px;white-space:nowrap" class="ft145">&#160;&#160;50.&#160;&#160;Levin, M., Pietak, A. M. &amp; Bischof, J. Planarian regeneration as a model of anatomical homeostasis: Recent progress in biophysical&#160;</p>
<p style="position:absolute;top:976px;left:257px;white-space:nowrap" class="ft145">and computational approaches.&#160;<i>Semin Cell Dev Biol</i>,&#160;<a href="https://doi.org/10.1016/j.semcdb.2018.04.003">https://doi.org/10.1016/j.semcdb.2018.04.003&#160;(2018).</a></p>
<p style="position:absolute;top:989px;left:233px;white-space:nowrap" class="ft145">&#160;&#160;51.&#160;&#160;Pietak, A. &amp; Levin, M. Bioelectric gene and reaction networks: computational modelling of genetic, biochemical and bioelectrical&#160;</p>
<p style="position:absolute;top:1002px;left:257px;white-space:nowrap" class="ft145">dynamics in pattern regulation.&#160;<i>J R Soc Interface</i>&#160;<b>14</b><a href="https://doi.org/10.1098/rsif.2017.0425">,&#160;https://doi.org/10.1098/rsif.2017.0425&#160;(2017).</a></p>
<p style="position:absolute;top:1015px;left:233px;white-space:nowrap" class="ft145">&#160;&#160;52.&#160;&#160;Mathews, J. &amp; Levin, M. Gap junctional signaling in pattern regulation: Physiological network connectivity instructs growth and&#160;</p>
<p style="position:absolute;top:1027px;left:257px;white-space:nowrap" class="ft145">form.&#160;<i>Developmental&#160;neurobiology</i>&#160;<b>77</b>, 643–673,&#160;<a href="https://doi.org/10.1002/dneu.22405">https://doi.org/10.1002/dneu.22405&#160;(2017).</a></p>
<p style="position:absolute;top:1040px;left:233px;white-space:nowrap" class="ft145">&#160;&#160;53.&#160;&#160;Koseska, A. &amp; Bastiaens, P. I. Cell signaling as a cognitive process.&#160;<i>EMBO J</i>&#160;<b>36</b><a href="https://doi.org/10.15252/embj.201695383">, 568–582,&#160;https://doi.org/10.15252/embj.201695383&#160;</a></p>
<p style="position:absolute;top:1053px;left:257px;white-space:nowrap" class="ft145">(2017).</p>
<p style="position:absolute;top:1066px;left:233px;white-space:nowrap" class="ft145">&#160;&#160;54.&#160;&#160;Barnard, A., Wolfe, A. &amp; Busby, S. Regulation at complex bacterial promoters: how bacteria use different promoter organizations&#160;</p>
<p style="position:absolute;top:1078px;left:257px;white-space:nowrap" class="ft145">to produce different regulatory outcomes.&#160;<i>Curr&#160;Opin&#160;Microbiol</i>&#160;<b>7</b>, 102–108,&#160;<a href="https://doi.org/10.1016/j.mib.2004.02.011">https://doi.org/10.1016/j.mib.2004.02.011&#160;(2004).</a></p>
</div>
</body>
</html>
<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="" xml:lang="">
<head>
<title>4b78c8d89b22cb315d90fd1274945de5-html.html</title>

<meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
 <br/>
<style type="text/css">
<!--
	p {margin: 0; padding: 0;}	.ft150{font-size:13px;font-family:Times;color:#cedde4;}
	.ft151{font-size:10px;font-family:Times;color:#000000;}
	.ft152{font-size:10px;font-family:Times;color:#000000;}
	.ft153{font-size:10px;font-family:Times;color:#000000;}
	.ft154{font-size:13px;font-family:Times;color:#000000;}
	.ft155{font-size:9px;font-family:Times;color:#000000;}
	.ft156{font-size:9px;font-family:Times;color:#000000;}
	.ft157{font-size:9px;font-family:Times;color:#000000;}
	.ft158{font-size:9px;font-family:Times;color:#0000ff;}
-->
</style>
</head>
<body bgcolor="#A0A0A0" vlink="blue" link="blue">
<div id="page15-div" style="position:relative;width:892px;height:1173px;">
<p style="position:absolute;top:1132px;left:817px;white-space:nowrap" class="ft150">15</p>
<p style="position:absolute;top:1135px;left:61px;white-space:nowrap" class="ft151"><b>Scientific</b>&#160;<b>RepoRtS</b>&#160;|&#160;</p>
<p style="position:absolute;top:1135px;left:192px;white-space:nowrap" class="ft153"><i>&#160; &#160; &#160; &#160; (2019) 9:18612&#160;</i>&#160;|&#160;<a href="https://doi.org/10.1038/s41598-019-54859-8">https://doi.org/10.1038/s41598-019-54859-8</a></p>
<p style="position:absolute;top:19px;left:585px;white-space:nowrap" class="ft154">www.nature.com/scientificreports</p>
<p style="position:absolute;top:19px;left:61px;white-space:nowrap" class="ft154">www.nature.com/scientificreports/</p>
<p style="position:absolute;top:71px;left:233px;white-space:nowrap" class="ft155">&#160;&#160;55.&#160;&#160;Brodskiy, P. A. &amp; Zartman, J. J. Calcium as a signal integrator in developing epithelial tissues.&#160;<i>Physical&#160;biology</i>&#160;<b>15</b><a href="https://doi.org/10.1088/1478-3975/aabb18">, 051001,&#160;https://</a></p>
<p style="position:absolute;top:84px;left:257px;white-space:nowrap" class="ft158"><a href="https://doi.org/10.1088/1478-3975/aabb18">doi.org/10.1088/1478-3975/aabb18&#160;(2018).</a></p>
<p style="position:absolute;top:97px;left:233px;white-space:nowrap" class="ft155">&#160;&#160;56.&#160;&#160;Guillermo Solovey, S. P. D. Intra-Cluster Percolation of Calcium Signals.&#160;<i>PLoS One</i>&#160;<b>5</b><a href="https://doi.org/10.1371/journal.pone.0008997">,&#160;https://doi.org/10.1371/journal.pone.0008997&#160;</a></p>
<p style="position:absolute;top:110px;left:257px;white-space:nowrap" class="ft155">(2010).</p>
<p style="position:absolute;top:122px;left:233px;white-space:nowrap" class="ft155">&#160;&#160;57.&#160;&#160;Lopez, L., Piegari, E., Sigaut, L. &amp; Ponce Dawson, S. Intracel&#160;ular calcium signals display an avalanche-like behavior over multiple&#160;</p>
<p style="position:absolute;top:135px;left:257px;white-space:nowrap" class="ft155">lengthscales.&#160;<i>Front&#160;Physiol</i>&#160;<b>3</b>, 350,&#160;<a href="https://doi.org/10.3389/fphys.2012.00350">https://doi.org/10.3389/fphys.2012.00350&#160;(2012).</a></p>
<p style="position:absolute;top:148px;left:233px;white-space:nowrap" class="ft155">&#160;&#160;58.&#160;&#160;Polsky, A., Mel, B. W. &amp; Schiller, J. Computational subunits in thin dendrites of pyramidal cel&#160;s.&#160;<i>Nat Neurosci</i>&#160;<b>7</b><a href="https://doi.org/10.1038/nn1253">, 621–627,&#160;https://</a></p>
<p style="position:absolute;top:161px;left:257px;white-space:nowrap" class="ft158"><a href="https://doi.org/10.1038/nn1253">doi.org/10.1038/nn1253&#160;(2004).</a></p>
<p style="position:absolute;top:173px;left:233px;white-space:nowrap" class="ft155">&#160;&#160;59.&#160;&#160;Panayiota Poirazi, T. B. &amp; Bartlett, W. Mel. Pyramidal neuron as two-layer neural network.&#160;<i>Neuron</i>&#160;<b>37</b>, 989–999 (2003).</p>
<p style="position:absolute;top:186px;left:233px;white-space:nowrap" class="ft155">&#160;&#160;60.&#160;&#160;Antic, S. D., Hines, M. &amp; Lytton, W. W. Embedded ensemble encoding hypothesis: The role of the “Prepared” cel&#160;.&#160;<i>J Neurosci Res</i>,&#160;</p>
<p style="position:absolute;top:199px;left:257px;white-space:nowrap" class="ft158"><a href="https://doi.org/10.1002/jnr.24240">https://doi.org/10.1002/jnr.24240</a>&#160;(2018).</p>
<p style="position:absolute;top:212px;left:233px;white-space:nowrap" class="ft155">&#160;&#160;61.&#160;&#160;Sperelakis, N.&#160;<i>Cell physiology source book: essentials of membrane biophysics</i>. (Elsevier, 2012).</p>
<p style="position:absolute;top:224px;left:233px;white-space:nowrap" class="ft155">&#160;&#160;62.&#160;&#160;Bukauskas, F. F. &amp; Verselis, V. K. Gap junction channel gating.&#160;<i>Biochim&#160;Biophys&#160;Acta</i>&#160;<b>1662</b><a href="https://doi.org/10.1016/j.bbamem.2004.01.008">, 42–60,&#160;https://doi.org/10.1016/j.</a></p>
<p style="position:absolute;top:237px;left:257px;white-space:nowrap" class="ft158"><a href="https://doi.org/10.1016/j.bbamem.2004.01.008">bbamem.2004.01.008</a>&#160;(2004).</p>
<p style="position:absolute;top:250px;left:233px;white-space:nowrap" class="ft155">&#160;&#160;63.&#160;&#160;Brink, P. Gap Junction Voltage Dependence - a clear picture emerges.&#160;<i>J. Gen. Physiol</i>&#160;<b>116</b>, 11–12 (2000).</p>
<p style="position:absolute;top:263px;left:233px;white-space:nowrap" class="ft155">&#160;&#160;64.&#160;&#160;Nielsen, M. A.&#160;<i>Neural networks and deep learning</i>. Vol. 25 (Determination press, 2015).</p>
<p style="position:absolute;top:275px;left:233px;white-space:nowrap" class="ft155">&#160;&#160;65.&#160;&#160;Brodmann, K.&#160;<i>Brodmann’s: Localisation in the cerebral cortex</i>. (Springer Science &amp; Business Media, 2007).</p>
<p style="position:absolute;top:288px;left:233px;white-space:nowrap" class="ft155">&#160;&#160;66.&#160;&#160;Hinczewski, M. &amp; Thirumalai, D. Cel&#160;ular signaling networks function as generalized Wiener-Kolmogorov filters to suppress noise.&#160;</p>
<p style="position:absolute;top:301px;left:257px;white-space:nowrap" class="ft156"><i>Physical Review X</i>&#160;<b>4</b>&#160;(2014).</p>
<p style="position:absolute;top:314px;left:233px;white-space:nowrap" class="ft155">&#160;&#160;67.&#160;&#160;Becker,&#160;N.&#160;B.,&#160;Mugler,&#160;A.&#160;&amp;&#160;Ten&#160;Wolde,&#160;P.&#160;R.&#160;Optimal&#160;Prediction&#160;by&#160;Cel&#160;ular&#160;Signaling&#160;Networks.&#160;<i>Phys&#160;Rev&#160;Lett</i>&#160;<b>115</b>,&#160;258103,&#160;<a href="https://doi.org/10.1103/PhysRevLett.115.258103">https://</a></p>
<p style="position:absolute;top:326px;left:257px;white-space:nowrap" class="ft158"><a href="https://doi.org/10.1103/PhysRevLett.115.258103">doi.org/10.1103/PhysRevLett.115.258103&#160;(2015).</a></p>
<p style="position:absolute;top:339px;left:233px;white-space:nowrap" class="ft155">&#160;&#160;68.&#160;&#160;Sagner, A. &amp; Briscoe, J. Morphogen interpretation: concentration, time, competence, and signaling dynamics.&#160;<i>Wiley Interdiscip&#160;Rev&#160;</i></p>
<p style="position:absolute;top:352px;left:257px;white-space:nowrap" class="ft156"><i>Dev&#160;Biol</i>&#160;<b>6</b>,&#160;<a href="https://doi.org/10.1002/wdev.271">https://doi.org/10.1002/wdev.271&#160;(2017).</a></p>
<p style="position:absolute;top:365px;left:233px;white-space:nowrap" class="ft155">&#160;&#160;69.&#160;&#160;Ebisuya, M. &amp; Briscoe, J. What does time mean in development?&#160;<i>Development</i>&#160;<b>145</b><a href="https://doi.org/10.1242/dev.164368">,&#160;https://doi.org/10.1242/dev.164368&#160;(2018).</a></p>
<p style="position:absolute;top:377px;left:233px;white-space:nowrap" class="ft155">&#160;&#160;70.&#160;&#160;Herrera-Rincon,&#160;C.&#160;<i>et&#160;al</i>. Brief Local Application of&#160;Progesterone&#160;via&#160;a&#160;Wearable Bioreactor&#160;Induces&#160;Long-Term Regenerative&#160;</p>
<p style="position:absolute;top:390px;left:257px;white-space:nowrap" class="ft155">Response in Adult Xenopus Hindlimb.&#160;<i>Cell&#160;Rep</i>&#160;<b>25</b><a href="https://doi.org/10.1016/j.celrep.2018.10.010">, 1593–1609 e1597,&#160;https://doi.org/10.1016/j.celrep.2018.10.010</a>&#160;(2018).</p>
<p style="position:absolute;top:403px;left:233px;white-space:nowrap" class="ft155">&#160;&#160;71.&#160;&#160;Amor, D. R., Ratzke, C. &amp; Gore, J. Transient invaders can induce shifts between alternative stable states of microbial communities.&#160;</p>
<p style="position:absolute;top:415px;left:257px;white-space:nowrap" class="ft156"><i>BioRxiv</i><a href="https://doi.org/10.1101/659052">,&#160;https://doi.org/10.1101/659052</a>&#160;(2019).</p>
<p style="position:absolute;top:428px;left:233px;white-space:nowrap" class="ft155">&#160;&#160;72.&#160;&#160;Durant, F.&#160;<i>et&#160;al</i>. The role of early bioelectric signals in the regeneration of planarian anterior-posterior polarity.&#160;<i>Biophys J</i><a href="https://doi.org/10.1016/j.bpj.2019.01.029">,&#160;https://</a></p>
<p style="position:absolute;top:441px;left:257px;white-space:nowrap" class="ft158"><a href="https://doi.org/10.1016/j.bpj.2019.01.029">doi.org/10.1016/j.bpj.2019.01.029&#160;(2019).</a></p>
<p style="position:absolute;top:454px;left:233px;white-space:nowrap" class="ft155">&#160;&#160;73.&#160;&#160;Davidson, S. I. A. E. H. Logic functions of the genomic cis-regulatory code.&#160;<i>PNAS</i>&#160;<b>102</b>, 4954–4959 (2004).</p>
<p style="position:absolute;top:467px;left:233px;white-space:nowrap" class="ft155">&#160;&#160;74.&#160;&#160;Palacios-Prado, N. &amp; Bukauskas, F. F. Heterotypic gap junction channels as voltage-sensitive valves for intercel&#160;ular signaling.&#160;<i>Proc&#160;</i></p>
<p style="position:absolute;top:479px;left:257px;white-space:nowrap" class="ft156"><i>Natl Acad Sci USA</i>&#160;<b>106</b>, 14855–14860, 0901923106 [pii],&#160;<a href="https://doi.org/10.1073/pnas.0901923106">https://doi.org/10.1073/pnas.0901923106</a>&#160;(2009).</p>
<p style="position:absolute;top:492px;left:233px;white-space:nowrap" class="ft155">&#160;&#160;75.&#160;&#160;Birnbaum, K. D. &amp; Alvarado, A. S. Slicing across kingdoms: regeneration in plants and animals.&#160;<i>Cell</i>&#160;<b>132</b>, 697–710 (2008).</p>
<p style="position:absolute;top:505px;left:233px;white-space:nowrap" class="ft155">&#160;&#160;76.&#160;&#160;Pinet, K. &amp; McLaughlin, K. A. Mechanisms of&#160;physiological&#160;tissue remodeling&#160;in&#160;animals: Manipulating tissue, organ, and&#160;</p>
<p style="position:absolute;top:517px;left:257px;white-space:nowrap" class="ft155">organism morphology.&#160;<i>Dev&#160;Biol</i><a href="https://doi.org/10.1016/j.ydbio.2019.04.001">,&#160;https://doi.org/10.1016/j.ydbio.2019.04.001</a>&#160;(2019).</p>
<p style="position:absolute;top:530px;left:233px;white-space:nowrap" class="ft155">&#160;&#160;77.&#160;&#160;Wolpert, L. Positional information and the spatial pattern of cel&#160;ular differentiation.&#160;<i>J.&#160;Theor.&#160;Biol</i>&#160;<b>25</b>, 1–47 (1969).</p>
<p style="position:absolute;top:543px;left:233px;white-space:nowrap" class="ft155">&#160;&#160;78.&#160;&#160;Dawkins, R.&#160;<i>The blind watchmaker: Why the evidence of evolution reveals a universe without design</i>. (WW Norton &amp; Company,&#160;</p>
<p style="position:absolute;top:556px;left:257px;white-space:nowrap" class="ft155">1996).</p>
<p style="position:absolute;top:568px;left:233px;white-space:nowrap" class="ft155">&#160;&#160;79.&#160;&#160;Espinosa-Soto, C. On the role of sparseness in the evolution of modularity in gene regulatory networks.&#160;<i>PLoS&#160;computational&#160;biology</i>&#160;</p>
<p style="position:absolute;top:581px;left:257px;white-space:nowrap" class="ft157"><b>14</b>&#160;(2018).</p>
<p style="position:absolute;top:594px;left:233px;white-space:nowrap" class="ft155">&#160;&#160;80.&#160;&#160;Hilger, K., Ekman, M., Fiebach, C. J. &amp; Basten, U. Intelligence is associated with the modular structure of intrinsic brain networks.&#160;</p>
<p style="position:absolute;top:607px;left:257px;white-space:nowrap" class="ft156"><i>Scientific&#160;reports</i>&#160;<b>7</b>&#160;(2017).</p>
<p style="position:absolute;top:619px;left:233px;white-space:nowrap" class="ft155">&#160;&#160;81.&#160;&#160;Meunier, D., Renaud, L. &amp; Edward T. Bullmore. Modular and hierarchical&#160;y modular organization of brain networks.&#160;<i>Frontiers&#160;in&#160;</i></p>
<p style="position:absolute;top:632px;left:257px;white-space:nowrap" class="ft156"><i>neuroscience</i>&#160;<b>4</b>&#160;(2010).</p>
<p style="position:absolute;top:645px;left:233px;white-space:nowrap" class="ft155">&#160;&#160;82.&#160;&#160;Marchisio, M. A. &amp; Stelling, J. Automatic design of digital synthetic gene circuits.&#160;<i>PLoS&#160;Comput&#160;Biol</i>&#160;<b>7</b>, e1001083,&#160;<a href="https://doi.org/10.1371/journal.pcbi.1001083">https://doi.</a></p>
<p style="position:absolute;top:658px;left:257px;white-space:nowrap" class="ft158"><a href="https://doi.org/10.1371/journal.pcbi.1001083">org/10.1371/journal.pcbi.1001083&#160;(2011).</a></p>
<p style="position:absolute;top:670px;left:233px;white-space:nowrap" class="ft155">&#160;&#160;83.&#160;&#160;Alon, U.&#160;<i>An Introduction to Systems Biology: Design Principles of Biological Circuits</i>. (Chapman and Hall/CRC Press, 2006).</p>
<p style="position:absolute;top:683px;left:233px;white-space:nowrap" class="ft155">&#160;&#160;84.&#160;&#160;Wang, B., Kitney, R. I., Joly, N. &amp; Buck, M. Engineering modular and orthogonal genetic logic gates for robust digital-like synthetic&#160;</p>
<p style="position:absolute;top:696px;left:257px;white-space:nowrap" class="ft155">biology.&#160;<i>Nat&#160;Commun</i>&#160;<b>2</b>, 508,&#160;<a href="https://doi.org/10.1038/ncomms1516">https://doi.org/10.1038/ncomms1516</a>&#160;(2011).</p>
<p style="position:absolute;top:709px;left:233px;white-space:nowrap" class="ft155">&#160;&#160;85.&#160;&#160;Amos, A. G. N.-M. A. M. A reconfigurable NAND-NOR genetic logic gate.&#160;<i>BMC Syst Biol</i>&#160;<b>6</b>&#160;(2012).</p>
<p style="position:absolute;top:721px;left:233px;white-space:nowrap" class="ft155">&#160;&#160;86.&#160;&#160;Sole, R., Moses, M. &amp; Forrest, S. Liquid brains, solid brains.&#160;<i>Philos&#160;Trans&#160;R&#160;Soc&#160;Lond&#160;B&#160;Biol&#160;Sci</i>&#160;<b>374</b>, 20190040,&#160;<a href="https://doi.org/10.1098/rstb.2019.0040">https://doi.</a></p>
<p style="position:absolute;top:734px;left:257px;white-space:nowrap" class="ft158"><a href="https://doi.org/10.1098/rstb.2019.0040">org/10.1098/rstb.2019.0040&#160;(2019).</a></p>
<p style="position:absolute;top:747px;left:233px;white-space:nowrap" class="ft155">&#160;&#160;87.&#160;&#160;Chaitin, G. J.&#160;<i>Proving Darwin: Making Biology Mathematical</i>. (Pantheon Books, 2012).</p>
<p style="position:absolute;top:760px;left:233px;white-space:nowrap" class="ft155">&#160;&#160;88.&#160;&#160;Adamatzky, A. A brief history of liquid computers.&#160;<i>Philos Trans R Soc Lond B Biol Sci</i>&#160;<b>374</b><a href="https://doi.org/10.1098/rstb.2018.0372">, 20180372,&#160;https://doi.org/10.1098/</a></p>
<p style="position:absolute;top:772px;left:257px;white-space:nowrap" class="ft158"><a href="https://doi.org/10.1098/rstb.2018.0372">rstb.2018.0372&#160;(2019).</a></p>
<p style="position:absolute;top:785px;left:233px;white-space:nowrap" class="ft155">&#160;&#160;89.&#160;&#160;Allen Hjelmfelt, E. D. W. &amp; Ross, J. Chemical implementation of finite-state machines.&#160;<i>PNAS</i>&#160;<b>89</b>, 383–387 (1992).</p>
<p style="position:absolute;top:798px;left:233px;white-space:nowrap" class="ft155">&#160;&#160;90.&#160;&#160;Allen Hjelmfelt, E. D. W. &amp; Ross, J. Chemical implementation of neural networks and Turing machines.&#160;<i>PNAS</i>&#160;<b>88</b>, 10983–10987&#160;</p>
<p style="position:absolute;top:811px;left:257px;white-space:nowrap" class="ft155">(1991).</p>
<p style="position:absolute;top:823px;left:233px;white-space:nowrap" class="ft155">&#160;&#160;91.&#160;&#160;Egbert,&#160;M.,&#160;Gagnon,&#160;J.&#160;S.&#160;&amp;&#160;Perez-Mercader,&#160;J.&#160;Dynamic&#160;modulation&#160;of&#160;external&#160;conditions&#160;can&#160;transform&#160;chemistry&#160;into&#160;logic&#160;</p>
<p style="position:absolute;top:836px;left:257px;white-space:nowrap" class="ft155">gates.&#160;<i>J R Soc Interface</i>&#160;<b>15</b>,&#160;<a href="https://doi.org/10.1098/rsif.2018.0169">https://doi.org/10.1098/rsif.2018.0169</a>&#160;(2018).</p>
<p style="position:absolute;top:849px;left:233px;white-space:nowrap" class="ft155">&#160;&#160;92.&#160;&#160;Holley, J., Jahan, I., Costello Bde, L., Bul&#160;, L. &amp; Adamatzky, A. Logical and arithmetic circuits in Belousov-Zhabotinsky encapsulated&#160;</p>
<p style="position:absolute;top:862px;left:257px;white-space:nowrap" class="ft155">disks.&#160;<i>Phys Rev E Stat Nonlin Soft Matter Phys</i>&#160;<b>84</b>, 056110 (2011).</p>
<p style="position:absolute;top:874px;left:233px;white-space:nowrap" class="ft155">&#160;&#160;93.&#160;&#160;Privman, V., Arugula, M. A., Halámek, J., Pita, M. &amp; Katz, E. Network Analysis of Biochemical Logic for Noise Reduction and&#160;</p>
<p style="position:absolute;top:887px;left:257px;white-space:nowrap" class="ft155">Stability - A System of Three Coupled Enzymatic AND Gates.&#160;<i>J. Phys. Chem. B</i>, 5301–5310 (2009).</p>
<p style="position:absolute;top:900px;left:233px;white-space:nowrap" class="ft155">&#160;&#160;94.&#160;&#160;McGregor, S., Vasas, V., Husbands, P. &amp; Fernando, C. Evolution of associative learning in chemical networks.&#160;<i>PLoS&#160;computational&#160;</i></p>
<p style="position:absolute;top:912px;left:257px;white-space:nowrap" class="ft156"><i>biology</i>&#160;<b>8</b><a href="https://doi.org/10.1371/journal.pcbi.1002739">, e1002739,&#160;https://doi.org/10.1371/journal.pcbi.1002739&#160;(2012).</a></p>
<p style="position:absolute;top:925px;left:233px;white-space:nowrap" class="ft155">&#160;&#160;95.&#160;&#160;Fernando, C. T.&#160;<i>et&#160;al</i>. Molecular circuits for associative learning in single-celled organisms.&#160;<i>J R Soc Interface</i>&#160;<b>6</b>, 463–469,&#160;<a href="https://doi.org/10.1098/rsif.2008.0344">https://doi.</a></p>
<p style="position:absolute;top:938px;left:257px;white-space:nowrap" class="ft158"><a href="https://doi.org/10.1098/rsif.2008.0344">org/10.1098/rsif.2008.0344&#160;(2009).</a></p>
<p style="position:absolute;top:951px;left:233px;white-space:nowrap" class="ft155">&#160;&#160;96.&#160;&#160;Cherry, K. M. &amp; Qian, L. Scaling up molecular pattern recognition with DNA-based winner-take-all neural networks.&#160;<i>Nature</i>&#160;<b>559</b>,&#160;</p>
<p style="position:absolute;top:964px;left:257px;white-space:nowrap" class="ft155">370–376,&#160;<a href="https://doi.org/10.1038/s41586-018-0289-6">https://doi.org/10.1038/s41586-018-0289-6&#160;(2018).</a></p>
<p style="position:absolute;top:976px;left:233px;white-space:nowrap" class="ft155">&#160;&#160;97.&#160;&#160;Nicolas E. B. U. G. &amp; Terence H. On schemes of combinatorial transcription logic.&#160;<i>PNAS</i>&#160;<b>100</b>, 5136–5141 (2003).</p>
<p style="position:absolute;top:989px;left:233px;white-space:nowrap" class="ft155">&#160;&#160;98.&#160;&#160;Woods, D.&#160;<i>et&#160;al</i>. Diverse and robust molecular algorithms using reprogrammable DNA self-assembly.&#160;<i>Nature</i>&#160;<b>567</b>, 366–372,&#160;<a href="https://doi.org/10.1038/s41586-019-1014-9">https://</a></p>
<p style="position:absolute;top:1002px;left:257px;white-space:nowrap" class="ft158"><a href="https://doi.org/10.1038/s41586-019-1014-9">doi.org/10.1038/s41586-019-1014-9</a>&#160;(2019).</p>
<p style="position:absolute;top:1015px;left:233px;white-space:nowrap" class="ft155">&#160;&#160;99.&#160;&#160;Sorek, M., Balaban, N. Q. &amp; Loewenstein, Y. Stochasticity, bistability and the wisdom of crowds: a model for associative learning in&#160;</p>
<p style="position:absolute;top:1027px;left:257px;white-space:nowrap" class="ft155">genetic regulatory networks.&#160;<i>PLoS&#160;computational&#160;biology</i>&#160;<b>9</b><a href="https://doi.org/10.1371/journal.pcbi.1003179">, e1003179,&#160;https://doi.org/10.1371/journal.pcbi.1003179&#160;(2013).</a></p>
<p style="position:absolute;top:1040px;left:233px;white-space:nowrap" class="ft155">&#160;100.&#160;&#160;Watson,&#160;R.&#160;A.,&#160;Wagner,&#160;G.&#160;P.,&#160;Pavlicev,&#160;M.,&#160;Weinreich,&#160;D.&#160;M.&#160;&amp;&#160;Mills,&#160;R.&#160;The&#160;evolution&#160;of&#160;phenotypic&#160;correlations&#160;and&#160;</p>
<p style="position:absolute;top:1053px;left:257px;white-space:nowrap" class="ft155">“developmental memory”.&#160;<i>Evolution</i>&#160;<b>68</b>, 1124–1138,&#160;<a href="https://doi.org/10.1111/evo.12337">https://doi.org/10.1111/evo.12337&#160;(2014).</a></p>
<p style="position:absolute;top:1066px;left:233px;white-space:nowrap" class="ft155">&#160;101.&#160;&#160;Goni-Moreno, A. &amp; Amos, M. Continuous computation in engineered gene circuits.&#160;<i>Biosystems</i>&#160;<b>109</b>, 52–56,&#160;<a href="https://doi.org/10.1016/j.biosystems.2012.02.001">https://doi.</a></p>
<p style="position:absolute;top:1078px;left:257px;white-space:nowrap" class="ft158"><a href="https://doi.org/10.1016/j.biosystems.2012.02.001">org/10.1016/j.biosystems.2012.02.001&#160;(2012).</a></p>
</div>
</body>
</html>
<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="" xml:lang="">
<head>
<title>4b78c8d89b22cb315d90fd1274945de5-html.html</title>

<meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
 <br/>
<style type="text/css">
<!--
	p {margin: 0; padding: 0;}	.ft160{font-size:13px;font-family:Times;color:#cedde4;}
	.ft161{font-size:10px;font-family:Times;color:#000000;}
	.ft162{font-size:10px;font-family:Times;color:#000000;}
	.ft163{font-size:10px;font-family:Times;color:#000000;}
	.ft164{font-size:13px;font-family:Times;color:#000000;}
	.ft165{font-size:9px;font-family:Times;color:#000000;}
	.ft166{font-size:9px;font-family:Times;color:#000000;}
	.ft167{font-size:9px;font-family:Times;color:#000000;}
	.ft168{font-size:9px;font-family:Times;color:#0000ff;}
	.ft169{font-size:14px;font-family:Times;color:#000000;}
	.ft1610{font-size:11px;font-family:Times;color:#000000;}
-->
</style>
</head>
<body bgcolor="#A0A0A0" vlink="blue" link="blue">
<div id="page16-div" style="position:relative;width:892px;height:1173px;">
<p style="position:absolute;top:1132px;left:816px;white-space:nowrap" class="ft160">16</p>
<p style="position:absolute;top:1135px;left:61px;white-space:nowrap" class="ft161"><b>Scientific</b>&#160;<b>RepoRtS</b>&#160;|&#160;</p>
<p style="position:absolute;top:1135px;left:192px;white-space:nowrap" class="ft163"><i>&#160; &#160; &#160; &#160; (2019) 9:18612&#160;</i>&#160;|&#160;<a href="https://doi.org/10.1038/s41598-019-54859-8">https://doi.org/10.1038/s41598-019-54859-8</a></p>
<p style="position:absolute;top:19px;left:585px;white-space:nowrap" class="ft164">www.nature.com/scientificreports</p>
<p style="position:absolute;top:19px;left:61px;white-space:nowrap" class="ft164">www.nature.com/scientificreports/</p>
<p style="position:absolute;top:71px;left:233px;white-space:nowrap" class="ft165">&#160;102.&#160;&#160;Bradley, R. W., Buck, M. &amp; Wang, B. Recognizing and engineering digital-like logic gates and switches in gene regulatory networks.&#160;</p>
<p style="position:absolute;top:84px;left:257px;white-space:nowrap" class="ft166"><i>Curr&#160;Opin&#160;Microbiol</i>&#160;<b>33</b>, 74–82,&#160;<a href="https://doi.org/10.1016/j.mib.2016.07.004">https://doi.org/10.1016/j.mib.2016.07.004&#160;(2016).</a></p>
<p style="position:absolute;top:97px;left:233px;white-space:nowrap" class="ft165">&#160;103.&#160;&#160;TerAvest,&#160;M.&#160;A.,&#160;Li,&#160;Z.&#160;&amp;&#160;Angenent,&#160;L.&#160;T.&#160;Bacteria-based&#160;biocomputing with Cel&#160;ular&#160;Computing&#160;Circuits to sense,&#160;decide, signal,&#160;</p>
<p style="position:absolute;top:110px;left:257px;white-space:nowrap" class="ft165">and act.&#160;<i>Energy &amp; Environmental Science</i>&#160;<b>4</b>,&#160;<a href="https://doi.org/10.1039/c1ee02455h">https://doi.org/10.1039/c1ee02455h</a>&#160;(2011).</p>
<p style="position:absolute;top:122px;left:233px;white-space:nowrap" class="ft165">&#160;104.&#160;&#160;Ayrinhac, S. Electric current solves mazes.&#160;<i>Phys. Educ</i>&#160;<b>49</b>&#160;(2014).</p>
<p style="position:absolute;top:135px;left:233px;white-space:nowrap" class="ft165">&#160;105.&#160;&#160;Safonov, A. A. Computing via natural erosion of sandstone.&#160;<i>International Journal of Paral&#160;el, Emergent and Distributed Systems</i>&#160;<b>33</b>,&#160;</p>
<p style="position:absolute;top:148px;left:257px;white-space:nowrap" class="ft165">742–751,&#160;<a href="https://doi.org/10.1080/17445760.2018.1455836">https://doi.org/10.1080/17445760.2018.1455836</a>&#160;(2018).</p>
<p style="position:absolute;top:161px;left:233px;white-space:nowrap" class="ft165">&#160;106.&#160;&#160;Ashikaga,&#160;&#160;H.&#160;&#160;<i>et&#160;al</i>.&#160;Model&#160;ing&#160;the&#160;heart&#160;as&#160;a&#160;communication&#160;system.&#160;<i>J&#160;R&#160;Soc&#160;Interface</i>&#160;<b>12</b>,&#160;<a href="https://doi.org/10.1098/rsif.2014.1201">https://doi.org/10.1098/rsif.2014.1201&#160;</a></p>
<p style="position:absolute;top:173px;left:257px;white-space:nowrap" class="ft165">(2015).</p>
<p style="position:absolute;top:186px;left:233px;white-space:nowrap" class="ft165">&#160;107.&#160;&#160;Srinivasa V. &amp; Chakravarthy, J. G. On Hebbian-like adaptation in heart muscle: a proposal for ‘cardiac memory’ (1997).</p>
<p style="position:absolute;top:199px;left:233px;white-space:nowrap" class="ft165">&#160;108.&#160;&#160;Gatenby, R. A. &amp; Frieden, B. R. Cel&#160;ular information dynamics through transmembrane flow of ions.&#160;<i>Sci&#160;Rep</i>&#160;<b>7</b><a href="https://doi.org/10.1038/s41598-017-15182-2">, 15075,&#160;https://doi.</a></p>
<p style="position:absolute;top:212px;left:257px;white-space:nowrap" class="ft168"><a href="https://doi.org/10.1038/s41598-017-15182-2">org/10.1038/s41598-017-15182-2&#160;(2017).</a></p>
<p style="position:absolute;top:224px;left:233px;white-space:nowrap" class="ft165">&#160;109.&#160;&#160;Couto,&#160;D.&#160;&amp;&#160;Zipfel,&#160;C.&#160;Regulation&#160;of&#160;pattern recognition&#160;receptor&#160;signal&#160;ing&#160;in plants.&#160;<i>Nat&#160;Rev&#160;Immunol</i>&#160;<b>16</b>, 537–552<a href="https://doi.org/10.1038/nri.2016.77">,&#160;https://doi.</a></p>
<p style="position:absolute;top:237px;left:257px;white-space:nowrap" class="ft168"><a href="https://doi.org/10.1038/nri.2016.77">org/10.1038/nri.2016.77&#160;(2016).</a></p>
<p style="position:absolute;top:250px;left:233px;white-space:nowrap" class="ft165">&#160;110.&#160;&#160;Stuckemann,&#160;&#160;T.&#160;&#160;<i>et&#160;al</i>. Antagonistic Self-Organizing Patterning Systems Control Maintenance and Regeneration of the&#160;</p>
<p style="position:absolute;top:263px;left:257px;white-space:nowrap" class="ft165">Anteroposterior Axis in Planarians.&#160;<i>Dev&#160;Cell</i>&#160;<b>40</b>, 248–263 e244,&#160;<a href="https://doi.org/10.1016/j.devcel.2016.12.024">https://doi.org/10.1016/j.devcel.2016.12.024&#160;(2017).</a></p>
<p style="position:absolute;top:275px;left:233px;white-space:nowrap" class="ft165">&#160;111.&#160;&#160;Glen, C. M., McDevitt, T. C. &amp; Kemp, M. L. Dynamic intercel&#160;ular transport modulates the spatial patterning of differentiation&#160;</p>
<p style="position:absolute;top:288px;left:257px;white-space:nowrap" class="ft165">during early neural commitment.&#160;<i>Nat&#160;Commun</i>&#160;<b>9</b><a href="https://doi.org/10.1038/s41467-018-06693-1">, 4111,&#160;https://doi.org/10.1038/s41467-018-06693-1&#160;(2018).</a></p>
<p style="position:absolute;top:301px;left:233px;white-space:nowrap" class="ft165">&#160;112.&#160;&#160;Gross,&#160;&#160;P.&#160;&#160;<i>et&#160;al</i>.&#160;Guiding&#160;self-organized&#160;pattern&#160;formation&#160;in&#160;cel&#160;&#160;polarity&#160;establishment.&#160;<i>Nature&#160;Physics</i>,&#160;<a href="https://doi.org/10.1038/s41567-018-0358-7">https://doi.org/10.1038/</a></p>
<p style="position:absolute;top:314px;left:257px;white-space:nowrap" class="ft168"><a href="https://doi.org/10.1038/s41567-018-0358-7">s41567-018-0358-7&#160;(2018).</a></p>
<p style="position:absolute;top:326px;left:233px;white-space:nowrap" class="ft165">&#160;113.&#160;&#160;Karig,&#160;&#160;D.&#160;&#160;<i>et&#160;al</i>. Stochastic Turing patterns in a synthetic bacterial population.&#160;<i>Proc Natl Acad Sci USA</i>,&#160;<a href="https://doi.org/10.1073/pnas.1720770115">https://doi.org/10.1073/</a></p>
<p style="position:absolute;top:339px;left:257px;white-space:nowrap" class="ft168"><a href="https://doi.org/10.1073/pnas.1720770115">pnas.1720770115&#160;(2018).</a></p>
<p style="position:absolute;top:352px;left:233px;white-space:nowrap" class="ft165">&#160;114.&#160;&#160;Ferreira, G. B., Matthias, S. &amp; Levin, M. In&#160;<i>European Conference on Artificial Life</i>. 170–177 (MIT Press).</p>
<p style="position:absolute;top:365px;left:233px;white-space:nowrap" class="ft165">&#160;115.&#160;&#160;Minh-Thai, T. N., Aryal, J., Samarasinghe, S. &amp; Levin, M. In&#160;<i>AI&#160;2018:&#160;Advances&#160;in&#160;Artificial&#160;Intel&#160;igence&#160;Lecture&#160;Notes&#160;in&#160;Computer&#160;</i></p>
<p style="position:absolute;top:377px;left:257px;white-space:nowrap" class="ft166"><i>Science</i>&#160;Ch. Chapter 16, 153–159 (2018).</p>
<p style="position:absolute;top:390px;left:233px;white-space:nowrap" class="ft165">&#160;116.&#160;&#160;De, A., Chakravarthy,&#160;V.&#160;S.&#160;&amp; Levin,&#160;M.&#160;A computational&#160;model&#160;of&#160;planarian&#160;regeneration.&#160;<i>International Journal of Paral&#160;el,&#160;</i></p>
<p style="position:absolute;top:403px;left:257px;white-space:nowrap" class="ft166"><i>Emergent and Distributed Systems</i>&#160;<b>32</b>, 331–347,&#160;<a href="https://doi.org/10.1080/17445760.2016.1185521">https://doi.org/10.1080/17445760.2016.1185521</a>&#160;(2016).</p>
<p style="position:absolute;top:416px;left:233px;white-space:nowrap" class="ft165">&#160;117.&#160;&#160;Levin, M. Z. B. A. M. In&#160;<i>Artificial&#160;Life</i>&#160;351–358 (MIT Press, 2018).</p>
<p style="position:absolute;top:428px;left:233px;white-space:nowrap" class="ft165">&#160;118.&#160;&#160;Cervera, J., Manzanares, J. A., Mafe, S. &amp; Levin, M. Synchronization of Bioelectric Oscil&#160;ations in Networks of Nonexcitable Cel&#160;s:&#160;</p>
<p style="position:absolute;top:441px;left:257px;white-space:nowrap" class="ft165">From Single-Cell to Multicel&#160;ular States.&#160;<i>J Phys Chem B</i>&#160;<b>123</b>, 3924–3934,&#160;<a href="https://doi.org/10.1021/acs.jpcb.9b01717">https://doi.org/10.1021/acs.jpcb.9b01717&#160;(2019).</a></p>
<p style="position:absolute;top:454px;left:233px;white-space:nowrap" class="ft165">&#160;119.&#160;&#160;Brodsky, M. Turing-like patterns can arise from purely bioelectric mechanisms.&#160;<i>bioRxiv</i><a href="https://doi.org/10.1101/336461">,&#160;https://doi.org/10.1101/336461</a>&#160;(2018).</p>
<p style="position:absolute;top:467px;left:233px;white-space:nowrap" class="ft165">&#160;120.&#160;&#160;Cervera, J.,&#160;Manzanares,&#160;J. A.&#160;&amp;&#160;Mafe, S.&#160;Cell-cell&#160;bioelectrical interactions&#160;and&#160;local&#160;heterogeneities&#160;in&#160;genetic networks: a&#160;model&#160;</p>
<p style="position:absolute;top:479px;left:257px;white-space:nowrap" class="ft165">for the&#160;stabilization&#160;of&#160;single-cell&#160;states&#160;and&#160;multicellular&#160;oscillations.&#160;<i>Phys Chem Chem&#160;Phys</i>&#160;<b>20</b>,&#160;<a href="https://doi.org/10.1039/C8CP00648B">9343–9354,&#160;https://doi.</a></p>
<p style="position:absolute;top:492px;left:257px;white-space:nowrap" class="ft168"><a href="https://doi.org/10.1039/C8CP00648B">org/10.1039/C8CP00648B</a>&#160;(2018).</p>
<p style="position:absolute;top:505px;left:233px;white-space:nowrap" class="ft165">&#160;121.&#160;&#160;Blackiston, D., Shomrat, T. &amp; Levin, M. The Stability of Memories During Brain Remodeling: a Perspective.&#160;<i>Communicative&#160;&amp;&#160;</i></p>
<p style="position:absolute;top:517px;left:257px;white-space:nowrap" class="ft166"><i>Integrative&#160;Biology</i>&#160;<b>8</b>, e1073424,&#160;<a href="https://doi.org/10.1080/19420889.2015.1073424">https://doi.org/10.1080/19420889.2015.1073424&#160;(2015).</a></p>
<p style="position:absolute;top:530px;left:233px;white-space:nowrap" class="ft165">&#160;122.&#160;&#160;Cooke, J. Scale of body pattern adjusts to available cell number in amphibian embryos.&#160;<i>Nature</i>&#160;<b>290</b>, 775–778 (1981).</p>
<p style="position:absolute;top:543px;left:233px;white-space:nowrap" class="ft165">&#160;123.&#160;&#160;Bellec,&#160;&#160;G.&#160;&#160;<i>et&#160;al</i>. Biological&#160;y inspired alternatives to backpropagation through time for learning in recurrent neural nets.&#160;<i>ArXiv</i>&#160;</p>
<p style="position:absolute;top:556px;left:257px;white-space:nowrap" class="ft165">(2019).</p>
<p style="position:absolute;top:568px;left:233px;white-space:nowrap" class="ft165">&#160;124.&#160;&#160;Stork, D. G. Is backpropagation biological&#160;y plausible?&#160;<i>International Joint Conference on Neural Networks</i>&#160;(1989).</p>
<p style="position:absolute;top:581px;left:233px;white-space:nowrap" class="ft165">&#160;125.&#160;&#160;Öfjäl&#160;, K. &amp; Felsberg, M. In&#160;<i>British&#160;Machine&#160;Vision&#160;Conference</i>&#160;137–156 (BMVA Press, Nottingham, UK, 2014).</p>
<p style="position:absolute;top:594px;left:233px;white-space:nowrap" class="ft165">&#160;126.&#160;&#160;Schil&#160;aci,&#160;G.,&#160;Hafner,&#160;V.&#160;V.&#160;&amp;&#160;Lara,&#160;B. In&#160;<i>4th International Conference on Development and Learning and on Epigenetic Robotics</i>&#160;</p>
<p style="position:absolute;top:607px;left:257px;white-space:nowrap" class="ft165">130–136 (IEEE, 2014).</p>
<p style="position:absolute;top:619px;left:233px;white-space:nowrap" class="ft165">&#160;127.&#160;&#160;Kak, S. C. On generalization by neural networks.&#160;<i>Information&#160;Sciences</i>&#160;<b>111</b>, 293–302 (1998).</p>
<p style="position:absolute;top:632px;left:233px;white-space:nowrap" class="ft165">&#160;128.&#160;&#160;Araque,&#160;A.&#160;&amp;&#160;Navarrete.,&#160;M.&#160;Glial&#160;cel&#160;s&#160;in&#160;neuronal&#160;network&#160;function.&#160;<i>Philosophical&#160;Transactions&#160;of&#160;the&#160;Royal&#160;Society&#160;B:&#160;Biological&#160;</i></p>
<p style="position:absolute;top:645px;left:257px;white-space:nowrap" class="ft166"><i>Sciences</i>&#160;<b>365</b>, 2375–2381 (2010).</p>
<p style="position:absolute;top:658px;left:233px;white-space:nowrap" class="ft165">&#160;129.&#160;&#160;De Pittà, M. B. H. In&#160;<i>C</i>omputati<i>o</i>na<i>l Glioscience</i>&#160;(ed. Berry H. &amp; De Pittà M.) (Springer, Cham, 2019).</p>
<p style="position:absolute;top:670px;left:233px;white-space:nowrap" class="ft165">&#160;130.&#160;&#160;Levin, M. &amp;&#160;Martyniuk, C. J. The bioelectric code: An ancient computational&#160;medium for dynamic control of growth and form.&#160;</p>
<p style="position:absolute;top:683px;left:257px;white-space:nowrap" class="ft166"><i>Biosystems</i>&#160;<b>164</b><a href="https://doi.org/10.1016/j.biosystems.2017.08.009">, 76–93,&#160;https://doi.org/10.1016/j.biosystems.2017.08.009&#160;(2018).</a></p>
<p style="position:absolute;top:696px;left:233px;white-space:nowrap" class="ft165">&#160;131.&#160;&#160;Pinet, K., Deolankar, M., Leung, B. &amp; McLaughlin, K. A. Adaptive correction of craniofacial defects in pre-metamorphic Xenopus&#160;</p>
<p style="position:absolute;top:709px;left:257px;white-space:nowrap" class="ft165">laevis tadpoles involves thyroid hormone-independent tissue remodeling.&#160;<i>Development</i>&#160;<b>146</b>,&#160;<a href="https://doi.org/10.1242/dev.175893">https://doi.org/10.1242/dev.175893&#160;</a></p>
<p style="position:absolute;top:721px;left:257px;white-space:nowrap" class="ft165">(2019).</p>
<p style="position:absolute;top:734px;left:233px;white-space:nowrap" class="ft165">&#160;132.&#160;&#160;Emmons-Bel&#160;,&#160;&#160;M.&#160;&#160;<i>et&#160;al</i>. Gap Junctional Blockade Stochastical&#160;y Induces Different Species-Specific Head Anatomies in Genetical&#160;y&#160;</p>
<p style="position:absolute;top:747px;left:257px;white-space:nowrap" class="ft165">Wild-Type Girardia dorotocephala Flatworms.&#160;<i>Int J Mol Sci</i>&#160;<b>16</b><a href="https://doi.org/10.3390/ijms161126065">, 27865–27896,&#160;https://doi.org/10.3390/ijms161126065&#160;(2015).</a></p>
<p style="position:absolute;top:760px;left:233px;white-space:nowrap" class="ft165">&#160;133.&#160;&#160;Harvey,&#160;I.&#160;In&#160;<i>Advances in Artificial Life. Darwin Meets von Neumann. ECAL 2009</i>. (ed Karsai, I. Kampis, G. &amp; Szathmáry, E.)&#160;</p>
<p style="position:absolute;top:772px;left:257px;white-space:nowrap" class="ft165">126–133 (Springer, Berlin, Heidelberg).</p>
<p style="position:absolute;top:801px;left:233px;white-space:nowrap" class="ft169"><b>Acknowledgements</b></p>
<p style="position:absolute;top:818px;left:233px;white-space:nowrap" class="ft1610">We&#160;thank&#160;Alexis&#160;Pietak&#160;and&#160;many&#160;members&#160;of&#160;the&#160;Levin&#160;lab&#160;for&#160;many&#160;helpful&#160;discussions,&#160;as&#160;well&#160;as&#160;Nima&#160;</p>
<p style="position:absolute;top:833px;left:233px;white-space:nowrap" class="ft1610">Dehghani and Joshua Finkelstein for helpful comments on a draft of the manuscript. This research was supported&#160;</p>
<p style="position:absolute;top:848px;left:233px;white-space:nowrap" class="ft1610">by the Allen Discovery Center program through The Paul G. Allen Frontiers Group (12171), and M. L. grateful&#160;y&#160;</p>
<p style="position:absolute;top:863px;left:233px;white-space:nowrap" class="ft1610">acknowledges support from the Templeton World Charity Foundation (TWCF0089/AB55), the Barton Family&#160;</p>
<p style="position:absolute;top:878px;left:233px;white-space:nowrap" class="ft1610">Foundation,&#160;and the&#160;National Science Foundation (DBI-1152279&#160;and Emergent&#160;Behaviors of&#160;Integrated Cel&#160;ular&#160;</p>
<p style="position:absolute;top:893px;left:233px;white-space:nowrap" class="ft1610">Systems subaward CBET-0939511). This research was also sponsored by the Defense Advanced Research&#160;</p>
<p style="position:absolute;top:908px;left:233px;white-space:nowrap" class="ft1610">Projects Agency (DARPA) under Cooperative Agreement Number HR0011-18-2-0022, the Lifelong Learning&#160;</p>
<p style="position:absolute;top:923px;left:233px;white-space:nowrap" class="ft1610">Machines program&#160;from&#160;DARPA/MTO. The&#160;content&#160;of&#160;the information does&#160;not necessarily&#160;reflect&#160;the&#160;position&#160;</p>
<p style="position:absolute;top:938px;left:233px;white-space:nowrap" class="ft1610">or the policy of the Government, and no official endorsement should be inferred. Approved for public release;&#160;</p>
<p style="position:absolute;top:953px;left:233px;white-space:nowrap" class="ft1610">distribution is unlimited.</p>
<p style="position:absolute;top:984px;left:233px;white-space:nowrap" class="ft169"><b>Author contributions</b></p>
<p style="position:absolute;top:1001px;left:233px;white-space:nowrap" class="ft1610">M.L. and S.M. conceptualized the project together. S.M. wrote all the code and performed the computational&#160;</p>
<p style="position:absolute;top:1016px;left:233px;white-space:nowrap" class="ft1610">experiments and data collection. M.L. and S.M. analyzed the data and wrote the paper.</p>
<p style="position:absolute;top:1047px;left:233px;white-space:nowrap" class="ft169"><b>competing interests</b></p>
<p style="position:absolute;top:1064px;left:233px;white-space:nowrap" class="ft1610">The authors declare no competing interests.</p>
</div>
</body>
</html>
<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="" xml:lang="">
<head>
<title>4b78c8d89b22cb315d90fd1274945de5-html.html</title>

<meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
 <br/>
<style type="text/css">
<!--
	p {margin: 0; padding: 0;}	.ft170{font-size:13px;font-family:Times;color:#cedde4;}
	.ft171{font-size:10px;font-family:Times;color:#000000;}
	.ft172{font-size:10px;font-family:Times;color:#000000;}
	.ft173{font-size:10px;font-family:Times;color:#000000;}
	.ft174{font-size:13px;font-family:Times;color:#000000;}
	.ft175{font-size:14px;font-family:Times;color:#000000;}
	.ft176{font-size:11px;font-family:Times;color:#000000;}
	.ft177{font-size:11px;font-family:Times;color:#000000;}
	.ft178{font-size:11px;font-family:Times;color:#0000ff;}
	.ft179{font-size:11px;line-height:21px;font-family:Times;color:#000000;}
-->
</style>
</head>
<body bgcolor="#A0A0A0" vlink="blue" link="blue">
<div id="page17-div" style="position:relative;width:892px;height:1173px;">
<p style="position:absolute;top:1132px;left:818px;white-space:nowrap" class="ft170">17</p>
<p style="position:absolute;top:1135px;left:61px;white-space:nowrap" class="ft171"><b>Scientific</b>&#160;<b>RepoRtS</b>&#160;|&#160;</p>
<p style="position:absolute;top:1135px;left:192px;white-space:nowrap" class="ft173"><i>&#160; &#160; &#160; &#160; (2019) 9:18612&#160;</i>&#160;|&#160;<a href="https://doi.org/10.1038/s41598-019-54859-8">https://doi.org/10.1038/s41598-019-54859-8</a></p>
<p style="position:absolute;top:19px;left:585px;white-space:nowrap" class="ft174">www.nature.com/scientificreports</p>
<p style="position:absolute;top:19px;left:61px;white-space:nowrap" class="ft174">www.nature.com/scientificreports/</p>
<p style="position:absolute;top:69px;left:233px;white-space:nowrap" class="ft175"><b>Additional&#160;information</b></p>
<p style="position:absolute;top:86px;left:233px;white-space:nowrap" class="ft179"><b>Supplementary information</b>&#160;is available for this paper at&#160;<a href="https://doi.org/10.1038/s41598-019-54859-8">https://doi.org/10.1038/s41598-019-54859-8</a>.<br/><b>Correspondence</b>&#160;and requests for materials should be addressed to M.L.<br/><b>Reprints and permissions information&#160;</b>is available a<a href="http://www.nature.com/reprints">t&#160;www.nature.com/reprints.<br/></a><b>Publisher’s note</b>&#160;Springer Nature remains neutral with regard to jurisdictional claims in published maps and&#160;</p>
<p style="position:absolute;top:167px;left:233px;white-space:nowrap" class="ft177">institutional affiliations.</p>
<p style="position:absolute;top:188px;left:301px;white-space:nowrap" class="ft176"><b>Open Access</b>&#160;This article is licensed under a Creative Commons Attribution 4.0 International&#160;</p>
<p style="position:absolute;top:203px;left:300px;white-space:nowrap" class="ft177">License, which permits use, sharing, adaptation, distribution and reproduction in any medium or&#160;</p>
<p style="position:absolute;top:218px;left:233px;white-space:nowrap" class="ft177">format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Cre-</p>
<p style="position:absolute;top:233px;left:233px;white-space:nowrap" class="ft177">ative Commons license, and indicate if changes were made. The images or other third party material in this&#160;</p>
<p style="position:absolute;top:248px;left:233px;white-space:nowrap" class="ft177">article are included in the article’s Creative Commons license, unless indicated otherwise in a credit line to the&#160;</p>
<p style="position:absolute;top:263px;left:233px;white-space:nowrap" class="ft177">material. If material is not included in the article’s Creative Commons license and your intended use is not per-</p>
<p style="position:absolute;top:278px;left:233px;white-space:nowrap" class="ft177">mitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the&#160;</p>
<p style="position:absolute;top:293px;left:233px;white-space:nowrap" class="ft177"><a href="http://creativecommons.org/licenses/by/4.0/">copyright holder. To view a copy of this license, visit&#160;http://creativecommons.org/licenses/by/4.0/.</a></p>
<p style="position:absolute;top:308px;left:233px;white-space:nowrap" class="ft177">&#160;</p>
<p style="position:absolute;top:323px;left:233px;white-space:nowrap" class="ft177">© The Author(s) 2019</p>
</div>
</body>
</html>
