<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="" xml:lang="">
<head>
<title>59fe6d0319faf365850c46f352541a9c-html.html</title>

<meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
 <br/>
<style type="text/css">
<!--
	p {margin: 0; padding: 0;}	.ft10{font-size:7px;font-family:Times;color:#000000;}
	.ft11{font-size:7px;font-family:Times;color:#000000;}
	.ft12{font-size:7px;font-family:Times;color:#2e3191;}
	.ft13{font-size:7px;font-family:Times;color:#231f20;}
	.ft14{font-size:10px;font-family:Times;color:#000000;}
	.ft15{font-size:10px;font-family:Times;color:#000000;}
	.ft16{font-size:8px;font-family:Times;color:#2e3191;}
	.ft17{font-size:10px;font-family:Times;color:#000000;}
	.ft18{font-size:25px;font-family:Times;color:#000000;}
	.ft19{font-size:9px;font-family:Times;color:#000000;}
	.ft110{font-size:6px;font-family:Times;color:#000000;}
	.ft111{font-size:4px;font-family:Times;color:#000000;}
	.ft112{font-size:11px;font-family:Times;color:#000000;}
	.ft113{font-size:9px;font-family:Times;color:#000000;}
	.ft114{font-size:14px;font-family:Times;color:#000000;}
	.ft115{font-size:11px;font-family:Times;color:#000000;}
	.ft116{font-size:11px;font-family:Times;color:#2e3191;}
	.ft117{font-size:10px;font-family:Times;color:#666666;-moz-transform: matrix(         0,          1,         -1,          0, 0, 0);-webkit-transform: matrix(         0,          1,         -1,          0, 0, 0);-o-transform: matrix(         0,          1,         -1,          0, 0, 0);-ms-transform: matrix(         0,          1,         -1,          0, 0, 0);-moz-transform-origin: left 75%;-webkit-transform-origin: left 75%;-o-transform-origin: left 75%;-ms-transform-origin: left 75%;}
	.ft118{font-size:7px;line-height:14px;font-family:Times;color:#000000;}
	.ft119{font-size:7px;line-height:14px;font-family:Times;color:#231f20;}
	.ft120{font-size:25px;line-height:34px;font-family:Times;color:#000000;}
	.ft121{font-size:7px;line-height:13px;font-family:Times;color:#000000;}
	.ft122{font-size:9px;line-height:14px;font-family:Times;color:#000000;}
	.ft123{font-size:9px;line-height:16px;font-family:Times;color:#000000;}
	.ft124{font-size:9px;line-height:16px;font-family:Times;color:#000000;}
	.ft125{font-size:11px;line-height:17px;font-family:Times;color:#000000;}
-->
</style>
</head>
<body bgcolor="#A0A0A0" vlink="blue" link="blue">
<div id="page1-div" style="position:relative;width:892px;height:1173px;">
<p style="position:absolute;top:1063px;left:72px;white-space:nowrap" class="ft119"><b>Received:&#160;</b>June&#160;25,&#160;2021.&#160;<b>Revised:&#160;</b>November&#160;16,&#160;2021.&#160;<b>Editorial&#160;decision:&#160;</b>November&#160;20,&#160;2021.&#160;<b>Accepted:&#160;</b>November&#160;25,&#160;2021<br/>©&#160;The&#160;Author(s)&#160;2021.&#160;Published&#160;by&#160;Oxford&#160;University&#160;Press.<br/>This&#160;is&#160;an&#160;Open&#160;Access&#160;article&#160;distributed&#160;under&#160;the&#160;terms&#160;of&#160;the&#160;Creative&#160;Commons&#160;Attribution&#160;License&#160;<a href="http://creativecommons.org/licenses/by/4.0/">(http://creativecommons.org/licenses/by/4.0/),&#160;</a>which<br/>permits&#160;unrestricted&#160;reuse,&#160;distribution,&#160;and&#160;reproduction&#160;in&#160;any&#160;medium,&#160;provided&#160;the&#160;original&#160;work&#160;is&#160;properly&#160;cited.</p>
<p style="position:absolute;top:60px;left:664px;white-space:nowrap" class="ft14"><i>Integrative&#160;Biology</i>,&#160;2021,&#160;1–12</p>
<p style="position:absolute;top:84px;left:631px;white-space:nowrap" class="ft16"><b>https://doi.org/10.1093/intbio/zyab020</b></p>
<p style="position:absolute;top:100px;left:739px;white-space:nowrap" class="ft17"><b>Original&#160;Article</b></p>
<p style="position:absolute;top:169px;left:72px;white-space:nowrap" class="ft120"><b>Evolution&#160;and&#160;emergence:&#160;higher&#160;order&#160;information<br/>structure&#160;in&#160;protein&#160;interactomes&#160;across&#160;the&#160;tree&#160;of&#160;life</b></p>
<p style="position:absolute;top:252px;left:72px;white-space:nowrap" class="ft19">Brennan&#160;<a href="https://orcid.org/0000-0001-8326-5044">Klein</a></p>
<p style="position:absolute;top:252px;left:171px;white-space:nowrap" class="ft110"><a href="https://orcid.org/0000-0001-8326-5044">1</a>,2,*,&#160;Erik&#160;<a href="https://orcid.org/0000-0002-2970-0057">Hoel</a></p>
<p style="position:absolute;top:252px;left:266px;white-space:nowrap" class="ft110"><a href="https://orcid.org/0000-0002-2970-0057">3</a>,&#160;Anshuman&#160;Swain</p>
<p style="position:absolute;top:252px;left:396px;white-space:nowrap" class="ft110"><a href="https://orcid.org/0000-0002-9180-2222">4</a>,&#160;Ross&#160;Griebenow5&#160;and&#160;Michael&#160;Levin3</p>
<p style="position:absolute;top:279px;left:72px;white-space:nowrap" class="ft118">1&#160;Network&#160;Science&#160;Institute,&#160;Northeastern&#160;University,&#160;Boston,&#160;MA,&#160;USA<br/>2&#160;Laboratory&#160;for&#160;the&#160;Modeling&#160;of&#160;Biological&#160;and&#160;Socio-Technical&#160;Systems,&#160;Northeastern&#160;University,&#160;Boston,&#160;MA,&#160;USA<br/>3&#160;Allen&#160;Discovery&#160;Center,&#160;Tufts&#160;University,&#160;Medford,&#160;MA,&#160;USA<br/>4&#160;Department&#160;of&#160;Biology,&#160;University&#160;of&#160;Maryland,&#160;College&#160;Park,&#160;MD,&#160;USA<br/>5&#160;Department&#160;of&#160;Computer&#160;Science,&#160;Drexel&#160;University,&#160;Philadelphia,&#160;PA,&#160;USA<br/>*<b>Corresponding&#160;author</b>.&#160;E-mail:&#160;b.klein@northeastern.edu</p>
<p style="position:absolute;top:388px;left:72px;white-space:nowrap" class="ft112"><b>Abstract</b></p>
<p style="position:absolute;top:411px;left:72px;white-space:nowrap" class="ft122">The&#160;internal&#160;workings&#160;of&#160;biological&#160;systems&#160;are&#160;notoriously&#160;difficult&#160;to&#160;understand.&#160;Due&#160;to&#160;the&#160;prevalence&#160;of&#160;noise&#160;and&#160;degeneracy&#160;in<br/>evolved&#160;systems,&#160;in&#160;many&#160;cases&#160;the&#160;workings&#160;of&#160;everything&#160;from&#160;gene&#160;regulatory&#160;networks&#160;to&#160;protein–protein&#160;interactome&#160;networks<br/>remain&#160;black&#160;boxes.&#160;One&#160;consequence&#160;of&#160;this&#160;black-box&#160;nature&#160;is&#160;that&#160;it&#160;is&#160;unclear&#160;at&#160;which&#160;scale&#160;to&#160;analyze&#160;biological&#160;systems&#160;to&#160;best<br/>understand&#160;their&#160;function.&#160;We&#160;analyzed&#160;the&#160;protein&#160;interactomes&#160;of&#160;over&#160;1800&#160;species,&#160;containing&#160;in&#160;total&#160;8&#160;782&#160;166&#160;protein–protein<br/>interactions,&#160;at&#160;different&#160;scales.&#160;We&#160;show&#160;the&#160;emergence&#160;of&#160;higher&#160;order&#160;‘macroscales’&#160;in&#160;these&#160;interactomes&#160;and&#160;that&#160;these&#160;biological<br/>macroscales&#160;are&#160;associated&#160;with&#160;lower&#160;noise&#160;and&#160;degeneracy&#160;and&#160;therefore&#160;lower&#160;uncertainty.&#160;Moreover,&#160;the&#160;nodes&#160;in&#160;the&#160;interactomes<br/>that&#160;make&#160;up&#160;the&#160;macroscale&#160;are&#160;more&#160;resilient&#160;compared&#160;with&#160;nodes&#160;that&#160;do&#160;not&#160;participate&#160;in&#160;the&#160;macroscale.&#160;These&#160;effects&#160;are<br/>more&#160;pronounced&#160;in&#160;interactomes&#160;of&#160;eukaryota,&#160;as&#160;compared&#160;with&#160;prokaryota;&#160;these&#160;results&#160;hold&#160;even&#160;after&#160;sensitivity&#160;tests&#160;where<br/>we&#160;recalculate&#160;the&#160;emergent&#160;macroscales&#160;under&#160;network&#160;simulations&#160;where&#160;we&#160;add&#160;different&#160;edge&#160;weights&#160;to&#160;the&#160;interactomes.&#160;This<br/>points&#160;to&#160;plausible&#160;evolutionary&#160;adaptation&#160;for&#160;macroscales:&#160;biological&#160;networks&#160;evolve&#160;informative&#160;macroscales&#160;to&#160;gain&#160;benefits&#160;of<br/>both&#160;being&#160;uncertain&#160;at&#160;lower&#160;scales&#160;to&#160;boost&#160;their&#160;resilience,&#160;and&#160;also&#160;being&#160;‘certain’&#160;at&#160;higher&#160;scales&#160;to&#160;increase&#160;their&#160;effectiveness&#160;at<br/>information&#160;transmission.&#160;Our&#160;work&#160;explains&#160;some&#160;of&#160;the&#160;difficulty&#160;in&#160;understanding&#160;the&#160;workings&#160;of&#160;biological&#160;networks,&#160;since&#160;they<br/>are&#160;often&#160;most&#160;informative&#160;at&#160;a&#160;hidden&#160;higher&#160;scale,&#160;and&#160;demonstrates&#160;the&#160;tools&#160;to&#160;make&#160;these&#160;informative&#160;higher&#160;scales&#160;explicit.</p>
<p style="position:absolute;top:610px;left:72px;white-space:nowrap" class="ft113"><b>Keywords:&#160;</b>emergence,&#160;evolution,&#160;interactome,&#160;network</p>
<p style="position:absolute;top:669px;left:86px;white-space:nowrap" class="ft124"><b>Insight&#160;Box<br/></b>Protein&#160;interactomes&#160;are&#160;an&#160;insight&#160;into&#160;the&#160;inner&#160;workings&#160;of&#160;cells.&#160;Here&#160;we&#160;analyze&#160;the&#160;protein&#160;interactomes&#160;of&#160;over&#160;1800&#160;different<br/>species&#160;using&#160;information&#160;theory&#160;to&#160;measure&#160;the&#160;amount&#160;of&#160;noise&#160;and&#160;uncertainty&#160;in&#160;protein&#160;interactions.&#160;We&#160;found&#160;that&#160;uncertainty<br/>associated&#160;with&#160;protein&#160;interactions&#160;is&#160;higher&#160;in&#160;eukaryotes&#160;compared&#160;with&#160;prokaryotes.&#160;To&#160;explore&#160;why&#160;this&#160;is&#160;the&#160;case,&#160;we&#160;also<br/>modeled&#160;the&#160;same&#160;protein&#160;interactomes&#160;at&#160;higher&#160;scales&#160;by&#160;coarse&#160;graining.&#160;We&#160;found&#160;that&#160;eukaryotes&#160;had&#160;much&#160;more&#160;informative<br/>higher&#160;scales&#160;in&#160;their&#160;interactomes&#160;(in&#160;the&#160;form&#160;of&#160;less&#160;noise&#160;and&#160;uncertainty).&#160;We&#160;explore&#160;the&#160;benefits&#160;of&#160;such&#160;higher&#160;scale&#160;structure<br/>and&#160;examine&#160;the&#160;relationship&#160;between&#160;evolution,&#160;information&#160;and&#160;scale&#160;in&#160;biological&#160;systems.</p>
<p style="position:absolute;top:823px;left:72px;white-space:nowrap" class="ft114"><b>INTRODUCTION</b></p>
<p style="position:absolute;top:849px;left:72px;white-space:nowrap" class="ft125">Interactions&#160;in&#160;biological&#160;systems&#160;are&#160;noisy&#160;and&#160;degen-<br/>erate&#160;in&#160;their&#160;functions,&#160;making&#160;them&#160;fundamentally<br/>noisier&#160;and&#160;fundamentally&#160;different&#160;from&#160;those&#160;in&#160;engi-<br/>neered&#160;systems&#160;<a href="59fe6d0319faf365850c46f352541a9cs.html#11">[1,&#160;2].&#160;</a>The&#160;sources&#160;of&#160;noise&#160;in&#160;biology&#160;are<br/>nearly&#160;ubiquitous&#160;and&#160;vary&#160;widely.&#160;Noise&#160;may&#160;exist&#160;a&#160;gene<br/>regulatory&#160;network,&#160;wherein&#160;a&#160;gene&#160;might&#160;upregulate<br/>another&#160;gene&#160;but&#160;only&#160;probabilistically,&#160;or&#160;they&#160;may&#160;be<br/>noisy&#160;in&#160;that&#160;a&#160;protein&#160;may&#160;bind&#160;randomly&#160;across&#160;a&#160;set<br/>of&#160;possible&#160;pairings.&#160;There&#160;are&#160;numerous&#160;sources&#160;of&#160;such<br/>indeterminism&#160;in&#160;cells&#160;and&#160;tissues,&#160;such&#160;as&#160;how&#160;cell</p>
<p style="position:absolute;top:831px;left:464px;white-space:nowrap" class="ft125">molecules&#160;are&#160;buffered&#160;by&#160;Brownian&#160;motion&#160;<a href="59fe6d0319faf365850c46f352541a9cs.html#12">[3],&#160;</a>to&#160;the<br/>stochastic&#160;opening&#160;and&#160;closing&#160;of&#160;ion&#160;channels&#160;<a href="59fe6d0319faf365850c46f352541a9cs.html#12">[4],&#160;</a>and<br/>even&#160;to&#160;the&#160;chaotic&#160;dynamics&#160;of&#160;neural&#160;activity&#160;[<a href="59fe6d0319faf365850c46f352541a9cs.html#12">5</a>].</p>
<p style="position:absolute;top:885px;left:480px;white-space:nowrap" class="ft115">There&#160;are&#160;also&#160;numerous&#160;sources&#160;of&#160;degeneracy&#160;within</p>
<p style="position:absolute;top:903px;left:464px;white-space:nowrap" class="ft125">the&#160;cellular,&#160;developmental&#160;and&#160;genetic&#160;operation&#160;of<br/>organisms&#160;<a href="59fe6d0319faf365850c46f352541a9cs.html#12">[6].&#160;</a>Degeneracy&#160;is&#160;when&#160;an&#160;end&#160;state&#160;or&#160;output,<br/>like&#160;a&#160;phenotype,&#160;can&#160;come&#160;from&#160;a&#160;large&#160;number&#160;of<br/>possible&#160;states&#160;or&#160;inputs&#160;<a href="59fe6d0319faf365850c46f352541a9cs.html#12">[7].</a></p>
<p style="position:absolute;top:975px;left:480px;white-space:nowrap" class="ft115">Due&#160;to&#160;this&#160;indeterminism&#160;and&#160;degeneracy,&#160;the</p>
<p style="position:absolute;top:993px;left:464px;white-space:nowrap" class="ft125">dynamics&#160;and&#160;function&#160;of&#160;biological&#160;systems&#160;are<br/>often&#160;uncertain.&#160;This&#160;hampers&#160;control&#160;of&#160;system-level</p>
<p style="position:absolute;top:229px;left:860px;white-space:nowrap" class="ft117">Downloaded from https://academic.oup.com/ib/advance-article/doi/10.1093/intbio/zyab020/6474503 by guest on 22 December 2021</p>
</div>
</body>
</html>
<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="" xml:lang="">
<head>
<title>59fe6d0319faf365850c46f352541a9c-html.html</title>

<meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
 <br/>
<style type="text/css">
<!--
	p {margin: 0; padding: 0;}	.ft20{font-size:9px;font-family:Times;color:#000000;}
	.ft21{font-size:9px;font-family:Times;color:#000000;}
	.ft22{font-size:9px;font-family:Times;color:#000000;}
	.ft23{font-size:11px;font-family:Times;color:#000000;}
	.ft24{font-size:11px;font-family:Times;color:#2e3191;}
	.ft25{font-size:11px;font-family:Times;color:#000000;}
	.ft26{font-size:10px;font-family:Times;color:#666666;-moz-transform: matrix(         0,          1,         -1,          0, 0, 0);-webkit-transform: matrix(         0,          1,         -1,          0, 0, 0);-o-transform: matrix(         0,          1,         -1,          0, 0, 0);-ms-transform: matrix(         0,          1,         -1,          0, 0, 0);-moz-transform-origin: left 75%;-webkit-transform-origin: left 75%;-o-transform-origin: left 75%;-ms-transform-origin: left 75%;}
	.ft27{font-size:11px;line-height:17px;font-family:Times;color:#000000;}
-->
</style>
</head>
<body bgcolor="#A0A0A0" vlink="blue" link="blue">
<div id="page2-div" style="position:relative;width:892px;height:1173px;">
<p style="position:absolute;top:42px;left:64px;white-space:nowrap" class="ft20"><b>2&#160;|&#160;</b><i>Integrative&#160;Biology</i>,&#160;2021</p>
<p style="position:absolute;top:79px;left:63px;white-space:nowrap" class="ft27">properties&#160;for&#160;biomedicine&#160;and&#160;synthetic&#160;bioengineering,<br/>as&#160;well&#160;as&#160;hampering&#160;the&#160;understanding&#160;of&#160;modelers&#160;and<br/>experimentalists&#160;who&#160;wish&#160;to&#160;build&#160;‘big&#160;data’&#160;approaches<br/>to&#160;biology&#160;like&#160;interactomes,&#160;connectomes&#160;and&#160;mapping<br/>molecular&#160;pathways&#160;[<a href="59fe6d0319faf365850c46f352541a9cs.html#12">8</a>,&#160;<a href="59fe6d0319faf365850c46f352541a9cs.html#12">9</a>].&#160;Although&#160;there&#160;have&#160;been<br/>many&#160;attempts&#160;to&#160;characterize&#160;and&#160;understand&#160;this<br/>uncertainty&#160;in&#160;biological&#160;systems&#160;<a href="59fe6d0319faf365850c46f352541a9cs.html#12">[7],&#160;</a>the&#160;explanations<br/>typically&#160;do&#160;not&#160;extend&#160;beyond&#160;the&#160;advantages&#160;of<br/>redundancy&#160;in&#160;these&#160;systems&#160;<a href="59fe6d0319faf365850c46f352541a9cs.html#12">[10</a>].</p>
<p style="position:absolute;top:241px;left:78px;white-space:nowrap" class="ft23">How&#160;do&#160;noise&#160;and&#160;uncertainty&#160;span&#160;the&#160;tree&#160;of&#160;life?</p>
<p style="position:absolute;top:259px;left:63px;white-space:nowrap" class="ft27">Here&#160;we&#160;examine&#160;this&#160;question&#160;in&#160;biological&#160;networks,&#160;a<br/>common&#160;type&#160;of&#160;model&#160;for&#160;biological&#160;systems&#160;<a href="59fe6d0319faf365850c46f352541a9cs.html#12">[11</a>,&#160;<a href="59fe6d0319faf365850c46f352541a9cs.html#12">12</a>].<br/>Specifically,&#160;we&#160;examine&#160;protein–protein&#160;interactomes<br/>from&#160;organisms&#160;across&#160;a&#160;wide&#160;range&#160;of&#160;organisms&#160;to<br/>investigate&#160;whether&#160;or&#160;not&#160;the&#160;noise&#160;and&#160;uncertainty&#160;in<br/>biological&#160;networks&#160;increases&#160;or&#160;decreases&#160;across&#160;evolu-<br/>tion.&#160;To&#160;quantify&#160;this&#160;noise&#160;and&#160;uncertainty,&#160;we&#160;make&#160;use<br/>of&#160;the&#160;<i>effective&#160;information&#160;</i>(<i>EI</i>),&#160;an&#160;information-theoretic<br/>network&#160;quantity&#160;based&#160;on&#160;the&#160;entropy&#160;of&#160;random&#160;walker<br/>behavior&#160;on&#160;a&#160;network.&#160;A&#160;lower&#160;<i>EI&#160;</i>indicates&#160;greater&#160;noise<br/>and&#160;uncertainty&#160;(a&#160;formal&#160;mathematical&#160;definition&#160;is<br/>given&#160;in&#160;the&#160;section&#160;‘Results’).&#160;Indeed,&#160;the&#160;<i>EI&#160;</i>of&#160;biological<br/>networks&#160;has&#160;already&#160;been&#160;shown&#160;to&#160;be&#160;lower&#160;in&#160;biologi-<br/>cal&#160;networks&#160;compared&#160;with&#160;technological&#160;networks&#160;<a href="59fe6d0319faf365850c46f352541a9cs.html#12">[13</a>],<br/>which&#160;opens&#160;the&#160;question&#160;of&#160;why&#160;this&#160;is&#160;the&#160;case.</p>
<p style="position:absolute;top:528px;left:78px;white-space:nowrap" class="ft23">To&#160;see&#160;how&#160;<i>EI&#160;</i>changes&#160;across&#160;evolution,&#160;we&#160;examined</p>
<p style="position:absolute;top:545px;left:63px;white-space:nowrap" class="ft27">networks&#160;of&#160;protein–protein&#160;interactions&#160;(PPIs)&#160;from<br/>organisms&#160;across&#160;the&#160;tree&#160;of&#160;life.&#160;The&#160;dataset&#160;consists<br/>of&#160;interactomes&#160;from&#160;1840&#160;species&#160;(1539&#160;bacteria,&#160;111<br/>archaea&#160;and&#160;190&#160;eukaryota)&#160;derived&#160;from&#160;the&#160;STRING<br/>database&#160;[<a href="59fe6d0319faf365850c46f352541a9cs.html#12">14,&#160;15].&#160;</a>These&#160;interactomes&#160;have&#160;been&#160;pre-<br/>viously&#160;used&#160;to&#160;study&#160;evolution&#160;of&#160;resilience,&#160;where<br/>researchers&#160;found&#160;that&#160;species&#160;tended&#160;to&#160;have&#160;higher<br/>values&#160;of&#160;network&#160;resilience&#160;with&#160;increasing&#160;evolution<br/>(wherein&#160;‘evolution’&#160;was&#160;defined&#160;as&#160;the&#160;number&#160;of<br/>nucleotide&#160;substitutions&#160;per&#160;site)&#160;<a href="59fe6d0319faf365850c46f352541a9cs.html#12">[16</a>].&#160;In&#160;our&#160;work,<br/>we&#160;take&#160;a&#160;similar&#160;approach,&#160;highlighting&#160;changes&#160;in<br/>interactome&#160;properties&#160;as&#160;evolution&#160;progresses.</p>
<p style="position:absolute;top:761px;left:78px;white-space:nowrap" class="ft23">In&#160;addition,&#160;we&#160;focus&#160;on&#160;identifying&#160;when&#160;interactomes</p>
<p style="position:absolute;top:779px;left:63px;white-space:nowrap" class="ft27">have&#160;informative&#160;macroscales.&#160;A&#160;<i>macroscale&#160;</i>refers&#160;to<br/>some&#160;dimension&#160;reduction,&#160;such&#160;as&#160;an&#160;aggregation,<br/>coarse&#160;graining&#160;or&#160;grouping,&#160;of&#160;states&#160;or&#160;elements&#160;of<br/>the&#160;biological&#160;system.&#160;In&#160;networks,&#160;this&#160;takes&#160;the&#160;form<br/>of&#160;replacing&#160;subgraphs&#160;of&#160;the&#160;network&#160;with&#160;individual<br/>nodes&#160;(macro-nodes).&#160;A&#160;network&#160;has&#160;an&#160;informative<br/>macroscale&#160;when&#160;subgraphs&#160;of&#160;the&#160;network&#160;can&#160;be<br/>grouped&#160;into&#160;macro-nodes&#160;such&#160;that&#160;the&#160;resulting<br/>dimensionally&#160;reduced&#160;network&#160;gains&#160;<i>EI&#160;</i>[<a href="59fe6d0319faf365850c46f352541a9cs.html#12">13].&#160;</a>When&#160;such<br/>grouping&#160;leads&#160;to&#160;an&#160;increase&#160;in&#160;EI,&#160;we&#160;describe&#160;the<br/>resulting&#160;macro-node&#160;as&#160;being&#160;part&#160;of&#160;an&#160;informative<br/>macroscale.&#160;Following&#160;previous&#160;work,&#160;we&#160;refer&#160;to&#160;any<br/>gain&#160;in&#160;<i>EI&#160;</i>at&#160;the&#160;macroscale&#160;as&#160;causal&#160;emergence<br/><a href="59fe6d0319faf365850c46f352541a9cs.html#12">[17</a>].&#160;With&#160;these&#160;techniques,&#160;we&#160;can&#160;identify&#160;which&#160;PPI<br/>networks&#160;have&#160;informative&#160;macroscales&#160;and&#160;which&#160;do<br/>not.&#160;By&#160;correlating&#160;this&#160;property&#160;with&#160;where&#160;(in&#160;time)<br/>each&#160;species&#160;lies&#160;in&#160;the&#160;evolutionary&#160;tree,&#160;we&#160;show<br/>that&#160;informative&#160;macroscales&#160;tend&#160;to&#160;emerge&#160;later<br/>in&#160;evolution,&#160;being&#160;associated&#160;more&#160;with&#160;eukaryota</p>
<p style="position:absolute;top:79px;left:455px;white-space:nowrap" class="ft27">than&#160;prokaryota&#160;(such&#160;as&#160;bacteria).&#160;Using&#160;a&#160;number&#160;of<br/>sensitivity&#160;and&#160;robustness&#160;tests,&#160;we&#160;show&#160;that&#160;these<br/>results&#160;are&#160;not&#160;explained&#160;by&#160;other&#160;network&#160;properties<br/>such&#160;as&#160;network&#160;density&#160;or&#160;size.</p>
<p style="position:absolute;top:151px;left:470px;white-space:nowrap" class="ft23">What&#160;is&#160;the&#160;evolutionary&#160;advantage&#160;of&#160;having&#160;informa-</p>
<p style="position:absolute;top:169px;left:455px;white-space:nowrap" class="ft27">tive&#160;higher&#160;scales?&#160;This&#160;question&#160;is&#160;important&#160;because<br/>higher&#160;scales&#160;minimize&#160;noise&#160;or&#160;uncertainty&#160;in&#160;biological<br/>networks.&#160;Yet&#160;such&#160;uncertainty&#160;or&#160;noise&#160;represents&#160;a<br/>fundamental&#160;paradox.&#160;The&#160;more&#160;noisy&#160;a&#160;network&#160;is,&#160;the<br/>more&#160;uncertain&#160;and&#160;the&#160;less&#160;effective&#160;that&#160;network&#160;is;<br/>effectiveness,&#160;here,&#160;refers&#160;to&#160;the&#160;ability&#160;to&#160;reliably&#160;trans-<br/>form&#160;inputs&#160;into&#160;outputs&#160;(similar&#160;to&#160;the&#160;notion&#160;of&#160;speci-<br/>ficity),&#160;such&#160;as&#160;being&#160;able&#160;to&#160;upregulate&#160;a&#160;particular&#160;gene<br/>in&#160;response&#160;to&#160;a&#160;detected&#160;chemical&#160;in&#160;the&#160;environment.<br/>In&#160;this&#160;sense,&#160;we&#160;might&#160;expect&#160;evolved&#160;networks&#160;to&#160;be<br/>highly&#160;effective—that&#160;is,&#160;we&#160;might&#160;expect&#160;them&#160;to&#160;have<br/>structures&#160;that&#160;reliably&#160;produce&#160;specific&#160;outputs&#160;given&#160;a<br/>certain&#160;set&#160;of&#160;inputs/causes.&#160;Yet&#160;this&#160;is&#160;the&#160;opposite&#160;of<br/>what&#160;we&#160;observe.&#160;Instead,&#160;we&#160;observe&#160;that&#160;effectiveness<br/>of&#160;lower&#160;scales&#160;decreases&#160;later&#160;in&#160;evolution,&#160;as&#160;higher<br/>scales&#160;that&#160;are&#160;effective&#160;emerge.</p>
<p style="position:absolute;top:456px;left:470px;white-space:nowrap" class="ft23">We&#160;argue&#160;here&#160;that&#160;this&#160;multiscale&#160;behavior&#160;is&#160;the&#160;res-</p>
<p style="position:absolute;top:474px;left:455px;white-space:nowrap" class="ft27">olution&#160;to&#160;a&#160;paradox:&#160;there&#160;are&#160;advantages&#160;to&#160;being&#160;effec-<br/>tive,&#160;but&#160;there&#160;are&#160;also&#160;advantages&#160;to&#160;being&#160;less&#160;effective<br/>and&#160;therefore&#160;more&#160;uncertain&#160;or&#160;noisy.&#160;For&#160;instance,&#160;less<br/>effective&#160;networks&#160;might&#160;be&#160;more&#160;resistant&#160;to&#160;attack&#160;or<br/>node&#160;failure&#160;due&#160;to&#160;redundancy.&#160;The&#160;paradox&#160;is&#160;that<br/>networks&#160;that&#160;are&#160;certain&#160;are&#160;effective&#160;yet&#160;are&#160;vulnerable<br/>to&#160;attacks&#160;or&#160;node&#160;failures,&#160;whereas&#160;networks&#160;that&#160;are<br/>uncertain&#160;are&#160;less&#160;effective&#160;but&#160;are&#160;resilient&#160;in&#160;the&#160;face<br/>of&#160;attacks&#160;or&#160;node&#160;failures.&#160;We&#160;argue&#160;that&#160;biological&#160;net-<br/>works&#160;have&#160;evolved&#160;to&#160;resolve&#160;this&#160;‘certainty&#160;paradox’&#160;by<br/>having&#160;informative&#160;higher&#160;scales.&#160;Specifically,&#160;we&#160;propose<br/>that&#160;the&#160;macroscales&#160;of&#160;a&#160;biological&#160;network&#160;evolve&#160;to<br/>have&#160;high&#160;effectiveness,&#160;but&#160;their&#160;underlying&#160;microscales<br/>may&#160;have&#160;low&#160;effectiveness,&#160;therefore&#160;making&#160;the&#160;system<br/>resilient&#160;without&#160;paying&#160;the&#160;price&#160;of&#160;a&#160;low&#160;effectiveness.</p>
<p style="position:absolute;top:743px;left:470px;white-space:nowrap" class="ft23">In&#160;a&#160;biological&#160;sense,&#160;node&#160;failures&#160;or&#160;attacks&#160;in&#160;a</p>
<p style="position:absolute;top:761px;left:455px;white-space:nowrap" class="ft27">cellular&#160;network&#160;may&#160;represent&#160;certain&#160;mutations&#160;in&#160;pro-<br/>teins&#160;or&#160;other&#160;biochemical&#160;entities,&#160;which&#160;in&#160;turn&#160;may<br/>prevent&#160;regular&#160;functioning&#160;of&#160;the&#160;system&#160;<a href="59fe6d0319faf365850c46f352541a9cs.html#12">[18</a>].&#160;Biologi-<br/>cal&#160;networks&#160;should&#160;then,&#160;over&#160;the&#160;course&#160;of&#160;evolution,<br/>develop&#160;degeneracy&#160;and&#160;noise&#160;at&#160;lower&#160;scales&#160;to&#160;maintain<br/>regular&#160;functioning,&#160;while&#160;at&#160;the&#160;same&#160;time&#160;developing<br/>effectiveness&#160;at&#160;a&#160;higher&#160;level.&#160;This&#160;transformation&#160;can<br/>be&#160;achieved&#160;by&#160;the&#160;action&#160;of&#160;both&#160;neutral&#160;and&#160;selective<br/>processes&#160;in&#160;evolution.&#160;Neutral&#160;processes&#160;such&#160;as&#160;pre-<br/>suppression,&#160;which,&#160;aided&#160;by&#160;mutations,&#160;increases&#160;the<br/>number&#160;of&#160;interactions&#160;<a href="59fe6d0319faf365850c46f352541a9cs.html#12">[19</a>]&#160;and&#160;can&#160;therefore&#160;decrease<br/>network&#160;effectiveness.&#160;On&#160;the&#160;other&#160;hand,&#160;selective&#160;pro-<br/>cesses&#160;can&#160;weed&#160;out&#160;the&#160;noise&#160;that&#160;interferes&#160;with&#160;the<br/>functioning&#160;and&#160;efficiency&#160;of&#160;the&#160;system&#160;<a href="59fe6d0319faf365850c46f352541a9cs.html#12">[20</a>].&#160;An&#160;inter-<br/>play&#160;of&#160;these&#160;evolutionary&#160;processes&#160;can&#160;lead&#160;to&#160;a&#160;reso-<br/>lution&#160;of&#160;the&#160;‘certainty&#160;paradox’&#160;in&#160;cellular&#160;networks&#160;by<br/>the&#160;development&#160;of&#160;informative&#160;macroscales.</p>
<p style="position:absolute;top:1066px;left:470px;white-space:nowrap" class="ft23">This&#160;work&#160;therefore&#160;presents&#160;an&#160;explanation&#160;for&#160;the</p>
<p style="position:absolute;top:1083px;left:455px;white-space:nowrap" class="ft27">observed&#160;trend&#160;in&#160;increased&#160;resiliency&#160;through&#160;evolu-<br/>tion&#160;<a href="59fe6d0319faf365850c46f352541a9cs.html#12">[16</a>]:&#160;informative&#160;macroscales&#160;make&#160;networks&#160;more</p>
<p style="position:absolute;top:229px;left:860px;white-space:nowrap" class="ft26">Downloaded from https://academic.oup.com/ib/advance-article/doi/10.1093/intbio/zyab020/6474503 by guest on 22 December 2021</p>
</div>
</body>
</html>
<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="" xml:lang="">
<head>
<title>59fe6d0319faf365850c46f352541a9c-html.html</title>

<meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
 <br/>
<style type="text/css">
<!--
	p {margin: 0; padding: 0;}	.ft30{font-size:9px;font-family:Times;color:#000000;}
	.ft31{font-size:9px;font-family:Times;color:#000000;}
	.ft32{font-size:11px;font-family:Times;color:#000000;}
	.ft33{font-size:14px;font-family:Times;color:#000000;}
	.ft34{font-size:13px;font-family:Times;color:#000000;}
	.ft35{font-size:11px;font-family:Times;color:#000000;}
	.ft36{font-size:11px;font-family:Times;color:#2e3191;}
	.ft37{font-size:7px;font-family:Times;color:#000000;}
	.ft38{font-size:8px;font-family:Times;color:#000000;}
	.ft39{font-size:7px;font-family:Times;color:#000000;}
	.ft310{font-size:10px;font-family:Times;color:#666666;-moz-transform: matrix(         0,          1,         -1,          0, 0, 0);-webkit-transform: matrix(         0,          1,         -1,          0, 0, 0);-o-transform: matrix(         0,          1,         -1,          0, 0, 0);-ms-transform: matrix(         0,          1,         -1,          0, 0, 0);-moz-transform-origin: left 75%;-webkit-transform-origin: left 75%;-o-transform-origin: left 75%;-ms-transform-origin: left 75%;}
	.ft311{font-size:11px;line-height:17px;font-family:Times;color:#000000;}
	.ft312{font-size:14px;line-height:20px;font-family:Times;color:#000000;}
	.ft313{font-size:13px;line-height:16px;font-family:Times;color:#000000;}
	.ft314{font-size:13px;line-height:22px;font-family:Times;color:#000000;}
-->
</style>
</head>
<body bgcolor="#A0A0A0" vlink="blue" link="blue">
<div id="page3-div" style="position:relative;width:892px;height:1173px;">
<p style="position:absolute;top:42px;left:194px;white-space:nowrap" class="ft30">Evolution&#160;and&#160;emergence:&#160;higher&#160;order&#160;information&#160;structure&#160;in&#160;protein&#160;interactomes&#160;across&#160;the&#160;tree&#160;of&#160;life&#160;<b>|&#160;3</b></p>
<p style="position:absolute;top:79px;left:72px;white-space:nowrap" class="ft311">resilient.&#160;Finally,&#160;we&#160;offer&#160;insights&#160;into&#160;biological&#160;pro-<br/>cesses&#160;at&#160;molecular&#160;level&#160;that&#160;might&#160;be&#160;responsible&#160;for<br/>the&#160;emergence&#160;of&#160;informative&#160;macroscales&#160;in&#160;PPI&#160;net-<br/>works,&#160;specifically&#160;looking&#160;at&#160;the&#160;differences&#160;between<br/>bacteria,&#160;which&#160;have&#160;a&#160;low&#160;rate&#160;of&#160;nucleotide&#160;substi-<br/>tutions&#160;per&#160;site,&#160;and&#160;eukaryota,&#160;which&#160;exhibit&#160;a&#160;higher<br/>rate.&#160;Understanding&#160;the&#160;basic&#160;principles&#160;governing&#160;the<br/>differences&#160;in&#160;efficiency&#160;and&#160;uncertainty&#160;between&#160;these<br/>major&#160;divisions&#160;of&#160;life&#160;can&#160;help&#160;us&#160;comprehend&#160;the&#160;trade-<br/>offs&#160;involved&#160;in&#160;information&#160;processes&#160;in&#160;PPIs&#160;across<br/>evolution.</p>
<p style="position:absolute;top:307px;left:72px;white-space:nowrap" class="ft311"><b>RESULTS<br/>Effectiveness&#160;of&#160;protein&#160;interactomes&#160;across<br/>the&#160;tree&#160;of&#160;life<br/></b><i>EI&#160;</i>is&#160;a&#160;network&#160;property&#160;reflecting&#160;the&#160;certainty&#160;(or&#160;uncer-<br/>tainty)&#160;contained&#160;in&#160;that&#160;network’s&#160;connectivity&#160;<a href="59fe6d0319faf365850c46f352541a9cs.html#12">[13</a>].&#160;It&#160;is<br/>a&#160;structural&#160;property&#160;of&#160;a&#160;network&#160;calculated&#160;by&#160;travers-<br/>ing&#160;its&#160;topology&#160;and&#160;is&#160;based&#160;on&#160;the&#160;uncertainty&#160;of&#160;a&#160;ran-<br/>dom&#160;walker’s&#160;transitions&#160;between&#160;pairs&#160;of&#160;nodes&#160;and&#160;the<br/>distribution&#160;of&#160;this&#160;uncertainty&#160;throughout&#160;the&#160;network.<br/>It&#160;is&#160;calculated&#160;by&#160;examining&#160;the&#160;network’s&#160;connectivity.</p>
<p style="position:absolute;top:492px;left:88px;white-space:nowrap" class="ft32">In&#160;a&#160;protein&#160;interactome,&#160;the&#160;nodes&#160;are&#160;individual</p>
<p style="position:absolute;top:510px;left:72px;white-space:nowrap" class="ft311">proteins,&#160;and&#160;the&#160;edges&#160;of&#160;the&#160;network&#160;are&#160;interactions,<br/>generally&#160;describing&#160;the&#160;possibility&#160;of&#160;binding&#160;between<br/>two&#160;proteins.&#160;Therefore,&#160;the&#160;uncertainty&#160;we&#160;analyze&#160;is<br/>uncertainty&#160;as&#160;to&#160;which&#160;protein(s)&#160;a&#160;given&#160;protein&#160;might<br/>interact&#160;with,&#160;e.g.&#160;bind&#160;with.&#160;Each&#160;node&#160;in&#160;the&#160;network<br/>has&#160;out-weights,&#160;which&#160;are&#160;represented&#160;by&#160;a&#160;vector,&#160;<i>Wiout</i>.<br/>For&#160;instance,&#160;protein&#160;<i>A&#160;</i>might&#160;share&#160;an&#160;edge&#160;with&#160;pro-<br/>tein&#160;<i>B&#160;</i>and&#160;also&#160;protein&#160;<i>C</i>.&#160;Therefore,&#160;<i>WAout&#160;</i>is&#160;[1/2,</p>
<p style="position:absolute;top:632px;left:417px;white-space:nowrap" class="ft38">1/2].</p>
<p style="position:absolute;top:653px;left:72px;white-space:nowrap" class="ft311">Since&#160;most&#160;protein&#160;interactomes&#160;are&#160;undirected,&#160;its&#160;edges<br/>are&#160;normalized&#160;for&#160;each&#160;node&#160;(such&#160;that&#160;the&#160;sum&#160;of<br/><i>Wiout&#160;</i>for&#160;each&#160;node&#160;is&#160;1.0).&#160;Note&#160;that&#160;this&#160;process&#160;of<br/>normalization&#160;implies&#160;that&#160;the&#160;probability&#160;of&#160;binding&#160;is<br/>uniform&#160;across&#160;the&#160;different&#160;possible&#160;interactions.&#160;This<br/>transformation&#160;into&#160;a&#160;direct&#160;network&#160;makes&#160;the&#160;networks<br/>amenable&#160;to&#160;standard&#160;tools&#160;of&#160;network&#160;science,&#160;such&#160;as<br/>analyzing&#160;random&#160;walk&#160;dynamics,&#160;and&#160;it&#160;is&#160;also&#160;nec-<br/>essary&#160;to&#160;calculate&#160;the&#160;<i>EI&#160;</i>of&#160;the&#160;network.&#160;In&#160;addition,<br/>the&#160;uniform&#160;distribution&#160;of&#160;1/n&#160;is&#160;the&#160;simplest&#160;a&#160;priori<br/>assumption.&#160;However,&#160;the&#160;actual&#160;probability&#160;of&#160;binding&#160;is<br/>dependent&#160;on&#160;biological&#160;background&#160;conditions&#160;such&#160;as<br/>protein&#160;prevalence&#160;and&#160;not&#160;included&#160;in&#160;most&#160;open-source<br/>models,&#160;and&#160;therefore&#160;our&#160;analysis&#160;could&#160;change&#160;if&#160;such<br/>detailed&#160;probabilities&#160;were&#160;known.</p>
<p style="position:absolute;top:922px;left:88px;white-space:nowrap" class="ft32">The&#160;uncertainty&#160;associated&#160;with&#160;each&#160;protein&#160;can&#160;be</p>
<p style="position:absolute;top:940px;left:72px;white-space:nowrap" class="ft311">mathematically&#160;captured&#160;by&#160;examining&#160;the&#160;entropy&#160;of<br/>the&#160;outputs&#160;of&#160;a&#160;node,&#160;<i>H</i>(<i>Wiout</i>),&#160;wherein&#160;a&#160;higher&#160;entropy<br/>indicates&#160;more&#160;uncertainty&#160;as&#160;to&#160;interactions&#160;<a href="59fe6d0319faf365850c46f352541a9cs.html#12">[21</a>].&#160;The<br/>entropy&#160;of&#160;the&#160;distribution&#160;of&#160;weight&#160;across&#160;the&#160;entire<br/>network,&#160;<i>H</i>(</p>
<p style="position:absolute;top:1009px;left:150px;white-space:nowrap" class="ft32"><i>Wiout</i>),&#160;reflects&#160;the&#160;spread&#160;of&#160;uncertainty</p>
<p style="position:absolute;top:1030px;left:72px;white-space:nowrap" class="ft32">across&#160;the&#160;network.&#160;A&#160;lower&#160;<i>H</i>(</p>
<p style="position:absolute;top:1027px;left:273px;white-space:nowrap" class="ft32"><i>Wiout</i>)&#160;means&#160;that&#160;infor-</p>
<p style="position:absolute;top:1048px;left:72px;white-space:nowrap" class="ft311">mation&#160;is&#160;distributed&#160;only&#160;over&#160;a&#160;small&#160;number&#160;of&#160;nodes.<br/>A&#160;high&#160;<i>H</i>(</p>
<p style="position:absolute;top:1063px;left:135px;white-space:nowrap" class="ft32"><i>Wiout</i>)&#160;signifies&#160;that&#160;information&#160;is&#160;dispersed</p>
<p style="position:absolute;top:1083px;left:72px;white-space:nowrap" class="ft311">throughout&#160;the&#160;network.&#160;The&#160;<i>EI&#160;</i>of&#160;a&#160;network&#160;can&#160;then&#160;be<br/>defined&#160;as&#160;the&#160;entropy&#160;of&#160;distribution&#160;of&#160;weights&#160;over&#160;the</p>
<p style="position:absolute;top:79px;left:464px;white-space:nowrap" class="ft311">network&#160;minus&#160;the&#160;average&#160;uncertainty&#160;inherent&#160;in&#160;the<br/>weight&#160;of&#160;each&#160;node,&#160;or:</p>
<p style="position:absolute;top:133px;left:568px;white-space:nowrap" class="ft32">EI</p>
<p style="position:absolute;top:131px;left:584px;white-space:nowrap" class="ft32">=&#160;H</p>
<p style="position:absolute;top:120px;left:611px;white-space:nowrap" class="ft32"></p>
<p style="position:absolute;top:133px;left:620px;white-space:nowrap" class="ft35"><i>Wout</i></p>
<p style="position:absolute;top:141px;left:633px;white-space:nowrap" class="ft37"><i>i</i></p>
<p style="position:absolute;top:120px;left:647px;white-space:nowrap" class="ft32"></p>
<p style="position:absolute;top:133px;left:659px;white-space:nowrap" class="ft32">–</p>
<p style="position:absolute;top:120px;left:668px;white-space:nowrap" class="ft32"></p>
<p style="position:absolute;top:133px;left:671px;white-space:nowrap" class="ft35"><i>H</i></p>
<p style="position:absolute;top:120px;left:683px;white-space:nowrap" class="ft32"></p>
<p style="position:absolute;top:133px;left:689px;white-space:nowrap" class="ft35"><i>Wout</i></p>
<p style="position:absolute;top:141px;left:703px;white-space:nowrap" class="ft37"><i>i</i></p>
<p style="position:absolute;top:120px;left:717px;white-space:nowrap" class="ft32"></p>
<p style="position:absolute;top:133px;left:813px;white-space:nowrap" class="ft32">(1)</p>
<p style="position:absolute;top:169px;left:464px;white-space:nowrap" class="ft311"><i>EI&#160;</i>can&#160;itself&#160;be&#160;further&#160;decomposed&#160;into&#160;the&#160;degeneracy<br/>and&#160;indeterminism&#160;of&#160;a&#160;network&#160;[<a href="59fe6d0319faf365850c46f352541a9cs.html#12">13,&#160;22],&#160;</a>where&#160;each&#160;indi-<br/>cates&#160;the&#160;lack&#160;of&#160;specificity&#160;in&#160;the&#160;network’s&#160;connectivity<br/>or&#160;interactions.&#160;Degeneracy&#160;indicates&#160;a&#160;lack&#160;of&#160;specificity<br/>in&#160;targeting&#160;nodes&#160;(many&#160;nodes&#160;target&#160;the&#160;same&#160;node),<br/>whereas&#160;indeterminism&#160;indicates&#160;a&#160;lack&#160;of&#160;specificity&#160;in<br/>targeted&#160;nodes&#160;(nodes&#160;target&#160;many&#160;nodes).&#160;In&#160;network<br/>science,&#160;a&#160;node&#160;that&#160;is&#160;connected&#160;to&#160;many&#160;other&#160;nodes&#160;is<br/>said&#160;to&#160;have&#160;a&#160;high&#160;degree,&#160;whereas&#160;here&#160;in&#160;this&#160;context<br/>we&#160;refer&#160;to&#160;it&#160;as&#160;having&#160;high&#160;indeterminism;&#160;both&#160;terms<br/>refer&#160;to&#160;the&#160;same&#160;structural&#160;property,&#160;but&#160;in&#160;this&#160;context,<br/>we&#160;view&#160;nodes&#160;with&#160;high&#160;degree&#160;as&#160;sources&#160;of&#160;uncertainty<br/>in&#160;the&#160;network&#160;(i.e.&#160;a&#160;random&#160;walker&#160;on&#160;a&#160;high-degree<br/>node&#160;is&#160;more&#160;uncertain&#160;about&#160;which&#160;node&#160;it&#160;will&#160;visit&#160;next<br/>compared&#160;with&#160;a&#160;random&#160;walker&#160;on&#160;a&#160;low-degree&#160;node).<br/>Note&#160;that,&#160;if&#160;networks&#160;are&#160;considered&#160;deterministic&#160;in&#160;the<br/>physical&#160;sense,&#160;the&#160;indeterminism&#160;term&#160;of&#160;<i>EI&#160;</i>still&#160;reflects<br/>the&#160;uniqueness&#160;of&#160;targets&#160;in&#160;the&#160;network.</p>
<p style="position:absolute;top:492px;left:480px;white-space:nowrap" class="ft32">A&#160;network&#160;where&#160;all&#160;the&#160;nodes&#160;target&#160;a&#160;single&#160;node&#160;will</p>
<p style="position:absolute;top:510px;left:464px;white-space:nowrap" class="ft311">have&#160;zero&#160;<i>EI&#160;</i>(since&#160;it&#160;has&#160;maximum&#160;degeneracy),&#160;as&#160;will&#160;a<br/>network&#160;where&#160;all&#160;nodes&#160;target&#160;all&#160;other&#160;nodes&#160;(complete<br/>indeterminism).&#160;<i>EI&#160;</i>will&#160;only&#160;be&#160;maximal&#160;if&#160;every&#160;node<br/>has&#160;a&#160;unique&#160;output.&#160;This&#160;forces&#160;the&#160;<i>EI&#160;</i>of&#160;a&#160;network<br/>to&#160;be&#160;bounded&#160;by&#160;log2(<i>n</i>),&#160;where&#160;<i>n&#160;</i>is&#160;the&#160;number&#160;of<br/>nodes&#160;in&#160;the&#160;network.&#160;Therefore,&#160;to&#160;compare&#160;networks&#160;of<br/>different&#160;sizes,&#160;<i>EI&#160;</i>can&#160;be&#160;normalized,&#160;and&#160;a&#160;new&#160;quantity,<br/>effectiveness,&#160;can&#160;be&#160;defined&#160;as:</p>
<p style="position:absolute;top:671px;left:564px;white-space:nowrap" class="ft32">effectiveness</p>
<p style="position:absolute;top:669px;left:654px;white-space:nowrap" class="ft32">=&#160;<i>EI/</i>log</p>
<p style="position:absolute;top:678px;left:706px;white-space:nowrap" class="ft39">2<i>(n)</i></p>
<p style="position:absolute;top:671px;left:813px;white-space:nowrap" class="ft32">(2)</p>
<p style="position:absolute;top:725px;left:480px;white-space:nowrap" class="ft32">Again,&#160;we&#160;use&#160;the&#160;term&#160;‘effectiveness’&#160;in&#160;a&#160;way&#160;that</p>
<p style="position:absolute;top:743px;left:464px;white-space:nowrap" class="ft311">is&#160;slightly&#160;different&#160;from&#160;its&#160;colloquial&#160;meaning;&#160;we<br/>instead&#160;use&#160;it&#160;to&#160;quantify&#160;the&#160;certainty&#160;that&#160;a&#160;given&#160;set&#160;of<br/>outputs&#160;will&#160;follow&#160;a&#160;particular&#160;set&#160;of&#160;inputs.&#160;To&#160;explore<br/>the&#160;change&#160;in&#160;effectiveness&#160;of&#160;biological&#160;networks,&#160;we<br/>examined&#160;protein&#160;interactomes&#160;of&#160;1840&#160;species&#160;divided<br/>between&#160;archaea,&#160;bacteria&#160;and&#160;eukaryota&#160;(see&#160;section<br/>‘Methods’&#160;for&#160;details&#160;on&#160;the&#160;origin&#160;and&#160;nature&#160;of&#160;these<br/>protein&#160;interactomes).&#160;We&#160;found&#160;a&#160;clear&#160;pattern&#160;in&#160;the<br/>effectiveness&#160;of&#160;the&#160;networks,&#160;based&#160;on&#160;where&#160;they&#160;are<br/>located&#160;in&#160;the&#160;tree&#160;of&#160;life&#160;(<a href="59fe6d0319faf365850c46f352541a9cs.html#4">Fig.&#160;1</a>),&#160;the&#160;position&#160;of&#160;which<br/>is&#160;based&#160;on&#160;each&#160;protein&#160;interactome’s&#160;small&#160;subunit<br/>ribosomal&#160;RNA&#160;gene&#160;sequence&#160;information&#160;[<a href="59fe6d0319faf365850c46f352541a9cs.html#12">23]&#160;</a>(see<br/>section&#160;‘Methods’&#160;for&#160;details).&#160;Overall,&#160;we&#160;found&#160;that&#160;the<br/>mean&#160;effectiveness&#160;of&#160;protein&#160;interactomes&#160;decreases<br/>later&#160;in&#160;the&#160;tree&#160;of&#160;life&#160;as&#160;nucleotide&#160;substitutions<br/>occurred.&#160;Specifically,&#160;bacteria&#160;were&#160;found&#160;to&#160;have&#160;a<br/>greater&#160;effectiveness&#160;(0.77)&#160;compared&#160;with&#160;eukaryota<br/>(0.72)&#160;on&#160;average&#160;(Student’s&#160;<i>t</i>-test,&#160;<i>P&#160;&lt;&#160;</i>10−8).&#160;Following<br/>Zitnik&#160;<i>et&#160;al</i>.,&#160;we&#160;restricted&#160;further&#160;statistical&#160;analysis<br/>to&#160;interactomes&#160;with&#160;<i>&gt;</i>1000&#160;citations,&#160;to&#160;use&#160;the&#160;most<br/>well-founded&#160;protein&#160;interactomes,&#160;but&#160;the&#160;directionality</p>
<p style="position:absolute;top:229px;left:860px;white-space:nowrap" class="ft310">Downloaded from https://academic.oup.com/ib/advance-article/doi/10.1093/intbio/zyab020/6474503 by guest on 22 December 2021</p>
</div>
</body>
</html>
<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="" xml:lang="">
<head>
<title>59fe6d0319faf365850c46f352541a9c-html.html</title>

<meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
 <br/>
<style type="text/css">
<!--
	p {margin: 0; padding: 0;}	.ft40{font-size:9px;font-family:Times;color:#000000;}
	.ft41{font-size:9px;font-family:Times;color:#000000;}
	.ft42{font-size:9px;font-family:Times;color:#000000;}
	.ft43{font-size:8px;font-family:Times;color:#000000;}
	.ft44{font-size:8px;font-family:Times;color:#000000;}
	.ft45{font-size:8px;font-family:Times;color:#000000;}
	.ft46{font-size:5px;font-family:Times;color:#000000;}
	.ft47{font-size:11px;font-family:Times;color:#000000;}
	.ft48{font-size:11px;font-family:Times;color:#000000;}
	.ft49{font-size:7px;font-family:Times;color:#000000;}
	.ft410{font-size:11px;font-family:Times;color:#2e3191;}
	.ft411{font-size:13px;font-family:Times;color:#000000;}
	.ft412{font-size:7px;font-family:Times;color:#000000;}
	.ft413{font-size:11px;font-family:Times;color:#2e3191;}
	.ft414{font-size:10px;font-family:Times;color:#666666;-moz-transform: matrix(         0,          1,         -1,          0, 0, 0);-webkit-transform: matrix(         0,          1,         -1,          0, 0, 0);-o-transform: matrix(         0,          1,         -1,          0, 0, 0);-ms-transform: matrix(         0,          1,         -1,          0, 0, 0);-moz-transform-origin: left 75%;-webkit-transform-origin: left 75%;-o-transform-origin: left 75%;-ms-transform-origin: left 75%;}
	.ft415{font-size:8px;line-height:13px;font-family:Times;color:#000000;}
	.ft416{font-size:11px;line-height:17px;font-family:Times;color:#000000;}
	.ft417{font-size:13px;line-height:22px;font-family:Times;color:#000000;}
	.ft418{font-size:13px;line-height:16px;font-family:Times;color:#000000;}
-->
</style>
</head>
<body bgcolor="#A0A0A0" vlink="blue" link="blue">
<div id="page4-div" style="position:relative;width:892px;height:1173px;">
<p style="position:absolute;top:42px;left:64px;white-space:nowrap" class="ft40"><b>4&#160;|&#160;</b><i>Integrative&#160;Biology</i>,&#160;2021</p>
<p style="position:absolute;top:418px;left:63px;white-space:nowrap" class="ft415"><b>Figure&#160;1.&#160;</b>Effectiveness&#160;of&#160;protein&#160;interactomes.&#160;(A)&#160;Effectiveness&#160;of&#160;all&#160;1840&#160;species&#160;with&#160;their&#160;superphylum&#160;association.&#160;Interactomes&#160;with&#160;a&#160;lower<br/>number&#160;of&#160;nucleotide&#160;substitutions&#160;per&#160;site&#160;tended&#160;to&#160;be&#160;prokaryota&#160;(yellow),&#160;whereas&#160;those&#160;higher&#160;tended&#160;to&#160;be&#160;eukaryota&#160;(blue).&#160;Solid&#160;line&#160;is&#160;a&#160;linear<br/>regression&#160;comparing&#160;the&#160;effectiveness&#160;of&#160;bacteria&#160;and&#160;eukaryota&#160;(<i>r&#160;=</i></p>
<p style="position:absolute;top:443px;left:412px;white-space:nowrap" class="ft44">−0.40,&#160;<i>P&#160;&lt;&#160;</i>10−5),&#160;due&#160;to&#160;the&#160;small&#160;number&#160;of&#160;archaea&#160;that&#160;passed&#160;the&#160;threshold</p>
<p style="position:absolute;top:458px;left:63px;white-space:nowrap" class="ft415">for&#160;reliable&#160;datasets&#160;(see&#160;section&#160;‘Results’).&#160;(B)&#160;The&#160;effectiveness&#160;of&#160;prokaryotic&#160;protein&#160;interactomes&#160;is&#160;greater&#160;than&#160;that&#160;of&#160;eukaryotic&#160;species,<br/>indicating&#160;that&#160;effectiveness&#160;might&#160;decrease&#160;with&#160;more&#160;nucleotide&#160;substitutions&#160;per&#160;site.</p>
<p style="position:absolute;top:505px;left:63px;white-space:nowrap" class="ft416">and&#160;significance&#160;of&#160;the&#160;result&#160;are&#160;unchanged&#160;when&#160;all<br/>interactomes&#160;are&#160;included&#160;(Student’s&#160;<i>t</i>-test,&#160;<i>P&#160;&lt;&#160;</i>10−11),<br/>indicating&#160;that&#160;the&#160;selection&#160;procedure&#160;of&#160;only&#160;using&#160;the<br/>most-cited&#160;organisms&#160;does&#160;not&#160;influence&#160;our&#160;results.&#160;Due<br/>to&#160;the&#160;small&#160;number&#160;of&#160;archaea&#160;interactomes&#160;based&#160;on<br/>above&#160;1000&#160;citations,&#160;we&#160;did&#160;not&#160;include&#160;those&#160;samples<br/>in&#160;<a href="59fe6d0319faf365850c46f352541a9cs.html#4">Figure&#160;1B</a>.</p>
<p style="position:absolute;top:649px;left:63px;white-space:nowrap" class="ft416"><b>Causal&#160;emergence&#160;across&#160;the&#160;tree&#160;of&#160;life<br/></b>At&#160;first&#160;the&#160;higher&#160;effectiveness&#160;in&#160;prokaryota&#160;interac-<br/>tomes&#160;as&#160;compared&#160;to&#160;that&#160;of&#160;eukaryota&#160;(as&#160;shown&#160;in<br/><a href="59fe6d0319faf365850c46f352541a9cs.html#4">Fig.&#160;1</a>)&#160;may&#160;seem&#160;counter&#160;intuitive.&#160;One&#160;might&#160;naively<br/>expect&#160;the&#160;effectiveness&#160;of&#160;cellular&#160;machinery,&#160;including<br/>or&#160;especially&#160;interactomes,&#160;to&#160;increase&#160;over&#160;evolutionary<br/>time,&#160;instead&#160;of&#160;decreasing&#160;as&#160;we&#160;have&#160;shown.</p>
<p style="position:absolute;top:779px;left:78px;white-space:nowrap" class="ft47">One&#160;hypothesis&#160;to&#160;explain&#160;these&#160;results&#160;is&#160;that,</p>
<p style="position:absolute;top:797px;left:63px;white-space:nowrap" class="ft416">although&#160;the&#160;protein&#160;interactomes&#160;get&#160;less&#160;effective&#160;in<br/>their&#160;microscales&#160;over&#160;evolutionary&#160;time,&#160;the&#160;interac-<br/>tomes&#160;are&#160;able&#160;to&#160;nonetheless&#160;be&#160;effective&#160;due&#160;to&#160;the<br/>emergence&#160;of&#160;informative&#160;macroscales&#160;as&#160;evolution<br/>proceeds.&#160;To&#160;examine&#160;this&#160;hypothesis,&#160;we&#160;must&#160;first<br/>define&#160;a&#160;procedure&#160;for&#160;finding&#160;macroscales&#160;in&#160;networks.</p>
<p style="position:absolute;top:904px;left:78px;white-space:nowrap" class="ft47">Network&#160;macroscales&#160;are&#160;defined&#160;as&#160;subgraphs&#160;(i.e.</p>
<p style="position:absolute;top:922px;left:63px;white-space:nowrap" class="ft416">connected&#160;sets&#160;of&#160;nodes&#160;and&#160;their&#160;associated&#160;links)&#160;that<br/>can&#160;be&#160;grouped&#160;into&#160;single&#160;macro-nodes&#160;such&#160;that&#160;the<br/>resulting&#160;network&#160;has&#160;a&#160;higher&#160;value&#160;of&#160;<i>EI&#160;</i>than&#160;the&#160;orig-<br/>inal&#160;microscale&#160;network&#160;[<a href="59fe6d0319faf365850c46f352541a9cs.html#12">13].&#160;</a>We&#160;denote&#160;the&#160;microscale<br/>of&#160;a&#160;network&#160;as&#160;<i>G&#160;</i>and&#160;the&#160;macroscale&#160;as&#160;<i>GM</i>,&#160;which&#160;is<br/>composed&#160;of&#160;both&#160;ungrouped&#160;nodes&#160;(micro-nodes)&#160;and<br/>macro-nodes,&#160;<i>μ</i>.&#160;The&#160;macroscale&#160;network,&#160;<i>GM</i>,&#160;is&#160;a&#160;dimen-<br/>sion&#160;reduction&#160;in&#160;that&#160;it&#160;always&#160;has&#160;fewer&#160;nodes&#160;than&#160;<i>G</i>.</p>
<p style="position:absolute;top:1066px;left:78px;white-space:nowrap" class="ft47">To&#160;recast&#160;a&#160;particular&#160;subgraph&#160;into&#160;a&#160;macro-node,</p>
<p style="position:absolute;top:1083px;left:63px;white-space:nowrap" class="ft416">its&#160;connectivity&#160;must&#160;be&#160;modified&#160;since&#160;the&#160;subgraph&#160;is<br/>being&#160;transformed&#160;into&#160;a&#160;single&#160;node.&#160;In&#160;terms&#160;of&#160;input<br/>to&#160;the&#160;new&#160;macro-node,&#160;<i>μ</i>,&#160;all&#160;out-weights&#160;that&#160;targeted</p>
<p style="position:absolute;top:505px;left:455px;white-space:nowrap" class="ft416">nodes&#160;in&#160;the&#160;subgraph&#160;now&#160;target&#160;the&#160;macro-node.&#160;In<br/>terms&#160;of&#160;output,&#160;each&#160;micro-node,&#160;<i>vi&#160;</i>inside&#160;the&#160;subgraph<br/>has&#160;some&#160;<i>Wiout</i>.&#160;To&#160;recast&#160;the&#160;nodes&#160;inside&#160;a&#160;subgraph&#160;into<br/>a&#160;macro-node,&#160;we&#160;replace&#160;the&#160;<i>Wiout&#160;</i>of&#160;the&#160;nodes&#160;in&#160;the<br/>subgraph&#160;with&#160;a&#160;single&#160;<i>Wμout</i>,&#160;which&#160;is&#160;a&#160;weighted&#160;average<br/>of&#160;the&#160;set&#160;of&#160;each&#160;<i>Wiout&#160;</i>in&#160;the&#160;subgraph.&#160;The&#160;weight&#160;is<br/>based&#160;on&#160;the&#160;probability&#160;<i>P&#160;</i>of&#160;each&#160;node&#160;<i>vi&#160;</i>in&#160;the&#160;sta-<br/>tionary&#160;distribution&#160;of&#160;the&#160;network,&#160;<i>π</i>.&#160;This&#160;forms&#160;macro-<br/>nodes&#160;(<i>μ</i>|<i>π</i>)&#160;that&#160;accurately&#160;recapitulate&#160;the&#160;microscale<br/>random&#160;walk&#160;dynamics&#160;at&#160;the&#160;new&#160;macroscale&#160;[<a href="59fe6d0319faf365850c46f352541a9cs.html#12">13].&#160;</a>A<br/>macroscale&#160;is&#160;informative&#160;if&#160;it&#160;increases&#160;the&#160;<i>EI&#160;</i>of&#160;the<br/>network&#160;compared&#160;with&#160;the&#160;original&#160;microscale.&#160;To&#160;find<br/>macro-nodes&#160;that&#160;maximally&#160;increase&#160;<i>EI</i>,&#160;we&#160;make&#160;use<br/>of&#160;a&#160;modified&#160;spectral&#160;algorithm&#160;to&#160;find&#160;locally&#160;opti-<br/>mal&#160;micro-to-macro&#160;mappings,&#160;originally&#160;described&#160;in<br/><a href="59fe6d0319faf365850c46f352541a9cs.html#12">Griebenow&#160;<i>et&#160;al</i>.&#160;[24].</a></p>
<p style="position:absolute;top:792px;left:470px;white-space:nowrap" class="ft47">Results&#160;from&#160;this&#160;analysis&#160;support&#160;our&#160;initial&#160;hypoth-</p>
<p style="position:absolute;top:810px;left:455px;white-space:nowrap" class="ft416">esis&#160;that&#160;effectiveness&#160;is&#160;actually&#160;being&#160;transitioned&#160;to<br/>macroscales&#160;of&#160;biological&#160;networks&#160;in&#160;eukaryota&#160;over<br/>evolutionary&#160;time,&#160;even&#160;though&#160;the&#160;microscales&#160;become<br/>noisy&#160;and&#160;less&#160;effective&#160;over&#160;evolutionary&#160;time.&#160;The&#160;total<br/>amount&#160;of&#160;causal&#160;emergence&#160;(the&#160;gain&#160;of&#160;<i>EI&#160;</i>by&#160;grouping<br/>subgraphs&#160;into&#160;macro-nodes)&#160;was&#160;identified&#160;for&#160;each&#160;pro-<br/>tein&#160;interactome&#160;from&#160;each&#160;species,&#160;normalized&#160;by&#160;the<br/>total&#160;size&#160;of&#160;that&#160;protein&#160;interactome&#160;(<a href="59fe6d0319faf365850c46f352541a9cs.html#5">Fig.&#160;2B</a>).&#160;Across&#160;the<br/>tree&#160;of&#160;life,&#160;we&#160;observe&#160;that&#160;eukaryota&#160;have&#160;more&#160;infor-<br/>mative&#160;macroscales&#160;and&#160;show&#160;a&#160;significant&#160;difference&#160;in<br/>the&#160;percentage&#160;of&#160;microscale&#160;nodes&#160;that&#160;get&#160;grouped&#160;into<br/>macro-nodes&#160;than&#160;prokaryota&#160;(<a href="59fe6d0319faf365850c46f352541a9cs.html#5">Fig.&#160;2A).</a></p>
<p style="position:absolute;top:1045px;left:455px;white-space:nowrap" class="ft416"><b>Macroscales&#160;of&#160;interactomes&#160;are&#160;more&#160;resilient<br/>than&#160;microscales<br/></b>Ultimately,&#160;although&#160;there&#160;may&#160;be&#160;many&#160;reasons&#160;to&#160;see<br/>an&#160;evolutionary&#160;increase&#160;in&#160;causal&#160;emergence,&#160;here,&#160;we<br/>explore&#160;the&#160;evidence&#160;for&#160;a&#160;specific&#160;benefit&#160;of&#160;having</p>
<p style="position:absolute;top:229px;left:860px;white-space:nowrap" class="ft414">Downloaded from https://academic.oup.com/ib/advance-article/doi/10.1093/intbio/zyab020/6474503 by guest on 22 December 2021</p>
</div>
</body>
</html>
<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="" xml:lang="">
<head>
<title>59fe6d0319faf365850c46f352541a9c-html.html</title>

<meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
 <br/>
<style type="text/css">
<!--
	p {margin: 0; padding: 0;}	.ft50{font-size:9px;font-family:Times;color:#000000;}
	.ft51{font-size:9px;font-family:Times;color:#000000;}
	.ft52{font-size:8px;font-family:Times;color:#000000;}
	.ft53{font-size:8px;font-family:Times;color:#000000;}
	.ft54{font-size:8px;font-family:Times;color:#000000;}
	.ft55{font-size:5px;font-family:Times;color:#000000;}
	.ft56{font-size:11px;font-family:Times;color:#000000;}
	.ft57{font-size:11px;font-family:Times;color:#2e3191;}
	.ft58{font-size:11px;font-family:Times;color:#000000;}
	.ft59{font-size:7px;font-family:Times;color:#000000;}
	.ft510{font-size:7px;font-family:Times;color:#000000;}
	.ft511{font-size:10px;font-family:Times;color:#666666;-moz-transform: matrix(         0,          1,         -1,          0, 0, 0);-webkit-transform: matrix(         0,          1,         -1,          0, 0, 0);-o-transform: matrix(         0,          1,         -1,          0, 0, 0);-ms-transform: matrix(         0,          1,         -1,          0, 0, 0);-moz-transform-origin: left 75%;-webkit-transform-origin: left 75%;-o-transform-origin: left 75%;-ms-transform-origin: left 75%;}
	.ft512{font-size:8px;line-height:13px;font-family:Times;color:#000000;}
	.ft513{font-size:11px;line-height:17px;font-family:Times;color:#000000;}
-->
</style>
</head>
<body bgcolor="#A0A0A0" vlink="blue" link="blue">
<div id="page5-div" style="position:relative;width:892px;height:1173px;">
<p style="position:absolute;top:42px;left:194px;white-space:nowrap" class="ft50">Evolution&#160;and&#160;emergence:&#160;higher&#160;order&#160;information&#160;structure&#160;in&#160;protein&#160;interactomes&#160;across&#160;the&#160;tree&#160;of&#160;life&#160;<b>|&#160;5</b></p>
<p style="position:absolute;top:306px;left:72px;white-space:nowrap" class="ft512"><b>Figure&#160;2.&#160;</b>Causal&#160;emergence&#160;in&#160;protein&#160;interactomes.&#160;(A)&#160;The&#160;protein&#160;interactomes&#160;of&#160;each&#160;species&#160;undergoes&#160;a&#160;modified&#160;spectral&#160;analysis&#160;to&#160;identify<br/>the&#160;scale&#160;with&#160;<i>EI</i>max.&#160;The&#160;total&#160;dimension&#160;reduction&#160;of&#160;the&#160;network&#160;is&#160;shown,&#160;with&#160;there&#160;being&#160;a&#160;greater&#160;effect&#160;in&#160;eukaryota&#160;as&#160;more&#160;subgraphs&#160;are<br/>grouped&#160;in&#160;macro-nodes.&#160;That&#160;is,&#160;as&#160;evolutionary&#160;time&#160;goes&#160;on&#160;the&#160;coarse-grained&#160;networks&#160;become&#160;a&#160;smaller&#160;fraction&#160;of&#160;their&#160;original&#160;microscale<br/>network&#160;size&#160;(<i>r&#160;</i>=</p>
<p style="position:absolute;top:344px;left:156px;white-space:nowrap" class="ft53">−0.46,&#160;<i>P&#160;&lt;&#160;</i>10−6).&#160;(B)&#160;To&#160;compare&#160;the&#160;degree&#160;of&#160;causal&#160;emergence&#160;in&#160;protein&#160;interactomes&#160;of&#160;different&#160;sizes,&#160;the&#160;total&#160;amount&#160;of&#160;causal</p>
<p style="position:absolute;top:359px;left:72px;white-space:nowrap" class="ft512">emergence&#160;is&#160;normalized&#160;by&#160;the&#160;size&#160;of&#160;the&#160;network,&#160;log2(<i>n</i>),&#160;and&#160;we&#160;see&#160;here&#160;a&#160;positive&#160;correlation&#160;between&#160;evolution&#160;and&#160;causal&#160;emergence&#160;(<i>r&#160;</i>=&#160;0.457,<br/><i>P&#160;&lt;&#160;</i>10−7),&#160;results&#160;which&#160;do&#160;not&#160;change&#160;in&#160;directionality&#160;or&#160;significance&#160;when&#160;using&#160;either&#160;full&#160;data&#160;set&#160;of&#160;organisms&#160;or&#160;just&#160;the&#160;top&#160;100&#160;most&#160;cited.&#160;(C)<br/>The&#160;amount&#160;of&#160;normalized&#160;causal&#160;emergence&#160;is&#160;significantly&#160;higher&#160;for&#160;eukaryota.</p>
<p style="position:absolute;top:437px;left:72px;white-space:nowrap" class="ft513">multiscale&#160;structure,&#160;which&#160;networks&#160;with&#160;only&#160;a&#160;single<br/>scale&#160;lack.&#160;This&#160;specific&#160;benefit&#160;comes&#160;from&#160;the&#160;fact&#160;that<br/>all&#160;networks&#160;face&#160;a&#160;‘certainty&#160;paradox.’&#160;The&#160;paradox&#160;is<br/>that&#160;uncertainty&#160;in&#160;connectivity&#160;is&#160;desirable&#160;since&#160;it&#160;is<br/>protective&#160;from&#160;node&#160;failures.&#160;For&#160;instance,&#160;a&#160;node&#160;failure<br/>could&#160;be&#160;the&#160;removal&#160;of&#160;a&#160;protein&#160;due&#160;to&#160;a&#160;nonsense<br/>mutation,&#160;or&#160;the&#160;inability&#160;to&#160;express&#160;a&#160;certain&#160;protein&#160;due<br/>to&#160;an&#160;environmental&#160;effect,&#160;such&#160;as&#160;a&#160;lack&#160;of&#160;resources,<br/>or&#160;even&#160;a&#160;viral&#160;attack.&#160;In&#160;turn,&#160;this&#160;could&#160;lead&#160;to&#160;a&#160;loss&#160;of<br/>biological&#160;function&#160;or&#160;the&#160;development&#160;of&#160;disease&#160;or&#160;even<br/>cell&#160;death.&#160;A&#160;protein&#160;interactome&#160;may&#160;be&#160;resilient&#160;to&#160;such<br/>node&#160;failures&#160;by&#160;being&#160;highly&#160;uncertain&#160;or&#160;degenerate<br/>in&#160;its&#160;protein–protein&#160;interactomes.&#160;However,&#160;this&#160;comes<br/>at&#160;a&#160;cost.&#160;A&#160;high&#160;uncertainty&#160;can&#160;lead&#160;to&#160;problems&#160;with<br/>reliability,&#160;uniqueness&#160;and&#160;control&#160;in&#160;terms&#160;of&#160;effects,<br/>such&#160;as&#160;an&#160;inability&#160;for&#160;a&#160;particular&#160;protein&#160;to&#160;deter-<br/>ministically&#160;bind&#160;with&#160;another&#160;protein.&#160;For&#160;instance,&#160;in<br/>a&#160;time&#160;of&#160;environmental&#160;restriction&#160;of&#160;resources,&#160;certain<br/>protein–protein&#160;interactomes&#160;may&#160;be&#160;necessary&#160;for&#160;con-<br/>tinued&#160;cellular&#160;function,&#160;but&#160;if&#160;there&#160;is&#160;large-scale&#160;uncer-<br/>tainty&#160;even&#160;significant&#160;upregulation&#160;of&#160;genes&#160;controlling<br/>expression&#160;may&#160;not&#160;lead&#160;reliably&#160;to&#160;a&#160;certain&#160;interaction.</p>
<p style="position:absolute;top:832px;left:88px;white-space:nowrap" class="ft56">Here&#160;we&#160;explore&#160;these&#160;issues&#160;by&#160;examining&#160;the&#160;net-</p>
<p style="position:absolute;top:850px;left:72px;white-space:nowrap" class="ft513">work&#160;resilience&#160;of&#160;protein&#160;interactomes&#160;in&#160;response&#160;to<br/>node&#160;removals,&#160;which&#160;represent&#160;either&#160;attacks&#160;or&#160;general<br/>node&#160;failures.&#160;To&#160;measure&#160;the&#160;resilience&#160;of&#160;the&#160;network<br/>in&#160;response&#160;to&#160;a&#160;node&#160;removal&#160;we&#160;follow&#160;<a href="59fe6d0319faf365850c46f352541a9cs.html#12">[16</a>]&#160;by&#160;using<br/>the&#160;change&#160;in&#160;the&#160;Shannon&#160;entropy&#160;of&#160;the&#160;component<br/>size&#160;distribution&#160;of&#160;the&#160;network&#160;following&#160;random&#160;node<br/>removal.&#160;That&#160;is,&#160;if&#160;<i>pc&#160;</i>is&#160;the&#160;probability&#160;that&#160;a&#160;randomly<br/>selected&#160;node&#160;is&#160;in&#160;connected&#160;component&#160;<i>c</i></p>
<p style="position:absolute;top:973px;left:353px;white-space:nowrap" class="ft56">∈&#160;<i>C&#160;</i>following</p>
<p style="position:absolute;top:993px;left:72px;white-space:nowrap" class="ft513">the&#160;removal&#160;of&#160;a&#160;fraction&#160;<i>f&#160;</i>of&#160;the&#160;nodes&#160;in&#160;the&#160;network,<br/>the&#160;entropy&#160;associated&#160;with&#160;the&#160;component&#160;size&#160;distribu-<br/>tion,&#160;<i>H</i>(<i>Gf</i>),&#160;is:</p>
<p style="position:absolute;top:1086px;left:149px;white-space:nowrap" class="ft58"><i>H</i></p>
<p style="position:absolute;top:1073px;left:161px;white-space:nowrap" class="ft56"></p>
<p style="position:absolute;top:1086px;left:166px;white-space:nowrap" class="ft58"><i>Gf</i></p>
<p style="position:absolute;top:1073px;left:181px;white-space:nowrap" class="ft56"></p>
<p style="position:absolute;top:1084px;left:190px;white-space:nowrap" class="ft56">=&#160;−</p>
<p style="position:absolute;top:1077px;left:236px;white-space:nowrap" class="ft56">1</p>
<p style="position:absolute;top:1096px;left:217px;white-space:nowrap" class="ft56">log</p>
<p style="position:absolute;top:1103px;left:237px;white-space:nowrap" class="ft510">2<i>(N)</i></p>
<p style="position:absolute;top:1071px;left:268px;white-space:nowrap" class="ft56"><i>n</i></p>
<p style="position:absolute;top:1081px;left:293px;white-space:nowrap" class="ft59"><i>c</i></p>
<p style="position:absolute;top:1098px;left:288px;white-space:nowrap" class="ft59"><i>c</i></p>
<p style="position:absolute;top:1086px;left:298px;white-space:nowrap" class="ft58"><i>pc</i>log2</p>
<p style="position:absolute;top:1073px;left:338px;white-space:nowrap" class="ft56"></p>
<p style="position:absolute;top:1086px;left:343px;white-space:nowrap" class="ft58"><i>pc</i></p>
<p style="position:absolute;top:1073px;left:355px;white-space:nowrap" class="ft56"></p>
<p style="position:absolute;top:1086px;left:421px;white-space:nowrap" class="ft56">(3)</p>
<p style="position:absolute;top:437px;left:464px;white-space:nowrap" class="ft513">where&#160;<i>nc&#160;</i>is&#160;the&#160;number&#160;of&#160;connected&#160;components<br/>remaining&#160;after&#160;<i>f&#160;</i>fraction&#160;nodes&#160;have&#160;been&#160;removed<br/>(note:&#160;‘removed’&#160;here&#160;indicates&#160;that&#160;the&#160;nodes&#160;become<br/>isolates,&#160;still&#160;contributing&#160;to&#160;the&#160;component&#160;size&#160;distri-<br/>bution&#160;though&#160;not&#160;retaining&#160;any&#160;of&#160;the&#160;original&#160;links).&#160;The<br/>change&#160;in&#160;entropy,&#160;<i>H</i>(<i>Gf</i>),&#160;as&#160;<i>f&#160;</i>from&#160;0.0&#160;to&#160;1.0&#160;corresponds<br/>to&#160;the&#160;resilience&#160;of&#160;the&#160;network&#160;in&#160;question.&#160;Specifically,<br/>this&#160;resilience&#160;is&#160;defined&#160;as&#160;follows:</p>
<p style="position:absolute;top:607px;left:544px;white-space:nowrap" class="ft56">Resilience<i>(G)</i></p>
<p style="position:absolute;top:604px;left:634px;white-space:nowrap" class="ft56">=&#160;1&#160;−</p>
<p style="position:absolute;top:592px;left:672px;white-space:nowrap" class="ft56">1</p>
<p style="position:absolute;top:618px;left:691px;white-space:nowrap" class="ft59"><i>f</i></p>
<p style="position:absolute;top:616px;left:695px;white-space:nowrap" class="ft510">=0</p>
<p style="position:absolute;top:596px;left:710px;white-space:nowrap" class="ft58"><i>H</i></p>
<p style="position:absolute;top:583px;left:723px;white-space:nowrap" class="ft56"></p>
<p style="position:absolute;top:596px;left:728px;white-space:nowrap" class="ft58"><i>Gf</i></p>
<p style="position:absolute;top:583px;left:743px;white-space:nowrap" class="ft56"></p>
<p style="position:absolute;top:616px;left:724px;white-space:nowrap" class="ft58"><i>rf</i></p>
<p style="position:absolute;top:607px;left:813px;white-space:nowrap" class="ft56">(4)</p>
<p style="position:absolute;top:653px;left:464px;white-space:nowrap" class="ft513">where&#160;<i>rf&#160;</i>is&#160;the&#160;rate&#160;of&#160;node&#160;removal&#160;(i.e.&#160;the&#160;increment<br/>that&#160;the&#160;fraction&#160;<i>f&#160;</i>increases&#160;from&#160;0&#160;to&#160;1).&#160;In&#160;this&#160;work,<br/>we&#160;default&#160;to&#160;a&#160;value&#160;of&#160;<i>rf&#160;=&#160;</i>100,&#160;which&#160;means&#160;that&#160;the<br/>calculation&#160;of&#160;a&#160;network’s&#160;resilience&#160;involves&#160;iteratively<br/>removing&#160;1,&#160;2,&#160;.&#160;.&#160;.&#160;100%&#160;of&#160;the&#160;nodes&#160;in&#160;the&#160;network.&#160;For<br/>each&#160;value&#160;of&#160;<i>f&#160;</i>,&#160;we&#160;simulate&#160;the&#160;node&#160;removal&#160;process&#160;20<br/>times.</p>
<p style="position:absolute;top:779px;left:480px;white-space:nowrap" class="ft56">Our&#160;hypothesis&#160;is&#160;that&#160;biological&#160;networks&#160;deal&#160;with</p>
<p style="position:absolute;top:797px;left:464px;white-space:nowrap" class="ft513">this&#160;‘certainty&#160;paradox’&#160;by&#160;maintaining&#160;uncertainty&#160;at<br/>their&#160;microscale.&#160;This&#160;gives&#160;a&#160;pool&#160;of&#160;noise&#160;and&#160;degener-<br/>acy,&#160;leading&#160;to&#160;resilience.&#160;Meanwhile,&#160;at&#160;the&#160;macroscale,<br/>the&#160;networks&#160;can&#160;develop&#160;a&#160;high&#160;effectiveness,&#160;wherein<br/>sets&#160;of&#160;proteins&#160;deterministically&#160;and&#160;non-degenerately<br/>interact.&#160;To&#160;explore&#160;this&#160;hypothesis,&#160;we&#160;compare&#160;the&#160;net-<br/>work’s&#160;resilience&#160;to&#160;removing&#160;micro-nodes&#160;that&#160;are&#160;mem-<br/>bers&#160;of&#160;subgraphs&#160;grouped&#160;into&#160;macro-nodes&#160;to&#160;the&#160;net-<br/>work’s&#160;resilience&#160;to&#160;removing&#160;micro-nodes&#160;that&#160;remain<br/>ungrouped&#160;(shown&#160;in&#160;<a href="59fe6d0319faf365850c46f352541a9cs.html#6">Fig.&#160;3</a>).</p>
<p style="position:absolute;top:976px;left:480px;white-space:nowrap" class="ft56">By&#160;isolating&#160;the&#160;calculation&#160;of&#160;network&#160;resilience&#160;to</p>
<p style="position:absolute;top:994px;left:464px;white-space:nowrap" class="ft513">only&#160;the&#160;micro-&#160;or&#160;macro-nodes&#160;of&#160;a&#160;network,&#160;we&#160;see&#160;a<br/>stark&#160;trend&#160;emerge&#160;wherein&#160;nodes&#160;inside&#160;highly&#160;informa-<br/>tive&#160;macro-nodes&#160;are&#160;more&#160;resilient&#160;than&#160;nodes&#160;outside.<br/>That&#160;is,&#160;nodes&#160;in&#160;the&#160;original&#160;interactome&#160;that&#160;were<br/>grouped&#160;into&#160;a&#160;macro-node&#160;contribute&#160;more&#160;to&#160;the&#160;overall<br/>resilience&#160;of&#160;the&#160;interactome.&#160;This&#160;not&#160;only&#160;supports&#160;our<br/>hypothesis&#160;that&#160;biological&#160;networks&#160;resolve&#160;the&#160;‘certainty</p>
<p style="position:absolute;top:229px;left:860px;white-space:nowrap" class="ft511">Downloaded from https://academic.oup.com/ib/advance-article/doi/10.1093/intbio/zyab020/6474503 by guest on 22 December 2021</p>
</div>
</body>
</html>
<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="" xml:lang="">
<head>
<title>59fe6d0319faf365850c46f352541a9c-html.html</title>

<meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
 <br/>
<style type="text/css">
<!--
	p {margin: 0; padding: 0;}	.ft60{font-size:9px;font-family:Times;color:#000000;}
	.ft61{font-size:9px;font-family:Times;color:#000000;}
	.ft62{font-size:9px;font-family:Times;color:#000000;}
	.ft63{font-size:8px;font-family:Times;color:#000000;}
	.ft64{font-size:8px;font-family:Times;color:#000000;}
	.ft65{font-size:8px;font-family:Times;color:#000000;}
	.ft66{font-size:5px;font-family:Times;color:#000000;}
	.ft67{font-size:11px;font-family:Times;color:#000000;}
	.ft68{font-size:11px;font-family:Times;color:#2e3191;}
	.ft69{font-size:14px;font-family:Times;color:#000000;}
	.ft610{font-size:11px;font-family:Times;color:#000000;}
	.ft611{font-size:10px;font-family:Times;color:#666666;-moz-transform: matrix(         0,          1,         -1,          0, 0, 0);-webkit-transform: matrix(         0,          1,         -1,          0, 0, 0);-o-transform: matrix(         0,          1,         -1,          0, 0, 0);-ms-transform: matrix(         0,          1,         -1,          0, 0, 0);-moz-transform-origin: left 75%;-webkit-transform-origin: left 75%;-o-transform-origin: left 75%;-ms-transform-origin: left 75%;}
	.ft612{font-size:8px;line-height:13px;font-family:Times;color:#000000;}
	.ft613{font-size:11px;line-height:17px;font-family:Times;color:#000000;}
-->
</style>
</head>
<body bgcolor="#A0A0A0" vlink="blue" link="blue">
<div id="page6-div" style="position:relative;width:892px;height:1173px;">
<p style="position:absolute;top:42px;left:64px;white-space:nowrap" class="ft60"><b>6&#160;|&#160;</b><i>Integrative&#160;Biology</i>,&#160;2021</p>
<p style="position:absolute;top:454px;left:63px;white-space:nowrap" class="ft612"><b>Figure&#160;3.&#160;</b>Resilience&#160;of&#160;micro-&#160;and&#160;macro-nodes&#160;following&#160;causal&#160;emergence&#160;in&#160;interactomes.&#160;The&#160;resilience&#160;of&#160;a&#160;species&#160;interactome&#160;changes&#160;across<br/>the&#160;tree&#160;of&#160;life,&#160;as&#160;shown&#160;in&#160;previous&#160;research&#160;[16].&#160;Using&#160;the&#160;mapping&#160;generated&#160;by&#160;computing&#160;causal&#160;emergence&#160;(Fig.&#160;2B),&#160;we&#160;calculate&#160;the&#160;resilience<br/>of&#160;the&#160;network,&#160;isolating&#160;the&#160;calculation&#160;to&#160;nodes&#160;that&#160;are&#160;either&#160;part&#160;of&#160;the&#160;macroscale&#160;or&#160;microscale.&#160;Points&#160;are&#160;color-coded&#160;according&#160;to&#160;the<br/>evolutionary&#160;domain;&#160;points&#160;with&#160;dark&#160;outlines&#160;are&#160;associated&#160;with&#160;micro-nodes&#160;that&#160;have&#160;been&#160;grouped&#160;into&#160;a&#160;macro-node&#160;(macroscale),&#160;whereas&#160;the<br/>points&#160;with&#160;light&#160;outlines&#160;have&#160;not&#160;been&#160;grouped&#160;into&#160;a&#160;macro-node&#160;(microscale).&#160;Nodes&#160;at&#160;the&#160;microscale&#160;contribute&#160;less&#160;to&#160;the&#160;overall&#160;resilience&#160;of&#160;a<br/>given&#160;network&#160;(0.331)&#160;compared&#160;with&#160;nodes&#160;that&#160;contribute&#160;to&#160;macro-nodes&#160;(0.543)&#160;on&#160;average&#160;(<i>t</i>-test,&#160;<i>P&#160;&lt;&#160;</i>10−10).&#160;Note:&#160;plotted&#160;are&#160;the&#160;microscale&#160;and<br/>macroscale&#160;resilience&#160;values&#160;for&#160;each&#160;interactome&#160;in&#160;the&#160;dataset,&#160;and&#160;the&#160;difference&#160;in&#160;resilience&#160;across&#160;scales&#160;holds&#160;even&#160;when&#160;only&#160;including&#160;species<br/>with&#160;<i>&gt;</i>10,&#160;100&#160;or&#160;1000&#160;citations.</p>
<p style="position:absolute;top:598px;left:63px;white-space:nowrap" class="ft613">paradox’&#160;by&#160;building&#160;multiscale&#160;structure,&#160;but&#160;also&#160;pro-<br/>vides&#160;further&#160;explanation&#160;and&#160;contextualization&#160;for&#160;the<br/>recent&#160;findings&#160;of&#160;increasing&#160;resilience&#160;across&#160;evolution-<br/>ary&#160;time&#160;<a href="59fe6d0319faf365850c46f352541a9cs.html#12">[16</a>].</p>
<p style="position:absolute;top:699px;left:63px;white-space:nowrap" class="ft69"><b>DISCUSSION</b></p>
<p style="position:absolute;top:725px;left:63px;white-space:nowrap" class="ft613">In&#160;this&#160;work,&#160;we&#160;analyzed&#160;how&#160;the&#160;informativeness&#160;of<br/>protein&#160;interactomes&#160;changed&#160;over&#160;evolutionary&#160;time.<br/>Specifically,&#160;we&#160;made&#160;use&#160;of&#160;the&#160;<i>EI&#160;</i>to&#160;analyze&#160;the&#160;amount<br/>of&#160;uncertainty&#160;(or&#160;noise)&#160;in&#160;the&#160;connectivity&#160;of&#160;protein<br/>interactomes.&#160;We&#160;found&#160;that&#160;the&#160;effectiveness&#160;(the&#160;nor-<br/>malized&#160;<i>EI</i>)&#160;of&#160;protein&#160;interactomes&#160;decreased&#160;over&#160;evo-<br/>lutionary&#160;time,&#160;indicating&#160;that&#160;uncertainty&#160;in&#160;the&#160;connec-<br/>tivity&#160;of&#160;the&#160;interactomes&#160;was&#160;increasing&#160;over&#160;evolution-<br/>ary&#160;time.&#160;However,&#160;we&#160;discovered&#160;that&#160;this&#160;was&#160;due&#160;to<br/>eukaryotic&#160;protein&#160;interactomes&#160;possessing&#160;higher&#160;(infor-<br/>mative)&#160;scales,&#160;such&#160;that&#160;they&#160;had&#160;more&#160;<i>EI&#160;</i>when&#160;recast<br/>as&#160;a&#160;coarse-grained&#160;network—a&#160;phenomenon&#160;known&#160;as<br/>causal&#160;emergence.&#160;This&#160;lower&#160;effectiveness&#160;and&#160;higher<br/>causal&#160;emergence&#160;in&#160;eukaryotic&#160;species&#160;was&#160;due&#160;to&#160;the<br/>indeterminism&#160;and&#160;degeneracy&#160;in&#160;the&#160;network&#160;structure<br/>of&#160;their&#160;PPIs.</p>
<p style="position:absolute;top:1012px;left:78px;white-space:nowrap" class="ft67">We&#160;used&#160;a&#160;dataset&#160;from&#160;the&#160;STRING&#160;database&#160;[<a href="59fe6d0319faf365850c46f352541a9cs.html#12">14</a>,&#160;<a href="59fe6d0319faf365850c46f352541a9cs.html#12">15]</a></p>
<p style="position:absolute;top:1030px;left:63px;white-space:nowrap" class="ft613">that&#160;spans&#160;<i>&gt;</i>1800&#160;species&#160;(1539&#160;bacteria,&#160;111&#160;archaea&#160;and<br/>190&#160;eukaryota),&#160;which&#160;has&#160;been&#160;shown&#160;to&#160;have&#160;consid-<br/>erable&#160;advantages&#160;compared&#160;with&#160;previous&#160;collections<br/>of&#160;protein&#160;interactomes&#160;[<a href="59fe6d0319faf365850c46f352541a9cs.html#12">16].&#160;</a>However,&#160;we&#160;cannot&#160;rule<br/>out&#160;the&#160;possibility&#160;that&#160;biases&#160;might&#160;exist&#160;in&#160;the&#160;specific</p>
<p style="position:absolute;top:598px;left:455px;white-space:nowrap" class="ft613">manner&#160;of&#160;data&#160;collection,&#160;such&#160;as&#160;high&#160;under&#160;repre-<br/>sentation&#160;of&#160;specific&#160;types&#160;of&#160;difficult-to-detect&#160;interac-<br/>tions,&#160;which&#160;could&#160;potentially&#160;introduce&#160;errors&#160;in&#160;the<br/>calculations&#160;of&#160;effectiveness&#160;in&#160;eukaryotic&#160;interactomes.<br/>As&#160;such,&#160;we&#160;conducted&#160;a&#160;series&#160;of&#160;statistical&#160;robustness<br/>tests&#160;that&#160;accounted&#160;for&#160;potential&#160;biases&#160;in&#160;both&#160;the<br/>data&#160;collection&#160;and&#160;network&#160;structures&#160;of&#160;interactomes<br/>in&#160;our&#160;dataset&#160;(see&#160;<a href="59fe6d0319faf365850c46f352541a9cs.html#7">Fig.&#160;4&#160;</a>in&#160;section&#160;‘Methods’&#160;for&#160;further<br/>details&#160;about&#160;these&#160;statistical&#160;tests).&#160;In&#160;short,&#160;the&#160;results<br/>we&#160;observed&#160;in&#160;this&#160;study&#160;cannot&#160;be&#160;explained&#160;by&#160;two<br/>plausible&#160;sources&#160;of&#160;bias:&#160;(1)&#160;Random&#160;rewiring&#160;of&#160;network<br/>edges&#160;does&#160;not&#160;produce&#160;similar&#160;results&#160;and&#160;(2)&#160;Network<br/>null&#160;models&#160;of&#160;each&#160;interactome&#160;in&#160;this&#160;study&#160;produce<br/>only&#160;a&#160;fraction&#160;of&#160;the&#160;observed&#160;causal&#160;emergence&#160;in<br/>our&#160;dataset&#160;(the&#160;maximum&#160;causal&#160;emergence&#160;values&#160;for<br/>a&#160;species’&#160;network&#160;null&#160;model&#160;only&#160;reached&#160;3%&#160;of&#160;the<br/>causal&#160;emergence&#160;of&#160;the&#160;original&#160;interactome).&#160;Notwith-<br/>standing&#160;these&#160;statistical&#160;tests,&#160;as&#160;technology&#160;and&#160;meth-<br/>ods&#160;continue&#160;to&#160;improve,&#160;these&#160;results&#160;and&#160;hypotheses<br/>should&#160;continue&#160;to&#160;be&#160;tested.</p>
<p style="position:absolute;top:957px;left:470px;white-space:nowrap" class="ft67">Macroscales&#160;themselves&#160;may&#160;be&#160;important&#160;both&#160;from</p>
<p style="position:absolute;top:975px;left:455px;white-space:nowrap" class="ft613">the&#160;perspective&#160;of&#160;the&#160;studying&#160;biologist&#160;understanding<br/>what&#160;the&#160;intrinsic&#160;or&#160;functionally&#160;relevant&#160;scales&#160;of&#160;a<br/>biological&#160;system&#160;are&#160;and&#160;also&#160;from&#160;the&#160;perspective&#160;of&#160;the<br/>system&#160;themselves.&#160;For&#160;instance,&#160;possessing&#160;informative<br/>macroscales&#160;might&#160;make&#160;for&#160;easier&#160;control&#160;of&#160;particular<br/>outcomes&#160;or&#160;processes,&#160;such&#160;as&#160;cell&#160;differentiation;&#160;they<br/>may&#160;also&#160;be&#160;easier&#160;for&#160;evolution&#160;to&#160;construct&#160;or&#160;evolve,<br/>since&#160;macroscales&#160;are&#160;by&#160;definition&#160;multiply&#160;realizable.</p>
<p style="position:absolute;top:229px;left:860px;white-space:nowrap" class="ft611">Downloaded from https://academic.oup.com/ib/advance-article/doi/10.1093/intbio/zyab020/6474503 by guest on 22 December 2021</p>
</div>
</body>
</html>
<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="" xml:lang="">
<head>
<title>59fe6d0319faf365850c46f352541a9c-html.html</title>

<meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
 <br/>
<style type="text/css">
<!--
	p {margin: 0; padding: 0;}	.ft70{font-size:9px;font-family:Times;color:#000000;}
	.ft71{font-size:9px;font-family:Times;color:#000000;}
	.ft72{font-size:8px;font-family:Times;color:#000000;}
	.ft73{font-size:8px;font-family:Times;color:#000000;}
	.ft74{font-size:11px;font-family:Times;color:#000000;}
	.ft75{font-size:11px;font-family:Times;color:#2e3191;}
	.ft76{font-size:10px;font-family:Times;color:#666666;-moz-transform: matrix(         0,          1,         -1,          0, 0, 0);-webkit-transform: matrix(         0,          1,         -1,          0, 0, 0);-o-transform: matrix(         0,          1,         -1,          0, 0, 0);-ms-transform: matrix(         0,          1,         -1,          0, 0, 0);-moz-transform-origin: left 75%;-webkit-transform-origin: left 75%;-o-transform-origin: left 75%;-ms-transform-origin: left 75%;}
	.ft77{font-size:8px;line-height:13px;font-family:Times;color:#000000;}
	.ft78{font-size:11px;line-height:17px;font-family:Times;color:#000000;}
-->
</style>
</head>
<body bgcolor="#A0A0A0" vlink="blue" link="blue">
<div id="page7-div" style="position:relative;width:892px;height:1173px;">
<p style="position:absolute;top:42px;left:194px;white-space:nowrap" class="ft70">Evolution&#160;and&#160;emergence:&#160;higher&#160;order&#160;information&#160;structure&#160;in&#160;protein&#160;interactomes&#160;across&#160;the&#160;tree&#160;of&#160;life&#160;<b>|&#160;7</b></p>
<p style="position:absolute;top:379px;left:72px;white-space:nowrap" class="ft77"><b>Figure&#160;4.&#160;</b>Statistical&#160;controls&#160;and&#160;network&#160;robustness&#160;tests.&#160;(A)&#160;As&#160;a&#160;greater&#160;fraction&#160;of&#160;network&#160;links&#160;are&#160;randomly&#160;rewired,&#160;we&#160;observe&#160;a&#160;decrease&#160;in<br/>the&#160;causal&#160;emergence&#160;values&#160;of&#160;the&#160;resulting&#160;networks&#160;(normalized&#160;by&#160;the&#160;causal&#160;emergence&#160;value&#160;of&#160;the&#160;original&#160;network).&#160;This&#160;appears&#160;not&#160;to&#160;be<br/>dependent&#160;on&#160;evolutionary&#160;domain,&#160;network&#160;size,&#160;density&#160;or&#160;other&#160;network&#160;properties.&#160;Error&#160;bands&#160;are&#160;95%&#160;confidence&#160;intervals.&#160;(B)&#160;A&#160;second<br/>statistical&#160;control&#160;known&#160;as&#160;a&#160;soft&#160;configuration&#160;model&#160;assesses&#160;whether&#160;there&#160;is&#160;anything&#160;intrinsic&#160;to&#160;the&#160;network’s&#160;degree&#160;distribution&#160;that&#160;could&#160;be<br/>driving&#160;a&#160;given&#160;result.&#160;Here,&#160;we&#160;divide&#160;the&#160;average&#160;causal&#160;emergence&#160;of&#160;10&#160;such&#160;configuration&#160;model&#160;networks&#160;by&#160;the&#160;causal&#160;emergence&#160;values&#160;of&#160;the<br/>original&#160;protein&#160;interactome&#160;and&#160;observe&#160;that&#160;the&#160;null&#160;model&#160;networks&#160;preserve&#160;only&#160;a&#160;small&#160;fraction&#160;of&#160;the&#160;original&#160;amount&#160;of&#160;information&#160;gain&#160;(at<br/>most,&#160;the&#160;configuration&#160;models&#160;may&#160;show&#160;3%&#160;of&#160;the&#160;original&#160;causal&#160;emergence).</p>
<p style="position:absolute;top:502px;left:88px;white-space:nowrap" class="ft74">To&#160;analyze&#160;one&#160;further&#160;specific&#160;possible&#160;cause&#160;for&#160;why</p>
<p style="position:absolute;top:520px;left:72px;white-space:nowrap" class="ft78">macroscales&#160;of&#160;biological&#160;networks&#160;evolved,&#160;we&#160;calcu-<br/>lated&#160;how&#160;resilience&#160;differed&#160;for&#160;nodes&#160;inside&#160;of&#160;or&#160;outside<br/>of&#160;macro-nodes.&#160;We&#160;found&#160;that&#160;resilience&#160;of&#160;nodes&#160;left<br/>outside&#160;the&#160;macro&#160;was&#160;far&#160;lower,&#160;on&#160;average,&#160;than&#160;the<br/>resilience&#160;of&#160;nodes&#160;grouped&#160;into&#160;macro-nodes.&#160;This&#160;indi-<br/>cates&#160;that&#160;there&#160;are&#160;direct&#160;measurable&#160;benefits&#160;of&#160;having<br/>macroscales,&#160;such&#160;as&#160;increased&#160;resilience,&#160;and&#160;that&#160;sys-<br/>tems&#160;with&#160;informative&#160;macroscales&#160;can&#160;still&#160;have&#160;a&#160;high<br/>effectiveness&#160;but&#160;also&#160;maintain&#160;the&#160;benefits&#160;of&#160;having<br/>low&#160;effectiveness&#160;at&#160;a&#160;microscale.&#160;This&#160;is&#160;in&#160;line&#160;with&#160;the<br/>existing&#160;research&#160;showing&#160;that&#160;resilience&#160;increases&#160;with<br/>evolution&#160;<a href="59fe6d0319faf365850c46f352541a9cs.html#12">[16</a>].</p>
<p style="position:absolute;top:735px;left:88px;white-space:nowrap" class="ft74">These&#160;findings&#160;present&#160;evidence&#160;that&#160;biological&#160;sys-</p>
<p style="position:absolute;top:753px;left:72px;white-space:nowrap" class="ft78">tems&#160;are&#160;sensitive&#160;to&#160;the&#160;tradeoff&#160;between&#160;effectiveness<br/>and&#160;robustness&#160;by&#160;examining&#160;whether&#160;evolution&#160;brings<br/>about&#160;multiscale&#160;structure&#160;in&#160;biological&#160;networks.&#160;Sys-<br/>tems&#160;with&#160;a&#160;single&#160;level&#160;of&#160;function&#160;face&#160;an&#160;irresolvable<br/>paradox:&#160;uncertainty&#160;in&#160;the&#160;connections&#160;and&#160;interactions<br/>between&#160;nodes&#160;leads&#160;to&#160;resilience&#160;to&#160;attack&#160;and&#160;robust-<br/>ness&#160;to&#160;node&#160;failures,&#160;but&#160;this&#160;decreases&#160;the&#160;effectiveness<br/>of&#160;that&#160;network.&#160;However,&#160;multiscale&#160;systems,&#160;defined<br/>as&#160;those&#160;with&#160;an&#160;informative&#160;higher&#160;scale,&#160;can&#160;solve<br/>this&#160;‘certainty&#160;paradox’&#160;by&#160;having&#160;high&#160;uncertainty&#160;in<br/>their&#160;connectivity&#160;at&#160;the&#160;microscale&#160;while&#160;having&#160;high<br/>certainty&#160;in&#160;their&#160;connectivity&#160;at&#160;the&#160;macroscale.&#160;The<br/>tradeoffs&#160;between&#160;being&#160;effective&#160;at&#160;a&#160;microscale&#160;(typ-<br/>ically&#160;in&#160;prokaryotes,&#160;e.g.&#160;bacteria)&#160;and&#160;being&#160;noisy&#160;at<br/>microscale&#160;while&#160;transitioning&#160;the&#160;information&#160;to&#160;higher<br/>scales&#160;(eukaryota)&#160;might&#160;have&#160;played&#160;a&#160;key&#160;role&#160;in&#160;evo-<br/>lutionary&#160;dynamics.&#160;Indeed,&#160;the&#160;drive&#160;from&#160;a&#160;prokary-<br/>otic&#160;ancestor&#160;to&#160;a&#160;eukaryotic&#160;one&#160;might&#160;have&#160;occurred<br/>based&#160;on&#160;this&#160;trade-off,&#160;however&#160;explaining&#160;such&#160;a&#160;phe-<br/>nomenon&#160;is&#160;outside&#160;the&#160;scope&#160;of&#160;the&#160;current&#160;work.</p>
<p style="position:absolute;top:502px;left:480px;white-space:nowrap" class="ft74">It&#160;is&#160;worth&#160;noting&#160;that&#160;our&#160;results&#160;on&#160;the&#160;‘certainty</p>
<p style="position:absolute;top:520px;left:464px;white-space:nowrap" class="ft78">paradox’&#160;may&#160;be&#160;similar&#160;to&#160;results&#160;showing&#160;a&#160;relationship<br/>between&#160;evolution&#160;and&#160;the&#160;criticality&#160;of&#160;biological&#160;sys-<br/>tems,&#160;wherein&#160;criticality&#160;reflects&#160;how&#160;close&#160;to&#160;the&#160;‘edge&#160;of<br/>chaos’&#160;a&#160;system&#160;is.&#160;Criticality&#160;has&#160;been&#160;shown&#160;in&#160;a&#160;diverse<br/>set&#160;of&#160;biological&#160;systems,&#160;such&#160;as&#160;uniquely&#160;distinguishing<br/>biological&#160;gene&#160;regulatory&#160;networks&#160;<a href="59fe6d0319faf365850c46f352541a9cs.html#12">[25</a>],&#160;or&#160;in&#160;protein<br/>evolution&#160;where&#160;criticality&#160;has&#160;been&#160;linked&#160;to&#160;robust-<br/>ness&#160;[<a href="59fe6d0319faf365850c46f352541a9cs.html#12">26],&#160;</a>and&#160;criticality&#160;is&#160;even&#160;an&#160;outcome&#160;of&#160;evolved<br/>artificial&#160;neural&#160;agents&#160;<a href="59fe6d0319faf365850c46f352541a9cs.html#12">[27</a>].&#160;Although&#160;mathematically<br/>possessing&#160;an&#160;informative&#160;macroscale&#160;is&#160;not&#160;the&#160;same<br/>as&#160;criticality&#160;for&#160;a&#160;biological&#160;system,&#160;they&#160;both&#160;may&#160;be<br/>important&#160;biological&#160;properties&#160;that&#160;drive&#160;evolutionary<br/>adaptations&#160;and&#160;are&#160;worthy&#160;of&#160;further&#160;investigation.</p>
<p style="position:absolute;top:753px;left:480px;white-space:nowrap" class="ft74">While&#160;we&#160;have&#160;illuminated&#160;many&#160;of&#160;the&#160;advantages</p>
<p style="position:absolute;top:771px;left:464px;white-space:nowrap" class="ft78">of&#160;biological&#160;macroscales&#160;and&#160;posited&#160;a&#160;functional&#160;rea-<br/>son&#160;for&#160;their&#160;existence&#160;as&#160;the&#160;solution&#160;to&#160;the&#160;‘certainty<br/>paradox,’&#160;what&#160;are&#160;the&#160;biological&#160;mechanisms&#160;behind<br/>the&#160;evolution&#160;of&#160;multiscale&#160;structure?&#160;We&#160;offer&#160;here&#160;a<br/>few&#160;hypotheses&#160;about&#160;biological&#160;mechanisms&#160;that&#160;are<br/>concordant&#160;with&#160;the&#160;hypothesis&#160;of&#160;multiscale&#160;advantages<br/>in&#160;terms&#160;of&#160;having&#160;both&#160;effectiveness&#160;and&#160;robustness.</p>
<p style="position:absolute;top:897px;left:480px;white-space:nowrap" class="ft74">Notably,&#160;evolution&#160;can&#160;proceed&#160;both&#160;via&#160;neutral&#160;pro-</p>
<p style="position:absolute;top:914px;left:464px;white-space:nowrap" class="ft78">cesses&#160;and&#160;selection-based&#160;contexts.&#160;A&#160;well-known&#160;neu-<br/>tral&#160;process&#160;that&#160;affects&#160;interactions&#160;at&#160;cellular&#160;scale,<br/>such&#160;as&#160;those&#160;between&#160;proteins,&#160;is&#160;presuppression&#160;(also<br/>termed&#160;constructive&#160;neutralism)&#160;[<a href="59fe6d0319faf365850c46f352541a9cs.html#12">20].&#160;</a>This&#160;refers&#160;to&#160;the<br/>complexity&#160;arising&#160;in&#160;the&#160;dependencies&#160;between&#160;inter-<br/>acting&#160;molecules&#160;in&#160;the&#160;absence&#160;of&#160;positive&#160;selection<br/><a href="59fe6d0319faf365850c46f352541a9cs.html#12">[19</a>].&#160;Simply&#160;put,&#160;the&#160;likelihood&#160;of&#160;maintaining&#160;indepen-<br/>dence&#160;between&#160;partners&#160;is&#160;less&#160;than&#160;that&#160;of&#160;moving&#160;away<br/>from&#160;the&#160;original&#160;state&#160;(by&#160;accumulating&#160;changes),&#160;and<br/>therefore&#160;random&#160;changes&#160;can&#160;increase&#160;the&#160;number&#160;of<br/>interactions&#160;between&#160;proteins&#160;in&#160;a&#160;system&#160;by&#160;chance</p>
<p style="position:absolute;top:229px;left:860px;white-space:nowrap" class="ft76">Downloaded from https://academic.oup.com/ib/advance-article/doi/10.1093/intbio/zyab020/6474503 by guest on 22 December 2021</p>
</div>
</body>
</html>
<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="" xml:lang="">
<head>
<title>59fe6d0319faf365850c46f352541a9c-html.html</title>

<meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
 <br/>
<style type="text/css">
<!--
	p {margin: 0; padding: 0;}	.ft80{font-size:9px;font-family:Times;color:#000000;}
	.ft81{font-size:9px;font-family:Times;color:#000000;}
	.ft82{font-size:9px;font-family:Times;color:#000000;}
	.ft83{font-size:11px;font-family:Times;color:#000000;}
	.ft84{font-size:11px;font-family:Times;color:#2e3191;}
	.ft85{font-size:14px;font-family:Times;color:#000000;}
	.ft86{font-size:13px;font-family:Times;color:#000000;}
	.ft87{font-size:11px;font-family:Times;color:#000000;}
	.ft88{font-size:11px;font-family:Times;color:#2e3191;}
	.ft89{font-size:10px;font-family:Times;color:#666666;-moz-transform: matrix(         0,          1,         -1,          0, 0, 0);-webkit-transform: matrix(         0,          1,         -1,          0, 0, 0);-o-transform: matrix(         0,          1,         -1,          0, 0, 0);-ms-transform: matrix(         0,          1,         -1,          0, 0, 0);-moz-transform-origin: left 75%;-webkit-transform-origin: left 75%;-o-transform-origin: left 75%;-ms-transform-origin: left 75%;}
	.ft810{font-size:11px;line-height:17px;font-family:Times;color:#000000;}
	.ft811{font-size:14px;line-height:20px;font-family:Times;color:#000000;}
	.ft812{font-size:13px;line-height:22px;font-family:Times;color:#000000;}
-->
</style>
</head>
<body bgcolor="#A0A0A0" vlink="blue" link="blue">
<div id="page8-div" style="position:relative;width:892px;height:1173px;">
<p style="position:absolute;top:42px;left:64px;white-space:nowrap" class="ft80"><b>8&#160;|&#160;</b><i>Integrative&#160;Biology</i>,&#160;2021</p>
<p style="position:absolute;top:79px;left:63px;white-space:nowrap" class="ft810">alone&#160;and&#160;result&#160;in&#160;‘noisiness’&#160;in&#160;the&#160;interactions.&#160;This<br/>may&#160;offer&#160;a&#160;biological&#160;mechanism&#160;behind&#160;the&#160;result&#160;in<br/>low&#160;effectiveness&#160;in&#160;an&#160;interactome.&#160;Because&#160;eukaryota<br/>have&#160;both&#160;a&#160;larger&#160;number&#160;of&#160;proteins&#160;and&#160;a&#160;higher<br/>substitution&#160;rate&#160;than&#160;bacteria&#160;[<a href="59fe6d0319faf365850c46f352541a9cs.html#12">16],&#160;</a>eukaryotic&#160;interac-<br/>tomes&#160;might&#160;be&#160;expected&#160;to&#160;feature&#160;a&#160;higher&#160;number<br/>of&#160;neutral&#160;processes,&#160;all&#160;of&#160;which&#160;would&#160;combine&#160;to<br/>make&#160;interaction&#160;networks&#160;noisier&#160;and&#160;less&#160;effective.&#160;One<br/>hypothesis&#160;is&#160;that&#160;neutral&#160;evolution&#160;specifically&#160;drives<br/>the&#160;noise&#160;at&#160;the&#160;microscale&#160;but&#160;not&#160;the&#160;macroscale.&#160;At&#160;the<br/>macroscale,&#160;interactomes&#160;would&#160;be&#160;trimmed&#160;and&#160;evolved<br/>under&#160;evolutionary&#160;constraints&#160;and&#160;selective&#160;pressures<br/><a href="59fe6d0319faf365850c46f352541a9cs.html#12">[28</a>],&#160;which&#160;would&#160;eventually&#160;reinforce&#160;beneficial&#160;rela-<br/>tionships,&#160;thinning&#160;out&#160;those&#160;that&#160;can&#160;cause&#160;negative<br/>effects&#160;on&#160;survival&#160;or&#160;growth&#160;[<a href="59fe6d0319faf365850c46f352541a9cs.html#12">20].&#160;</a>These&#160;processes&#160;may<br/>lead&#160;to&#160;formation&#160;of&#160;subgroups&#160;of&#160;proteins&#160;in&#160;the&#160;network<br/>with&#160;more&#160;and&#160;stronger&#160;interactions&#160;within&#160;the&#160;group<br/>compared&#160;with&#160;fewer&#160;or&#160;weaker&#160;interactions&#160;between<br/>those&#160;in&#160;different&#160;subgroups&#160;<a href="59fe6d0319faf365850c46f352541a9cs.html#12">[20</a>,&#160;<a href="59fe6d0319faf365850c46f352541a9cs.html#12">29</a>]—thereby&#160;leading<br/>to&#160;the&#160;emergence&#160;of&#160;modular,&#160;macroscale&#160;structures&#160;in<br/>these&#160;networks,&#160;which&#160;we&#160;hypothesize&#160;to&#160;be&#160;correlated<br/>with&#160;organismal&#160;function&#160;[<a href="59fe6d0319faf365850c46f352541a9cs.html#12">11].</a></p>
<p style="position:absolute;top:474px;left:78px;white-space:nowrap" class="ft83">Another&#160;possible&#160;explanation&#160;as&#160;to&#160;the&#160;biological</p>
<p style="position:absolute;top:492px;left:63px;white-space:nowrap" class="ft810">mechanism&#160;behind&#160;our&#160;observed&#160;results&#160;of&#160;a&#160;decrease&#160;in<br/>effectiveness&#160;is&#160;that&#160;prokaryotes&#160;are&#160;more&#160;metabolically<br/>diverse&#160;than&#160;eukaryotes,&#160;possessing&#160;more&#160;metabolic<br/>processing&#160;pathways&#160;<a href="59fe6d0319faf365850c46f352541a9cs.html#12">[30</a>].&#160;Together&#160;with&#160;changed&#160;usage<br/>patterns&#160;(such&#160;as&#160;carbon&#160;catabolite&#160;repression&#160;in<br/>bacteria),&#160;this&#160;specificity&#160;of&#160;metabolite&#160;processing<br/>reduces&#160;energy&#160;demand&#160;and&#160;allows&#160;for&#160;more&#160;effective<br/>usage&#160;of&#160;resources&#160;<a href="59fe6d0319faf365850c46f352541a9cs.html#12">[31</a>].&#160;These&#160;processes&#160;would&#160;make<br/>biochemical&#160;inputs&#160;and&#160;outputs&#160;more&#160;streamlined<br/>and&#160;efficient&#160;in&#160;prokaryotes,&#160;which&#160;in&#160;turn&#160;should<br/>increase&#160;the&#160;effectiveness&#160;of&#160;their&#160;protein&#160;interactomes,<br/>given&#160;energy&#160;and&#160;genomic&#160;size&#160;constraints&#160;<a href="59fe6d0319faf365850c46f352541a9cs.html#12">[32</a>].&#160;In<br/>contrast,&#160;eukaryota,&#160;as&#160;a&#160;group,&#160;are&#160;less&#160;constrained&#160;by<br/>energy&#160;than&#160;prokaryotes&#160;[<a href="59fe6d0319faf365850c46f352541a9cs.html#12">33]&#160;</a>but&#160;must&#160;contend&#160;with&#160;a<br/>constrained&#160;number&#160;of&#160;metabolites,&#160;channelizing&#160;them<br/>to&#160;perform&#160;cellular&#160;functions&#160;in&#160;morphologically&#160;more<br/>complex&#160;environments&#160;<a href="59fe6d0319faf365850c46f352541a9cs.html#12">[30</a>,&#160;<a href="59fe6d0319faf365850c46f352541a9cs.html#12">33].&#160;</a>Eukaryotic&#160;cells&#160;are<br/>about&#160;three&#160;orders&#160;of&#160;magnitude&#160;larger&#160;than&#160;prokaryotes<br/><a href="59fe6d0319faf365850c46f352541a9cs.html#12">[33</a>],&#160;requiring&#160;more&#160;and&#160;different&#160;sets&#160;of&#160;controls&#160;and<br/>organizational&#160;processes.&#160;Prokaryotes&#160;depend&#160;on&#160;free<br/>diffusion&#160;for&#160;intracellular&#160;transport,&#160;whereas&#160;eukaryota<br/>have&#160;elaborate&#160;mechanisms&#160;for&#160;targeted&#160;transfers&#160;<a href="59fe6d0319faf365850c46f352541a9cs.html#12">[34</a>].<br/>This&#160;reliance&#160;on&#160;cellular&#160;transport&#160;mechanisms&#160;can<br/>lead&#160;to&#160;higher&#160;modular&#160;(and&#160;thus&#160;more&#160;degenerate&#160;or<br/>indeterministic)&#160;structure&#160;in&#160;protein&#160;interactomes&#160;and<br/>other&#160;intracellular&#160;entities,&#160;which,&#160;as&#160;we&#160;show&#160;here,<br/>can&#160;be&#160;associated&#160;with&#160;less&#160;noise&#160;at&#160;higher&#160;scales&#160;of<br/>interaction.&#160;These&#160;higher&#160;scale&#160;inter&#160;module&#160;transfer<br/>mechanisms&#160;ensure&#160;the&#160;proper&#160;and&#160;less&#160;noisy&#160;flow&#160;of<br/>important&#160;molecules&#160;among&#160;these&#160;modules&#160;(such&#160;as<br/>protein&#160;or&#160;metabolite&#160;transport&#160;among&#160;organelles)&#160;<a href="59fe6d0319faf365850c46f352541a9cs.html#12">[11</a>].<br/>Each&#160;of&#160;these&#160;larger&#160;scale&#160;processes,&#160;such&#160;as&#160;transport<br/>among&#160;organelles,&#160;relies&#160;on&#160;only&#160;a&#160;handful&#160;of&#160;inputs<br/>and&#160;outputs&#160;from&#160;outside&#160;its&#160;module,&#160;as&#160;compared&#160;with<br/>much&#160;more&#160;diverse&#160;interactions&#160;within&#160;the&#160;modules</p>
<p style="position:absolute;top:79px;left:455px;white-space:nowrap" class="ft810">themselves&#160;<a href="59fe6d0319faf365850c46f352541a9cs.html#12">[11</a>],&#160;which&#160;arise&#160;due&#160;to&#160;both&#160;functional&#160;and<br/>neutral&#160;processes.&#160;In&#160;terms&#160;of&#160;networks,&#160;this&#160;hierarchical<br/>organizational&#160;structure&#160;is&#160;apt&#160;to&#160;lead&#160;to&#160;a&#160;higher<br/>network&#160;effectiveness&#160;score&#160;at&#160;the&#160;module/process&#160;scale<br/>compared&#160;with&#160;the&#160;microscale.</p>
<p style="position:absolute;top:169px;left:470px;white-space:nowrap" class="ft83">Such&#160;mechanistic&#160;biological&#160;explanations&#160;for&#160;why&#160;we</p>
<p style="position:absolute;top:187px;left:455px;white-space:nowrap" class="ft810">might&#160;observe&#160;these&#160;differences&#160;in&#160;effectiveness&#160;are&#160;in<br/>line&#160;with&#160;the&#160;theoretical&#160;reasoning&#160;that&#160;biological&#160;sys-<br/>tems&#160;need&#160;to&#160;resolve&#160;the&#160;paradox&#160;they&#160;face&#160;at&#160;indi-<br/>vidual&#160;scales&#160;and&#160;therefore&#160;construct&#160;multiscale&#160;struc-<br/>ture.&#160;We&#160;seek&#160;to&#160;tie&#160;the&#160;‘certainty&#160;paradox’&#160;directly&#160;to<br/>the&#160;notion&#160;of&#160;scale&#160;in&#160;biological&#160;systems&#160;and&#160;provide&#160;a<br/>means&#160;for&#160;researchers&#160;to&#160;reduce&#160;the&#160;‘black&#160;box’&#160;nature<br/>of&#160;these&#160;systems&#160;by&#160;searching&#160;across&#160;scales&#160;for&#160;mod-<br/>els&#160;with&#160;low&#160;uncertainty.&#160;Understanding&#160;the&#160;mechan-<br/>ics&#160;of&#160;information&#160;transfer&#160;and&#160;noise&#160;in&#160;biological&#160;sys-<br/>tems,&#160;and&#160;how&#160;they&#160;affect&#160;functionality,&#160;remains&#160;a&#160;major<br/>challenge&#160;in&#160;biology&#160;today.&#160;One&#160;can&#160;imagine&#160;that&#160;the<br/>drive&#160;from&#160;unicellular&#160;to&#160;multicellular&#160;life&#160;was&#160;based<br/>on&#160;some&#160;form&#160;of&#160;similar&#160;trade-offs,&#160;as&#160;those&#160;between<br/>prokaryotes&#160;and&#160;eukaryotes,&#160;that&#160;allowed&#160;multicellular<br/>life&#160;to&#160;operate&#160;via&#160;effective&#160;macro-states&#160;while&#160;reserving<br/>a&#160;pool&#160;of&#160;noise&#160;and&#160;degeneracy.&#160;Thus,&#160;understanding&#160;the<br/>information&#160;structure&#160;of&#160;these&#160;interactomes&#160;lends&#160;us&#160;an<br/>eye&#160;into&#160;the&#160;inner&#160;workings&#160;of&#160;long-term&#160;evolutionary<br/>processes&#160;and&#160;trade-offs&#160;that&#160;might&#160;have&#160;resulted&#160;in&#160;the<br/>two&#160;biggest&#160;phenotypic&#160;splits&#160;in&#160;evolutionary&#160;history—<br/>that&#160;of&#160;prokaryotic&#160;and&#160;eukaryotic&#160;cells,&#160;and&#160;of&#160;uni-<br/>cellular&#160;and&#160;multicellular&#160;life.&#160;We&#160;hope&#160;this&#160;developed<br/>framework&#160;is&#160;applied&#160;to&#160;other&#160;interactomes&#160;and&#160;other<br/>biological&#160;networks,&#160;such&#160;as&#160;gene&#160;regulatory&#160;networks,<br/>or&#160;even&#160;functional&#160;brain&#160;networks,&#160;to&#160;examine&#160;both&#160;how<br/>uncertainty&#160;plays&#160;a&#160;role&#160;in&#160;robustness,&#160;how&#160;informative<br/>higher&#160;scales&#160;change&#160;across&#160;evolution&#160;and&#160;what&#160;funda-<br/>mental&#160;tradeoffs&#160;biological&#160;systems&#160;face.</p>
<p style="position:absolute;top:736px;left:455px;white-space:nowrap" class="ft810"><b>METHODS<br/>Protein&#160;interactomes<br/></b>Protein&#160;interactomes&#160;are&#160;complex&#160;models&#160;of&#160;intracellular<br/>activity,&#160;often&#160;based&#160;on&#160;high-throughput&#160;experiments<br/><a href="59fe6d0319faf365850c46f352541a9cs.html#12">[35</a>,&#160;<a href="59fe6d0319faf365850c46f352541a9cs.html#12">36].&#160;</a>Here&#160;protein&#160;interactomes&#160;formed&#160;from&#160;a&#160;curated<br/>set&#160;of&#160;high-quality&#160;interactions&#160;between&#160;proteins&#160;(&#160;PPIs)<br/>are&#160;taken&#160;from&#160;the&#160;STRING&#160;database&#160;[<a href="59fe6d0319faf365850c46f352541a9cs.html#12">14,&#160;15</a>]&#160;the&#160;curation<br/>of&#160;which&#160;is&#160;outlined&#160;in&#160;Zitnik&#160;<i>et&#160;al</i>.&#160;[<a href="59fe6d0319faf365850c46f352541a9cs.html#12">16].&#160;</a>In&#160;this&#160;curation,<br/>the&#160;STRING&#160;database&#160;(Search&#160;Tool&#160;for&#160;the&#160;Retrieval&#160;of<br/>Interacting&#160;Genes/Proteins,&#160;found&#160;at&#160;<a href="http://string-db.org">http://string-db.org)<br/></a>is&#160;used&#160;to&#160;derive&#160;a&#160;protein&#160;interactome&#160;for&#160;each&#160;species.<br/>Each&#160;PPIs&#160;in&#160;the&#160;protein&#160;interactome&#160;is&#160;an&#160;undirected<br/>edge&#160;where&#160;the&#160;edges&#160;are&#160;based&#160;on&#160;experimentally&#160;doc-<br/>umented&#160;physical&#160;interactions&#160;in&#160;the&#160;species&#160;itself&#160;or&#160;on<br/>human&#160;expert-curated&#160;interactions&#160;(e.g.&#160;no&#160;interactions<br/>are&#160;based&#160;on&#160;text-mining&#160;or&#160;associations).&#160;The&#160;dataset&#160;is<br/>curated&#160;to&#160;only&#160;include&#160;interactions&#160;derived&#160;from&#160;direct<br/>biophysical&#160;PPIs,&#160;metabolic&#160;pathway&#160;interactions,&#160;reg-<br/>ulatory&#160;protein–DNA&#160;interactions&#160;and&#160;kinase–substrate<br/>interactions.&#160;The&#160;details&#160;of&#160;the&#160;curation&#160;of&#160;these&#160;interac-<br/>tomes&#160;can&#160;be&#160;found&#160;in&#160;<a href="59fe6d0319faf365850c46f352541a9cs.html#12">Zitnik&#160;<i>et&#160;al</i>.&#160;[16</a>].</p>
<p style="position:absolute;top:229px;left:860px;white-space:nowrap" class="ft89">Downloaded from https://academic.oup.com/ib/advance-article/doi/10.1093/intbio/zyab020/6474503 by guest on 22 December 2021</p>
</div>
</body>
</html>
<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="" xml:lang="">
<head>
<title>59fe6d0319faf365850c46f352541a9c-html.html</title>

<meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
 <br/>
<style type="text/css">
<!--
	p {margin: 0; padding: 0;}	.ft90{font-size:9px;font-family:Times;color:#000000;}
	.ft91{font-size:9px;font-family:Times;color:#000000;}
	.ft92{font-size:11px;font-family:Times;color:#000000;}
	.ft93{font-size:11px;font-family:Times;color:#000000;}
	.ft94{font-size:11px;font-family:Times;color:#2e3191;}
	.ft95{font-size:11px;font-family:Times;color:#2e3191;}
	.ft96{font-size:13px;font-family:Times;color:#000000;}
	.ft97{font-size:7px;font-family:Times;color:#000000;}
	.ft98{font-size:7px;font-family:Times;color:#000000;}
	.ft99{font-size:10px;font-family:Times;color:#666666;-moz-transform: matrix(         0,          1,         -1,          0, 0, 0);-webkit-transform: matrix(         0,          1,         -1,          0, 0, 0);-o-transform: matrix(         0,          1,         -1,          0, 0, 0);-ms-transform: matrix(         0,          1,         -1,          0, 0, 0);-moz-transform-origin: left 75%;-webkit-transform-origin: left 75%;-o-transform-origin: left 75%;-ms-transform-origin: left 75%;}
	.ft910{font-size:11px;line-height:17px;font-family:Times;color:#000000;}
	.ft911{font-size:11px;line-height:17px;font-family:Times;color:#2e3191;}
	.ft912{font-size:13px;line-height:16px;font-family:Times;color:#000000;}
	.ft913{font-size:13px;line-height:22px;font-family:Times;color:#000000;}
	.ft914{font-size:11px;line-height:17px;font-family:Times;color:#000000;}
-->
</style>
</head>
<body bgcolor="#A0A0A0" vlink="blue" link="blue">
<div id="page9-div" style="position:relative;width:892px;height:1173px;">
<p style="position:absolute;top:42px;left:194px;white-space:nowrap" class="ft90">Evolution&#160;and&#160;emergence:&#160;higher&#160;order&#160;information&#160;structure&#160;in&#160;protein&#160;interactomes&#160;across&#160;the&#160;tree&#160;of&#160;life&#160;<b>|&#160;9</b></p>
<p style="position:absolute;top:79px;left:88px;white-space:nowrap" class="ft92">The&#160;evolutionary&#160;history&#160;of&#160;the&#160;set&#160;of&#160;PPIs&#160;was&#160;obtained</p>
<p style="position:absolute;top:97px;left:72px;white-space:nowrap" class="ft910">by&#160;Zitnik&#160;<i>et&#160;al</i>.&#160;<a href="59fe6d0319faf365850c46f352541a9cs.html#12">[16</a>]&#160;and&#160;is&#160;derived&#160;from&#160;a&#160;high-resolution<br/>phylogenetic&#160;tree&#160;[<a href="59fe6d0319faf365850c46f352541a9cs.html#12">23].&#160;</a>The&#160;tree&#160;is&#160;composed&#160;of&#160;archaea,<br/>bacteria&#160;and&#160;eukaryota&#160;and&#160;captures&#160;a&#160;diversity&#160;of<br/>species&#160;in&#160;each&#160;lineage.&#160;The&#160;phylogenetic&#160;tree&#160;is&#160;used&#160;to<br/>characterize&#160;the&#160;evolution&#160;of&#160;each&#160;species&#160;based&#160;on&#160;the<br/>total&#160;branch&#160;length&#160;(which&#160;takes&#160;the&#160;form&#160;of&#160;nucleotide<br/>substitutions&#160;per&#160;site)&#160;from&#160;the&#160;root&#160;of&#160;the&#160;tree&#160;to&#160;the&#160;leaf<br/>of&#160;the&#160;species.&#160;The&#160;phylogenetic&#160;taxonomy,&#160;the&#160;names&#160;of<br/>species&#160;and&#160;lineages&#160;of&#160;each&#160;species&#160;were&#160;taken&#160;from<br/>the&#160;NCBI&#160;Taxonomy&#160;database&#160;[<a href="59fe6d0319faf365850c46f352541a9cs.html#12">37].&#160;</a>Details&#160;of&#160;how&#160;this<br/>is&#160;associated&#160;with&#160;each&#160;species&#160;can&#160;be&#160;found&#160;at&#160;<a href="http://snap.stanford.edu/tree-of-life">(http://<br/>snap.stanford.edu/tree-of-life),&#160;</a>and&#160;we&#160;refer&#160;to&#160;<a href="59fe6d0319faf365850c46f352541a9cs.html#12">Zitnik<br/></a><i>et&#160;al</i>.&#160;[16]&#160;for&#160;further&#160;specifics&#160;on&#160;how&#160;each&#160;species<br/>was&#160;assigned&#160;an&#160;average&#160;nucleotide&#160;substitution&#160;rate.<br/>Ultimately&#160;these&#160;protein&#160;interactomes&#160;are&#160;incomplete<br/>models&#160;that&#160;may&#160;change&#160;as&#160;time&#160;goes&#160;on.&#160;Because&#160;we<br/>do&#160;not&#160;wish&#160;to&#160;bias&#160;our&#160;results,&#160;our&#160;statistical&#160;analyses<br/>were&#160;performed&#160;only&#160;over&#160;the&#160;interactomes&#160;of&#160;the&#160;species<br/>based&#160;on&#160;<i>&gt;</i>1000&#160;citations&#160;in&#160;the&#160;literature.</p>
<p style="position:absolute;top:453px;left:72px;white-space:nowrap" class="ft910"><b>Spectral&#160;analysis&#160;for&#160;identifying&#160;causal<br/>emergence<br/></b>Spectral&#160;methods&#160;have&#160;proved&#160;to&#160;be&#160;successful&#160;in&#160;iden-<br/>tifying&#160;good&#160;graph&#160;partitions&#160;in&#160;a&#160;wide&#160;variety&#160;of&#160;appli-<br/>cations&#160;[<a href="59fe6d0319faf365850c46f352541a9cs.html#12">38].&#160;</a>Given&#160;an&#160;undirected&#160;network,&#160;we&#160;take&#160;the<br/>degree&#160;normalized&#160;adjacency&#160;matrix&#160;<i>A&#160;</i>and&#160;compute&#160;the<br/>eigendecomposition&#160;<i>A&#160;</i>=&#160;<i>EΛE</i>T,&#160;where&#160;the&#160;<i>i</i>th&#160;column&#160;of&#160;<i>E<br/></i>is&#160;the&#160;normalized&#160;eigenvector&#160;corresponding&#160;to&#160;the&#160;<i>i</i>th<br/>eigenvalue,&#160;and&#160;<i>Λ&#160;</i>is&#160;the&#160;matrix&#160;with&#160;the&#160;<i>i</i>th&#160;eigenvalue<br/>on&#160;the&#160;<i>i</i>th&#160;diagonal&#160;and&#160;zeros&#160;elsewhere.&#160;The&#160;eigenvector<br/>matrix&#160;<i>E&#160;</i>contains&#160;rich&#160;information&#160;about&#160;the&#160;structure<br/>of&#160;the&#160;network,&#160;including&#160;information&#160;about&#160;the&#160;optimal<br/>scale&#160;of&#160;a&#160;network.&#160;The&#160;rows&#160;of&#160;<i>E&#160;</i>correspond&#160;to&#160;nodes<br/>in&#160;the&#160;network,&#160;so&#160;we&#160;construct&#160;a&#160;vector&#160;representation<br/>of&#160;each&#160;node’s&#160;contribution&#160;to&#160;the&#160;network&#160;topology&#160;by<br/>weighting&#160;the&#160;columns&#160;of&#160;<i>E&#160;</i>by&#160;their&#160;corresponding&#160;eigen-<br/>values,&#160;removing&#160;columns&#160;that&#160;correspond&#160;to&#160;null&#160;eigen-<br/>values,&#160;and&#160;associating&#160;the&#160;resulting&#160;row&#160;vectors&#160;with<br/>the&#160;nodes&#160;of&#160;the&#160;network.&#160;We&#160;construct&#160;a&#160;distance&#160;met-<br/>ric&#160;that&#160;reflects&#160;similarity&#160;in&#160;causal&#160;structure&#160;between<br/>pairs&#160;of&#160;nodes&#160;by&#160;taking&#160;the&#160;cosine&#160;similarity&#160;between<br/>the&#160;vectors&#160;corresponding&#160;to&#160;nodes.&#160;If&#160;a&#160;pair&#160;of&#160;nodes<br/>are&#160;not&#160;in&#160;each&#160;other’s&#160;Markov&#160;blankets,&#160;coarse&#160;graining<br/>them&#160;together&#160;cannot&#160;increase&#160;the&#160;<i>EI</i>,&#160;so&#160;we&#160;define&#160;the<br/>distance&#160;between&#160;them&#160;to&#160;be</p>
<p style="position:absolute;top:884px;left:275px;white-space:nowrap" class="ft92">∞&#160;(or&#160;simply&#160;very&#160;large,</p>
<p style="position:absolute;top:904px;left:72px;white-space:nowrap" class="ft910">in&#160;this&#160;case,&#160;1000).&#160;We&#160;use&#160;this&#160;metric&#160;to&#160;cluster&#160;the<br/>nodes&#160;of&#160;the&#160;network&#160;using&#160;the&#160;OPTICS&#160;algorithm&#160;[<a href="59fe6d0319faf365850c46f352541a9cs.html#12">39]<br/></a>which&#160;we&#160;can&#160;interpret&#160;as&#160;a&#160;coarse&#160;graining&#160;to&#160;construct<br/>a&#160;macroscale&#160;network,&#160;where&#160;micro-nodes&#160;are&#160;placed&#160;in<br/>the&#160;same&#160;macro-node&#160;if&#160;they&#160;are&#160;placed&#160;in&#160;the&#160;same&#160;clus-<br/>ter.&#160;Note&#160;that&#160;this&#160;method&#160;for&#160;detecting&#160;causal&#160;emergence<br/>in&#160;networks&#160;is&#160;explored&#160;in&#160;detail&#160;in&#160;other&#160;sources&#160;<a href="59fe6d0319faf365850c46f352541a9cs.html#12">[13</a>].</p>
<p style="position:absolute;top:1045px;left:72px;white-space:nowrap" class="ft910"><b>Robustness&#160;of&#160;causal&#160;emergence&#160;differences<br/>across&#160;species<br/></b>To&#160;ensure&#160;that&#160;the&#160;differences&#160;observed&#160;in&#160;the&#160;causal<br/>emergence&#160;values&#160;of&#160;the&#160;PPI&#160;networks&#160;were&#160;not&#160;merely&#160;a</p>
<p style="position:absolute;top:79px;left:464px;white-space:nowrap" class="ft910">statistical&#160;artifact,&#160;we&#160;conducted&#160;a&#160;series&#160;of&#160;robustness<br/>tests&#160;of&#160;our&#160;analysis.&#160;These&#160;tests&#160;were&#160;necessary&#160;for<br/>two&#160;key&#160;reasons.&#160;First,&#160;the&#160;nature&#160;of&#160;interaction&#160;data&#160;in<br/>biology&#160;is&#160;inherently&#160;difficult&#160;to&#160;obtain.&#160;Although&#160;many&#160;of<br/>the&#160;tools&#160;we&#160;use&#160;to&#160;collect,&#160;clean&#160;and&#160;interpret&#160;biological<br/>systems&#160;are&#160;sophisticated,&#160;they&#160;are&#160;nonetheless&#160;subject<br/>to&#160;potential&#160;biases.&#160;However,&#160;if&#160;there&#160;were&#160;systematic<br/>biases&#160;in&#160;the&#160;network&#160;construction&#160;process&#160;for&#160;the&#160;pro-<br/>tein&#160;interactomes&#160;used&#160;in&#160;this&#160;study&#160;(for&#160;example,&#160;if<br/>the&#160;interaction&#160;networks&#160;of&#160;eukaryotic&#160;species&#160;system-<br/>atically&#160;over&#160;estimated&#160;certain&#160;interactions),&#160;randomiza-<br/>tion&#160;procedures&#160;should&#160;clarify&#160;the&#160;extent&#160;to&#160;which&#160;the<br/><a href="59fe6d0319faf365850c46f352541a9cs.html#12">results&#160;</a>we&#160;observed&#160;are&#160;truly&#160;a&#160;property&#160;of&#160;the&#160;species<br/>themselves.</p>
<p style="position:absolute;top:330px;left:480px;white-space:nowrap" class="ft92">Second,&#160;these&#160;robustness&#160;tests&#160;offer&#160;insights&#160;into</p>
<p style="position:absolute;top:348px;left:464px;white-space:nowrap" class="ft910">whether&#160;there&#160;is&#160;anything&#160;intrinsic&#160;to&#160;the&#160;network<br/>structures&#160;of&#160;the&#160;eukaryotic&#160;or&#160;prokaryotic&#160;species&#160;that<br/>could&#160;be&#160;contributing&#160;to&#160;their&#160;causal&#160;emergence&#160;values.<br/>For&#160;example,&#160;the&#160;protein-interaction&#160;networks&#160;of&#160;the<br/>eukaryote,&#160;<i>Rattus&#160;norvegicus&#160;</i>(the&#160;common&#160;sewer&#160;rat),<br/>have&#160;a&#160;certain&#160;amount&#160;of&#160;causal&#160;emergence.&#160;Would&#160;an<br/>arbitrary,&#160;simulated&#160;network&#160;with&#160;the&#160;same&#160;number<br/>of&#160;nodes&#160;and&#160;edges,&#160;connected&#160;randomly,&#160;also&#160;have&#160;a<br/>similar&#160;amount&#160;of&#160;causal&#160;emergence?&#160;By&#160;performing<br/>a&#160;series&#160;of&#160;robustness&#160;tests&#160;on&#160;the&#160;protein&#160;interaction<br/>networks&#160;in&#160;our&#160;study,&#160;we&#160;can&#160;get&#160;closer&#160;to&#160;the&#160;question<br/>of&#160;whether&#160;or&#160;not&#160;there&#160;is&#160;anything&#160;intrinsic&#160;to&#160;the&#160;protein<br/>interaction&#160;network&#160;of&#160;<i>R.&#160;norvegicus</i>,&#160;or&#160;any&#160;other&#160;species,<br/>that&#160;makes&#160;it&#160;particularly&#160;prone&#160;to&#160;displaying&#160;higher<br/>scale&#160;informative&#160;structures?</p>
<p style="position:absolute;top:617px;left:480px;white-space:nowrap" class="ft92">To&#160;address&#160;the&#160;two&#160;concerns&#160;above,&#160;we&#160;performed&#160;two</p>
<p style="position:absolute;top:635px;left:464px;white-space:nowrap" class="ft910">separate&#160;but&#160;similar&#160;robustness&#160;tests.&#160;The&#160;first&#160;uses&#160;a<br/>network&#160;null&#160;model&#160;known&#160;as&#160;the&#160;configuration&#160;model&#160;to<br/>randomize&#160;the&#160;connectivity&#160;of&#160;the&#160;protein&#160;interactomes<br/>while&#160;also&#160;preserving&#160;the&#160;number&#160;of&#160;nodes,&#160;edges&#160;and<br/>distribution&#160;of&#160;node&#160;degree&#160;<a href="59fe6d0319faf365850c46f352541a9cs.html#12">[40</a>].&#160;The&#160;second&#160;robustness<br/>test&#160;involves&#160;random&#160;edge&#160;rewiring&#160;[<a href="59fe6d0319faf365850c46f352541a9cs.html#12">41].&#160;</a>For&#160;each&#160;network<br/>in&#160;our&#160;study,&#160;we&#160;iteratively&#160;increased&#160;the&#160;fraction&#160;of<br/>random&#160;edges&#160;to&#160;rewire&#160;in&#160;the&#160;network;&#160;an&#160;edge,&#160;<i>eij</i>,&#160;that<br/>connects&#160;nodes&#160;<i>vi&#160;</i>and&#160;<i>vj</i>,&#160;becomes&#160;reconnected&#160;to&#160;a&#160;new<br/>node,&#160;<i>vk</i>,&#160;forming&#160;a&#160;new&#160;edge,&#160;<i>eik</i>,&#160;instead&#160;of&#160;the&#160;original&#160;<i>eij</i>.<br/>We&#160;do&#160;this&#160;with&#160;an&#160;iteratively&#160;increasing&#160;fraction&#160;of&#160;edges,<br/>starting&#160;with&#160;1%&#160;of&#160;edges&#160;and&#160;increasing&#160;until&#160;100%&#160;of&#160;the<br/>network’s&#160;edges&#160;are&#160;rewired.</p>
<p style="position:absolute;top:868px;left:480px;white-space:nowrap" class="ft92">If&#160;the&#160;causal&#160;emergence&#160;values&#160;of&#160;the&#160;networks</p>
<p style="position:absolute;top:886px;left:464px;white-space:nowrap" class="ft910">in&#160;this&#160;study&#160;decrease&#160;following&#160;the&#160;robustness&#160;tests<br/>above—and&#160;in&#160;particular,&#160;if&#160;they&#160;decrease&#160;differently<br/>for&#160;eukaryota&#160;and&#160;prokaryotes—then&#160;the&#160;differences&#160;we<br/>observe&#160;are&#160;unlikely&#160;to&#160;have&#160;arisen&#160;simply&#160;from&#160;chance,<br/>noisy/biased&#160;data,&#160;or&#160;otherwise&#160;coincidental,&#160;<i>ad&#160;hoc<br/></i>network&#160;properties.&#160;Instead,&#160;our&#160;testing&#160;of&#160;the&#160;robustness<br/>of&#160;our&#160;analysis&#160;lends&#160;credence&#160;to&#160;the&#160;main&#160;finding&#160;of<br/>this&#160;paper,&#160;which&#160;is&#160;that&#160;species&#160;that&#160;emerged&#160;later&#160;in<br/>evolutionary&#160;time&#160;are&#160;associated&#160;with&#160;more&#160;informative<br/>macroscale&#160;protein&#160;interaction&#160;networks.</p>
<p style="position:absolute;top:1066px;left:480px;white-space:nowrap" class="ft92">In&#160;<a href="59fe6d0319faf365850c46f352541a9cs.html#7">Figure&#160;4A</a>,&#160;we&#160;show&#160;how&#160;the&#160;causal&#160;emergence</p>
<p style="position:absolute;top:1083px;left:464px;white-space:nowrap" class="ft910">of&#160;archaea,&#160;bacteria&#160;and&#160;eukaryota&#160;interactomes&#160;all<br/>decreases&#160;as&#160;a&#160;higher&#160;and&#160;higher&#160;amount&#160;of&#160;network</p>
<p style="position:absolute;top:229px;left:860px;white-space:nowrap" class="ft99">Downloaded from https://academic.oup.com/ib/advance-article/doi/10.1093/intbio/zyab020/6474503 by guest on 22 December 2021</p>
</div>
</body>
</html>
<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="" xml:lang="">
<head>
<title>59fe6d0319faf365850c46f352541a9c-html.html</title>

<meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
 <br/>
<style type="text/css">
<!--
	p {margin: 0; padding: 0;}	.ft100{font-size:9px;font-family:Times;color:#000000;}
	.ft101{font-size:9px;font-family:Times;color:#000000;}
	.ft102{font-size:9px;font-family:Times;color:#000000;}
	.ft103{font-size:8px;font-family:Times;color:#000000;}
	.ft104{font-size:8px;font-family:Times;color:#000000;}
	.ft105{font-size:8px;font-family:Times;color:#000000;}
	.ft106{font-size:5px;font-family:Times;color:#000000;}
	.ft107{font-size:4px;font-family:Times;color:#000000;}
	.ft108{font-size:5px;font-family:Times;color:#000000;}
	.ft109{font-size:4px;font-family:Times;color:#000000;}
	.ft1010{font-size:11px;font-family:Times;color:#000000;}
	.ft1011{font-size:11px;font-family:Times;color:#2e3191;}
	.ft1012{font-size:11px;font-family:Times;color:#000000;}
	.ft1013{font-size:7px;font-family:Times;color:#000000;}
	.ft1014{font-size:10px;font-family:Times;color:#666666;-moz-transform: matrix(         0,          1,         -1,          0, 0, 0);-webkit-transform: matrix(         0,          1,         -1,          0, 0, 0);-o-transform: matrix(         0,          1,         -1,          0, 0, 0);-ms-transform: matrix(         0,          1,         -1,          0, 0, 0);-moz-transform-origin: left 75%;-webkit-transform-origin: left 75%;-o-transform-origin: left 75%;-ms-transform-origin: left 75%;}
	.ft1015{font-size:8px;line-height:13px;font-family:Times;color:#000000;}
	.ft1016{font-size:5px;line-height:13px;font-family:Times;color:#000000;}
	.ft1017{font-size:8px;line-height:13px;font-family:Times;color:#000000;}
	.ft1018{font-size:8px;line-height:16px;font-family:Times;color:#000000;}
	.ft1019{font-size:11px;line-height:17px;font-family:Times;color:#000000;}
-->
</style>
</head>
<body bgcolor="#A0A0A0" vlink="blue" link="blue">
<div id="page10-div" style="position:relative;width:892px;height:1173px;">
<p style="position:absolute;top:42px;left:64px;white-space:nowrap" class="ft100"><b>10&#160;|&#160;</b><i>Integrative&#160;Biology</i>,&#160;2021</p>
<p style="position:absolute;top:535px;left:63px;white-space:nowrap" class="ft1018"><b>Figure&#160;5.&#160;</b>Schematic&#160;of&#160;edge&#160;reweighting&#160;procedure.&#160;Consider&#160;a&#160;network,&#160;<i>G&#160;</i>(A),&#160;and&#160;its&#160;transition&#160;probability&#160;matrix,&#160;<i>W&#160;</i>(A).&#160;Each&#160;element&#160;<i>wij&#160;</i>of&#160;<i>W<br/></i>corresponds&#160;to&#160;the&#160;probability&#160;that&#160;a&#160;random&#160;walker&#160;on&#160;node&#160;<i>vi&#160;</i>will&#160;transition&#160;to&#160;node&#160;<i>vj&#160;</i>at&#160;the&#160;following&#160;timestep;&#160;for&#160;each&#160;node&#160;<i>vi</i>,&#160;<i>wij&#160;</i>is&#160;1/<i>ki</i>,&#160;where&#160;<i>ki<br/></i>is&#160;the&#160;out-degree&#160;of&#160;node&#160;<i>vi</i>.&#160;(C,&#160;D,&#160;E)&#160;Examples&#160;of&#160;the&#160;transition&#160;probabilities&#160;(i.e.&#160;the&#160;rows&#160;of&#160;<i>W</i>)&#160;for&#160;three&#160;nodes&#160;in&#160;<i>G</i>.&#160;The&#160;edge&#160;reweighting&#160;procedure<br/>introduced&#160;here&#160;involves&#160;two&#160;steps:&#160;(F)&#160;First,&#160;we&#160;select&#160;a&#160;row&#160;of&#160;<i>W&#160;</i>and&#160;add&#160;small&#160;uniform&#160;noise&#160;to&#160;it.&#160;Next,&#160;we&#160;apply&#160;a&#160;softmax&#160;function&#160;to&#160;the&#160;noisy<br/>vector,&#160;which&#160;is&#160;defined&#160;as&#160;<i>σ</i></p>
<p style="position:absolute;top:578px;left:206px;white-space:nowrap" class="ft104"></p>
<p style="position:absolute;top:592px;left:212px;white-space:nowrap" class="ft105"><i>Wi</i></p>
<p style="position:absolute;top:578px;left:225px;white-space:nowrap" class="ft104"></p>
<p style="position:absolute;top:590px;left:234px;white-space:nowrap" class="ft104">=</p>
<p style="position:absolute;top:590px;left:257px;white-space:nowrap" class="ft106"><i>e</i></p>
<p style="position:absolute;top:588px;left:261px;white-space:nowrap" class="ft107"><i>θ</i></p>
<p style="position:absolute;top:587px;left:264px;white-space:nowrap" class="ft107"><i>Wi</i></p>
<p style="position:absolute;top:594px;left:247px;white-space:nowrap" class="ft108">	<i>n</i></p>
<p style="position:absolute;top:605px;left:255px;white-space:nowrap" class="ft107"><i>j</i></p>
<p style="position:absolute;top:604px;left:257px;white-space:nowrap" class="ft109">=1&#160;<i>e</i></p>
<p style="position:absolute;top:598px;left:271px;white-space:nowrap" class="ft107"><i>θ</i></p>
<p style="position:absolute;top:597px;left:274px;white-space:nowrap" class="ft107"><i>wij&#160;</i>.&#160;In&#160;(G,&#160;H,&#160;I)&#160;we&#160;show&#160;how&#160;the&#160;output&#160;of&#160;a&#160;softmax&#160;function&#160;relies&#160;on&#160;the&#160;<i>θ&#160;</i>parameter—as&#160;<i>θ&#160;</i>increases,&#160;the</p>
<p style="position:absolute;top:613px;left:63px;white-space:nowrap" class="ft1017">slightest&#160;differences&#160;become&#160;accentuated&#160;in&#160;the&#160;resulting&#160;value,&#160;which&#160;we&#160;show&#160;using&#160;the&#160;noisy&#160;<i>W0&#160;</i>vector&#160;and&#160;three&#160;different&#160;values&#160;of&#160;<i>θ</i>.&#160;(J)&#160;We<br/>iteratively&#160;span&#160;values&#160;of&#160;<i>θ</i>,&#160;and&#160;we&#160;are&#160;able&#160;to&#160;approximate&#160;the&#160;causal&#160;emergence&#160;of&#160;networks&#160;with&#160;(tunable)&#160;non-uniform&#160;edge&#160;weights;&#160;here&#160;we&#160;plot<br/>the&#160;resulting&#160;causal&#160;emergence&#160;values&#160;as&#160;a&#160;percentage&#160;of&#160;the&#160;original&#160;causal&#160;emergence.&#160;Error&#160;bands&#160;are&#160;95%&#160;confidence&#160;intervals.</p>
<p style="position:absolute;top:683px;left:63px;white-space:nowrap" class="ft1019">edges&#160;are&#160;rewired,&#160;indicating&#160;that&#160;random&#160;rewiring&#160;has&#160;a<br/>similar&#160;effect&#160;on&#160;all&#160;datasets.&#160;This&#160;analysis&#160;suggests&#160;that<br/>if&#160;there&#160;were&#160;significant&#160;noise&#160;in&#160;the&#160;network&#160;data&#160;itself<br/>(i.e.&#160;connections&#160;between&#160;proteins&#160;where&#160;there&#160;otherwise<br/>should&#160;not&#160;be&#160;or&#160;a&#160;lack&#160;of&#160;connections&#160;where&#160;there<br/>should&#160;be),&#160;we&#160;should&#160;not&#160;expect&#160;to&#160;see&#160;the&#160;magnitude<br/>of&#160;causal&#160;emergence&#160;values&#160;that&#160;we&#160;indeed&#160;do&#160;see.<br/>This&#160;adds&#160;evidence&#160;that&#160;the&#160;inherent&#160;noise&#160;in&#160;the&#160;data<br/>collection&#160;process&#160;is&#160;not&#160;sufficient&#160;to&#160;produce&#160;the&#160;results<br/>we&#160;see.</p>
<p style="position:absolute;top:862px;left:78px;white-space:nowrap" class="ft1010">In&#160;<a href="59fe6d0319faf365850c46f352541a9cs.html#7">Figure&#160;4B</a>,&#160;we&#160;show&#160;that&#160;random&#160;null&#160;models&#160;of</p>
<p style="position:absolute;top:880px;left:63px;white-space:nowrap" class="ft1019">the&#160;networks&#160;used&#160;in&#160;this&#160;study&#160;are&#160;characteristically<br/>unlikely&#160;to&#160;have&#160;values&#160;for&#160;causal&#160;emergence&#160;values&#160;that<br/>are&#160;at&#160;all&#160;similar&#160;to&#160;the&#160;original&#160;interactomes.&#160;On&#160;the<br/>contrary,&#160;the&#160;maximum&#160;average&#160;causal&#160;emergence&#160;value<br/>for&#160;any&#160;of&#160;the&#160;networks&#160;used&#160;here&#160;reaches&#160;only&#160;3%&#160;of&#160;the<br/>original&#160;network’s&#160;values.&#160;This&#160;suggests&#160;that&#160;random&#160;null<br/>models&#160;of&#160;networks&#160;are&#160;less&#160;likely&#160;to&#160;contain&#160;higher&#160;scale<br/>structure&#160;but&#160;also&#160;that&#160;the&#160;observed&#160;differences&#160;in&#160;the<br/>causal&#160;emergence&#160;values&#160;for&#160;prokaryotic&#160;and&#160;eukaryotic<br/>species&#160;are&#160;unlikely&#160;to&#160;be&#160;driven&#160;merely&#160;due&#160;to&#160;basic<br/>properties&#160;like&#160;their&#160;edge&#160;density&#160;or&#160;degree&#160;distribution.</p>
<p style="position:absolute;top:1077px;left:78px;white-space:nowrap" class="ft1010">Lastly,&#160;<i>EI&#160;</i>is&#160;sensitive&#160;not&#160;only&#160;to&#160;the&#160;network&#160;structure</p>
<p style="position:absolute;top:1095px;left:63px;white-space:nowrap" class="ft1010">but&#160;also&#160;to&#160;the&#160;distribution&#160;of&#160;edge&#160;weights&#160;within&#160;the</p>
<p style="position:absolute;top:683px;left:455px;white-space:nowrap" class="ft1019">network.&#160;However,&#160;we&#160;do&#160;not&#160;have&#160;edge&#160;weights&#160;data&#160;for<br/>the&#160;particular&#160;networks&#160;included&#160;in&#160;this&#160;work;&#160;as&#160;such,<br/>we&#160;assign&#160;uniform&#160;weights&#160;of&#160;1/<i>ki&#160;</i>to&#160;the&#160;<i>ki&#160;</i>edges&#160;of<br/>node&#160;<i>vi</i>,&#160;which&#160;means&#160;that&#160;edge&#160;weights&#160;correspond&#160;to<br/>the&#160;probability&#160;that&#160;a&#160;random&#160;walker&#160;will&#160;traverse&#160;from<br/>node&#160;<i>vi&#160;</i>to&#160;node&#160;<i>vj&#160;</i>in&#160;the&#160;next&#160;time&#160;step.&#160;This&#160;assumption<br/>likely&#160;distorts&#160;the&#160;true-&#160;(likely&#160;non-uniform)&#160;weighted<br/>interaction&#160;patterns&#160;between&#160;proteins.&#160;This&#160;could&#160;affect<br/>the&#160;main&#160;results&#160;of&#160;this&#160;work,&#160;and&#160;as&#160;such,&#160;we&#160;devised<br/>an&#160;edge&#160;weight&#160;randomization&#160;procedure&#160;that&#160;allows&#160;us<br/>to&#160;systematically&#160;vary&#160;the&#160;shape&#160;of&#160;the&#160;distribution&#160;of<br/>edge&#160;weights&#160;in&#160;the&#160;networks&#160;we&#160;study.&#160;That&#160;is,&#160;whereas<br/>we&#160;usually&#160;assume&#160;that&#160;nodes&#160;have&#160;‘flat’&#160;edge&#160;weights<br/>(uniform,&#160;summing&#160;to&#160;1.0)&#160;to&#160;neighboring&#160;nodes,&#160;under<br/>the&#160;edge&#160;weight&#160;randomization,&#160;we&#160;can&#160;create&#160;versions<br/>of&#160;a&#160;given&#160;network&#160;where&#160;nodes’&#160;edge&#160;weights&#160;distribu-<br/>tions&#160;are&#160;more&#160;or&#160;less&#160;heavy-tailed.&#160;Take,&#160;for&#160;example,<br/>a&#160;node&#160;connected&#160;to&#160;four&#160;other&#160;nodes;&#160;originally,&#160;its<br/>vector&#160;of&#160;edge&#160;weights&#160;would&#160;be&#160;[0.25,&#160;0.25,&#160;0.25,&#160;0.25].<br/>The&#160;procedure&#160;introduced&#160;here&#160;allows&#160;us&#160;to&#160;continuously<br/>vary&#160;the&#160;shape&#160;of&#160;the&#160;edge&#160;weight&#160;distribution&#160;to&#160;some-<br/>thing&#160;more&#160;heavy-tailed,&#160;such&#160;as&#160;[0.025,&#160;0.05,&#160;0.025,&#160;0.9].<br/>A&#160;schematic&#160;of&#160;the&#160;randomization&#160;procedure&#160;is&#160;shown<br/>in&#160;<a href="59fe6d0319faf365850c46f352541a9cs.html#10">Figure&#160;5,&#160;</a>the&#160;results&#160;of&#160;which&#160;are&#160;shown&#160;in&#160;<a href="59fe6d0319faf365850c46f352541a9cs.html#11">Figure&#160;6</a></p>
<p style="position:absolute;top:229px;left:860px;white-space:nowrap" class="ft1014">Downloaded from https://academic.oup.com/ib/advance-article/doi/10.1093/intbio/zyab020/6474503 by guest on 22 December 2021</p>
</div>
</body>
</html>
<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="" xml:lang="">
<head>
<title>59fe6d0319faf365850c46f352541a9c-html.html</title>

<meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
 <br/>
<style type="text/css">
<!--
	p {margin: 0; padding: 0;}	.ft110{font-size:9px;font-family:Times;color:#000000;}
	.ft111{font-size:9px;font-family:Times;color:#000000;}
	.ft112{font-size:8px;font-family:Times;color:#000000;}
	.ft113{font-size:8px;font-family:Times;color:#000000;}
	.ft114{font-size:11px;font-family:Times;color:#000000;}
	.ft115{font-size:11px;font-family:Times;color:#000000;}
	.ft116{font-size:7px;font-family:Times;color:#000000;}
	.ft117{font-size:7px;font-family:Times;color:#000000;}
	.ft118{font-size:11px;font-family:Times;color:#2e3191;}
	.ft119{font-size:14px;font-family:Times;color:#000000;}
	.ft1110{font-size:9px;font-family:Times;color:#2e3191;}
	.ft1111{font-size:9px;font-family:Times;color:#000000;}
	.ft1112{font-size:10px;font-family:Times;color:#666666;-moz-transform: matrix(         0,          1,         -1,          0, 0, 0);-webkit-transform: matrix(         0,          1,         -1,          0, 0, 0);-o-transform: matrix(         0,          1,         -1,          0, 0, 0);-ms-transform: matrix(         0,          1,         -1,          0, 0, 0);-moz-transform-origin: left 75%;-webkit-transform-origin: left 75%;-o-transform-origin: left 75%;-ms-transform-origin: left 75%;}
	.ft1113{font-size:8px;line-height:13px;font-family:Times;color:#000000;}
	.ft1114{font-size:11px;line-height:17px;font-family:Times;color:#000000;}
	.ft1115{font-size:7px;line-height:10px;font-family:Times;color:#000000;}
	.ft1116{font-size:11px;line-height:19px;font-family:Times;color:#000000;}
	.ft1117{font-size:7px;line-height:17px;font-family:Times;color:#000000;}
	.ft1118{font-size:9px;line-height:16px;font-family:Times;color:#000000;}
-->
</style>
</head>
<body bgcolor="#A0A0A0" vlink="blue" link="blue">
<div id="page11-div" style="position:relative;width:892px;height:1173px;">
<p style="position:absolute;top:42px;left:187px;white-space:nowrap" class="ft110">Evolution&#160;and&#160;emergence:&#160;higher&#160;order&#160;information&#160;structure&#160;in&#160;protein&#160;interactomes&#160;across&#160;the&#160;tree&#160;of&#160;life&#160;<b>|&#160;11</b></p>
<p style="position:absolute;top:289px;left:72px;white-space:nowrap" class="ft1113"><b>Figure&#160;6.&#160;</b>Comparison&#160;of&#160;causal&#160;emergence&#160;of&#160;bacteria&#160;and&#160;eukaryota.<br/>Using&#160;the&#160;procedure&#160;in&#160;Figure&#160;5,&#160;we&#160;select&#160;a&#160;random&#160;sample&#160;of&#160;bacteria<br/>species&#160;and&#160;eukaryota&#160;species&#160;and&#160;perform&#160;the&#160;edge&#160;reweighting&#160;on<br/>their&#160;interactomes.&#160;Notably,&#160;the&#160;two&#160;domains&#160;do&#160;not&#160;substantially&#160;differ<br/>under&#160;this&#160;re-weighting&#160;scheme,&#160;at&#160;least&#160;partially&#160;validating&#160;the&#160;results<br/>from&#160;the&#160;main&#160;text.&#160;Error&#160;bands&#160;are&#160;95%&#160;confidence&#160;intervals.</p>
<p style="position:absolute;top:401px;left:72px;white-space:nowrap" class="ft1114">and&#160;a&#160;full&#160;description&#160;of&#160;the&#160;randomization&#160;procedure<br/>below.</p>
<p style="position:absolute;top:437px;left:88px;white-space:nowrap" class="ft114">To&#160;reweight&#160;the&#160;edges&#160;in&#160;a&#160;given&#160;network,&#160;we&#160;first&#160;mul-</p>
<p style="position:absolute;top:455px;left:72px;white-space:nowrap" class="ft1114">tiply&#160;each&#160;row&#160;of&#160;<i>Wout&#160;</i>by&#160;vectors&#160;of&#160;uniformly&#160;distributed<br/>noise.&#160;This&#160;multiplication&#160;ensures&#160;that&#160;the&#160;vectors&#160;of<br/>noise&#160;will&#160;preserve&#160;the&#160;presence&#160;and&#160;absence&#160;of&#160;outgoing<br/>edges&#160;for&#160;each&#160;node.&#160;After&#160;this,&#160;we&#160;renormalize&#160;each&#160;row<br/>so&#160;that&#160;they&#160;sum&#160;to&#160;1.0.&#160;Next,&#160;we&#160;want&#160;to&#160;introduce<br/>tunable&#160;non-uniformity&#160;to&#160;each&#160;node’s&#160;edge&#160;weights.&#160;For<br/>example,&#160;if&#160;node&#160;<i>vi&#160;</i>connects&#160;to&#160;nodes&#160;<i>vj&#160;</i>and&#160;<i>vk&#160;</i>each&#160;with<br/>a&#160;weight&#160;<i>wij&#160;=&#160;wik&#160;=&#160;</i>0.5,&#160;we&#160;want&#160;to&#160;track&#160;how&#160;the&#160;net-<br/>work’s&#160;causal&#160;emergence&#160;changes&#160;if&#160;these&#160;edge&#160;weights<br/>were,&#160;for&#160;example,&#160;[0.45,&#160;0.55]&#160;or&#160;[0.7,&#160;0.3]&#160;or&#160;[0.9,&#160;0.1],&#160;etc.<br/>For&#160;this,&#160;we&#160;use&#160;a&#160;<i>softmax&#160;</i>function—a&#160;common&#160;tool&#160;in<br/>machine&#160;learning&#160;and&#160;statistics.&#160;Softmax&#160;functions&#160;have<br/>many&#160;purposes&#160;and&#160;can&#160;be&#160;used&#160;to&#160;exaggerate&#160;or&#160;depress<br/>probabilities&#160;in&#160;vectors;&#160;the&#160;function&#160;is&#160;defined&#160;as:</p>
<p style="position:absolute;top:741px;left:196px;white-space:nowrap" class="ft115"><i>σ&#160;(</i></p>
<p style="position:absolute;top:740px;left:212px;white-space:nowrap" class="ft115"><i>Wi)&#160;</i>=</p>
<p style="position:absolute;top:731px;left:271px;white-space:nowrap" class="ft115"><i>eθWi</i></p>
<p style="position:absolute;top:741px;left:254px;white-space:nowrap" class="ft114">	</p>
<p style="position:absolute;top:750px;left:268px;white-space:nowrap" class="ft1115"><i>N<br/>j</i></p>
<p style="position:absolute;top:758px;left:271px;white-space:nowrap" class="ft117">=1&#160;<i>e</i></p>
<p style="position:absolute;top:749px;left:292px;white-space:nowrap" class="ft116"><i>θ</i></p>
<p style="position:absolute;top:748px;left:297px;white-space:nowrap" class="ft116"><i>wij</i></p>
<p style="position:absolute;top:740px;left:421px;white-space:nowrap" class="ft114">(5)</p>
<p style="position:absolute;top:797px;left:72px;white-space:nowrap" class="ft1116">where&#160;<i>θ&#160;</i>modulates&#160;the&#160;extent&#160;to&#160;which&#160;higher&#160;probability<br/>values&#160;will&#160;become&#160;even&#160;higher.&#160;For&#160;example,&#160;the&#160;vector<br/>[0.1,&#160;0.2,&#160;0.3,&#160;0.4],&#160;when&#160;passed&#160;through&#160;a&#160;softmax&#160;func-<br/>tion&#160;with&#160;<i>θ&#160;=&#160;</i>10,&#160;becomes&#160;[0.03,&#160;0.09,&#160;0.24,&#160;0.64];&#160;when<br/><i>θ</i></p>
<p style="position:absolute;top:868px;left:85px;white-space:nowrap" class="ft115"><i>=&#160;</i>0,&#160;it&#160;becomes&#160;[0.25,&#160;0.25,&#160;0.25,&#160;0.25].&#160;This&#160;<i>θ&#160;</i>parame-</p>
<p style="position:absolute;top:886px;left:72px;white-space:nowrap" class="ft1114">ter,&#160;when&#160;varied,&#160;can&#160;generate&#160;a&#160;range&#160;of&#160;uniform&#160;and<br/>non-uniform&#160;vectors—precisely&#160;what&#160;this&#160;edge&#160;reweight-<br/>ing&#160;procedure&#160;requires.&#160;We&#160;repeatedly&#160;create&#160;noisy&#160;<i>Wout<br/></i>matrices&#160;and&#160;apply&#160;the&#160;softmax&#160;function&#160;under&#160;a&#160;variety<br/>of&#160;<i>θ&#160;</i>values.&#160;We&#160;can&#160;then&#160;compute&#160;the&#160;causal&#160;emergence<br/>of&#160;this&#160;new&#160;matrix,&#160;<i>ω</i>,&#160;and&#160;calculate&#160;its&#160;percentage&#160;of&#160;the<br/>original&#160;network’s&#160;causal&#160;emergence&#160;value.&#160;We&#160;plot&#160;this<br/>for&#160;an&#160;example&#160;network&#160;under&#160;a&#160;range&#160;of&#160;<i>θ&#160;</i>values&#160;in<br/><a href="59fe6d0319faf365850c46f352541a9cs.html#10">Figure&#160;5J</a>.</p>
<p style="position:absolute;top:1048px;left:88px;white-space:nowrap" class="ft114">To&#160;compare&#160;the&#160;effect&#160;of&#160;uniform&#160;versus&#160;non-uniform</p>
<p style="position:absolute;top:1066px;left:72px;white-space:nowrap" class="ft1114">edge&#160;weights&#160;in&#160;the&#160;protein&#160;interaction&#160;networks&#160;studied<br/>in&#160;this&#160;work,&#160;we&#160;randomly&#160;sample&#160;50&#160;bacteria&#160;and&#160;50<br/>eukaryota&#160;to&#160;perform&#160;this&#160;procedure&#160;on.&#160;We&#160;compare</p>
<p style="position:absolute;top:79px;left:464px;white-space:nowrap" class="ft1114">the&#160;effect&#160;of&#160;this&#160;reweighting&#160;procedure&#160;in&#160;<a href="59fe6d0319faf365850c46f352541a9cs.html#11">Figure&#160;6,&#160;</a>find-<br/>ing&#160;little&#160;differences&#160;between&#160;bacteria&#160;and&#160;eukaryota.<br/>This&#160;result&#160;and&#160;procedure&#160;are&#160;important&#160;controls&#160;for<br/>characterizing&#160;the&#160;higher&#160;informative&#160;scales&#160;of&#160;different<br/>protein&#160;networks,&#160;as&#160;there&#160;does&#160;not&#160;seem&#160;to&#160;be&#160;especially<br/>consistent&#160;descriptions&#160;of&#160;the&#160;expected&#160;shape&#160;of&#160;indi-<br/>vidual&#160;proteins’&#160;edge&#160;weight&#160;distributions&#160;across&#160;species.<br/>Note&#160;that&#160;here&#160;we&#160;make&#160;the&#160;minimal&#160;assumption&#160;that<br/>there&#160;should&#160;not&#160;be&#160;domain-specific&#160;differences&#160;between<br/>how&#160;this&#160;reweighting&#160;scheme&#160;impacts&#160;the&#160;causal&#160;emer-<br/>gence&#160;of&#160;bacteria&#160;versus&#160;eukaryota.&#160;The&#160;introduction&#160;of<br/>this&#160;technique&#160;opens&#160;a&#160;wide&#160;number&#160;of&#160;novel&#160;research<br/>questions&#160;for&#160;future&#160;work.&#160;Importantly,&#160;we&#160;do&#160;not&#160;want<br/>to&#160;introduce&#160;additional&#160;assumptions&#160;about&#160;how&#160;this&#160;non-<br/>uniform&#160;weight&#160;is&#160;distributed&#160;(e.g.&#160;we&#160;do&#160;not&#160;want&#160;to<br/>artificially&#160;impose&#160;correlations&#160;between&#160;edge&#160;weight&#160;and<br/>degree&#160;of&#160;incident&#160;nodes,&#160;as&#160;we&#160;did&#160;not&#160;find&#160;evidence&#160;for<br/>this&#160;in&#160;the&#160;literature).</p>
<p style="position:absolute;top:402px;left:480px;white-space:nowrap" class="ft114">Although&#160;it&#160;is&#160;impossible&#160;to&#160;exhaust&#160;all&#160;possible</p>
<p style="position:absolute;top:420px;left:464px;white-space:nowrap" class="ft1114">sources&#160;of&#160;bias&#160;or&#160;confounding&#160;variables&#160;in&#160;biological<br/>networks,&#160;the&#160;two&#160;statistical&#160;controls&#160;performed&#160;here&#160;get<br/>us&#160;closer&#160;to&#160;validating&#160;the&#160;hypotheses&#160;underlying&#160;this<br/>work:&#160;that&#160;evolution&#160;brings&#160;about&#160;higher&#160;informative<br/>scales&#160;in&#160;protein&#160;networks.</p>
<p style="position:absolute;top:534px;left:464px;white-space:nowrap" class="ft119"><b>Author&#160;contributions</b></p>
<p style="position:absolute;top:560px;left:464px;white-space:nowrap" class="ft1114">E.H.,&#160;B.K.&#160;and&#160;M.L.&#160;conceived&#160;the&#160;project.&#160;E.H.,&#160;B.K.,&#160;A.S.<br/>and&#160;M.L.&#160;wrote&#160;the&#160;article.&#160;E.H.,&#160;B.K.&#160;and&#160;R.G.&#160;performed<br/>the&#160;analyses.</p>
<p style="position:absolute;top:639px;left:464px;white-space:nowrap" class="ft119"><b>Funding</b></p>
<p style="position:absolute;top:665px;left:464px;white-space:nowrap" class="ft1114">Allen&#160;Discovery&#160;Center&#160;Program&#160;through&#160;The&#160;Paul&#160;G.<br/>Allen&#160;Frontiers&#160;Group&#160;(12171&#160;to&#160;M.L.);Templeton&#160;World<br/>Charity&#160;Foundation,&#160;Inc.&#160;(TWCFG0273);&#160;Army&#160;Research<br/>Office&#160;(proposal&#160;77111-PH-II);&#160;National&#160;Science&#160;Founda-<br/>tion&#160;(DGE-1632976&#160;to&#160;A.S.).&#160;John&#160;Templeton&#160;Foundation<br/>(61780&#160;to&#160;B.K.&#160;and&#160;62212&#160;to&#160;M.L.).</p>
<p style="position:absolute;top:798px;left:464px;white-space:nowrap" class="ft119"><b>Software&#160;and&#160;data&#160;availability</b></p>
<p style="position:absolute;top:824px;left:464px;white-space:nowrap" class="ft1114">The&#160;dataset&#160;and&#160;Python&#160;code&#160;to&#160;reproduce&#160;these&#160;analyses<br/>is&#160;available&#160;at&#160;<a href="https://github.com/jkbren/einet">https://github.com/jkbren/einet&#160;</a><a href="59fe6d0319faf365850c46f352541a9cs.html#12">[42</a>].</p>
<p style="position:absolute;top:884px;left:464px;white-space:nowrap" class="ft119"><b>Conflict&#160;of&#160;interest&#160;statement</b></p>
<p style="position:absolute;top:911px;left:464px;white-space:nowrap" class="ft1114">The&#160;opinions&#160;expressed&#160;in&#160;this&#160;publication&#160;are&#160;those&#160;of<br/>the&#160;author(s)&#160;and&#160;do&#160;not&#160;necessarily&#160;reflect&#160;the&#160;views<br/>of&#160;the&#160;John&#160;Templeton&#160;Foundation&#160;and&#160;Templeton&#160;World<br/>Charity&#160;Foundation,&#160;Inc..</p>
<p style="position:absolute;top:1007px;left:464px;white-space:nowrap" class="ft119"><b>References</b></p>
<p style="position:absolute;top:1037px;left:470px;white-space:nowrap" class="ft1110"><a href="59fe6d0319faf365850c46f352541a9cs.html#1">1.&#160;</a>Edelman&#160;GM,&#160;Gally&#160;JA.&#160;Degeneracy&#160;and&#160;complexity&#160;in&#160;bio-</p>
<p style="position:absolute;top:1054px;left:487px;white-space:nowrap" class="ft1118">logical&#160;systems.&#160;<i>Proc&#160;Natl&#160;Acad&#160;Sci&#160;U&#160;S&#160;A&#160;</i>2001;<b>98</b>:13763–8.<br/><a href="https://doi.org/10.1073/pnas.231499798">10.1073/pnas.231499798.</a></p>
<p style="position:absolute;top:1087px;left:470px;white-space:nowrap" class="ft1110"><a href="59fe6d0319faf365850c46f352541a9cs.html#1">2.&#160;</a>Tsimring&#160;LS.&#160;Noise&#160;in&#160;biology.&#160;<i>Rep&#160;Prog&#160;Phys&#160;</i>2014;<b>77</b>:026601.</p>
<p style="position:absolute;top:1103px;left:487px;white-space:nowrap" class="ft1110"><a href="https://doi.org/10.1088/0034-4885/77/2/026601">10.1088/0034-4885/77/2/026601.</a></p>
<p style="position:absolute;top:229px;left:860px;white-space:nowrap" class="ft1112">Downloaded from https://academic.oup.com/ib/advance-article/doi/10.1093/intbio/zyab020/6474503 by guest on 22 December 2021</p>
</div>
</body>
</html>
<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="" xml:lang="">
<head>
<title>59fe6d0319faf365850c46f352541a9c-html.html</title>

<meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
 <br/>
<style type="text/css">
<!--
	p {margin: 0; padding: 0;}	.ft120{font-size:9px;font-family:Times;color:#000000;}
	.ft121{font-size:9px;font-family:Times;color:#000000;}
	.ft122{font-size:9px;font-family:Times;color:#000000;}
	.ft123{font-size:9px;font-family:Times;color:#2e3191;}
	.ft124{font-size:10px;font-family:Times;color:#666666;-moz-transform: matrix(         0,          1,         -1,          0, 0, 0);-webkit-transform: matrix(         0,          1,         -1,          0, 0, 0);-o-transform: matrix(         0,          1,         -1,          0, 0, 0);-ms-transform: matrix(         0,          1,         -1,          0, 0, 0);-moz-transform-origin: left 75%;-webkit-transform-origin: left 75%;-o-transform-origin: left 75%;-ms-transform-origin: left 75%;}
	.ft125{font-size:9px;line-height:16px;font-family:Times;color:#2e3191;}
	.ft126{font-size:9px;line-height:16px;font-family:Times;color:#000000;}
	.ft127{font-size:9px;line-height:16px;font-family:Times;color:#000000;}
-->
</style>
</head>
<body bgcolor="#A0A0A0" vlink="blue" link="blue">
<div id="page12-div" style="position:relative;width:892px;height:1173px;">
<p style="position:absolute;top:42px;left:64px;white-space:nowrap" class="ft120"><b>12&#160;|&#160;</b><i>Integrative&#160;Biology</i>,&#160;2021</p>
<p style="position:absolute;top:81px;left:69px;white-space:nowrap" class="ft123"><a href="59fe6d0319faf365850c46f352541a9cs.html#1">3.&#160;</a>Einstein&#160;A.&#160;On&#160;the&#160;theory&#160;of&#160;the&#160;Brownian&#160;movement.&#160;<i>Ann</i></p>
<p style="position:absolute;top:97px;left:86px;white-space:nowrap" class="ft125"><i>Phys&#160;</i>1906;<b>19</b>:371–81.&#160;<a href="https://einsteinpapers.press.princeton.edu/vol2-trans/194">https://einsteinpapers.press.princeton.e<br/>du/vol2-trans/194.</a></p>
<p style="position:absolute;top:130px;left:69px;white-space:nowrap" class="ft123"><a href="59fe6d0319faf365850c46f352541a9cs.html#1">4.&#160;</a>Colquhoun&#160;D,&#160;Hawkes&#160;AG.&#160;On&#160;the&#160;stochastic&#160;properties</p>
<p style="position:absolute;top:147px;left:86px;white-space:nowrap" class="ft126">of&#160;single&#160;ion&#160;channels.&#160;<i>Proc&#160;R&#160;Soc&#160;B&#160;</i>1981;<b>211</b>:205–35.<br/><a href="https://doi.org/10.1098/rspb.1981.0003">10.1098/rspb.1981.0003.</a></p>
<p style="position:absolute;top:180px;left:69px;white-space:nowrap" class="ft123"><a href="59fe6d0319faf365850c46f352541a9cs.html#1">5.&#160;</a>Ba¸sar&#160;E.&#160;<i>Chaos&#160;in&#160;Brain&#160;Function:&#160;Containing&#160;Original&#160;Chapters&#160;by&#160;E.</i></p>
<p style="position:absolute;top:196px;left:86px;white-space:nowrap" class="ft126"><i>Basar&#160;and&#160;TH&#160;Bullock&#160;and&#160;Topical&#160;Articles&#160;Reprinted&#160;from&#160;the&#160;Springer<br/>Series&#160;in&#160;Brain&#160;Dynamics</i>.&#160;Berlin,&#160;Germany:&#160;Springer&#160;Science&#160;&amp;<br/>Business&#160;Media,&#160;2012.</p>
<p style="position:absolute;top:245px;left:69px;white-space:nowrap" class="ft123"><a href="59fe6d0319faf365850c46f352541a9cs.html#1">6.&#160;</a>Brennan&#160;MD,&#160;Cheong&#160;R,&#160;Levchenko&#160;A.&#160;How&#160;information&#160;theory</p>
<p style="position:absolute;top:262px;left:86px;white-space:nowrap" class="ft126">handles&#160;cell&#160;signaling&#160;and&#160;uncertainty.&#160;<i>Science&#160;</i>2012;<b>338</b>:334–5.<br/><a href="https://doi.org/10.1126/science.1227946">10.1126/science.1227946.</a></p>
<p style="position:absolute;top:295px;left:69px;white-space:nowrap" class="ft123"><a href="59fe6d0319faf365850c46f352541a9cs.html#1">7.&#160;</a>Tononi&#160;G,&#160;Sporns&#160;O,&#160;Edelman&#160;GM.&#160;Measures&#160;of&#160;degeneracy&#160;and</p>
<p style="position:absolute;top:311px;left:86px;white-space:nowrap" class="ft127">redundancy&#160;in&#160;biological&#160;networks.&#160;<i>Proc&#160;Natl&#160;Acad&#160;Sci&#160;U&#160;S&#160;A<br/></i>1999;<b>96</b>:3257–62.&#160;<a href="https://doi.org/10.1073/pnas.96.6.3257">10.1073/pnas.96.6.3257.</a></p>
<p style="position:absolute;top:344px;left:69px;white-space:nowrap" class="ft123"><a href="59fe6d0319faf365850c46f352541a9cs.html#2">8.&#160;</a>Dolinski&#160;K,&#160;Troyanskaya&#160;OG.&#160;Implications&#160;of&#160;big&#160;data&#160;for</p>
<p style="position:absolute;top:360px;left:86px;white-space:nowrap" class="ft126">cell&#160;biology.&#160;<i>Mol&#160;Biol&#160;Cell&#160;</i>2015;<b>26</b>:2575–8issn:&#160;19394586.<br/><a href="https://doi.org/10.1091/mbc.E13-12-0756">10.1091/mbc.E13-12-0756.</a></p>
<p style="position:absolute;top:393px;left:69px;white-space:nowrap" class="ft123"><a href="59fe6d0319faf365850c46f352541a9cs.html#2">9.&#160;</a>Marx&#160;V.&#160;The&#160;big&#160;challenges&#160;of&#160;big&#160;data.&#160;<i>Nature&#160;</i>2013;<b>498</b>:</p>
<p style="position:absolute;top:410px;left:86px;white-space:nowrap" class="ft122">255–60issn:&#160;00280836.&#160;<a href="https://doi.org/10.1038/498255a">10.1038/498255a.</a></p>
<p style="position:absolute;top:426px;left:63px;white-space:nowrap" class="ft123"><a href="59fe6d0319faf365850c46f352541a9cs.html#2">10.&#160;</a>Whitacre&#160;JM.&#160;Degeneracy:&#160;a&#160;link&#160;between&#160;evolvability,&#160;robust-</p>
<p style="position:absolute;top:443px;left:86px;white-space:nowrap" class="ft127">ness&#160;and&#160;complexity&#160;in&#160;biological&#160;systems.&#160;<i>Theor&#160;Biol&#160;Med&#160;Model<br/></i>2010;<b>7</b>:1–17.&#160;<a href="https://doi.org/10.1186/1742-4682-7-6">10.1186/1742-4682-7-6.</a></p>
<p style="position:absolute;top:476px;left:63px;white-space:nowrap" class="ft123"><a href="59fe6d0319faf365850c46f352541a9cs.html#2">11.&#160;</a>Alon&#160;U.&#160;Biological&#160;networks:&#160;the&#160;tinkerer&#160;as&#160;an&#160;engineer.&#160;<i>Science</i></p>
<p style="position:absolute;top:492px;left:86px;white-space:nowrap" class="ft122">2003;<b>301</b>:1866–7.&#160;<a href="https://doi.org/10.1126/science.1089072">10.1126/science.1089072.</a></p>
<p style="position:absolute;top:508px;left:63px;white-space:nowrap" class="ft123"><a href="59fe6d0319faf365850c46f352541a9cs.html#2">12.&#160;</a>Bray&#160;D.&#160;Molecular&#160;networks:&#160;the&#160;top-down&#160;view.&#160;<i>Science</i></p>
<p style="position:absolute;top:525px;left:86px;white-space:nowrap" class="ft122">2003;<b>301</b>:1864–5.&#160;<a href="https://doi.org/10.1126/science.1089118">10.1126/science.1089118.</a></p>
<p style="position:absolute;top:541px;left:63px;white-space:nowrap" class="ft123"><a href="59fe6d0319faf365850c46f352541a9cs.html#2">13.&#160;</a>Klein&#160;B,&#160;Hoel&#160;E.&#160;The&#160;emergence&#160;of&#160;informative&#160;higher</p>
<p style="position:absolute;top:558px;left:86px;white-space:nowrap" class="ft122">scales</p>
<p style="position:absolute;top:558px;left:131px;white-space:nowrap" class="ft122">in</p>
<p style="position:absolute;top:558px;left:153px;white-space:nowrap" class="ft122">complex</p>
<p style="position:absolute;top:558px;left:211px;white-space:nowrap" class="ft122">networks.&#160;<i>Complexity</i></p>
<p style="position:absolute;top:558px;left:343px;white-space:nowrap" class="ft122">2020;<b>2020</b>:1–12.</p>
<p style="position:absolute;top:574px;left:86px;white-space:nowrap" class="ft123"><a href="https://doi.org/10.1155/2020/8932526">10.1155/2020/8932526.</a></p>
<p style="position:absolute;top:591px;left:63px;white-space:nowrap" class="ft123"><a href="59fe6d0319faf365850c46f352541a9cs.html#2">14.&#160;</a>Szklarczyk&#160;D,&#160;Franceschini&#160;A,&#160;Kuhn&#160;M&#160;et&#160;al.&#160;The&#160;STRING</p>
<p style="position:absolute;top:607px;left:86px;white-space:nowrap" class="ft126">database&#160;in&#160;2011:&#160;functional&#160;interaction&#160;networks&#160;of&#160;proteins,<br/>globally&#160;integrated&#160;and&#160;scored.&#160;<i>Nucleic&#160;Acids&#160;Res&#160;</i>2011;<b>39</b>:561–8.<br/><a href="https://doi.org/10.1093/nar/gkq973">10.1093/nar/gkq973.</a></p>
<p style="position:absolute;top:656px;left:63px;white-space:nowrap" class="ft123"><a href="59fe6d0319faf365850c46f352541a9cs.html#2">15.&#160;</a>Szklarczyk&#160;D,&#160;Morris&#160;JH,&#160;Cook&#160;H&#160;et&#160;al.&#160;The&#160;STRING&#160;database&#160;in</p>
<p style="position:absolute;top:673px;left:86px;white-space:nowrap" class="ft126">2017:&#160;quality-controlled&#160;protein-protein&#160;association&#160;networks,<br/>made&#160;broadly&#160;accessible.&#160;<i>Nucleic&#160;Acids&#160;Res&#160;</i>2017;<b>45</b>:D362–8.<br/><a href="https://doi.org/10.1093/nar/gkw937">10.1093/nar/gkw937.</a></p>
<p style="position:absolute;top:722px;left:63px;white-space:nowrap" class="ft123"><a href="59fe6d0319faf365850c46f352541a9cs.html#2">16.&#160;</a>Zitnik&#160;M,&#160;Sosi</p>
<p style="position:absolute;top:719px;left:162px;white-space:nowrap" class="ft122">ˇc&#160;R,&#160;Feldman&#160;MW&#160;et&#160;al.&#160;Evolution&#160;of&#160;resilience&#160;in</p>
<p style="position:absolute;top:739px;left:86px;white-space:nowrap" class="ft127">protein&#160;interactomes&#160;across&#160;the&#160;tree&#160;of&#160;life.&#160;<i>Proc&#160;Natl&#160;Acad&#160;Sci&#160;U<br/>S&#160;A&#160;</i>2019;<b>116</b>:4426–33.&#160;<a href="https://doi.org/10.1073/pnas.1818013116">10.1073/pnas.1818013116.</a></p>
<p style="position:absolute;top:771px;left:63px;white-space:nowrap" class="ft123"><a href="59fe6d0319faf365850c46f352541a9cs.html#2">17.&#160;</a>Hoel&#160;E,&#160;Albantakis&#160;L,&#160;Tononi&#160;G.&#160;Quantifying&#160;causal&#160;emergence</p>
<p style="position:absolute;top:788px;left:86px;white-space:nowrap" class="ft127">shows&#160;that&#160;macro&#160;can&#160;beat&#160;micro.&#160;<i>Proc&#160;Natl&#160;Acad&#160;Sci&#160;U&#160;S&#160;A<br/></i>2013;<b>110</b>:19790–5.&#160;<a href="https://doi.org/10.1073/pnas.1314922110">10.1073/pnas.1314922110.</a></p>
<p style="position:absolute;top:821px;left:63px;white-space:nowrap" class="ft123"><a href="59fe6d0319faf365850c46f352541a9cs.html#2">18.&#160;</a>Barabási&#160;A-L,&#160;Oltvai&#160;ZN.&#160;Network&#160;biology:&#160;understanding&#160;the</p>
<p style="position:absolute;top:837px;left:86px;white-space:nowrap" class="ft126">cell’s&#160;functional&#160;organization.&#160;<i>Nat&#160;Rev&#160;Genet&#160;</i>2004;<b>5</b>:101–13.<br/><a href="https://doi.org/10.1038/nrg1272">10.1038/nrg1272.</a></p>
<p style="position:absolute;top:870px;left:63px;white-space:nowrap" class="ft123"><a href="59fe6d0319faf365850c46f352541a9cs.html#2">19.&#160;</a>Luke</p>
<p style="position:absolute;top:867px;left:113px;white-space:nowrap" class="ft122">ˇs&#160;J,&#160;Archibald&#160;JM,&#160;Keeling&#160;PJ&#160;et&#160;al.&#160;How&#160;a&#160;neutral&#160;evolutionary</p>
<p style="position:absolute;top:886px;left:86px;white-space:nowrap" class="ft126">ratchet&#160;can&#160;build&#160;cellular&#160;complexity.&#160;<i>IUBMB&#160;Life&#160;</i>2011;<b>63</b>:528–37.<br/><a href="https://doi.org/10.1002/iub.489">10.1002/iub.489.</a></p>
<p style="position:absolute;top:919px;left:63px;white-space:nowrap" class="ft123"><a href="59fe6d0319faf365850c46f352541a9cs.html#2">20.&#160;</a>Brunet</p>
<p style="position:absolute;top:919px;left:139px;white-space:nowrap" class="ft122">TDP,</p>
<p style="position:absolute;top:919px;left:179px;white-space:nowrap" class="ft122">Ford</p>
<p style="position:absolute;top:919px;left:219px;white-space:nowrap" class="ft122">Doolittle</p>
<p style="position:absolute;top:919px;left:281px;white-space:nowrap" class="ft122">W.</p>
<p style="position:absolute;top:919px;left:311px;white-space:nowrap" class="ft122">The</p>
<p style="position:absolute;top:919px;left:347px;white-space:nowrap" class="ft122">generality</p>
<p style="position:absolute;top:919px;left:417px;white-space:nowrap" class="ft122">of</p>
<p style="position:absolute;top:936px;left:86px;white-space:nowrap" class="ft122">constructive</p>
<p style="position:absolute;top:936px;left:170px;white-space:nowrap" class="ft122">neutral</p>
<p style="position:absolute;top:936px;left:225px;white-space:nowrap" class="ft122">evolution.</p>
<p style="position:absolute;top:936px;left:295px;white-space:nowrap" class="ft121"><i>Biol</i></p>
<p style="position:absolute;top:936px;left:329px;white-space:nowrap" class="ft121"><i>Philos</i></p>
<p style="position:absolute;top:936px;left:374px;white-space:nowrap" class="ft122">2018;<b>33</b>:2.</p>
<p style="position:absolute;top:952px;left:86px;white-space:nowrap" class="ft123"><a href="https://doi.org/10.1007/s10539-018-9614-6">10.1007/s10539-018-9614-6.</a></p>
<p style="position:absolute;top:969px;left:63px;white-space:nowrap" class="ft123"><a href="59fe6d0319faf365850c46f352541a9cs.html#3">21.&#160;</a>Shannon&#160;CE.&#160;A&#160;mathematical&#160;theory&#160;of&#160;communication.&#160;<i>Bell&#160;Syst</i></p>
<p style="position:absolute;top:985px;left:86px;white-space:nowrap" class="ft121"><i>Tech&#160;J&#160;</i>1948;<b>27</b>:379–423.&#160;<a href="https://doi.org/10.1145/584091.584093">10.1145/584091.584093.</a></p>
<p style="position:absolute;top:1002px;left:63px;white-space:nowrap" class="ft123"><a href="59fe6d0319faf365850c46f352541a9cs.html#3">22.&#160;</a>Hoel&#160;E.&#160;When&#160;the&#160;map&#160;is&#160;better&#160;than&#160;the&#160;territory.&#160;<i>Entropy</i></p>
<p style="position:absolute;top:1018px;left:86px;white-space:nowrap" class="ft122">2017;<b>19</b>:188.&#160;<a href="https://doi.org/10.3390/e19050188">10.3390/e19050188.</a></p>
<p style="position:absolute;top:81px;left:455px;white-space:nowrap" class="ft123"><a href="59fe6d0319faf365850c46f352541a9cs.html#3">23.&#160;</a>Hug&#160;LA,&#160;Baker&#160;BJ,&#160;Anantharaman&#160;K&#160;et&#160;al.&#160;A&#160;new&#160;view&#160;of&#160;the&#160;tree</p>
<p style="position:absolute;top:97px;left:478px;white-space:nowrap" class="ft122">of&#160;life.&#160;<i>Nat&#160;Microbiol&#160;</i>2016;<b>1</b>:1–6.&#160;<a href="https://doi.org/10.1038/nmicrobiol.2016.48">10.1038/nmicrobiol.2016.48.</a></p>
<p style="position:absolute;top:114px;left:455px;white-space:nowrap" class="ft123"><a href="59fe6d0319faf365850c46f352541a9cs.html#4">24.&#160;</a>Griebenow&#160;R,&#160;Klein&#160;B,&#160;Hoel&#160;E.&#160;Finding&#160;the&#160;right&#160;scale&#160;of&#160;a&#160;network:</p>
<p style="position:absolute;top:130px;left:478px;white-space:nowrap" class="ft126">efficient&#160;identification&#160;of&#160;causal&#160;emergence&#160;through&#160;spectral<br/>clustering.&#160;<i>arXiv&#160;</i>2019.&#160;<a href="https://arxiv.org/abs/1908.07565">https://arxiv.org/abs/1908.07565.</a></p>
<p style="position:absolute;top:163px;left:455px;white-space:nowrap" class="ft123"><a href="59fe6d0319faf365850c46f352541a9cs.html#7">25.&#160;</a>Daniels&#160;BC,&#160;Kim&#160;H,&#160;Moore&#160;D&#160;et&#160;al.&#160;Criticality&#160;distinguishes</p>
<p style="position:absolute;top:180px;left:478px;white-space:nowrap" class="ft127">the&#160;ensemble&#160;of&#160;biological&#160;regulatory&#160;networks.&#160;<i>Phys&#160;Rev&#160;Lett<br/></i>2018;<b>121</b>:138102.&#160;<a href="https://doi.org/10.1103/PhysRevLett.121.138102">10.1103/PhysRevLett.121.138102.</a></p>
<p style="position:absolute;top:212px;left:455px;white-space:nowrap" class="ft123"><a href="59fe6d0319faf365850c46f352541a9cs.html#7">26.&#160;</a>Tang&#160;Q-Y,&#160;Hatakeyama&#160;TS,&#160;Kaneko&#160;K.&#160;Functional&#160;sensitivity&#160;and</p>
<p style="position:absolute;top:229px;left:478px;white-space:nowrap" class="ft126">mutational&#160;robustness&#160;of&#160;proteins.&#160;<i>Phys&#160;Rev&#160;Res&#160;</i>2020;<b>2</b>:033452.<br/><a href="https://doi.org/10.1103/PhysRevResearch.2.033452">10.1103/PhysRevResearch.2.033452.</a></p>
<p style="position:absolute;top:262px;left:455px;white-space:nowrap" class="ft123"><a href="59fe6d0319faf365850c46f352541a9cs.html#7">27.&#160;</a>Khajehabdollahi</p>
<p style="position:absolute;top:262px;left:583px;white-space:nowrap" class="ft122">S,</p>
<p style="position:absolute;top:262px;left:609px;white-space:nowrap" class="ft122">Witkowski</p>
<p style="position:absolute;top:262px;left:682px;white-space:nowrap" class="ft122">O.</p>
<p style="position:absolute;top:262px;left:709px;white-space:nowrap" class="ft122">Evolution</p>
<p style="position:absolute;top:262px;left:777px;white-space:nowrap" class="ft122">towards</p>
<p style="position:absolute;top:278px;left:478px;white-space:nowrap" class="ft126">criticality&#160;in&#160;Ising&#160;neural&#160;agents.&#160;<i>Artif&#160;Life&#160;</i>2020;<b>26</b>:112–29.<br/><a href="https://doi.org/10.1162/artl_a_00309">10.1162/artl_a_00309.</a></p>
<p style="position:absolute;top:311px;left:455px;white-space:nowrap" class="ft123"><a href="59fe6d0319faf365850c46f352541a9cs.html#8">28.&#160;</a>Jain&#160;R,&#160;Rivera&#160;MC,&#160;Lake&#160;JA.&#160;Horizontal&#160;gene&#160;transfer&#160;among</p>
<p style="position:absolute;top:328px;left:478px;white-space:nowrap" class="ft127">genomes:&#160;the&#160;complexity&#160;hypothesis.&#160;<i>Proc&#160;Natl&#160;Acad&#160;Sci&#160;U&#160;S&#160;A<br/></i>1999;<b>96</b>:3801–6.&#160;<a href="https://doi.org/10.1073/pnas.96.7.3801">10.1073/pnas.96.7.3801.</a></p>
<p style="position:absolute;top:360px;left:455px;white-space:nowrap" class="ft123"><a href="59fe6d0319faf365850c46f352541a9cs.html#8">29.&#160;</a>Martin&#160;W,&#160;Koonin&#160;EV.&#160;Introns&#160;and&#160;the&#160;origin&#160;of&#160;nucleus-</p>
<p style="position:absolute;top:377px;left:478px;white-space:nowrap" class="ft122">cytosol</p>
<p style="position:absolute;top:377px;left:538px;white-space:nowrap" class="ft122">compartmentalization.</p>
<p style="position:absolute;top:377px;left:685px;white-space:nowrap" class="ft121"><i>Nature</i></p>
<p style="position:absolute;top:377px;left:741px;white-space:nowrap" class="ft122">2006;<b>440</b>:41–5.</p>
<p style="position:absolute;top:393px;left:478px;white-space:nowrap" class="ft123"><a href="https://doi.org/10.1038/nature04531">10.1038/nature04531.</a></p>
<p style="position:absolute;top:410px;left:455px;white-space:nowrap" class="ft123"><a href="59fe6d0319faf365850c46f352541a9cs.html#8">30.&#160;</a>Carlile&#160;M.&#160;Prokaryotes&#160;and&#160;eukaryotes:&#160;strategies&#160;and&#160;successes.</p>
<p style="position:absolute;top:426px;left:478px;white-space:nowrap" class="ft121"><i>Trends&#160;Biochem&#160;Sci&#160;</i>1982;<b>7</b>:128–30.&#160;<a href="https://doi.org/10.1016/0968-0004(82)90199-2">10.1016/0968-0004(82)90199-2.</a></p>
<p style="position:absolute;top:443px;left:455px;white-space:nowrap" class="ft123"><a href="59fe6d0319faf365850c46f352541a9cs.html#8">31.&#160;</a>Görke&#160;B,&#160;Stülke&#160;J.&#160;Carbon&#160;catabolite&#160;repression&#160;in&#160;bacteria:&#160;many</p>
<p style="position:absolute;top:459px;left:478px;white-space:nowrap" class="ft127">ways&#160;to&#160;make&#160;the&#160;most&#160;out&#160;of&#160;nutrients.&#160;<i>Nat&#160;Rev&#160;Microbiol<br/></i>2008;<b>6</b>:613.&#160;<a href="https://doi.org/10.1038/nrmicro1932">10.1038/nrmicro1932.</a></p>
<p style="position:absolute;top:492px;left:455px;white-space:nowrap" class="ft123"><a href="59fe6d0319faf365850c46f352541a9cs.html#8">32.&#160;</a>Giovannoni&#160;SJ,&#160;Cameron&#160;Thrash&#160;J,&#160;Temperton&#160;B.&#160;Implications&#160;of</p>
<p style="position:absolute;top:508px;left:478px;white-space:nowrap" class="ft126">streamlining&#160;theory&#160;for&#160;microbial&#160;ecology.&#160;<i>ISME&#160;J&#160;</i>2014;<b>8</b>:1553.<br/><a href="https://doi.org/10.1038/ismej.2014.60">10.1038/ismej.2014.60.</a></p>
<p style="position:absolute;top:541px;left:455px;white-space:nowrap" class="ft123"><a href="59fe6d0319faf365850c46f352541a9cs.html#8">33.&#160;</a>Lane&#160;N.&#160;Energetics&#160;and&#160;genetics&#160;across&#160;the&#160;prokaryote-</p>
<p style="position:absolute;top:558px;left:478px;white-space:nowrap" class="ft122">eukaryote&#160;divide.&#160;<i>Biol&#160;Direct&#160;</i>2011;<b>6</b>:35.&#160;<a href="https://doi.org/10.1186/1745-6150-6-35">10.1186/1745-6150-6-35.</a></p>
<p style="position:absolute;top:574px;left:455px;white-space:nowrap" class="ft123"><a href="59fe6d0319faf365850c46f352541a9cs.html#8">34.&#160;</a>Dacks&#160;JB,&#160;Peden&#160;AA,&#160;Field&#160;MC.&#160;Evolution&#160;of&#160;specificity&#160;in&#160;the</p>
<p style="position:absolute;top:591px;left:478px;white-space:nowrap" class="ft126">eukaryotic&#160;endomembrane&#160;system.&#160;<i>Int&#160;J&#160;Biochem&#160;Cell&#160;Biol&#160;</i>2009;<b>41</b>:<br/>330–40.&#160;<a href="https://doi.org/10.1016/j.biocel.2008.08.041">10.1016/j.biocel.2008.08.041.</a></p>
<p style="position:absolute;top:623px;left:455px;white-space:nowrap" class="ft123"><a href="59fe6d0319faf365850c46f352541a9cs.html#8">35.&#160;</a>Rual&#160;JF,&#160;Venkatesan&#160;K,&#160;Hao&#160;T&#160;et&#160;al.&#160;Towards&#160;a&#160;proteome-scale</p>
<p style="position:absolute;top:640px;left:478px;white-space:nowrap" class="ft127">map&#160;of&#160;the&#160;human&#160;protein-protein&#160;interaction&#160;network.&#160;<i>Nature<br/></i>2005;<b>437</b>:1173–8.&#160;<a href="https://doi.org/10.1038/nature04209">10.1038/nature04209.</a></p>
<p style="position:absolute;top:673px;left:455px;white-space:nowrap" class="ft123"><a href="59fe6d0319faf365850c46f352541a9cs.html#8">36.&#160;</a>Rolland&#160;T,&#160;Ta¸san&#160;M,&#160;Charloteaux&#160;B&#160;et&#160;al.&#160;A&#160;proteome-scale</p>
<p style="position:absolute;top:689px;left:478px;white-space:nowrap" class="ft126">map&#160;of&#160;the&#160;human&#160;interactome&#160;network.&#160;<i>Cell&#160;</i>2014;<b>159</b>:1212–26.<br/><a href="https://doi.org/10.1016/j.cell.2014.10.050">10.1016/j.cell.2014.10.050.</a></p>
<p style="position:absolute;top:722px;left:455px;white-space:nowrap" class="ft123"><a href="59fe6d0319faf365850c46f352541a9cs.html#9">37.&#160;</a>Federhen&#160;S.&#160;The&#160;NCBI&#160;taxonomy&#160;database.&#160;<i>Nucleic&#160;Acids&#160;Res</i></p>
<p style="position:absolute;top:739px;left:478px;white-space:nowrap" class="ft122">2012;<b>40</b>:136–43.&#160;<a href="https://doi.org/10.1093/nar/gkr1178">10.1093/nar/gkr1178.</a></p>
<p style="position:absolute;top:755px;left:455px;white-space:nowrap" class="ft123"><a href="59fe6d0319faf365850c46f352541a9cs.html#9">38.&#160;</a>Guattery&#160;S,&#160;Miller&#160;GL.&#160;On&#160;the&#160;performance&#160;of&#160;spectral&#160;graph</p>
<p style="position:absolute;top:771px;left:478px;white-space:nowrap" class="ft126">partitioning&#160;methods.&#160;<i>Proceedings&#160;of&#160;the&#160;Sixth&#160;Annual&#160;ACM-SIAM<br/>Symposium&#160;on&#160;Discrete&#160;Algorithms.&#160;SODA&#160;‘95</i>.&#160;Philadelphia,&#160;PA,<br/>USA:&#160;Society&#160;for&#160;Industrial&#160;and&#160;Applied&#160;Mathematics,&#160;1995,<br/>233–242.&#160;doi:&#160;<a href="https://doi.org/10.5555/313651.313702">10.5555/313651.313702.</a></p>
<p style="position:absolute;top:837px;left:455px;white-space:nowrap" class="ft123"><a href="59fe6d0319faf365850c46f352541a9cs.html#9">39.&#160;</a>Ankerst&#160;M,&#160;Breunig&#160;MM,&#160;Kriegel&#160;H-P&#160;<i>et&#160;al</i>.&#160;OPTICS:&#160;ordering&#160;points</p>
<p style="position:absolute;top:854px;left:478px;white-space:nowrap" class="ft126">to&#160;identify&#160;the&#160;clustering&#160;structure.&#160;<i>Proc.&#160;ACM&#160;SIGMOD’99&#160;Int.<br/>Conf.&#160;on&#160;Management&#160;of&#160;Data</i>.&#160;Philadelphia,&#160;PA,&#160;USA:&#160;ACM&#160;Press,<br/>1999,&#160;49–60.&#160;doi:&#160;<a href="https://doi.org/10.1145/304182. \ignorespaces 304187">10.1145/304182.&#160;304187.</a></p>
<p style="position:absolute;top:903px;left:455px;white-space:nowrap" class="ft123"><a href="59fe6d0319faf365850c46f352541a9cs.html#9">40.&#160;</a>Garlaschelli&#160;D,&#160;Loffredo&#160;MI.&#160;Maximum&#160;likelihood:&#160;extracting</p>
<p style="position:absolute;top:919px;left:478px;white-space:nowrap" class="ft127">unbiased&#160;information&#160;from&#160;complex&#160;networks.&#160;<i>Phys&#160;Rev&#160;E<br/></i>2008;<b>78</b>:015101.&#160;<a href="https://doi.org/10.1103/PhysRevE.78.015101">10.1103/PhysRevE.78.015101.</a></p>
<p style="position:absolute;top:952px;left:455px;white-space:nowrap" class="ft123"><a href="59fe6d0319faf365850c46f352541a9cs.html#9">41.&#160;</a>Karrer&#160;B,&#160;Levina&#160;E,&#160;Newman&#160;MEJ.&#160;Robustness&#160;of&#160;community</p>
<p style="position:absolute;top:969px;left:478px;white-space:nowrap" class="ft126">structure&#160;in&#160;networks.&#160;<i>Phys&#160;Rev&#160;E&#160;</i>2007;<b>77</b>:1–9issn:&#160;1539-3755.<br/><a href="https://doi.org/10.1103/PhysRevE.77.046119">10.1103/PhysRevE.77.046119.</a></p>
<p style="position:absolute;top:1002px;left:455px;white-space:nowrap" class="ft123"><a href="59fe6d0319faf365850c46f352541a9cs.html#11">42.&#160;</a><a href="https://doi.org/10.5281/zenodo.5236550">Klein&#160;B.&#160;<i>jkbren/einet:&#160;einet</i>.&#160;Version&#160;v1.0.&#160;2021.&#160;doi:&#160;10.5281/zen-</a></p>
<p style="position:absolute;top:1018px;left:478px;white-space:nowrap" class="ft123"><a href="https://doi.org/10.5281/zenodo.5236550">odo.5236550.</a></p>
<p style="position:absolute;top:229px;left:860px;white-space:nowrap" class="ft124">Downloaded from https://academic.oup.com/ib/advance-article/doi/10.1093/intbio/zyab020/6474503 by guest on 22 December 2021</p>
</div>
</body>
</html>
