<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="" xml:lang="">
<head>
<title>e1aa43c837743d72c31dbd52d6a9caac-html.html</title>

<meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
 <br/>
<style type="text/css">
<!--
	p {margin: 0; padding: 0;}	.ft10{font-size:10px;font-family:Times;color:#000000;}
	.ft11{font-size:10px;font-family:Times;color:#000000;}
	.ft12{font-size:10px;font-family:Times;color:#000000;}
	.ft13{font-size:17px;font-family:Times;color:#ffffff;}
	.ft14{font-size:34px;font-family:Times;color:#000000;}
	.ft15{font-size:14px;font-family:Times;color:#000000;}
	.ft16{font-size:13px;font-family:Times;color:#000000;}
	.ft17{font-size:13px;font-family:Times;color:#000000;}
	.ft18{font-size:9px;font-family:Times;color:#000000;}
	.ft19{font-size:8px;font-family:Times;color:#000000;}
	.ft110{font-size:10px;line-height:14px;font-family:Times;color:#000000;}
	.ft111{font-size:34px;line-height:38px;font-family:Times;color:#000000;}
	.ft112{font-size:34px;line-height:39px;font-family:Times;color:#000000;}
	.ft113{font-size:34px;line-height:46px;font-family:Times;color:#000000;}
	.ft114{font-size:13px;line-height:17px;font-family:Times;color:#000000;}
	.ft115{font-size:13px;line-height:18px;font-family:Times;color:#000000;}
	.ft116{font-size:9px;line-height:17px;font-family:Times;color:#000000;}
	.ft117{font-size:9px;line-height:13px;font-family:Times;color:#000000;}
	.ft118{font-size:8px;line-height:12px;font-family:Times;color:#000000;}
-->
</style>
</head>
<body bgcolor="#A0A0A0" vlink="blue" link="blue">
<div id="page1-div" style="position:relative;width:892px;height:1191px;">
<p style="position:absolute;top:53px;left:86px;white-space:nowrap" class="ft110">Budnikova&#160;<i>et&#160;al.&#160;BMC&#160;Bioinformatics&#160;</i>2014,&#160;<b>15</b>:178<br/>http://www.biomedcentral.com/1471-2105/15/178</p>
<p style="position:absolute;top:141px;left:92px;white-space:nowrap" class="ft13"><b>M&#160;E&#160;T&#160;H&#160;O&#160;D&#160;O&#160;L&#160;O&#160;G&#160;Y&#160;A&#160;R&#160;T&#160;I&#160;C&#160;L&#160;E</b></p>
<p style="position:absolute;top:141px;left:694px;white-space:nowrap" class="ft13"><b>Open&#160;Access</b></p>
<p style="position:absolute;top:187px;left:86px;white-space:nowrap" class="ft113">Design&#160;of&#160;a&#160;flexible&#160;component&#160;gathering<br/>algorithm&#160;for&#160;converting&#160;cell-based&#160;models&#160;to<br/>graph&#160;representations&#160;for&#160;use&#160;in&#160;evolutionary<br/>search<br/>Marianna&#160;Budnikova1,&#160;Jeffrey&#160;W&#160;Habig1*,&#160;Daniel&#160;Lobo2,&#160;Nicolas&#160;Cornia1,&#160;Michael&#160;Levin2&#160;and&#160;Tim&#160;Andersen1*</p>
<p style="position:absolute;top:408px;left:96px;white-space:nowrap" class="ft16"><b>Abstract</b></p>
<p style="position:absolute;top:431px;left:96px;white-space:nowrap" class="ft115"><b>Background:&#160;</b>The&#160;ability&#160;of&#160;science&#160;to&#160;produce&#160;experimental&#160;data&#160;has&#160;outpaced&#160;the&#160;ability&#160;to&#160;effectively&#160;visualize&#160;and<br/>integrate&#160;the&#160;data&#160;into&#160;a&#160;conceptual&#160;framework&#160;that&#160;can&#160;further&#160;higher&#160;order&#160;understanding.&#160;Multidimensional&#160;and<br/>shape-based&#160;observational&#160;data&#160;of&#160;regenerative&#160;biology&#160;presents&#160;a&#160;particularly&#160;daunting&#160;challenge&#160;in&#160;this&#160;regard.<br/>Large&#160;amounts&#160;of&#160;data&#160;are&#160;available&#160;in&#160;regenerative&#160;biology,&#160;but&#160;little&#160;progress&#160;has&#160;been&#160;made&#160;in&#160;understanding&#160;how<br/>organisms&#160;such&#160;as&#160;planaria&#160;robustly&#160;achieve&#160;and&#160;maintain&#160;body&#160;form.&#160;An&#160;example&#160;of&#160;this&#160;kind&#160;of&#160;data&#160;can&#160;be&#160;found&#160;in<br/>a&#160;new&#160;repository&#160;(PlanformDB)&#160;that&#160;encodes&#160;descriptions&#160;of&#160;planaria&#160;experiments&#160;and&#160;morphological&#160;outcomes&#160;using<br/>a&#160;graph&#160;formalism.</p>
<p style="position:absolute;top:561px;left:96px;white-space:nowrap" class="ft114"><b>Results:&#160;</b>We&#160;are&#160;developing&#160;a&#160;model&#160;discovery&#160;framework&#160;that&#160;uses&#160;a&#160;cell-based&#160;modeling&#160;platform&#160;combined&#160;with<br/>evolutionary&#160;search&#160;to&#160;automatically&#160;search&#160;for&#160;and&#160;identify&#160;plausible&#160;mechanisms&#160;for&#160;the&#160;biological&#160;behavior<br/>described&#160;in&#160;PlanformDB.&#160;To&#160;automate&#160;the&#160;evolutionary&#160;search&#160;we&#160;developed&#160;a&#160;way&#160;to&#160;compare&#160;the&#160;output&#160;of&#160;the<br/>modeling&#160;platform&#160;to&#160;the&#160;morphological&#160;descriptions&#160;stored&#160;in&#160;PlanformDB.&#160;We&#160;used&#160;a&#160;flexible&#160;connected<br/>component&#160;algorithm&#160;to&#160;create&#160;a&#160;graph&#160;representation&#160;of&#160;the&#160;virtual&#160;worm&#160;from&#160;the&#160;robust,&#160;cell-based&#160;simulation<br/>data.&#160;These&#160;graphs&#160;can&#160;then&#160;be&#160;validated&#160;and&#160;compared&#160;with&#160;target&#160;data&#160;from&#160;PlanformDB&#160;using&#160;the&#160;well-known<br/>graph-edit&#160;distance&#160;calculation,&#160;which&#160;provides&#160;a&#160;quantitative&#160;metric&#160;of&#160;similarity&#160;between&#160;graphs.&#160;The&#160;graph&#160;edit<br/>distance&#160;calculation&#160;was&#160;integrated&#160;into&#160;a&#160;fitness&#160;function&#160;that&#160;was&#160;able&#160;to&#160;guide&#160;automated&#160;searches&#160;for&#160;unbiased<br/>models&#160;of&#160;planarian&#160;regeneration.&#160;We&#160;present&#160;a&#160;cell-based&#160;model&#160;of&#160;planarian&#160;that&#160;can&#160;regenerate&#160;anatomical&#160;regions<br/>following&#160;bisection&#160;of&#160;the&#160;organism,&#160;and&#160;show&#160;that&#160;the&#160;automated&#160;model&#160;discovery&#160;framework&#160;is&#160;capable&#160;of<br/>searching&#160;for&#160;and&#160;finding&#160;models&#160;of&#160;planarian&#160;regeneration&#160;that&#160;match&#160;experimental&#160;data&#160;stored&#160;in&#160;PlanformDB.</p>
<p style="position:absolute;top:764px;left:96px;white-space:nowrap" class="ft114"><b>Conclusion:&#160;</b>The&#160;work&#160;presented&#160;here,&#160;including&#160;our&#160;algorithm&#160;for&#160;converting&#160;cell-based&#160;models&#160;into&#160;graphs&#160;for<br/>comparison&#160;with&#160;data&#160;stored&#160;in&#160;an&#160;external&#160;data&#160;repository,&#160;has&#160;made&#160;feasible&#160;the&#160;automated&#160;development,&#160;training,<br/>and&#160;validation&#160;of&#160;computational&#160;models&#160;using&#160;morphology-based&#160;data.&#160;This&#160;work&#160;is&#160;part&#160;of&#160;an&#160;ongoing&#160;project&#160;to<br/>automate&#160;the&#160;search&#160;process,&#160;which&#160;will&#160;greatly&#160;expand&#160;our&#160;ability&#160;to&#160;identify,&#160;consider,&#160;and&#160;test&#160;biological<br/>mechanisms&#160;in&#160;the&#160;field&#160;of&#160;regenerative&#160;biology.</p>
<p style="position:absolute;top:983px;left:87px;white-space:nowrap" class="ft18">*Correspondence:&#160;<a href="mailto:jeffreyhabig@boisestate.edu">jeffreyhabig@boisestate.edu;&#160;</a><a href="mailto:tim@cs.boisestate.edu">tim@cs.boisestate.edu</a></p>
<p style="position:absolute;top:992px;left:86px;white-space:nowrap" class="ft117">1Department&#160;of&#160;Computer&#160;Science,&#160;Boise&#160;State&#160;University,&#160;1910&#160;University<br/>Drive,&#160;Boise,&#160;ID&#160;83725,&#160;USA<br/>Full&#160;list&#160;of&#160;author&#160;information&#160;is&#160;available&#160;at&#160;the&#160;end&#160;of&#160;the&#160;article</p>
<p style="position:absolute;top:1068px;left:272px;white-space:nowrap" class="ft118">Â©&#160;2014&#160;Budnikova&#160;et&#160;al.;&#160;licensee&#160;BioMed&#160;Central&#160;Ltd.&#160;This&#160;is&#160;an&#160;Open&#160;Access&#160;article&#160;distributed&#160;under&#160;the&#160;terms&#160;of&#160;the&#160;Creative<br/>Commons&#160;Attribution&#160;License&#160;<a href="http://creativecommons.org/licenses/by/2.0">(http://creativecommons.org/licenses/by/2.0),&#160;</a>which&#160;permits&#160;unrestricted&#160;use,&#160;distribution,&#160;and<br/>reproduction&#160;in&#160;any&#160;medium,&#160;provided&#160;the&#160;original&#160;work&#160;is&#160;properly&#160;credited.</p>
</div>
</body>
</html>
<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="" xml:lang="">
<head>
<title>e1aa43c837743d72c31dbd52d6a9caac-html.html</title>

<meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
 <br/>
<style type="text/css">
<!--
	p {margin: 0; padding: 0;}	.ft20{font-size:10px;font-family:Times;color:#000000;}
	.ft21{font-size:10px;font-family:Times;color:#000000;}
	.ft22{font-size:10px;font-family:Times;color:#000000;}
	.ft23{font-size:13px;font-family:Times;color:#000000;}
	.ft24{font-size:12px;font-family:Times;color:#000000;}
	.ft25{font-size:13px;line-height:18px;font-family:Times;color:#000000;}
	.ft26{font-size:12px;line-height:17px;font-family:Times;color:#000000;}
-->
</style>
</head>
<body bgcolor="#A0A0A0" vlink="blue" link="blue">
<div id="page2-div" style="position:relative;width:892px;height:1191px;">
<p style="position:absolute;top:53px;left:84px;white-space:nowrap" class="ft20">Budnikova&#160;<i>et&#160;al.&#160;BMC&#160;Bioinformatics&#160;</i>2014,&#160;<b>15</b>:178</p>
<p style="position:absolute;top:53px;left:746px;white-space:nowrap" class="ft20">Page&#160;2&#160;of&#160;14</p>
<p style="position:absolute;top:68px;left:84px;white-space:nowrap" class="ft20">http://www.biomedcentral.com/1471-2105/15/178</p>
<p style="position:absolute;top:135px;left:84px;white-space:nowrap" class="ft26"><b>Background<br/></b>High-throughput&#160;technologies&#160;have&#160;led&#160;to&#160;an&#160;accumula-<br/>tion&#160;of&#160;large&#160;amounts&#160;of&#160;data&#160;that&#160;can&#160;be&#160;used&#160;to&#160;advance<br/>scientific&#160;inquiry&#160;given&#160;the&#160;appropriate&#160;tools.&#160;However,<br/>our&#160;inability&#160;to&#160;effectively&#160;visualize&#160;or&#160;conceptualize&#160;these<br/>data,&#160;particularly&#160;multidimensional&#160;data,&#160;is&#160;one&#160;of&#160;the<br/>factors&#160;preventing&#160;its&#160;integration&#160;into&#160;the&#160;scientific&#160;pro-<br/>cess.&#160;One&#160;of&#160;the&#160;promising&#160;means&#160;of&#160;using&#160;these&#160;data<br/>is&#160;to&#160;develop,&#160;train,&#160;and&#160;validate&#160;computational&#160;mod-<br/>els,&#160;preferably&#160;those&#160;with&#160;interactive&#160;visual&#160;interfaces.<br/>Advances&#160;in&#160;computational&#160;modeling&#160;platforms&#160;are&#160;begin-<br/>ning&#160;to&#160;allow&#160;simulation&#160;of&#160;biological&#160;systems&#160;from&#160;the<br/>single&#160;cell&#160;biochemical&#160;level&#160;to&#160;more&#160;abstract&#160;multicellu-<br/>lar&#160;environments,&#160;such&#160;as&#160;representative&#160;tissues,&#160;organs,<br/>or&#160;even&#160;organisms.&#160;These&#160;emerging&#160;computational&#160;tools<br/>are&#160;poised&#160;to&#160;put&#160;the&#160;power&#160;of&#160;bioinformatics&#160;and&#160;data<br/>interpretation&#160;back&#160;into&#160;the&#160;wet-bench&#160;biologists&#160;hands<br/>by&#160;automatically&#160;incorporating&#160;data&#160;from&#160;the&#160;aforemen-<br/>tioned&#160;datasets&#160;with&#160;tools&#160;for&#160;visualization,&#160;experimenta-<br/>tion,&#160;and&#160;data&#160;analysis.</p>
<p style="position:absolute;top:494px;left:96px;white-space:nowrap" class="ft24">Many&#160;high-throughput&#160;technologies&#160;collect&#160;large</p>
<p style="position:absolute;top:512px;left:84px;white-space:nowrap" class="ft26">amounts&#160;of&#160;measurement&#160;data&#160;that&#160;are&#160;conducive&#160;to&#160;being<br/>stored&#160;in&#160;databases.&#160;For&#160;example,&#160;a&#160;database&#160;can&#160;easily<br/>house&#160;multi-scale&#160;gene&#160;expression&#160;data&#160;obtained&#160;from&#160;a<br/>single&#160;cell&#160;to&#160;a&#160;whole&#160;organism&#160;while&#160;also&#160;documenting<br/>the&#160;source&#160;and&#160;experimental&#160;methods&#160;associated&#160;with&#160;the<br/>data.&#160;Such&#160;repositories&#160;are&#160;well&#160;suited&#160;for&#160;data&#160;consist-<br/>ing&#160;of&#160;lists&#160;of&#160;gene&#160;and&#160;protein&#160;abundance,&#160;for&#160;example.<br/>However,&#160;new&#160;ontologies&#160;and&#160;formalisms&#160;are&#160;required&#160;for<br/>collecting&#160;and&#160;describing&#160;certain&#160;kinds&#160;of&#160;higher-order<br/>data.&#160;For&#160;instance,&#160;the&#160;outcome&#160;of&#160;experiments&#160;involv-<br/>ing&#160;shape&#160;or&#160;morphology&#160;can&#160;be&#160;challenging&#160;to&#160;describe<br/>accurately,&#160;particularly&#160;in&#160;a&#160;way&#160;that&#160;others&#160;can&#160;search<br/>for&#160;or&#160;interpret&#160;computationally.&#160;This&#160;problem&#160;has&#160;been<br/>particularly&#160;challenging&#160;in&#160;areas&#160;of&#160;development&#160;and<br/>regeneration&#160;where&#160;a&#160;description&#160;of&#160;the&#160;organ,&#160;appendage,<br/>or&#160;organism&#160;is&#160;one&#160;of&#160;the&#160;key&#160;reported&#160;observations.</p>
<p style="position:absolute;top:799px;left:96px;white-space:nowrap" class="ft24">The&#160;planarian&#160;worm&#160;is&#160;a&#160;model&#160;organism&#160;in&#160;regenerative</p>
<p style="position:absolute;top:817px;left:84px;white-space:nowrap" class="ft26">biology&#160;that&#160;perfectly&#160;illustrates&#160;the&#160;problem&#160;of&#160;storing<br/>shape-based&#160;experimental&#160;results&#160;in&#160;a&#160;formal&#160;database.<br/>These&#160;free-living&#160;flatworms&#160;have&#160;exceptional&#160;regenerative<br/>properties&#160;that&#160;have&#160;fascinated&#160;biologists&#160;for&#160;centuries&#160;<a href="e1aa43c837743d72c31dbd52d6a9caacs.html#13">[1].<br/></a>They&#160;are&#160;able&#160;to&#160;regenerate&#160;aged,&#160;damaged,&#160;or&#160;lost&#160;tis-<br/>sues&#160;with&#160;the&#160;help&#160;of&#160;a&#160;large&#160;adult&#160;stem&#160;cell&#160;population<br/><a href="e1aa43c837743d72c31dbd52d6a9caacs.html#13">[2].&#160;</a>Despite&#160;being&#160;complex&#160;organisms&#160;possessing&#160;bilateral<br/>symmetry,&#160;musculature,&#160;intestine,&#160;and&#160;a&#160;central&#160;nervous<br/>system&#160;including&#160;a&#160;true&#160;brain&#160;<a href="e1aa43c837743d72c31dbd52d6a9caacs.html#13">[3,4],&#160;</a>fragments&#160;smaller&#160;than<br/>1/200th&#160;of&#160;the&#160;adult&#160;size&#160;can&#160;remodel&#160;and&#160;regenerate&#160;an<br/>intact&#160;worm&#160;<a href="e1aa43c837743d72c31dbd52d6a9caacs.html#14">[5].&#160;</a>This&#160;astonishing&#160;regenerative&#160;ability&#160;has<br/>stimulated&#160;an&#160;effort&#160;to&#160;understand&#160;its&#160;underlying&#160;mecha-<br/>nisms&#160;<a href="e1aa43c837743d72c31dbd52d6a9caacs.html#14">[6],&#160;</a>producing&#160;an&#160;extensive&#160;number&#160;of&#160;experiments<br/>based&#160;on&#160;amputations&#160;<a href="e1aa43c837743d72c31dbd52d6a9caacs.html#13">[4],&#160;</a>drug-induced&#160;phenotypes&#160;<a href="e1aa43c837743d72c31dbd52d6a9caacs.html#14">[7,8],<br/></a>and&#160;RNAi&#160;gene-knockdowns&#160;<a href="e1aa43c837743d72c31dbd52d6a9caacs.html#14">[9-13].&#160;</a>However,&#160;despite<br/>these&#160;important&#160;efforts,&#160;we&#160;still&#160;lack&#160;a&#160;comprehensive</p>
<p style="position:absolute;top:136px;left:456px;white-space:nowrap" class="ft26">model&#160;that&#160;can&#160;explain&#160;more&#160;than&#160;one&#160;or&#160;two&#160;aspects&#160;of<br/>planarian&#160;regeneration&#160;<a href="e1aa43c837743d72c31dbd52d6a9caacs.html#14">[14].</a></p>
<p style="position:absolute;top:171px;left:468px;white-space:nowrap" class="ft24">Recently,&#160;the&#160;Levin&#160;lab&#160;has&#160;developed&#160;a&#160;new&#160;tool&#160;(Plan-</p>
<p style="position:absolute;top:189px;left:456px;white-space:nowrap" class="ft26">form)&#160;to&#160;aid&#160;in&#160;the&#160;assimilation&#160;of&#160;these&#160;data&#160;using&#160;a&#160;graph-<br/>based&#160;formalism&#160;to&#160;describe&#160;anatomy&#160;and&#160;morphology<br/>along&#160;with&#160;a&#160;new&#160;ontology&#160;for&#160;describing&#160;experimen-<br/>tal&#160;manipulations&#160;and&#160;observations&#160;<a href="e1aa43c837743d72c31dbd52d6a9caacs.html#14">[15,16].&#160;</a>The&#160;flexible<br/>and&#160;extensible&#160;graph&#160;notation&#160;allows&#160;worm&#160;regions&#160;and<br/>organs&#160;to&#160;be&#160;described&#160;as&#160;nodes&#160;connected&#160;by&#160;linkages<br/>with&#160;associated&#160;angles&#160;and&#160;length&#160;parameters.&#160;Based&#160;on<br/>this&#160;approach,&#160;the&#160;Planform&#160;Database&#160;(PlanformDB)&#160;was<br/>designed&#160;and&#160;curated&#160;to&#160;include&#160;a&#160;complete&#160;description<br/>of&#160;the&#160;many&#160;planarian&#160;experiments&#160;and&#160;outcomes&#160;that<br/>exist&#160;throughout&#160;the&#160;literature.&#160;Such&#160;a&#160;resource&#160;does&#160;not<br/>only&#160;make&#160;it&#160;possible&#160;for&#160;scientists&#160;to&#160;search&#160;and&#160;compare<br/>worm&#160;morphologies,&#160;but&#160;it&#160;also&#160;provides&#160;an&#160;extractable<br/>resource&#160;for&#160;bioinformatics&#160;applications.</p>
<p style="position:absolute;top:440px;left:468px;white-space:nowrap" class="ft24">We&#160;are&#160;currently&#160;combining&#160;Planform,&#160;agent-based</p>
<p style="position:absolute;top:458px;left:456px;white-space:nowrap" class="ft26">modeling,&#160;and&#160;an&#160;evolutionary&#160;search&#160;engine&#160;to&#160;develop<br/>an&#160;automated&#160;system&#160;for&#160;searching&#160;and&#160;validating&#160;com-<br/>putational&#160;models&#160;of&#160;regeneration.&#160;Agent-based&#160;model-<br/>ing&#160;holds&#160;promise&#160;for&#160;studying&#160;the&#160;emergent&#160;behavior<br/>and&#160;complex&#160;interactions&#160;between&#160;signaling&#160;networks<br/>involved&#160;in&#160;directing&#160;regeneration,&#160;when&#160;multi-scale&#160;or<br/>multi-cellular&#160;systems&#160;are&#160;supported.&#160;To&#160;this&#160;end,&#160;we&#160;are<br/>using&#160;a&#160;modeling&#160;platform&#160;(CellSim)&#160;where&#160;the&#160;central<br/>agents&#160;are&#160;autonomous&#160;cells&#160;containing&#160;many&#160;of&#160;the&#160;bio-<br/>logical&#160;primitives&#160;necessary&#160;for&#160;simulating&#160;living&#160;systems<br/><a href="e1aa43c837743d72c31dbd52d6a9caacs.html#14">[17].&#160;</a>The&#160;current&#160;version&#160;of&#160;this&#160;software&#160;contains&#160;a&#160;num-<br/>ber&#160;of&#160;useful&#160;features&#160;to&#160;support&#160;this&#160;endeavor,&#160;including<br/>a&#160;3-D&#160;interface&#160;for&#160;visualization&#160;and&#160;tools&#160;for&#160;perform-<br/>ing&#160;experimental&#160;manipulations&#160;within&#160;the&#160;client-server<br/>architecture.&#160;The&#160;process&#160;of&#160;developing,&#160;testing,&#160;and&#160;val-<br/>idating&#160;a&#160;complex&#160;model&#160;by&#160;hand&#160;can&#160;be&#160;a&#160;daunting<br/>task,&#160;particularly&#160;when&#160;many&#160;individual&#160;experimental&#160;out-<br/>comes&#160;are&#160;combined.&#160;To&#160;simplify&#160;this&#160;process,&#160;we&#160;have<br/>incorporated&#160;an&#160;evolutionary&#160;search&#160;engine&#160;that&#160;can&#160;auto-<br/>mate&#160;this&#160;process&#160;using&#160;a&#160;genetic&#160;algorithm&#160;driven&#160;by<br/>appropriate&#160;fitness&#160;metrics&#160;that&#160;are&#160;informed&#160;by&#160;the&#160;Plan-<br/>form&#160;Database&#160;(PlanformDB).&#160;Our&#160;ultimate&#160;goal&#160;is&#160;for&#160;this<br/>integrated&#160;system&#160;to&#160;identify&#160;computational&#160;models&#160;that<br/>can&#160;account&#160;for&#160;many,&#160;if&#160;not&#160;all,&#160;of&#160;the&#160;available&#160;exper-<br/>imental&#160;outcomes&#160;related&#160;to&#160;planarian&#160;regeneration.&#160;We<br/>believe&#160;that&#160;this&#160;general&#160;approach&#160;holds&#160;the&#160;promise&#160;to<br/>spur&#160;biological&#160;discovery,&#160;develop&#160;novel&#160;insights&#160;into&#160;long-<br/>standing&#160;problems&#160;and&#160;biases,&#160;and&#160;elucidate&#160;previously<br/>unobserved&#160;biological&#160;behaviors.</p>
<p style="position:absolute;top:978px;left:468px;white-space:nowrap" class="ft24">This&#160;paper&#160;presents&#160;a&#160;novel&#160;agent-based&#160;planarian</p>
<p style="position:absolute;top:996px;left:456px;white-space:nowrap" class="ft26">model&#160;capable&#160;of&#160;simulating&#160;basic&#160;biological&#160;behavior.&#160;The<br/>model&#160;is&#160;suitable&#160;for&#160;automated&#160;and&#160;varied&#160;experimen-<br/>tal&#160;manipulations&#160;akin&#160;to&#160;those&#160;traditionally&#160;performed<br/>by&#160;wet-bench&#160;biologists&#160;and&#160;represented&#160;in&#160;the&#160;Plan-<br/>formDB.&#160;This&#160;model&#160;includes&#160;a&#160;reaction&#160;network&#160;that<br/>responds&#160;to&#160;manipulations&#160;by&#160;initiating&#160;appropriate&#160;head</p>
</div>
</body>
</html>
<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="" xml:lang="">
<head>
<title>e1aa43c837743d72c31dbd52d6a9caac-html.html</title>

<meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
 <br/>
<style type="text/css">
<!--
	p {margin: 0; padding: 0;}	.ft30{font-size:10px;font-family:Times;color:#000000;}
	.ft31{font-size:10px;font-family:Times;color:#000000;}
	.ft32{font-size:10px;font-family:Times;color:#000000;}
	.ft33{font-size:12px;font-family:Times;color:#000000;}
	.ft34{font-size:13px;font-family:Times;color:#000000;}
	.ft35{font-size:11px;font-family:Times;color:#000000;}
	.ft36{font-size:12px;font-family:Times;color:#000000;}
	.ft37{font-size:12px;line-height:17px;font-family:Times;color:#000000;}
	.ft38{font-size:13px;line-height:19px;font-family:Times;color:#000000;}
	.ft39{font-size:11px;line-height:17px;font-family:Times;color:#000000;}
	.ft310{font-size:12px;line-height:17px;font-family:Times;color:#000000;}
-->
</style>
</head>
<body bgcolor="#A0A0A0" vlink="blue" link="blue">
<div id="page3-div" style="position:relative;width:892px;height:1191px;">
<p style="position:absolute;top:53px;left:86px;white-space:nowrap" class="ft30">Budnikova&#160;<i>et&#160;al.&#160;BMC&#160;Bioinformatics&#160;</i>2014,&#160;<b>15</b>:178</p>
<p style="position:absolute;top:53px;left:748px;white-space:nowrap" class="ft30">Page&#160;3&#160;of&#160;14</p>
<p style="position:absolute;top:68px;left:86px;white-space:nowrap" class="ft30">http://www.biomedcentral.com/1471-2105/15/178</p>
<p style="position:absolute;top:136px;left:86px;white-space:nowrap" class="ft37">and&#160;tail&#160;regeneration.&#160;Importantly,&#160;we&#160;describe&#160;an&#160;algo-<br/>rithm&#160;that&#160;allows&#160;translation&#160;of&#160;multicellular&#160;simulation<br/>output&#160;into&#160;a&#160;formal&#160;graph&#160;representation&#160;equivalent&#160;to<br/>that&#160;described&#160;by&#160;Lobo&#160;and&#160;colleagues&#160;<a href="e1aa43c837743d72c31dbd52d6a9caacs.html#14">[15,16].&#160;</a>This&#160;real-<br/>time&#160;translation&#160;is&#160;central&#160;to&#160;the&#160;automation&#160;of&#160;model<br/>discovery&#160;as&#160;it&#160;enables&#160;use&#160;of&#160;a&#160;fitness&#160;metric&#160;based&#160;upon&#160;a<br/>graph-edit&#160;distance&#160;calculation,&#160;which&#160;quantitatively&#160;com-<br/>pares&#160;simulation&#160;output&#160;and&#160;target&#160;morphologies&#160;stored<br/>in&#160;the&#160;PlanformDB.&#160;The&#160;combination&#160;of&#160;the&#160;model,&#160;trans-<br/>lation&#160;algorithm,&#160;and&#160;fitness&#160;metric&#160;provide&#160;the&#160;basis<br/>for&#160;future&#160;automated&#160;model&#160;discovery&#160;in&#160;regeneration<br/>biology.</p>
<p style="position:absolute;top:368px;left:86px;white-space:nowrap" class="ft37"><b>Results<br/>Modeling&#160;a&#160;classic&#160;planaria&#160;regeneration&#160;experiment<br/></b>As&#160;shown&#160;in&#160;the&#160;classic&#160;regeneration&#160;experiment&#160;pre-<br/>sented&#160;in&#160;Figure&#160;<a href="e1aa43c837743d72c31dbd52d6a9caacs.html#4">1a</a>,&#160;when&#160;a&#160;worm&#160;is&#160;bisected&#160;laterally<br/>the&#160;resulting&#160;fragments&#160;will&#160;naturally&#160;lack&#160;a&#160;head&#160;or&#160;tail<br/>region.&#160;Normally,&#160;each&#160;fragment&#160;will&#160;regenerate&#160;into&#160;inde-<br/>pendent,&#160;intact&#160;worms&#160;with&#160;the&#160;appropriate&#160;shape&#160;and<br/>architecture&#160;over&#160;the&#160;course&#160;of&#160;roughly&#160;ten&#160;days.&#160;We<br/>sought&#160;to&#160;develop&#160;an&#160;agent-based&#160;representation&#160;of&#160;a<br/>planarian&#160;that&#160;could&#160;simulate&#160;these&#160;experiments.&#160;Such<br/>a&#160;model&#160;would&#160;(1)&#160;validate&#160;the&#160;chosen&#160;modeling&#160;plat-<br/>form&#160;(Cellsim,&#160;see&#160;<a href="e1aa43c837743d72c31dbd52d6a9caacs.html#13">Methods&#160;</a>and&#160;<a href="e1aa43c837743d72c31dbd52d6a9caacs.html#14">[17])&#160;</a>for&#160;this&#160;project,&#160;(2)<br/>provide&#160;a&#160;working&#160;model&#160;for&#160;testing&#160;our&#160;translation&#160;algo-<br/>rithm&#160;and&#160;experimental&#160;manipulations,&#160;and&#160;(3)&#160;provide<br/>a&#160;starting&#160;network&#160;description&#160;to&#160;be&#160;used&#160;by&#160;individu-<br/>als&#160;in&#160;the&#160;population&#160;of&#160;automated&#160;searches.&#160;The&#160;model<br/>developed&#160;was&#160;dependent&#160;upon&#160;competitive&#160;inhibition<br/>using&#160;a&#160;signaling&#160;mechanism&#160;consisting&#160;of&#160;long-range<br/>morphogen&#160;gradients&#160;emanating&#160;from&#160;existing&#160;head&#160;and<br/>tail&#160;regions.&#160;As&#160;the&#160;morphogen&#160;is&#160;subject&#160;to&#160;molecu-<br/>lar&#160;decay&#160;and/or&#160;consumption&#160;in&#160;chemical&#160;reactions,&#160;its<br/>long-term&#160;presence&#160;is&#160;dependent&#160;upon&#160;the&#160;existence&#160;of<br/>its&#160;source&#160;(e.g.&#160;head&#160;or&#160;tail)&#160;in&#160;a&#160;given&#160;worm&#160;fragment.<br/>Thus,&#160;a&#160;regeneration&#160;signal&#160;can&#160;simultaneously&#160;trigger<br/>head&#160;and&#160;tail&#160;regeneration&#160;in&#160;response&#160;to&#160;a&#160;cut&#160;where<br/>the&#160;morphogens&#160;competitively&#160;inhibit&#160;their&#160;own&#160;develop-<br/>mental&#160;paths.&#160;For&#160;instance,&#160;the&#160;morphogen&#160;derived&#160;from<br/>head&#160;cells&#160;represses&#160;head&#160;regeneration&#160;in&#160;worm&#160;fragments<br/>possessing&#160;a&#160;head&#160;after&#160;a&#160;manipulation,&#160;whereas&#160;the&#160;lack<br/>of&#160;a&#160;tail&#160;will&#160;lead&#160;to&#160;decay&#160;of&#160;the&#160;tail&#160;morphogen&#160;and<br/>allow&#160;tail&#160;regeneration&#160;to&#160;proceed.&#160;A&#160;schematic&#160;of&#160;the&#160;net-<br/>work&#160;design&#160;used&#160;in&#160;these&#160;experiments&#160;is&#160;presented&#160;in<br/>Figure&#160;<a href="e1aa43c837743d72c31dbd52d6a9caacs.html#5">2(a).</a></p>
<p style="position:absolute;top:961px;left:98px;white-space:nowrap" class="ft33">Our&#160;representation&#160;of&#160;this&#160;work&#160;included&#160;a&#160;simple&#160;archi-</p>
<p style="position:absolute;top:979px;left:86px;white-space:nowrap" class="ft37">tecture&#160;of&#160;420&#160;planar&#160;cells&#160;arranged&#160;as&#160;a&#160;rectangular<br/>abstraction&#160;of&#160;an&#160;intact&#160;worm&#160;(Figure&#160;<a href="e1aa43c837743d72c31dbd52d6a9caacs.html#4">1b).&#160;</a>The&#160;num-<br/>ber&#160;of&#160;cells&#160;was&#160;chosen&#160;empirically&#160;to&#160;provide&#160;a&#160;robust<br/>system&#160;that&#160;could&#160;be&#160;reasonably&#160;manipulated&#160;by&#160;one&#160;or<br/>more&#160;simultaneous&#160;or&#160;sequential&#160;cuts.&#160;The&#160;implemen-<br/>tation&#160;of&#160;our&#160;morphogen-based&#160;model&#160;was&#160;represented<br/>in&#160;Cellsim&#160;using&#160;a&#160;series&#160;of&#160;metabolic&#160;and&#160;transcription</p>
<p style="position:absolute;top:136px;left:458px;white-space:nowrap" class="ft37">reactions&#160;(Figure&#160;<a href="e1aa43c837743d72c31dbd52d6a9caacs.html#5">2(</a>b))&#160;where&#160;every&#160;cell&#160;was&#160;controlled<br/>autonomously&#160;by&#160;this&#160;same&#160;network.&#160;In&#160;response&#160;to&#160;a&#160;sim-<br/>ulated&#160;cut,&#160;a&#160;<i>Regeneration&#160;signal&#160;</i>activates&#160;a&#160;<i>Regeneration<br/></i>pathway,&#160;which&#160;simultaneously&#160;promotes&#160;head&#160;and&#160;tail<br/>development&#160;responses.&#160;The&#160;head&#160;and&#160;tail&#160;development<br/>pathways&#160;are&#160;constitutively&#160;repressed&#160;by&#160;the&#160;presence&#160;of&#160;a<br/>morphogen&#160;(i.e.&#160;<i>Head&#160;gradient&#160;</i>and&#160;<i>Tail&#160;gradient</i>)&#160;emanat-<br/>ing&#160;from&#160;existing&#160;head&#160;or&#160;tail&#160;cells&#160;in&#160;the&#160;simulation&#160;and<br/>spread&#160;to&#160;neighboring&#160;cells&#160;through&#160;gap&#160;junctions.&#160;The<br/>morphogen&#160;gradients&#160;will&#160;disappear&#160;or&#160;be&#160;diminished&#160;fol-<br/>lowing&#160;a&#160;transverse&#160;cut&#160;when&#160;the&#160;source&#160;(head&#160;or&#160;tail)&#160;is<br/>physically&#160;removed,&#160;causing&#160;one&#160;developmental&#160;pathway<br/>to&#160;be&#160;favored&#160;over&#160;the&#160;other.&#160;Furthermore,&#160;the&#160;head&#160;and<br/>tail&#160;resources&#160;repress&#160;each&#160;other&#160;to&#160;ensure&#160;a&#160;unique&#160;cell<br/>state&#160;is&#160;ultimately&#160;achieved.</p>
<p style="position:absolute;top:405px;left:470px;white-space:nowrap" class="ft33">At&#160;the&#160;start&#160;of&#160;a&#160;simulation,&#160;the&#160;head,&#160;trunk,&#160;and&#160;tail</p>
<p style="position:absolute;top:423px;left:458px;white-space:nowrap" class="ft310">regions&#160;were&#160;defined&#160;by&#160;introducing&#160;one&#160;of&#160;three&#160;cell-<br/>state&#160;resources&#160;(head,&#160;<i>hCell</i>;&#160;trunk,&#160;<i>iCell</i>;&#160;tail,&#160;<i>tCell</i>)&#160;into<br/>each&#160;cell.&#160;Simulations&#160;were&#160;then&#160;run&#160;for&#160;approximately<br/>200&#160;steps&#160;to&#160;allow&#160;the&#160;network&#160;to&#160;reach&#160;homeostasis&#160;and<br/>provide&#160;sufficient&#160;time&#160;to&#160;develop&#160;long-range&#160;morphogen<br/>gradients.&#160;As&#160;shown&#160;in&#160;panel&#160;1&#160;of&#160;Figure&#160;<a href="e1aa43c837743d72c31dbd52d6a9caacs.html#4">1b,&#160;</a>worms&#160;con-<br/>sisted&#160;of&#160;head&#160;(blue)&#160;and&#160;tail&#160;(purple)&#160;regions&#160;separated<br/>by&#160;a&#160;trunk&#160;(orange).&#160;Next,&#160;a&#160;transverse&#160;cut&#160;was&#160;simulated<br/>by&#160;injecting&#160;a&#160;resource,&#160;<i>Lysis</i>,&#160;into&#160;a&#160;cross-section&#160;of&#160;cells<br/>located&#160;at&#160;or&#160;near&#160;the&#160;mid-line&#160;of&#160;the&#160;worm.&#160;The&#160;pres-<br/>ence&#160;of&#160;<i>Lysis&#160;</i>results&#160;in&#160;a&#160;localized&#160;cell&#160;death&#160;response&#160;that<br/>results&#160;in&#160;separation&#160;of&#160;the&#160;initial&#160;worm&#160;into&#160;two&#160;worm<br/>fragments&#160;lacking&#160;either&#160;a&#160;head&#160;or&#160;tail&#160;(panel&#160;2).&#160;Nearby<br/>cells&#160;respond&#160;to&#160;the&#160;<i>cut&#160;</i>by&#160;inducing&#160;a&#160;localized&#160;<i>Regen-<br/>eration&#160;signal</i></p>
<p style="position:absolute;top:674px;left:542px;white-space:nowrap" class="ft33">,&#160;which&#160;in&#160;turn&#160;activates&#160;a&#160;cellâs&#160;<i>Regenera-</i></p>
<p style="position:absolute;top:692px;left:458px;white-space:nowrap" class="ft36"><i>tion</i></p>
<p style="position:absolute;top:692px;left:486px;white-space:nowrap" class="ft33">pathway.&#160;At&#160;this&#160;point,&#160;simulations&#160;consisting&#160;of&#160;two</p>
<p style="position:absolute;top:709px;left:458px;white-space:nowrap" class="ft37">worm&#160;fragments&#160;were&#160;advanced&#160;another&#160;200&#160;steps&#160;prior&#160;to<br/>evaluating&#160;their&#160;emergent&#160;outcomes.&#160;The&#160;regulatory&#160;net-<br/>work&#160;parameters&#160;were&#160;optimized&#160;by&#160;hand&#160;for&#160;the&#160;network<br/>(Figure&#160;<a href="e1aa43c837743d72c31dbd52d6a9caacs.html#5">2(</a>b)),&#160;which&#160;resulted&#160;in&#160;proper&#160;regeneration&#160;of<br/>head&#160;and&#160;tail&#160;regions&#160;as&#160;shown&#160;in&#160;panel&#160;2&#160;of&#160;Figure&#160;<a href="e1aa43c837743d72c31dbd52d6a9caacs.html#4">1b.&#160;</a>For<br/>simplicity,&#160;these&#160;simulations&#160;did&#160;not&#160;include&#160;cell&#160;growth,<br/>division,&#160;or&#160;rearrangements,&#160;but&#160;these&#160;properties&#160;will&#160;be<br/>introduced&#160;in&#160;future&#160;studies.</p>
<p style="position:absolute;top:853px;left:470px;white-space:nowrap" class="ft33">These&#160;results&#160;showed&#160;that&#160;we&#160;could&#160;develop&#160;a&#160;simple</p>
<p style="position:absolute;top:871px;left:458px;white-space:nowrap" class="ft37">model&#160;of&#160;planarian&#160;regeneration&#160;using&#160;a&#160;long-range&#160;mor-<br/>phogen&#160;gradient&#160;that&#160;could&#160;faithfully&#160;respond&#160;to&#160;at&#160;least<br/>simple&#160;manipulations.&#160;However,&#160;it&#160;was&#160;clear&#160;that&#160;hand-<br/>design&#160;and&#160;tuning&#160;a&#160;single&#160;model&#160;to&#160;represent&#160;the&#160;many<br/>experimental&#160;outcomes&#160;described&#160;in&#160;the&#160;literature&#160;would<br/>be&#160;a&#160;daunting&#160;task&#160;without&#160;computational&#160;automation.<br/>This&#160;challenge&#160;could&#160;be&#160;alleviated&#160;using&#160;an&#160;automated<br/>method&#160;of&#160;model&#160;creation&#160;and&#160;evaluation,&#160;such&#160;as&#160;per-<br/>formed&#160;by&#160;genetic&#160;algorithms&#160;<a href="e1aa43c837743d72c31dbd52d6a9caacs.html#14">[18].&#160;</a>These&#160;algorithms&#160;are<br/>based&#160;upon&#160;the&#160;principles&#160;of&#160;evolution&#160;where&#160;individu-<br/>als&#160;in&#160;a&#160;population&#160;are&#160;generationally-&#160;modified&#160;through<br/>random&#160;genetic&#160;mutations&#160;and&#160;crossovers&#160;during&#160;repro-<br/>duction.&#160;Individuals&#160;chosen&#160;to&#160;contribute&#160;to&#160;the&#160;offspring</p>
</div>
</body>
</html>
<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="" xml:lang="">
<head>
<title>e1aa43c837743d72c31dbd52d6a9caac-html.html</title>

<meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
 <br/>
<style type="text/css">
<!--
	p {margin: 0; padding: 0;}	.ft40{font-size:10px;font-family:Times;color:#000000;}
	.ft41{font-size:10px;font-family:Times;color:#000000;}
	.ft42{font-size:10px;font-family:Times;color:#000000;}
	.ft43{font-size:12px;font-family:Times;color:#000000;}
	.ft44{font-size:10px;line-height:15px;font-family:Times;color:#000000;}
	.ft45{font-size:10px;line-height:14px;font-family:Times;color:#000000;}
	.ft46{font-size:12px;line-height:17px;font-family:Times;color:#000000;}
-->
</style>
</head>
<body bgcolor="#A0A0A0" vlink="blue" link="blue">
<div id="page4-div" style="position:relative;width:892px;height:1191px;">
<p style="position:absolute;top:53px;left:84px;white-space:nowrap" class="ft40">Budnikova&#160;<i>et&#160;al.&#160;BMC&#160;Bioinformatics&#160;</i>2014,&#160;<b>15</b>:178</p>
<p style="position:absolute;top:53px;left:746px;white-space:nowrap" class="ft40">Page&#160;4&#160;of&#160;14</p>
<p style="position:absolute;top:68px;left:84px;white-space:nowrap" class="ft40">http://www.biomedcentral.com/1471-2105/15/178</p>
<p style="position:absolute;top:941px;left:94px;white-space:nowrap" class="ft45"><b>Figure&#160;1&#160;This&#160;figure&#160;depicts&#160;a&#160;classic&#160;planaria&#160;regeneration&#160;experiment&#160;involving&#160;a&#160;transverse&#160;cut&#160;of&#160;an&#160;intact&#160;worm,&#160;followed&#160;by&#160;the<br/>regeneration&#160;products&#160;for&#160;each&#160;fragment.&#160;</b>The&#160;real&#160;experiment&#160;is&#160;shown&#160;in&#160;<b>(a)&#160;</b>along&#160;with&#160;the&#160;<b>(b)&#160;</b>simulation&#160;and&#160;<b>(c)&#160;</b>graph&#160;representations.&#160;In<br/>each&#160;case,&#160;the&#160;second&#160;panel&#160;represents&#160;the&#160;worms&#160;immediately&#160;following&#160;the&#160;cut,&#160;whereas&#160;the&#160;third&#160;panel&#160;depicts&#160;the&#160;regeneration&#160;outcome&#160;at&#160;a<br/>later&#160;time.</p>
<p style="position:absolute;top:1050px;left:84px;white-space:nowrap" class="ft46">of&#160;the&#160;subsequent&#160;generation&#160;are&#160;selected,&#160;in&#160;part,&#160;based<br/>upon&#160;a&#160;fitness&#160;metric,&#160;which&#160;quantitatively&#160;defines&#160;how<br/>well&#160;each&#160;individual&#160;matches&#160;the&#160;characteristics&#160;of&#160;the</p>
<p style="position:absolute;top:1050px;left:456px;white-space:nowrap" class="ft46">target.&#160;This&#160;evolutionary&#160;search&#160;technique&#160;continues&#160;in<br/>an&#160;automated&#160;fashion&#160;until&#160;an&#160;individual&#160;matching&#160;the<br/>desired&#160;target&#160;(fitness&#160;value&#160;of&#160;1.0)&#160;is&#160;generated.</p>
</div>
</body>
</html>
<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="" xml:lang="">
<head>
<title>e1aa43c837743d72c31dbd52d6a9caac-html.html</title>

<meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
 <br/>
<style type="text/css">
<!--
	p {margin: 0; padding: 0;}	.ft50{font-size:10px;font-family:Times;color:#000000;}
	.ft51{font-size:10px;font-family:Times;color:#000000;}
	.ft52{font-size:10px;font-family:Times;color:#000000;}
	.ft53{font-size:5px;font-family:Times;color:#000000;}
	.ft54{font-size:16px;font-family:Times;color:#000000;}
	.ft55{font-size:5px;font-family:Times;color:#0000ff;}
	.ft56{font-size:11px;font-family:Times;color:#000000;}
	.ft57{font-size:12px;font-family:Times;color:#000000;}
	.ft58{font-size:10px;line-height:14px;font-family:Times;color:#000000;}
	.ft59{font-size:11px;line-height:17px;font-family:Times;color:#000000;}
	.ft510{font-size:12px;line-height:17px;font-family:Times;color:#000000;}
-->
</style>
</head>
<body bgcolor="#A0A0A0" vlink="blue" link="blue">
<div id="page5-div" style="position:relative;width:892px;height:1191px;">
<p style="position:absolute;top:53px;left:86px;white-space:nowrap" class="ft50">Budnikova&#160;<i>et&#160;al.&#160;BMC&#160;Bioinformatics&#160;</i>2014,&#160;<b>15</b>:178</p>
<p style="position:absolute;top:53px;left:748px;white-space:nowrap" class="ft50">Page&#160;5&#160;of&#160;14</p>
<p style="position:absolute;top:68px;left:86px;white-space:nowrap" class="ft50">http://www.biomedcentral.com/1471-2105/15/178</p>
<p style="position:absolute;top:396px;left:206px;white-space:nowrap" class="ft53">GENE 0</p>
<p style="position:absolute;top:377px;left:178px;white-space:nowrap" class="ft54">b</p>
<p style="position:absolute;top:144px;left:178px;white-space:nowrap" class="ft54">a</p>
<p style="position:absolute;top:437px;left:211px;white-space:nowrap" class="ft53">hCell</p>
<p style="position:absolute;top:609px;left:251px;white-space:nowrap" class="ft53">Regeneration</p>
<p style="position:absolute;top:481px;left:173px;white-space:nowrap" class="ft53">GENE 1</p>
<p style="position:absolute;top:565px;left:259px;white-space:nowrap" class="ft53">GENE 4</p>
<p style="position:absolute;top:481px;left:239px;white-space:nowrap" class="ft53">GENE 2</p>
<p style="position:absolute;top:565px;left:101px;white-space:nowrap" class="ft53">GENE 5</p>
<p style="position:absolute;top:523px;left:396px;white-space:nowrap" class="ft53">Head gradient</p>
<p style="position:absolute;top:565px;left:409px;white-space:nowrap" class="ft53">EQ 0</p>
<p style="position:absolute;top:523px;left:154px;white-space:nowrap" class="ft53">tCell</p>
<p style="position:absolute;top:565px;left:146px;white-space:nowrap" class="ft53">GENE 3</p>
<p style="position:absolute;top:609px;left:173px;white-space:nowrap" class="ft53">Tail gradient</p>
<p style="position:absolute;top:653px;left:252px;white-space:nowrap" class="ft53">EQ 1</p>
<p style="position:absolute;top:523px;left:502px;white-space:nowrap" class="ft53">Regeneration signal</p>
<p style="position:absolute;top:609px;left:108px;white-space:nowrap" class="ft53">iCell</p>
<p style="position:absolute;top:609px;left:365px;white-space:nowrap" class="ft53">Head gradient</p>
<p style="position:absolute;top:653px;left:389px;white-space:nowrap" class="ft53">EQ 6</p>
<p style="position:absolute;top:698px;left:318px;white-space:nowrap" class="ft53">Tail gradient</p>
<p style="position:absolute;top:742px;left:394px;white-space:nowrap" class="ft53">EQ 7</p>
<p style="position:absolute;top:481px;left:411px;white-space:nowrap" class="ft53">EQ 2</p>
<p style="position:absolute;top:437px;left:465px;white-space:nowrap" class="ft53">Head gradient</p>
<p style="position:absolute;top:437px;left:378px;white-space:nowrap" class="ft53">Gap junctions</p>
<p style="position:absolute;top:481px;left:307px;white-space:nowrap" class="ft53">EQ 3</p>
<p style="position:absolute;top:437px;left:278px;white-space:nowrap" class="ft53">Tail gradient</p>
<p style="position:absolute;top:481px;left:542px;white-space:nowrap" class="ft53">EQ 4</p>
<p style="position:absolute;top:437px;left:546px;white-space:nowrap" class="ft53">cut</p>
<p style="position:absolute;top:437px;left:589px;white-space:nowrap" class="ft53">membrane</p>
<p style="position:absolute;top:481px;left:696px;white-space:nowrap" class="ft53">EQ 5</p>
<p style="position:absolute;top:523px;left:700px;white-space:nowrap" class="ft53">cut</p>
<p style="position:absolute;top:437px;left:730px;white-space:nowrap" class="ft53">Lysis</p>
<p style="position:absolute;top:481px;left:768px;white-space:nowrap" class="ft55">Die</p>
<p style="position:absolute;top:609px;left:453px;white-space:nowrap" class="ft53">Gap junctions</p>
<p style="position:absolute;top:761px;left:95px;white-space:nowrap" class="ft58"><b>Figure&#160;2&#160;General&#160;network&#160;scheme&#160;of&#160;the&#160;virtual&#160;planaria.&#160;</b>(a)&#160;Depiction&#160;of&#160;a&#160;regulatory&#160;model&#160;based&#160;upon&#160;a&#160;competitive&#160;development&#160;pathway<br/>initiated&#160;by&#160;a&#160;regeneration&#160;signal&#160;and&#160;repressed&#160;by&#160;active&#160;long-range&#160;morphogen&#160;gradients.&#160;(b)&#160;Representation&#160;of&#160;the&#160;principal&#160;components&#160;of&#160;the<br/>network&#160;as&#160;described&#160;in&#160;the&#160;Cellsim&#160;architecture.&#160;<b>(a)&#160;</b>Overview&#160;of&#160;general&#160;network&#160;scheme.&#160;<b>(b)&#160;</b>Regulatory&#160;network&#160;defined&#160;in&#160;Cellsim&#160;modeling<br/>platform.</p>
<p style="position:absolute;top:872px;left:86px;white-space:nowrap" class="ft510"><b>Graph&#160;formalism&#160;provides&#160;a&#160;convenient&#160;means&#160;of&#160;storing<br/>morphologies&#160;and&#160;comparing&#160;worms<br/></b>The&#160;challenge&#160;of&#160;automating&#160;searches&#160;to&#160;identify&#160;possi-<br/>ble&#160;planarian&#160;regeneration&#160;mechanisms&#160;was&#160;made&#160;more<br/>tractable&#160;by&#160;the&#160;database&#160;and&#160;formalism&#160;developed&#160;to<br/>describe&#160;wet-bench&#160;experiments&#160;and&#160;outcomes&#160;<a href="e1aa43c837743d72c31dbd52d6a9caacs.html#14">[15,16].<br/></a>Within&#160;PlanformDB,&#160;worm&#160;morphologies&#160;are&#160;described<br/>using&#160;a&#160;graph-based&#160;formalism&#160;as&#160;part&#160;of&#160;a&#160;more&#160;gen-<br/>eral&#160;ontology&#160;for&#160;describing&#160;regeneration&#160;experiments.<br/>Briefly,&#160;a&#160;graph&#160;defines&#160;anatomic&#160;regions&#160;and&#160;organs&#160;as<br/>nodes&#160;where&#160;their&#160;size,&#160;spatial&#160;orientation,&#160;and&#160;connec-<br/>tions&#160;are&#160;defined&#160;by&#160;parameters&#160;and&#160;linkages&#160;between<br/>adjoined&#160;nodes.&#160;For&#160;example,&#160;a&#160;simple&#160;description&#160;of&#160;the</p>
<p style="position:absolute;top:871px;left:458px;white-space:nowrap" class="ft510">regions&#160;within&#160;a&#160;normal&#160;planarian&#160;consists&#160;of&#160;three&#160;con-<br/>nected&#160;nodes&#160;(head,&#160;trunk,&#160;and&#160;tail)&#160;as&#160;shown&#160;in&#160;Figure&#160;<a href="e1aa43c837743d72c31dbd52d6a9caacs.html#4">1c</a>.<br/>Although&#160;the&#160;formalism&#160;supports&#160;more&#160;complex&#160;descrip-<br/>tions&#160;of&#160;worms&#160;including&#160;organs,&#160;those&#160;aspects&#160;of&#160;worm<br/>anatomy&#160;were&#160;not&#160;considered&#160;in&#160;the&#160;current&#160;work.&#160;A<br/>particular&#160;experiment&#160;may&#160;include&#160;a&#160;description&#160;of&#160;the<br/>observed&#160;starting,&#160;intermediate,&#160;and&#160;ending&#160;morphologies<br/>of&#160;worms&#160;along&#160;with&#160;the&#160;physical&#160;or&#160;chemical&#160;manipula-<br/>tions&#160;performed&#160;in&#160;the&#160;laboratory.&#160;The&#160;database&#160;currently<br/>describes&#160;most,&#160;if&#160;not&#160;all,&#160;of&#160;the&#160;published&#160;planarian&#160;regen-<br/>eration&#160;experiments&#160;for&#160;use&#160;by&#160;this&#160;and&#160;other&#160;projects.</p>
<p style="position:absolute;top:1068px;left:470px;white-space:nowrap" class="ft57">In&#160;this&#160;study,&#160;we&#160;extended&#160;and&#160;adapted&#160;an&#160;existing</p>
<p style="position:absolute;top:1086px;left:458px;white-space:nowrap" class="ft57">genetic&#160;algorithm&#160;(CSGA)&#160;to&#160;fit&#160;our&#160;needs&#160;to&#160;model&#160;and</p>
</div>
</body>
</html>
<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="" xml:lang="">
<head>
<title>e1aa43c837743d72c31dbd52d6a9caac-html.html</title>

<meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
 <br/>
<style type="text/css">
<!--
	p {margin: 0; padding: 0;}	.ft60{font-size:10px;font-family:Times;color:#000000;}
	.ft61{font-size:10px;font-family:Times;color:#000000;}
	.ft62{font-size:10px;font-family:Times;color:#000000;}
	.ft63{font-size:12px;font-family:Times;color:#000000;}
	.ft64{font-size:11px;font-family:Times;color:#000000;}
	.ft65{font-size:12px;line-height:17px;font-family:Times;color:#000000;}
	.ft66{font-size:11px;line-height:17px;font-family:Times;color:#000000;}
	.ft67{font-size:11px;line-height:18px;font-family:Times;color:#000000;}
	.ft68{font-size:10px;line-height:14px;font-family:Times;color:#000000;}
	.ft69{font-size:10px;line-height:15px;font-family:Times;color:#000000;}
	.ft610{font-size:10px;line-height:14px;font-family:Times;color:#000000;}
-->
</style>
</head>
<body bgcolor="#A0A0A0" vlink="blue" link="blue">
<div id="page6-div" style="position:relative;width:892px;height:1191px;">
<p style="position:absolute;top:53px;left:84px;white-space:nowrap" class="ft60">Budnikova&#160;<i>et&#160;al.&#160;BMC&#160;Bioinformatics&#160;</i>2014,&#160;<b>15</b>:178</p>
<p style="position:absolute;top:53px;left:746px;white-space:nowrap" class="ft60">Page&#160;6&#160;of&#160;14</p>
<p style="position:absolute;top:68px;left:84px;white-space:nowrap" class="ft60">http://www.biomedcentral.com/1471-2105/15/178</p>
<p style="position:absolute;top:136px;left:84px;white-space:nowrap" class="ft65">evaluate&#160;planarian&#160;regeneration.&#160;One&#160;of&#160;the&#160;key&#160;adap-<br/>tations&#160;was&#160;providing&#160;the&#160;CSGA&#160;access&#160;to&#160;the&#160;Plan-<br/>formDB&#160;to&#160;facilitate&#160;simulation&#160;and&#160;fitness&#160;evaluation.<br/>However,&#160;the&#160;challenge&#160;of&#160;comparing&#160;our&#160;agent-based<br/>simulation&#160;output&#160;to&#160;a&#160;graph-based&#160;representation&#160;pre-<br/>sented&#160;a&#160;significant&#160;challenge.&#160;In&#160;order&#160;to&#160;facilitate&#160;these<br/>comparisons,&#160;we&#160;chose&#160;to&#160;convert&#160;simulation&#160;results&#160;into<br/>a&#160;graph&#160;representation&#160;for&#160;a&#160;number&#160;of&#160;reasons,&#160;includ-<br/>ing&#160;increased&#160;flexibility&#160;as&#160;the&#160;CSGA&#160;could&#160;be&#160;extended<br/>to&#160;support&#160;additional&#160;modeling&#160;platforms&#160;as&#160;long&#160;as&#160;their<br/>output&#160;could&#160;also&#160;be&#160;translated&#160;into&#160;this&#160;graph&#160;formal-<br/>ism.&#160;More&#160;importantly,&#160;many&#160;methods&#160;currently&#160;exist<br/>for&#160;operating&#160;on,&#160;transforming,&#160;and&#160;comparing&#160;graphs<br/>which&#160;can&#160;be&#160;included&#160;as&#160;part&#160;of&#160;the&#160;fitness&#160;evalua-<br/>tion&#160;step&#160;of&#160;an&#160;automated&#160;evolutionary&#160;search&#160;<a href="e1aa43c837743d72c31dbd52d6a9caacs.html#14">[19-21].<br/></a>Included&#160;in&#160;this&#160;repertoire&#160;are&#160;a&#160;number&#160;of&#160;algorithms<br/>suited&#160;for&#160;measuring&#160;similarity&#160;between&#160;two&#160;graphs&#160;<a href="e1aa43c837743d72c31dbd52d6a9caacs.html#14">[22].<br/></a>Of&#160;these,&#160;the&#160;graph&#160;edit&#160;distance&#160;algorithm&#160;is&#160;the&#160;most<br/>flexible&#160;and&#160;powerful&#160;and&#160;was&#160;chosen&#160;here&#160;as&#160;it&#160;deals&#160;with<br/>structural&#160;errors&#160;and&#160;any&#160;type&#160;of&#160;graph&#160;node&#160;and&#160;edge<br/>labels&#160;<a href="e1aa43c837743d72c31dbd52d6a9caacs.html#14">[23,24].</a></p>
<p style="position:absolute;top:512px;left:96px;white-space:nowrap" class="ft63">The&#160;graph&#160;edit&#160;distance&#160;is&#160;defined&#160;as&#160;the&#160;minimum</p>
<p style="position:absolute;top:530px;left:84px;white-space:nowrap" class="ft65">number&#160;of&#160;distortions&#160;required&#160;for&#160;transforming&#160;one<br/>graph&#160;into&#160;another.&#160;These&#160;distortions&#160;are&#160;referred&#160;to&#160;as<br/>graph&#160;edit&#160;operations,&#160;where&#160;each&#160;edit&#160;has&#160;a&#160;defined&#160;cost<br/>associated&#160;with&#160;it&#160;<a href="e1aa43c837743d72c31dbd52d6a9caacs.html#14">[23].&#160;</a>A&#160;particular&#160;sequence&#160;of&#160;edit&#160;oper-<br/>ations&#160;is&#160;called&#160;an&#160;edit&#160;path,&#160;and&#160;the&#160;total&#160;cost&#160;of&#160;the&#160;edit<br/>path&#160;is&#160;the&#160;graph&#160;edit&#160;distance.&#160;Graphs&#160;that&#160;are&#160;similar<br/>to&#160;each&#160;other&#160;typically&#160;have&#160;small&#160;edit&#160;distances,&#160;whereas<br/>dissimilar&#160;graphs&#160;have&#160;large&#160;edit&#160;distances.&#160;The&#160;cost&#160;of<br/>each&#160;type&#160;of&#160;graph&#160;edit&#160;operation&#160;varies&#160;and&#160;is&#160;dependent<br/>upon&#160;the&#160;perceived&#160;severity&#160;of&#160;the&#160;operation.&#160;For&#160;example,<br/>the&#160;deletion&#160;of&#160;a&#160;node&#160;from&#160;a&#160;graph&#160;is&#160;generally&#160;viewed<br/>as&#160;having&#160;a&#160;higher&#160;cost&#160;than&#160;a&#160;node&#160;parameter&#160;change.<br/>Thus,&#160;the&#160;graph&#160;edit&#160;distance&#160;can&#160;be&#160;used&#160;as&#160;a&#160;quantita-<br/>tive&#160;similarity&#160;measure&#160;to&#160;compare&#160;and&#160;order&#160;individuals<br/>within&#160;a&#160;population,&#160;and&#160;thus&#160;serve&#160;as&#160;a&#160;metric&#160;within<br/>a&#160;fitness&#160;evaluation&#160;to&#160;guide&#160;the&#160;evolutionary&#160;search<br/>process.</p>
<p style="position:absolute;top:854px;left:84px;white-space:nowrap" class="ft65"><b>Design&#160;of&#160;a&#160;connected&#160;component&#160;analysis&#160;algorithm<br/>to&#160;convert&#160;cell&#160;simulation&#160;output&#160;into&#160;graph<br/>representations<br/></b>The&#160;worms&#160;in&#160;our&#160;simulation&#160;are&#160;composed&#160;of&#160;a&#160;collection<br/>of&#160;discrete&#160;cells&#160;rather&#160;than&#160;interconnected&#160;regions.&#160;Thus,<br/>the&#160;initial&#160;step&#160;in&#160;deriving&#160;a&#160;graph-based&#160;representation&#160;is<br/>to&#160;translate&#160;cells&#160;within&#160;a&#160;simulation&#160;snapshot&#160;into&#160;discrete<br/>regions&#160;(e.g.&#160;head,&#160;trunk,&#160;or&#160;tail),&#160;and&#160;determine&#160;how&#160;they<br/>are&#160;interconnected.&#160;To&#160;this&#160;end,&#160;we&#160;designed&#160;and&#160;imple-<br/>mented&#160;an&#160;algorithm&#160;based&#160;upon&#160;connected&#160;component<br/>analysis&#160;similar&#160;to&#160;methods&#160;used&#160;in&#160;computer&#160;vision&#160;and<br/>document&#160;analysis&#160;<a href="e1aa43c837743d72c31dbd52d6a9caacs.html#14">[25].</a></p>
<p style="position:absolute;top:1068px;left:96px;white-space:nowrap" class="ft63">A&#160;simulation&#160;is&#160;processed&#160;as&#160;a&#160;series&#160;of&#160;discrete&#160;steps</p>
<p style="position:absolute;top:1086px;left:84px;white-space:nowrap" class="ft63">where&#160;each&#160;step&#160;consists&#160;of&#160;a&#160;complete&#160;description&#160;of</p>
<p style="position:absolute;top:136px;left:456px;white-space:nowrap" class="ft65">cellular&#160;locations&#160;and&#160;their&#160;individual&#160;resources&#160;(snap-<br/>shot).&#160;The&#160;snapshot&#160;associated&#160;with&#160;each&#160;step&#160;can&#160;be<br/>independently&#160;analyzed&#160;by&#160;the&#160;algorithm&#160;to&#160;identify&#160;cell<br/>states&#160;and&#160;identify&#160;regions&#160;over&#160;time,&#160;or&#160;a&#160;single&#160;endpoint<br/>can&#160;be&#160;examined.&#160;The&#160;algorithm&#160;first&#160;iterates&#160;through&#160;all<br/>the&#160;cells&#160;in&#160;a&#160;snapshot&#160;and&#160;assigns&#160;each&#160;cell&#160;a&#160;region&#160;type<br/>(e.g.&#160;head,&#160;trunk,&#160;or&#160;tail).&#160;In&#160;general,&#160;the&#160;assignment&#160;of<br/>a&#160;cell&#160;type&#160;could&#160;be&#160;complex&#160;as&#160;there&#160;are&#160;many&#160;differ-<br/>ent&#160;components&#160;(e.g.&#160;proteins/resources&#160;or&#160;neighboring<br/>cell&#160;interactions)&#160;associated&#160;with&#160;each&#160;cell.&#160;In&#160;our&#160;case,<br/>we&#160;decided&#160;to&#160;simply&#160;define&#160;each&#160;cellâs&#160;state&#160;based&#160;upon<br/>the&#160;molecular&#160;concentrations&#160;of&#160;three&#160;resources,&#160;which<br/>we&#160;labelled&#160;hCell&#160;(head),&#160;iCell&#160;(trunk),&#160;and&#160;tCell&#160;(tail).<br/>Our&#160;algorithm&#160;assigns&#160;a&#160;region&#160;type&#160;to&#160;each&#160;cell&#160;based<br/>upon&#160;the&#160;highest&#160;total&#160;concentration&#160;of&#160;these&#160;indicator<br/>resources.&#160;For&#160;example,&#160;a&#160;cell&#160;is&#160;assigned&#160;a&#160;head&#160;state&#160;if<br/>its&#160;concentration&#160;of&#160;hCell&#160;is&#160;greater&#160;than&#160;iCell&#160;and&#160;tCell.<br/>In&#160;the&#160;modeling&#160;platform,&#160;transcriptional&#160;noise&#160;and&#160;cellu-<br/>lar&#160;autonomy&#160;result&#160;in&#160;cells&#160;with&#160;varying&#160;concentrations&#160;of<br/>resources,&#160;even&#160;those&#160;located&#160;near&#160;each&#160;other&#160;(Figure&#160;<a href="e1aa43c837743d72c31dbd52d6a9caacs.html#6">3).<br/></a>Since&#160;a&#160;resource&#160;may&#160;be&#160;found&#160;on&#160;the&#160;cell&#160;surface&#160;(S)<br/>or&#160;internally&#160;(I),&#160;the&#160;algorithm&#160;was&#160;designed&#160;with&#160;the<br/>flexibility&#160;to&#160;allow&#160;the&#160;user&#160;to&#160;define&#160;whether&#160;to&#160;con-<br/>sider&#160;the&#160;total&#160;or&#160;localized&#160;concentrations&#160;of&#160;resources.<br/>We&#160;used&#160;total&#160;cellular&#160;concentration&#160;of&#160;each&#160;of&#160;the&#160;three<br/>indicator&#160;resources&#160;to&#160;determine&#160;a&#160;cellâs&#160;state&#160;in&#160;this<br/>work.</p>
<p style="position:absolute;top:941px;left:466px;white-space:nowrap" class="ft610"><b>Figure&#160;3&#160;The&#160;molecular&#160;concentration&#160;of&#160;resources&#160;varies<br/>between&#160;and&#160;within&#160;cells&#160;during&#160;a&#160;simulation.&#160;</b>The&#160;concentration<br/>of&#160;a&#160;particular&#160;resource&#160;is&#160;governed&#160;by&#160;spatial&#160;and&#160;environmental<br/>cues,&#160;such&#160;as&#160;signals&#160;from&#160;neighboring&#160;cells.&#160;Within&#160;a&#160;given&#160;cell&#160;the<br/>location&#160;of&#160;a&#160;given&#160;resource&#160;can&#160;be&#160;distributed&#160;between&#160;the&#160;internal<br/>compartment&#160;(e.g.&#160;cytosol)&#160;and&#160;the&#160;surface&#160;(e.g.&#160;membrane).&#160;The<br/>differentiate&#160;state&#160;of&#160;cells&#160;are&#160;color-coded&#160;to&#160;enable&#160;visual&#160;distinction<br/>of&#160;cells&#160;and&#160;the&#160;composition&#160;of&#160;a&#160;region:&#160;head&#160;(blue),&#160;trunk&#160;(yellow)<br/>and&#160;tail&#160;(purple).&#160;Concentrations&#160;of&#160;representative&#160;resources&#160;inside&#160;(I)<br/>and&#160;on&#160;the&#160;surface&#160;(S)&#160;are&#160;provided.</p>
</div>
</body>
</html>
<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="" xml:lang="">
<head>
<title>e1aa43c837743d72c31dbd52d6a9caac-html.html</title>

<meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
 <br/>
<style type="text/css">
<!--
	p {margin: 0; padding: 0;}	.ft70{font-size:10px;font-family:Times;color:#000000;}
	.ft71{font-size:10px;font-family:Times;color:#000000;}
	.ft72{font-size:10px;font-family:Times;color:#000000;}
	.ft73{font-size:12px;font-family:Times;color:#000000;}
	.ft74{font-size:11px;font-family:Times;color:#000000;}
	.ft75{font-size:12px;font-family:Times;color:#000000;}
	.ft76{font-size:12px;line-height:17px;font-family:Times;color:#000000;}
	.ft77{font-size:11px;line-height:18px;font-family:Times;color:#000000;}
	.ft78{font-size:11px;line-height:17px;font-family:Times;color:#000000;}
-->
</style>
</head>
<body bgcolor="#A0A0A0" vlink="blue" link="blue">
<div id="page7-div" style="position:relative;width:892px;height:1191px;">
<p style="position:absolute;top:53px;left:86px;white-space:nowrap" class="ft70">Budnikova&#160;<i>et&#160;al.&#160;BMC&#160;Bioinformatics&#160;</i>2014,&#160;<b>15</b>:178</p>
<p style="position:absolute;top:53px;left:748px;white-space:nowrap" class="ft70">Page&#160;7&#160;of&#160;14</p>
<p style="position:absolute;top:68px;left:86px;white-space:nowrap" class="ft70">http://www.biomedcentral.com/1471-2105/15/178</p>
<p style="position:absolute;top:136px;left:98px;white-space:nowrap" class="ft73">Once&#160;each&#160;cellâs&#160;state&#160;has&#160;been&#160;determined,&#160;the&#160;algo-</p>
<p style="position:absolute;top:153px;left:86px;white-space:nowrap" class="ft76">rithm&#160;identifies&#160;regions&#160;based&#160;upon&#160;spatially&#160;cohesive&#160;cells<br/>sharing&#160;the&#160;same&#160;state&#160;using&#160;connected&#160;component&#160;analy-<br/>sis.&#160;All&#160;cells&#160;located&#160;adjacent&#160;to&#160;cells&#160;containing&#160;the&#160;same<br/>state&#160;are&#160;considered&#160;part&#160;of&#160;the&#160;same&#160;region,&#160;where&#160;the<br/>outermost&#160;cells&#160;define&#160;the&#160;region&#160;border.&#160;The&#160;connected<br/>component&#160;analysis&#160;algorithm&#160;(Algorithm&#160;<a href="e1aa43c837743d72c31dbd52d6a9caacs.html#7">1)&#160;</a>initiates&#160;with<br/>a&#160;call&#160;to&#160;the&#160;ProcessConnectedComponents&#160;function&#160;pro-<br/>viding&#160;a&#160;simulation&#160;snapshot&#160;as&#160;a&#160;parameter.&#160;The&#160;Pro-<br/>cessConnectedComponents&#160;function&#160;cycles&#160;through&#160;all<br/>cells&#160;and&#160;calls&#160;the&#160;GatherConnected&#160;function&#160;for&#160;each<br/>unassigned/marked&#160;cell.&#160;The&#160;GatherConnected&#160;function<br/>recursively&#160;collects&#160;and&#160;marks&#160;all&#160;other&#160;cells&#160;in&#160;the&#160;snap-<br/>shot&#160;that&#160;belong&#160;to&#160;the&#160;same&#160;spatially&#160;cohesive&#160;region&#160;as<br/>the&#160;starting&#160;cell.&#160;A&#160;cell&#160;is&#160;defined&#160;to&#160;be&#160;in&#160;the&#160;same&#160;spa-<br/>tially&#160;cohesive&#160;region&#160;as&#160;the&#160;starting&#160;cell&#160;if&#160;it&#160;is&#160;of&#160;the<br/>same&#160;type&#160;as&#160;the&#160;starting&#160;cell&#160;and&#160;is&#160;either&#160;connected&#160;to<br/>the&#160;starting&#160;cell&#160;or&#160;to&#160;some&#160;other&#160;cell&#160;already&#160;determined<br/>to&#160;be&#160;in&#160;the&#160;starting&#160;cellâs&#160;region.&#160;Two&#160;cells&#160;are&#160;consid-<br/>ered&#160;to&#160;be&#160;connected&#160;if&#160;the&#160;Euclidean&#160;distance&#160;between<br/>them&#160;is&#160;below&#160;a&#160;user-specified&#160;threshold&#160;(discussed&#160;in<br/>section&#160;<a href="e1aa43c837743d72c31dbd52d6a9caacs.html#9">âA&#160;cell&#160;connectivity&#160;distance&#160;threshold&#160;effects<br/>region&#160;determinationâ).&#160;</a>Additionally,&#160;if&#160;two&#160;cells&#160;are&#160;close<br/>enough&#160;to&#160;each&#160;other&#160;to&#160;be&#160;considered&#160;connected,&#160;but&#160;are<br/>assigned&#160;to&#160;different&#160;regions&#160;because&#160;they&#160;are&#160;of&#160;differ-<br/>ent&#160;types,&#160;those&#160;cells&#160;are&#160;identified&#160;as&#160;border&#160;cells.&#160;Border<br/>cells&#160;are&#160;used&#160;to&#160;determine&#160;which&#160;regions&#160;are&#160;linked&#160;to<br/>each&#160;other.</p>
<p style="position:absolute;top:136px;left:470px;white-space:nowrap" class="ft73">Once&#160;each&#160;cell&#160;in&#160;the&#160;snapshot&#160;is&#160;assigned&#160;to&#160;a&#160;specific</p>
<p style="position:absolute;top:154px;left:458px;white-space:nowrap" class="ft76">region,&#160;the&#160;algorithm&#160;determines&#160;the&#160;number&#160;of&#160;neigh-<br/>boring&#160;regions&#160;using&#160;the&#160;border&#160;cells&#160;found&#160;during&#160;the<br/>recursive&#160;process&#160;and&#160;establishes&#160;links&#160;between&#160;nodes<br/>where&#160;regions&#160;are&#160;considered&#160;linked&#160;if&#160;their&#160;border&#160;cells<br/>are&#160;adjacent&#160;to&#160;each&#160;other.&#160;Other&#160;necessary&#160;parameters<br/>for&#160;a&#160;complete&#160;graph&#160;representation&#160;include&#160;the&#160;distance<br/>between&#160;the&#160;connected&#160;regionsâ&#160;centers&#160;(length&#160;of&#160;link),<br/>orientation&#160;with&#160;respect&#160;to&#160;each&#160;other&#160;(angle&#160;of&#160;the&#160;link<br/>relative&#160;to&#160;the&#160;x-axis),&#160;and&#160;the&#160;border&#160;between&#160;the&#160;two<br/>regions&#160;(location&#160;along&#160;the&#160;link&#160;where&#160;the&#160;two&#160;regions<br/>meet).&#160;The&#160;center&#160;of&#160;a&#160;region&#160;is&#160;calculated&#160;by&#160;averaging<br/>the&#160;spatial&#160;centers&#160;of&#160;every&#160;cell&#160;within&#160;a&#160;particular&#160;region.<br/>The&#160;Euclidian&#160;distance&#160;between&#160;these&#160;points&#160;of&#160;neighbor-<br/>ing&#160;regions&#160;is&#160;used&#160;to&#160;define&#160;the&#160;length&#160;and&#160;orientation&#160;of<br/>each&#160;link.&#160;Finally,&#160;the&#160;graph&#160;component&#160;parameter&#160;defin-<br/>ing&#160;the&#160;borders&#160;of&#160;regions&#160;in&#160;each&#160;direction&#160;is&#160;calculated<br/>from&#160;the&#160;location&#160;of&#160;the&#160;most&#160;distantly&#160;located&#160;cell&#160;in&#160;a<br/>specific&#160;direction.&#160;The&#160;number&#160;of&#160;parameters&#160;for&#160;a&#160;region<br/>depends&#160;on&#160;the&#160;number&#160;of&#160;links&#160;it&#160;has&#160;with&#160;other&#160;regions.<br/>Figures&#160;<a href="e1aa43c837743d72c31dbd52d6a9caacs.html#4">1(</a>b&#160;to&#160;c)&#160;show&#160;examples&#160;of&#160;simulation&#160;morpholo-<br/>gies&#160;that&#160;have&#160;been&#160;converted&#160;to&#160;a&#160;graph&#160;formalism&#160;using<br/>this&#160;algorithm.</p>
<p style="position:absolute;top:567px;left:458px;white-space:nowrap" class="ft76"><b>Simulation&#160;snapshots&#160;are&#160;converted&#160;to&#160;well-ordered&#160;graphs<br/>using&#160;our&#160;conversion&#160;and&#160;graph-edit&#160;distance&#160;algorithms<br/></b>During&#160;an&#160;evolutionary&#160;search,&#160;large&#160;numbers&#160;of&#160;unique<br/>individuals&#160;are&#160;generated&#160;and&#160;must&#160;be&#160;evaluated&#160;against</p>
<p style="position:absolute;top:682px;left:86px;white-space:nowrap" class="ft75"><b>Algorithm&#160;1</b></p>
<p style="position:absolute;top:682px;left:170px;white-space:nowrap" class="ft73">Recursive&#160;pseudocode&#160;for&#160;connected&#160;component&#160;analysis&#160;algorithm&#160;to&#160;separate&#160;a&#160;list&#160;of&#160;cells&#160;taken&#160;from</p>
<p style="position:absolute;top:700px;left:86px;white-space:nowrap" class="ft73">the&#160;simulation&#160;snapshots&#160;into&#160;discrete&#160;morphology&#160;regions.</p>
<p style="position:absolute;top:731px;left:149px;white-space:nowrap" class="ft73">ProcessConnectedComponents&#160;(&#160;s&#160;n&#160;a&#160;p&#160;s&#160;h&#160;o&#160;t&#160;)&#160;:</p>
<p style="position:absolute;top:749px;left:221px;white-space:nowrap" class="ft73">l&#160;i&#160;s&#160;t&#160;=&#160;new&#160;l&#160;i&#160;s&#160;t&#160;o&#160;f&#160;c&#160;o&#160;n&#160;n&#160;e&#160;c&#160;t&#160;e&#160;d&#160;components</p>
<p style="position:absolute;top:767px;left:220px;white-space:nowrap" class="ft73">f&#160;o&#160;r&#160;e&#160;a&#160;c&#160;h&#160;u&#160;n&#160;p&#160;r&#160;o&#160;c&#160;e&#160;s&#160;s&#160;e&#160;d&#160;c&#160;e&#160;l&#160;l&#160;c&#160;i&#160;n&#160;s&#160;n&#160;a&#160;p&#160;s&#160;h&#160;o&#160;t&#160;:</p>
<p style="position:absolute;top:785px;left:288px;white-space:nowrap" class="ft76">comp&#160;=&#160;new&#160;c&#160;o&#160;n&#160;n&#160;e&#160;c&#160;t&#160;e&#160;d&#160;component<br/>comp&#160;.&#160;a&#160;d&#160;d&#160;C&#160;e&#160;l&#160;l&#160;(&#160;c&#160;)</p>
<p style="position:absolute;top:821px;left:289px;white-space:nowrap" class="ft73">c&#160;.&#160;s&#160;e&#160;t&#160;P&#160;r&#160;o&#160;c&#160;e&#160;s&#160;s&#160;e&#160;d&#160;(&#160;)</p>
<p style="position:absolute;top:839px;left:149px;white-space:nowrap" class="ft73">GatherConnected&#160;(&#160;c&#160;,&#160;comp&#160;,&#160;s&#160;n&#160;a&#160;p&#160;s&#160;h&#160;o&#160;t&#160;)</p>
<p style="position:absolute;top:857px;left:151px;white-space:nowrap" class="ft73">l&#160;i&#160;s&#160;t&#160;.&#160;add&#160;(&#160;comp&#160;)</p>
<p style="position:absolute;top:875px;left:150px;white-space:nowrap" class="ft73">f&#160;o&#160;r&#160;comp&#160;i&#160;n&#160;components&#160;:</p>
<p style="position:absolute;top:893px;left:148px;white-space:nowrap" class="ft73">comp&#160;.&#160;c&#160;a&#160;l&#160;c&#160;u&#160;l&#160;a&#160;t&#160;e&#160;P&#160;a&#160;r&#160;a&#160;m&#160;e&#160;t&#160;e&#160;r&#160;s&#160;(&#160;)</p>
<p style="position:absolute;top:929px;left:96px;white-space:nowrap" class="ft73">GatherConnected&#160;(&#160;c1&#160;,&#160;comp&#160;,&#160;s&#160;n&#160;a&#160;p&#160;s&#160;h&#160;o&#160;t&#160;)&#160;:</p>
<p style="position:absolute;top:947px;left:167px;white-space:nowrap" class="ft73">f&#160;o&#160;r&#160;e&#160;a&#160;c&#160;h&#160;u&#160;n&#160;p&#160;r&#160;o&#160;c&#160;e&#160;s&#160;s&#160;e&#160;d&#160;c&#160;e&#160;l&#160;l&#160;c2&#160;i&#160;n&#160;s&#160;n&#160;a&#160;p&#160;s&#160;h&#160;o&#160;t&#160;:</p>
<p style="position:absolute;top:965px;left:194px;white-space:nowrap" class="ft73">i&#160;f&#160;c&#160;o&#160;n&#160;n&#160;e&#160;c&#160;t&#160;e&#160;d&#160;(&#160;c1&#160;,&#160;c2&#160;)&#160;:</p>
<p style="position:absolute;top:982px;left:265px;white-space:nowrap" class="ft73">i&#160;f&#160;c1&#160;.&#160;t&#160;y&#160;p&#160;e&#160;i&#160;s&#160;no&#160;t&#160;c2&#160;.&#160;t&#160;y&#160;p&#160;e&#160;:</p>
<p style="position:absolute;top:1000px;left:333px;white-space:nowrap" class="ft73">mark&#160;c1&#160;and&#160;c2&#160;a&#160;s&#160;b&#160;o&#160;r&#160;d&#160;e&#160;r&#160;c&#160;e&#160;l&#160;l&#160;s</p>
<p style="position:absolute;top:1018px;left:264px;white-space:nowrap" class="ft73">e&#160;l&#160;s&#160;e&#160;:</p>
<p style="position:absolute;top:1036px;left:332px;white-space:nowrap" class="ft73">comp&#160;.&#160;a&#160;d&#160;d&#160;C&#160;e&#160;l&#160;l&#160;(&#160;c2&#160;)</p>
<p style="position:absolute;top:1054px;left:333px;white-space:nowrap" class="ft76">c2&#160;.&#160;s&#160;e&#160;t&#160;P&#160;r&#160;o&#160;c&#160;e&#160;s&#160;s&#160;e&#160;d&#160;(&#160;)<br/>GatherConnected&#160;(&#160;c2&#160;,&#160;comp&#160;,&#160;s&#160;n&#160;a&#160;p&#160;s&#160;h&#160;o&#160;t&#160;)</p>
</div>
</body>
</html>
<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="" xml:lang="">
<head>
<title>e1aa43c837743d72c31dbd52d6a9caac-html.html</title>

<meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
 <br/>
<style type="text/css">
<!--
	p {margin: 0; padding: 0;}	.ft80{font-size:10px;font-family:Times;color:#000000;}
	.ft81{font-size:10px;font-family:Times;color:#000000;}
	.ft82{font-size:10px;font-family:Times;color:#000000;}
	.ft83{font-size:12px;font-family:Times;color:#000000;}
	.ft84{font-size:12px;line-height:17px;font-family:Times;color:#000000;}
-->
</style>
</head>
<body bgcolor="#A0A0A0" vlink="blue" link="blue">
<div id="page8-div" style="position:relative;width:892px;height:1191px;">
<p style="position:absolute;top:53px;left:84px;white-space:nowrap" class="ft80">Budnikova&#160;<i>et&#160;al.&#160;BMC&#160;Bioinformatics&#160;</i>2014,&#160;<b>15</b>:178</p>
<p style="position:absolute;top:53px;left:746px;white-space:nowrap" class="ft80">Page&#160;8&#160;of&#160;14</p>
<p style="position:absolute;top:68px;left:84px;white-space:nowrap" class="ft80">http://www.biomedcentral.com/1471-2105/15/178</p>
<p style="position:absolute;top:136px;left:84px;white-space:nowrap" class="ft84">the&#160;target&#160;individual&#160;encoded&#160;in&#160;the&#160;database.&#160;Thus,&#160;we<br/>sought&#160;to&#160;evaluate&#160;our&#160;conversion&#160;algorithm&#160;manually&#160;to<br/>ensure&#160;the&#160;graph&#160;representations&#160;were&#160;intuitive&#160;and&#160;to<br/>evaluate&#160;the&#160;use&#160;of&#160;the&#160;graph&#160;edit&#160;distance&#160;metric&#160;for<br/>ordering&#160;individuals&#160;in&#160;the&#160;population.&#160;To&#160;this&#160;end,&#160;we<br/>generated&#160;a&#160;number&#160;of&#160;worms&#160;with&#160;distinct&#160;morpholo-<br/>gies&#160;by&#160;hand&#160;using&#160;the&#160;modeling&#160;platform&#160;and&#160;converted<br/>their&#160;snapshots&#160;into&#160;graph&#160;representations&#160;using&#160;our&#160;algo-<br/>rithm.&#160;The&#160;simplest&#160;individuals&#160;that&#160;can&#160;be&#160;represented<br/>by&#160;the&#160;simulation&#160;platform&#160;include&#160;worms&#160;with&#160;discrete<br/>regions,&#160;whereas&#160;more&#160;complicated&#160;morphologies&#160;con-<br/>sisting&#160;of&#160;regions&#160;contained&#160;within&#160;other&#160;regions&#160;could<br/>also&#160;exist.&#160;Just&#160;considering&#160;the&#160;basic&#160;morphologies,&#160;the<br/>number&#160;of&#160;individuals&#160;that&#160;can&#160;be&#160;formed&#160;and&#160;the&#160;search<br/>space&#160;for&#160;the&#160;genetic&#160;algorithm&#160;are&#160;infinite,&#160;and&#160;therefore<br/>our&#160;algorithm&#160;was&#160;tested&#160;using&#160;simple&#160;individuals&#160;before<br/>considering&#160;more&#160;complicated&#160;morphologies.</p>
<p style="position:absolute;top:440px;left:96px;white-space:nowrap" class="ft83">As&#160;shown&#160;in&#160;Figure&#160;<a href="e1aa43c837743d72c31dbd52d6a9caacs.html#8">4,&#160;</a>we&#160;generated&#160;a&#160;series&#160;of&#160;distinct</p>
<p style="position:absolute;top:458px;left:84px;white-space:nowrap" class="ft83">worms&#160;(ID&#160;1-13)&#160;for&#160;comparison&#160;with&#160;a&#160;desired&#160;target&#160;(ID</p>
<p style="position:absolute;top:136px;left:456px;white-space:nowrap" class="ft84">0).&#160;In&#160;each&#160;case,&#160;the&#160;worm&#160;representations&#160;included&#160;two<br/>fragments&#160;to&#160;simulate&#160;the&#160;state&#160;of&#160;worms&#160;following&#160;a&#160;sin-<br/>gle&#160;transverse&#160;cut.&#160;Each&#160;worm&#160;was&#160;generated&#160;by&#160;injection<br/>of&#160;the&#160;appropriate&#160;cell-state&#160;resource&#160;(i.e.&#160;hCell,&#160;tCell&#160;and<br/>iCell)&#160;to&#160;generate&#160;the&#160;desired&#160;regions&#160;within&#160;the&#160;worm<br/>fragments,&#160;resulting&#160;in&#160;different&#160;permutations&#160;of&#160;head,&#160;tail<br/>and&#160;trunk&#160;regions.&#160;Every&#160;test&#160;morphology&#160;was&#160;converted<br/>to&#160;a&#160;graph&#160;(Figure&#160;<a href="e1aa43c837743d72c31dbd52d6a9caacs.html#8">4,&#160;</a>Morphology&#160;Graph)&#160;using&#160;our&#160;con-<br/>version&#160;algorithm.&#160;We&#160;did&#160;not&#160;find&#160;discordance&#160;between<br/>the&#160;graphs&#160;generated&#160;by&#160;the&#160;conversion&#160;algorithm&#160;and<br/>those&#160;expected&#160;upon&#160;visual&#160;inspection&#160;of&#160;the&#160;simulation<br/>output.&#160;Thus,&#160;the&#160;algorithm&#160;was&#160;working&#160;as&#160;expected&#160;on<br/>these&#160;simple&#160;morphologies.</p>
<p style="position:absolute;top:369px;left:468px;white-space:nowrap" class="ft83">During&#160;an&#160;evolutionary&#160;search,&#160;the&#160;genetic&#160;algorithm</p>
<p style="position:absolute;top:387px;left:456px;white-space:nowrap" class="ft84">needs&#160;to&#160;compare&#160;individuals&#160;to&#160;the&#160;target&#160;and&#160;reward<br/>those&#160;individuals&#160;with&#160;morphologies&#160;most&#160;similar&#160;to&#160;the<br/>target&#160;individual&#160;as&#160;their&#160;offspring&#160;are&#160;more&#160;likely&#160;to<br/>possess&#160;the&#160;a&#160;reaction&#160;network&#160;capable&#160;of&#160;proper&#160;regen-<br/>eration.&#160;The&#160;genetic&#160;algorithm&#160;thus&#160;assigns&#160;fitness&#160;values</p>
<p style="position:absolute;top:1080px;left:94px;white-space:nowrap" class="ft82"><b>Figure&#160;4&#160;Single&#160;cut&#160;morphology&#160;experiment&#160;results&#160;for&#160;simulation&#160;snapshot&#160;to&#160;graph&#160;conversion&#160;and&#160;graph&#160;edit&#160;distance&#160;comparison.</b></p>
</div>
</body>
</html>
<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="" xml:lang="">
<head>
<title>e1aa43c837743d72c31dbd52d6a9caac-html.html</title>

<meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
 <br/>
<style type="text/css">
<!--
	p {margin: 0; padding: 0;}	.ft90{font-size:10px;font-family:Times;color:#000000;}
	.ft91{font-size:10px;font-family:Times;color:#000000;}
	.ft92{font-size:10px;font-family:Times;color:#000000;}
	.ft93{font-size:12px;font-family:Times;color:#000000;}
	.ft94{font-size:12px;font-family:Times;color:#000000;}
	.ft95{font-size:9px;font-family:Times;color:#000000;}
	.ft96{font-size:11px;font-family:Times;color:#000000;}
	.ft97{font-size:12px;line-height:17px;font-family:Times;color:#000000;}
	.ft98{font-size:11px;line-height:16px;font-family:Times;color:#000000;}
	.ft99{font-size:11px;line-height:18px;font-family:Times;color:#000000;}
	.ft910{font-size:11px;line-height:17px;font-family:Times;color:#000000;}
-->
</style>
</head>
<body bgcolor="#A0A0A0" vlink="blue" link="blue">
<div id="page9-div" style="position:relative;width:892px;height:1191px;">
<p style="position:absolute;top:53px;left:86px;white-space:nowrap" class="ft90">Budnikova&#160;<i>et&#160;al.&#160;BMC&#160;Bioinformatics&#160;</i>2014,&#160;<b>15</b>:178</p>
<p style="position:absolute;top:53px;left:748px;white-space:nowrap" class="ft90">Page&#160;9&#160;of&#160;14</p>
<p style="position:absolute;top:68px;left:86px;white-space:nowrap" class="ft90">http://www.biomedcentral.com/1471-2105/15/178</p>
<p style="position:absolute;top:136px;left:86px;white-space:nowrap" class="ft97">based&#160;upon&#160;how&#160;similar&#160;the&#160;individual&#160;is&#160;to&#160;the&#160;target,&#160;with<br/>more&#160;similar&#160;individuals&#160;getting&#160;higher&#160;fitness&#160;values.&#160;A<br/>fitness&#160;value&#160;of&#160;1.0&#160;is&#160;awarded&#160;to&#160;an&#160;individual&#160;with&#160;a&#160;per-<br/>fect&#160;match&#160;to&#160;the&#160;target&#160;and&#160;is&#160;the&#160;ultimate&#160;goal&#160;of&#160;a&#160;search.<br/>Thus,&#160;we&#160;calculated&#160;the&#160;graph&#160;edit&#160;distance&#160;between&#160;each<br/>test&#160;individual&#160;and&#160;the&#160;target&#160;and&#160;converted&#160;those&#160;values<br/>into&#160;a&#160;fitness&#160;value&#160;(Figure&#160;<a href="e1aa43c837743d72c31dbd52d6a9caacs.html#8">4).</a></p>
<p style="position:absolute;top:261px;left:98px;white-space:nowrap" class="ft93">The&#160;graph&#160;edit&#160;distance&#160;penalties&#160;used&#160;in&#160;our&#160;algorithm</p>
<p style="position:absolute;top:279px;left:86px;white-space:nowrap" class="ft97">and&#160;this&#160;manuscript&#160;are&#160;described&#160;in&#160;Table&#160;<a href="e1aa43c837743d72c31dbd52d6a9caacs.html#9">1,&#160;</a>but&#160;can&#160;be<br/>modified&#160;by&#160;the&#160;user.&#160;The&#160;penalties&#160;are&#160;most&#160;severe&#160;when<br/>differences&#160;exist&#160;between&#160;region&#160;numbers&#160;and&#160;connectiv-<br/>ity&#160;than&#160;for&#160;region&#160;size&#160;and&#160;linkage&#160;parameters.&#160;While<br/>the&#160;optimization&#160;of&#160;the&#160;graph&#160;edit&#160;costs&#160;is&#160;beyond&#160;the<br/>scope&#160;of&#160;this&#160;paper,&#160;we&#160;will&#160;explore&#160;methods&#160;for&#160;automated<br/>optimization&#160;of&#160;the&#160;graph&#160;edit&#160;costs&#160;in&#160;future&#160;work.</p>
<p style="position:absolute;top:405px;left:98px;white-space:nowrap" class="ft93">The&#160;target&#160;individual,&#160;when&#160;compared&#160;to&#160;itself,&#160;yielded</p>
<p style="position:absolute;top:422px;left:86px;white-space:nowrap" class="ft97">a&#160;value&#160;of&#160;0.0,&#160;because&#160;when&#160;two&#160;individuals&#160;are&#160;identi-<br/>cal,&#160;the&#160;distance&#160;between&#160;them&#160;measured&#160;by&#160;the&#160;graph<br/>edit&#160;distance&#160;algorithm&#160;is&#160;0.&#160;Using&#160;Equation&#160;<a href="e1aa43c837743d72c31dbd52d6a9caacs.html#10">1,&#160;</a>the&#160;dis-<br/>tance&#160;of&#160;0.0&#160;translates&#160;to&#160;a&#160;fitness&#160;value&#160;of&#160;1.0,&#160;which&#160;in&#160;a<br/>genetic&#160;algorithm&#160;search&#160;would&#160;indicate&#160;the&#160;target&#160;mor-<br/>phology&#160;has&#160;been&#160;found.&#160;Morphology&#160;13&#160;in&#160;Figure&#160;<a href="e1aa43c837743d72c31dbd52d6a9caacs.html#8">4&#160;</a>is&#160;a<br/>slight&#160;variation&#160;of&#160;the&#160;target&#160;morphology,&#160;where&#160;its&#160;heads<br/>are&#160;several&#160;cell&#160;layers&#160;thinner&#160;than&#160;the&#160;heads&#160;of&#160;the&#160;target,<br/>and&#160;as&#160;expected,&#160;has&#160;the&#160;next&#160;best&#160;fitness&#160;value&#160;(0.998).<br/>In&#160;general,&#160;high&#160;fitness&#160;values&#160;for&#160;morphologies&#160;such&#160;as<br/>number&#160;13&#160;are&#160;expected&#160;as&#160;their&#160;regions&#160;are&#160;connected<br/>and&#160;oriented&#160;the&#160;same&#160;as&#160;the&#160;target.&#160;When&#160;compared&#160;to<br/>the&#160;target,&#160;morphologies&#160;that&#160;consist&#160;of&#160;three&#160;regions&#160;in<br/>each&#160;worm&#160;fragment&#160;received&#160;higher&#160;fitness&#160;values&#160;than<br/>morphologies&#160;having&#160;one&#160;or&#160;two&#160;regions.&#160;For&#160;example,<br/>morphology&#160;6&#160;was&#160;rated&#160;higher&#160;than&#160;morphology&#160;4.&#160;Again,<br/>this&#160;is&#160;because&#160;the&#160;graph&#160;edit&#160;distance&#160;costs&#160;included&#160;a<br/>much&#160;larger&#160;penalty&#160;for&#160;the&#160;deletion&#160;of&#160;a&#160;region&#160;than&#160;with<br/>a&#160;change&#160;to&#160;the&#160;type&#160;of&#160;region.</p>
<p style="position:absolute;top:763px;left:98px;white-space:nowrap" class="ft93">Conversion&#160;of&#160;a&#160;simulation&#160;snapshot&#160;into&#160;the&#160;graph&#160;is</p>
<p style="position:absolute;top:781px;left:86px;white-space:nowrap" class="ft97">an&#160;<i>O(N</i>2<i>)&#160;</i>algorithm&#160;where&#160;<i>N&#160;</i>is&#160;the&#160;number&#160;of&#160;cells&#160;in&#160;an<br/>individual.&#160;The&#160;wild&#160;type&#160;morphology&#160;had&#160;420&#160;cells,&#160;but<br/>since&#160;the&#160;transverse&#160;cut&#160;removed&#160;four&#160;rows&#160;of&#160;cells,&#160;the<br/>morphologies&#160;used&#160;in&#160;this&#160;experiment&#160;consisted&#160;of&#160;364</p>
<p style="position:absolute;top:893px;left:86px;white-space:nowrap" class="ft98"><b>Table&#160;1&#160;This&#160;table&#160;presents&#160;the&#160;graph&#160;edit&#160;costs&#160;used&#160;for<br/>graph&#160;edit&#160;distance&#160;calculations&#160;in&#160;this&#160;manuscript</b></p>
<p style="position:absolute;top:932px;left:86px;white-space:nowrap" class="ft92"><b>Operation</b></p>
<p style="position:absolute;top:932px;left:337px;white-space:nowrap" class="ft92"><b>Cost</b></p>
<p style="position:absolute;top:954px;left:86px;white-space:nowrap" class="ft90">Insert/delete&#160;region</p>
<p style="position:absolute;top:954px;left:337px;white-space:nowrap" class="ft90">1500</p>
<p style="position:absolute;top:975px;left:86px;white-space:nowrap" class="ft90">Change&#160;region&#160;type</p>
<p style="position:absolute;top:975px;left:337px;white-space:nowrap" class="ft90">1000</p>
<p style="position:absolute;top:996px;left:86px;white-space:nowrap" class="ft90">Change&#160;region&#160;parameter</p>
<p style="position:absolute;top:996px;left:337px;white-space:nowrap" class="ft90">0.1&#160;per&#160;unit&#160;changed</p>
<p style="position:absolute;top:1017px;left:86px;white-space:nowrap" class="ft90">Insert/delete&#160;link</p>
<p style="position:absolute;top:1017px;left:337px;white-space:nowrap" class="ft90">1000</p>
<p style="position:absolute;top:1038px;left:86px;white-space:nowrap" class="ft90">Change&#160;link&#160;distance</p>
<p style="position:absolute;top:1038px;left:337px;white-space:nowrap" class="ft90">0.1&#160;per&#160;unit&#160;moved</p>
<p style="position:absolute;top:1059px;left:86px;white-space:nowrap" class="ft90">Change&#160;link&#160;angle</p>
<p style="position:absolute;top:1059px;left:337px;white-space:nowrap" class="ft90">0â100</p>
<p style="position:absolute;top:1079px;left:86px;white-space:nowrap" class="ft90">Change&#160;link&#160;angle&#160;<i>&gt;&#160;</i>90&#160;penalty</p>
<p style="position:absolute;top:1079px;left:337px;white-space:nowrap" class="ft90">750</p>
<p style="position:absolute;top:136px;left:458px;white-space:nowrap" class="ft97">cells.&#160;The&#160;conversion&#160;algorithm&#160;ran&#160;in&#160;less&#160;than&#160;1.3&#160;sec-<br/>onds&#160;for&#160;every&#160;morphology&#160;in&#160;Figure&#160;<a href="e1aa43c837743d72c31dbd52d6a9caacs.html#10">5.&#160;</a>It&#160;is&#160;well-known<br/>that&#160;the&#160;run&#160;time&#160;of&#160;the&#160;graph&#160;edit&#160;distance&#160;calculation<br/>grows&#160;exponentially&#160;with&#160;the&#160;size&#160;of&#160;the&#160;graphs&#160;(number&#160;of<br/>nodes,&#160;or&#160;in&#160;our&#160;case&#160;to&#160;the&#160;number&#160;of&#160;regions&#160;in&#160;the&#160;two<br/>morphologies)&#160;<a href="e1aa43c837743d72c31dbd52d6a9caacs.html#14">[23].&#160;</a>However,&#160;since&#160;the&#160;number&#160;of&#160;regions<br/>in&#160;each&#160;morphology&#160;tested&#160;was&#160;at&#160;most&#160;6,&#160;the&#160;graph&#160;dis-<br/>tance&#160;algorithm&#160;finished&#160;in&#160;less&#160;than&#160;1&#160;millisecond&#160;for&#160;all<br/>morphologies.</p>
<p style="position:absolute;top:316px;left:458px;white-space:nowrap" class="ft97"><b>A&#160;cell&#160;connectivity&#160;distance&#160;threshold&#160;effects&#160;region<br/>determination<br/></b>A&#160;comparison&#160;of&#160;more&#160;complicated&#160;morphologies&#160;high-<br/>lighted&#160;the&#160;need&#160;for&#160;a&#160;flexible&#160;distance&#160;threshold&#160;in&#160;the<br/>component&#160;gathering&#160;algorithm.&#160;Since&#160;the&#160;cells&#160;in&#160;our<br/>simulation&#160;have&#160;radii&#160;of&#160;0.5&#160;units,&#160;the&#160;Euclidian&#160;distance<br/>between&#160;two&#160;adjacent&#160;cells&#160;can&#160;be&#160;as&#160;low&#160;as&#160;one.&#160;However,<br/>using&#160;a&#160;very&#160;rigid&#160;measure&#160;for&#160;identifying&#160;neighboring<br/>cells&#160;and&#160;determining&#160;the&#160;borders&#160;of&#160;regions&#160;can&#160;have&#160;dra-<br/>matic&#160;effects&#160;on&#160;the&#160;graph&#160;conversion.&#160;For&#160;example,&#160;con-<br/>sider&#160;the&#160;morphology&#160;of&#160;the&#160;second&#160;individual&#160;shown&#160;in<br/>Figure&#160;<a href="e1aa43c837743d72c31dbd52d6a9caacs.html#10">5.&#160;</a>In&#160;this&#160;individual&#160;thin&#160;lines&#160;of&#160;trunk&#160;cells&#160;dissect<br/>the&#160;head&#160;and&#160;tail&#160;regions&#160;into&#160;a&#160;number&#160;of&#160;potentially&#160;dis-<br/>tinct&#160;heads&#160;and&#160;tails&#160;if&#160;the&#160;borders&#160;are&#160;considered&#160;rigidly.<br/>Comparison&#160;of&#160;this&#160;individual&#160;with&#160;the&#160;target&#160;results&#160;in&#160;a<br/>very&#160;high&#160;graph&#160;edit&#160;distance&#160;due&#160;to&#160;the&#160;cost&#160;associated<br/>with&#160;having&#160;multiple&#160;heads.&#160;However,&#160;in&#160;the&#160;context&#160;of&#160;a<br/>evolutionary&#160;search,&#160;this&#160;individual&#160;may&#160;be&#160;very&#160;close&#160;to<br/>producing&#160;the&#160;target&#160;morphology.</p>
<p style="position:absolute;top:656px;left:470px;white-space:nowrap" class="ft93">A&#160;flexible&#160;threshold&#160;parameter&#160;was&#160;introduced&#160;to&#160;reduce</p>
<p style="position:absolute;top:674px;left:458px;white-space:nowrap" class="ft97">the&#160;rigidity&#160;of&#160;region&#160;definitions,&#160;which&#160;allowed&#160;neighbor-<br/>ing&#160;regions&#160;separated&#160;by&#160;thin&#160;regions&#160;to&#160;be&#160;merged&#160;in&#160;the<br/>final&#160;graph&#160;representation.&#160;Increasing&#160;the&#160;threshold&#160;value<br/>reduces&#160;the&#160;stringency&#160;by&#160;increasing&#160;the&#160;search&#160;distance<br/>between&#160;cells&#160;for&#160;neighbors&#160;with&#160;the&#160;same&#160;state.&#160;Thus,&#160;in<br/>the&#160;example&#160;just&#160;discussed,&#160;increasing&#160;the&#160;threshold&#160;value<br/>allowed&#160;the&#160;multiple&#160;head&#160;regions&#160;to&#160;be&#160;lumped&#160;into&#160;a&#160;sin-<br/>gle&#160;head&#160;region.&#160;The&#160;graph&#160;edit&#160;distance&#160;of&#160;this&#160;worm&#160;is<br/>much&#160;lower&#160;resulting&#160;in&#160;a&#160;fitness&#160;value&#160;close&#160;to&#160;one.</p>
<p style="position:absolute;top:835px;left:470px;white-space:nowrap" class="ft93">A&#160;second&#160;example&#160;highlighting&#160;the&#160;importance&#160;of&#160;this</p>
<p style="position:absolute;top:853px;left:458px;white-space:nowrap" class="ft97">parameter&#160;to&#160;component&#160;gathering&#160;is&#160;presented&#160;at&#160;the<br/>bottom&#160;of&#160;Figure&#160;<a href="e1aa43c837743d72c31dbd52d6a9caacs.html#10">5.&#160;</a>This&#160;worm&#160;represents&#160;a&#160;classic&#160;exper-<br/>iment&#160;that&#160;involves&#160;bifurcating&#160;the&#160;head&#160;region&#160;into<br/>two&#160;fully-developed&#160;heads.&#160;These&#160;two&#160;heads&#160;are&#160;sepa-<br/>rated&#160;physically&#160;and&#160;should&#160;be&#160;classified&#160;as&#160;two-headed.<br/>A&#160;threshold&#160;parameter&#160;of&#160;less&#160;than&#160;three&#160;results&#160;in&#160;the<br/>desired&#160;graph&#160;conversion&#160;in&#160;our&#160;algorithm,&#160;whereas&#160;the<br/>larger&#160;value&#160;results&#160;in&#160;a&#160;worm&#160;with&#160;a&#160;single&#160;head.</p>
<p style="position:absolute;top:996px;left:470px;white-space:nowrap" class="ft93">These&#160;two&#160;examples&#160;show&#160;the&#160;necessity&#160;of&#160;a&#160;flexible</p>
<p style="position:absolute;top:1014px;left:458px;white-space:nowrap" class="ft97">parameter&#160;for&#160;determining&#160;local&#160;regions&#160;during&#160;a&#160;GA&#160;run.<br/>In&#160;the&#160;first&#160;case,&#160;a&#160;low&#160;threshold&#160;was&#160;shown&#160;to&#160;penalize&#160;a<br/>morphology&#160;that&#160;was&#160;very&#160;similar&#160;to&#160;the&#160;target,&#160;whereas&#160;a<br/>high&#160;threshold&#160;inappropriately&#160;favored&#160;a&#160;morphology&#160;con-<br/>taining&#160;a&#160;physical&#160;gap&#160;between&#160;head&#160;regions.&#160;An&#160;optimal</p>
</div>
</body>
</html>
<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="" xml:lang="">
<head>
<title>e1aa43c837743d72c31dbd52d6a9caac-html.html</title>

<meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
 <br/>
<style type="text/css">
<!--
	p {margin: 0; padding: 0;}	.ft100{font-size:10px;font-family:Times;color:#000000;}
	.ft101{font-size:10px;font-family:Times;color:#000000;}
	.ft102{font-size:10px;font-family:Times;color:#000000;}
	.ft103{font-size:12px;font-family:Times;color:#000000;}
	.ft104{font-size:11px;font-family:Times;color:#000000;}
	.ft105{font-size:12px;font-family:Times;color:#000000;}
	.ft106{font-size:12px;line-height:17px;font-family:Times;color:#000000;}
	.ft107{font-size:11px;line-height:17px;font-family:Times;color:#000000;}
	.ft108{font-size:12px;line-height:17px;font-family:Times;color:#000000;}
-->
</style>
</head>
<body bgcolor="#A0A0A0" vlink="blue" link="blue">
<div id="page10-div" style="position:relative;width:892px;height:1191px;">
<p style="position:absolute;top:53px;left:84px;white-space:nowrap" class="ft100">Budnikova&#160;<i>et&#160;al.&#160;BMC&#160;Bioinformatics&#160;</i>2014,&#160;<b>15</b>:178</p>
<p style="position:absolute;top:53px;left:740px;white-space:nowrap" class="ft100">Page&#160;10&#160;of&#160;14</p>
<p style="position:absolute;top:68px;left:84px;white-space:nowrap" class="ft100">http://www.biomedcentral.com/1471-2105/15/178</p>
<p style="position:absolute;top:353px;left:94px;white-space:nowrap" class="ft102"><b>Figure&#160;5&#160;Threshold&#160;value&#160;influence&#160;on&#160;snapshot&#160;to&#160;graph&#160;conversion&#160;algorithm.</b></p>
<p style="position:absolute;top:405px;left:84px;white-space:nowrap" class="ft106">threshold&#160;will&#160;depend&#160;upon&#160;the&#160;modeling&#160;platform&#160;and<br/>project,&#160;but&#160;in&#160;this&#160;work&#160;and&#160;from&#160;an&#160;evolutionary&#160;per-<br/>spective&#160;a&#160;threshold&#160;of&#160;two&#160;was&#160;optimal.</p>
<p style="position:absolute;top:477px;left:84px;white-space:nowrap" class="ft106"><b>Validation&#160;of&#160;component&#160;gathering&#160;and&#160;graph&#160;edit<br/>distance&#160;during&#160;evolutionary&#160;search<br/></b>As&#160;our&#160;ultimate&#160;goal&#160;is&#160;the&#160;generation&#160;of&#160;an&#160;automated<br/>model&#160;discovery&#160;tool,&#160;we&#160;tested&#160;the&#160;utility&#160;of&#160;our&#160;conver-<br/>sion&#160;algorithm&#160;and&#160;the&#160;graph&#160;edit&#160;distance&#160;as&#160;a&#160;fitness<br/>metric&#160;as&#160;part&#160;of&#160;an&#160;evolutionary&#160;search&#160;for&#160;a&#160;target<br/>described&#160;in&#160;the&#160;PlanformDB.&#160;As&#160;shown&#160;in&#160;Figure&#160;<a href="e1aa43c837743d72c31dbd52d6a9caacs.html#11">6a</a>,&#160;we<br/>selected&#160;an&#160;experiment&#160;from&#160;the&#160;database&#160;where&#160;either&#160;the<br/>anterior&#160;or&#160;posterior&#160;end&#160;of&#160;an&#160;intact&#160;worm&#160;was&#160;removed.<br/>Functionality&#160;was&#160;added&#160;to&#160;the&#160;GA&#160;to&#160;interact&#160;directly<br/>with&#160;the&#160;PlanformDB&#160;so&#160;that&#160;it&#160;could&#160;extract&#160;the&#160;tar-<br/>get&#160;individual&#160;as&#160;a&#160;graph&#160;representation&#160;for&#160;comparison.<br/>The&#160;target&#160;individual&#160;for&#160;this&#160;experiment&#160;was&#160;a&#160;single<br/>normal&#160;worm&#160;consisting&#160;of&#160;head,&#160;trunk,&#160;and&#160;tail&#160;regions<br/>connected&#160;in&#160;that&#160;order&#160;(see&#160;graph&#160;in&#160;Figure&#160;<a href="e1aa43c837743d72c31dbd52d6a9caacs.html#11">6a).</a></p>
<p style="position:absolute;top:745px;left:96px;white-space:nowrap" class="ft103">The&#160;starting&#160;population&#160;of&#160;individuals&#160;in&#160;the&#160;GA&#160;search</p>
<p style="position:absolute;top:763px;left:84px;white-space:nowrap" class="ft108">contained&#160;a&#160;version&#160;of&#160;the&#160;hand-designed&#160;model&#160;descrip-<br/>tion&#160;shown&#160;in&#160;Figure&#160;<a href="e1aa43c837743d72c31dbd52d6a9caacs.html#5">2&#160;</a>after&#160;it&#160;was&#160;purposely&#160;modified&#160;to<br/>become&#160;non-functional.&#160;The&#160;<i>Regeneration</i>,&#160;<i>Head&#160;develop-<br/>ment</i></p>
<p style="position:absolute;top:817px;left:115px;white-space:nowrap" class="ft103">,&#160;and&#160;<i>Tail&#160;development&#160;</i>resources&#160;were&#160;removed&#160;from</p>
<p style="position:absolute;top:835px;left:84px;white-space:nowrap" class="ft106">the&#160;regulatory&#160;network,&#160;thus&#160;preventing&#160;proper&#160;regener-<br/>ation&#160;(data&#160;not&#160;shown).&#160;We&#160;were&#160;interested&#160;in&#160;whether<br/>the&#160;GA&#160;could&#160;find&#160;solutions&#160;that&#160;properly&#160;regenerated&#160;the<br/>tail&#160;region&#160;using&#160;our&#160;conversion&#160;algorithm&#160;and&#160;a&#160;fitness<br/>metric&#160;based&#160;solely&#160;on&#160;the&#160;graph&#160;edit&#160;distance&#160;calcula-<br/>tion.&#160;Since&#160;the&#160;GA&#160;is&#160;designed&#160;to&#160;evaluate&#160;fitness&#160;values<br/>in&#160;the&#160;range&#160;of&#160;0.0&#160;to&#160;1.0,&#160;we&#160;converted&#160;the&#160;graph&#160;edit<br/>value&#160;as&#160;shown&#160;in&#160;Equation&#160;<a href="e1aa43c837743d72c31dbd52d6a9caacs.html#10">1.&#160;</a>Initially,&#160;we&#160;used&#160;the&#160;sim-<br/>ple&#160;inverse&#160;function&#160;of&#160;1/(graph&#160;edit&#160;distance)&#160;to&#160;obtain<br/>the&#160;GA&#160;fitness&#160;values.&#160;However,&#160;the&#160;fitness&#160;function&#160;values<br/>in&#160;such&#160;cases&#160;tended&#160;to&#160;be&#160;very&#160;small&#160;even&#160;for&#160;relatively<br/>similar&#160;graphs&#160;due&#160;to&#160;sensitivity&#160;caused&#160;by&#160;the&#160;large&#160;edit<br/>penalties&#160;in&#160;Table&#160;<a href="e1aa43c837743d72c31dbd52d6a9caacs.html#9">1.&#160;</a>To&#160;reduce&#160;the&#160;sensitivity&#160;to&#160;the&#160;large<br/>edit&#160;distance&#160;penalties,&#160;we&#160;introduced&#160;a&#160;constant&#160;(5000)<br/>to&#160;our&#160;equation&#160;after&#160;empirically&#160;testing&#160;values&#160;between</p>
<p style="position:absolute;top:405px;left:456px;white-space:nowrap" class="ft106">1000&#160;and&#160;10000.&#160;Constants&#160;below&#160;1000&#160;were&#160;excluded<br/>as&#160;they&#160;yielded&#160;very&#160;small&#160;fitness&#160;values,&#160;while&#160;constants<br/>above&#160;10000&#160;were&#160;excluded&#160;on&#160;the&#160;basis&#160;of&#160;generating&#160;high<br/>fitness&#160;values&#160;for&#160;individuals&#160;with&#160;morphologies&#160;very&#160;dis-<br/>similar&#160;to&#160;the&#160;target.&#160;Although&#160;other&#160;numbers&#160;in&#160;this&#160;range<br/>could&#160;have&#160;sufficed,&#160;the&#160;5000&#160;value&#160;maximized&#160;the&#160;differ-<br/>ences&#160;between&#160;fitness&#160;values&#160;and&#160;thus&#160;was&#160;used&#160;in&#160;this<br/>work.</p>
<p style="position:absolute;top:574px;left:479px;white-space:nowrap" class="ft105"><i>fitness</i></p>
<p style="position:absolute;top:570px;left:521px;white-space:nowrap" class="ft103">=</p>
<p style="position:absolute;top:563px;left:578px;white-space:nowrap" class="ft103">5000</p>
<p style="position:absolute;top:584px;left:538px;white-space:nowrap" class="ft105"><i>(</i></p>
<p style="position:absolute;top:584px;left:543px;white-space:nowrap" class="ft105"><i>distance</i></p>
<p style="position:absolute;top:581px;left:597px;white-space:nowrap" class="ft103">+&#160;5000<i>)</i></p>
<p style="position:absolute;top:574px;left:790px;white-space:nowrap" class="ft103">(1)</p>
<p style="position:absolute;top:620px;left:468px;white-space:nowrap" class="ft103">Our&#160;searches&#160;included&#160;populations&#160;of&#160;only&#160;30&#160;individu-</p>
<p style="position:absolute;top:638px;left:456px;white-space:nowrap" class="ft106">als,&#160;but&#160;much&#160;larger&#160;populations&#160;are&#160;feasible.&#160;In&#160;order&#160;to<br/>generate&#160;variability&#160;in&#160;the&#160;individuals,&#160;a&#160;set&#160;of&#160;mutation<br/>and&#160;crossover&#160;parameters&#160;were&#160;introduced&#160;and&#160;applied<br/>to&#160;each&#160;new&#160;generation&#160;of&#160;offspring.&#160;These&#160;operators<br/>and&#160;parameters&#160;were&#160;hand&#160;designed&#160;for&#160;this&#160;experiment,<br/>but&#160;parameter&#160;definition&#160;will&#160;eventually&#160;become&#160;auto-<br/>mated.&#160;Searches&#160;were&#160;performed&#160;for&#160;individuals&#160;with<br/>proper&#160;regeneration&#160;of&#160;head&#160;or&#160;tail&#160;following&#160;removal&#160;of<br/>the&#160;anterior&#160;or&#160;posterior&#160;regions,&#160;respectively.&#160;During&#160;the<br/>evolutionary&#160;search,&#160;the&#160;GA&#160;pauses&#160;each&#160;simulation&#160;at&#160;a<br/>predefined&#160;step&#160;(e.g.&#160;200&#160;in&#160;this&#160;experiment)&#160;and&#160;requests<br/>a&#160;simulated&#160;experiment&#160;be&#160;performed&#160;(e.g.&#160;cutting&#160;off&#160;the<br/>head&#160;or&#160;tail).&#160;Each&#160;individual&#160;simulation&#160;is&#160;resumed&#160;and<br/>continues&#160;until&#160;the&#160;GA&#160;requests&#160;a&#160;snapshot&#160;to&#160;evaluate<br/>(e.g.&#160;step&#160;400).&#160;At&#160;this&#160;point,&#160;the&#160;simulation&#160;snapshot&#160;is<br/>used&#160;to&#160;create&#160;a&#160;graph&#160;representation&#160;using&#160;our&#160;compo-<br/>nent&#160;gathering&#160;algorithm,&#160;which&#160;is&#160;then&#160;compared&#160;to&#160;the<br/>target&#160;individual&#160;from&#160;the&#160;PlanformDB&#160;to&#160;determine&#160;the<br/>individualâs&#160;fitness&#160;value.&#160;High&#160;scoring&#160;individuals&#160;were<br/>chosen&#160;for&#160;reproduction&#160;to&#160;generate&#160;the&#160;next&#160;generation&#160;of<br/>individuals&#160;which&#160;were&#160;independently&#160;mutated&#160;to&#160;increase<br/>variability&#160;in&#160;the&#160;population.</p>
<p style="position:absolute;top:1033px;left:468px;white-space:nowrap" class="ft103">The&#160;GA&#160;was&#160;successful&#160;in&#160;identifying&#160;individuals&#160;with</p>
<p style="position:absolute;top:1051px;left:456px;white-space:nowrap" class="ft106">fitness&#160;values&#160;very&#160;close&#160;to&#160;the&#160;target&#160;value&#160;of&#160;1.0.&#160;Two<br/>such&#160;regulatory&#160;networks&#160;are&#160;shown&#160;in&#160;Figure&#160;<a href="e1aa43c837743d72c31dbd52d6a9caacs.html#11">6.&#160;</a>The&#160;net-<br/>work&#160;shown&#160;in&#160;Figure&#160;<a href="e1aa43c837743d72c31dbd52d6a9caacs.html#11">6(</a>b)&#160;is&#160;a&#160;representative&#160;evolved</p>
</div>
</body>
</html>
<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="" xml:lang="">
<head>
<title>e1aa43c837743d72c31dbd52d6a9caac-html.html</title>

<meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
 <br/>
<style type="text/css">
<!--
	p {margin: 0; padding: 0;}	.ft110{font-size:10px;font-family:Times;color:#000000;}
	.ft111{font-size:10px;font-family:Times;color:#000000;}
	.ft112{font-size:10px;font-family:Times;color:#000000;}
	.ft113{font-size:5px;font-family:Times;color:#000000;}
	.ft114{font-size:14px;font-family:Times;color:#000000;}
	.ft115{font-size:5px;font-family:Times;color:#0000ff;}
	.ft116{font-size:10px;line-height:14px;font-family:Times;color:#000000;}
	.ft117{font-size:10px;line-height:14px;font-family:Times;color:#000000;}
	.ft118{font-size:10px;line-height:15px;font-family:Times;color:#000000;}
-->
</style>
</head>
<body bgcolor="#A0A0A0" vlink="blue" link="blue">
<div id="page11-div" style="position:relative;width:892px;height:1191px;">
<p style="position:absolute;top:53px;left:86px;white-space:nowrap" class="ft110">Budnikova&#160;<i>et&#160;al.&#160;BMC&#160;Bioinformatics&#160;</i>2014,&#160;<b>15</b>:178</p>
<p style="position:absolute;top:53px;left:742px;white-space:nowrap" class="ft110">Page&#160;11&#160;of&#160;14</p>
<p style="position:absolute;top:68px;left:86px;white-space:nowrap" class="ft110">http://www.biomedcentral.com/1471-2105/15/178</p>
<p style="position:absolute;top:405px;left:503px;white-space:nowrap" class="ft113">GENE 0</p>
<p style="position:absolute;top:449px;left:640px;white-space:nowrap" class="ft113">hCell</p>
<p style="position:absolute;top:493px;left:400px;white-space:nowrap" class="ft113">GENE 1</p>
<p style="position:absolute;top:493px;left:585px;white-space:nowrap" class="ft113">GENE 2</p>
<p style="position:absolute;top:577px;left:550px;white-space:nowrap" class="ft113">GENE 4</p>
<p style="position:absolute;top:535px;left:486px;white-space:nowrap" class="ft113">Head gradient</p>
<p style="position:absolute;top:535px;left:406px;white-space:nowrap" class="ft113">tCell</p>
<p style="position:absolute;top:577px;left:229px;white-space:nowrap" class="ft113">GENE 3</p>
<p style="position:absolute;top:449px;left:386px;white-space:nowrap" class="ft113">adhesions</p>
<p style="position:absolute;top:618px;left:394px;white-space:nowrap" class="ft113">Tail gradient</p>
<p style="position:absolute;top:449px;left:338px;white-space:nowrap" class="ft113">Lysis</p>
<p style="position:absolute;top:493px;left:256px;white-space:nowrap" class="ft113">EQ 5</p>
<p style="position:absolute;top:493px;left:341px;white-space:nowrap" class="ft113">action Die</p>
<p style="position:absolute;top:618px;left:211px;white-space:nowrap" class="ft113">membrane</p>
<p style="position:absolute;top:618px;left:556px;white-space:nowrap" class="ft113">iCell</p>
<p style="position:absolute;top:493px;left:501px;white-space:nowrap" class="ft113">EQ 2</p>
<p style="position:absolute;top:449px;left:476px;white-space:nowrap" class="ft113">Head gradient</p>
<p style="position:absolute;top:449px;left:562px;white-space:nowrap" class="ft113">Gap junctions</p>
<p style="position:absolute;top:577px;left:391px;white-space:nowrap" class="ft113">EQ 3</p>
<p style="position:absolute;top:577px;left:311px;white-space:nowrap" class="ft113">EQ 4</p>
<p style="position:absolute;top:618px;left:287px;white-space:nowrap" class="ft113">Regeneration signal</p>
<p style="position:absolute;top:535px;left:260px;white-space:nowrap" class="ft113">cut</p>
<p style="position:absolute;top:449px;left:247px;white-space:nowrap" class="ft113">membrane</p>
<p style="position:absolute;top:411px;left:195px;white-space:nowrap" class="ft114">b</p>
<p style="position:absolute;top:662px;left:195px;white-space:nowrap" class="ft114">c</p>
<p style="position:absolute;top:146px;left:195px;white-space:nowrap" class="ft114">a</p>
<p style="position:absolute;top:405px;left:640px;white-space:nowrap" class="ft113">EQ 8</p>
<p style="position:absolute;top:363px;left:644px;white-space:nowrap" class="ft113">cut</p>
<p style="position:absolute;top:682px;left:297px;white-space:nowrap" class="ft113">GENE 0</p>
<p style="position:absolute;top:722px;left:302px;white-space:nowrap" class="ft113">hCell</p>
<p style="position:absolute;top:766px;left:329px;white-space:nowrap" class="ft113">GENE 1</p>
<p style="position:absolute;top:766px;left:264px;white-space:nowrap" class="ft113">GENE 2</p>
<p style="position:absolute;top:850px;left:284px;white-space:nowrap" class="ft113">GENE 4</p>
<p style="position:absolute;top:808px;left:194px;white-space:nowrap" class="ft113">Head gradient</p>
<p style="position:absolute;top:850px;left:210px;white-space:nowrap" class="ft113">EQ 0</p>
<p style="position:absolute;top:808px;left:368px;white-space:nowrap" class="ft113">tCell</p>
<p style="position:absolute;top:850px;left:372px;white-space:nowrap" class="ft113">GENE 3</p>
<p style="position:absolute;top:894px;left:407px;white-space:nowrap" class="ft113">Tail gradient</p>
<p style="position:absolute;top:937px;left:418px;white-space:nowrap" class="ft113">EQ 1</p>
<p style="position:absolute;top:894px;left:277px;white-space:nowrap" class="ft113">iCell</p>
<p style="position:absolute;top:894px;left:195px;white-space:nowrap" class="ft113">Head gradient</p>
<p style="position:absolute;top:937px;left:234px;white-space:nowrap" class="ft113">EQ 6</p>
<p style="position:absolute;top:981px;left:407px;white-space:nowrap" class="ft113">Tail gradient</p>
<p style="position:absolute;top:766px;left:206px;white-space:nowrap" class="ft113">EQ 2</p>
<p style="position:absolute;top:722px;left:195px;white-space:nowrap" class="ft113">Head gradient</p>
<p style="position:absolute;top:722px;left:405px;white-space:nowrap" class="ft113">Gap junctions</p>
<p style="position:absolute;top:766px;left:419px;white-space:nowrap" class="ft113">EQ 3</p>
<p style="position:absolute;top:722px;left:490px;white-space:nowrap" class="ft113">Tail gradient</p>
<p style="position:absolute;top:766px;left:553px;white-space:nowrap" class="ft113">EQ 4</p>
<p style="position:absolute;top:808px;left:528px;white-space:nowrap" class="ft113">Regeneration signal</p>
<p style="position:absolute;top:722px;left:562px;white-space:nowrap" class="ft113">cut</p>
<p style="position:absolute;top:722px;left:617px;white-space:nowrap" class="ft113">membrane</p>
<p style="position:absolute;top:766px;left:632px;white-space:nowrap" class="ft113">EQ 5</p>
<p style="position:absolute;top:808px;left:636px;white-space:nowrap" class="ft113">cut</p>
<p style="position:absolute;top:722px;left:690px;white-space:nowrap" class="ft113">Lysis</p>
<p style="position:absolute;top:766px;left:703px;white-space:nowrap" class="ft115">Die</p>
<p style="position:absolute;top:894px;left:324px;white-space:nowrap" class="ft113">Gap junctions</p>
<p style="position:absolute;top:937px;left:338px;white-space:nowrap" class="ft113">EQ 7</p>
<p style="position:absolute;top:1005px;left:95px;white-space:nowrap" class="ft118"><b>Figure&#160;6&#160;An&#160;evolutionary&#160;search&#160;experiment&#160;where&#160;simulated&#160;individuals&#160;are&#160;evaluated&#160;against&#160;a&#160;target&#160;morphology&#160;extracted&#160;from&#160;the<br/>PlanformDB.&#160;</b>The&#160;simulation&#160;output&#160;is&#160;converted&#160;to&#160;a&#160;graph&#160;using&#160;our&#160;component&#160;gathering&#160;analysis&#160;algorithm&#160;followed&#160;by&#160;a&#160;comparison&#160;using<br/>the&#160;graph&#160;edit&#160;distance&#160;fitness&#160;function.&#160;The&#160;process&#160;is&#160;repeated&#160;with&#160;new&#160;individuals&#160;following&#160;selection,&#160;reproduction,&#160;and&#160;mutation&#160;operations<br/>until&#160;a&#160;suitable&#160;solution&#160;is&#160;identified.&#160;Solutions&#160;from&#160;simple&#160;searches&#160;are&#160;presented&#160;following&#160;removal&#160;of&#160;the&#160;anterior&#160;(b)&#160;or&#160;posterior&#160;(c),&#160;respectively.<br/><b>(a)&#160;</b>Genetic&#160;Algorithm&#160;(GA)&#160;experimental&#160;flow.&#160;<b>(b)&#160;</b>Representative&#160;GA&#160;head&#160;regeneration&#160;solution&#160;network.&#160;<b>(c)&#160;</b>Representative&#160;GA&#160;tail&#160;regeneration<br/>solution&#160;network.</p>
</div>
</body>
</html>
<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="" xml:lang="">
<head>
<title>e1aa43c837743d72c31dbd52d6a9caac-html.html</title>

<meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
 <br/>
<style type="text/css">
<!--
	p {margin: 0; padding: 0;}	.ft120{font-size:10px;font-family:Times;color:#000000;}
	.ft121{font-size:10px;font-family:Times;color:#000000;}
	.ft122{font-size:10px;font-family:Times;color:#000000;}
	.ft123{font-size:12px;font-family:Times;color:#000000;}
	.ft124{font-size:13px;font-family:Times;color:#000000;}
	.ft125{font-size:12px;line-height:17px;font-family:Times;color:#000000;}
	.ft126{font-size:13px;line-height:18px;font-family:Times;color:#000000;}
-->
</style>
</head>
<body bgcolor="#A0A0A0" vlink="blue" link="blue">
<div id="page12-div" style="position:relative;width:892px;height:1191px;">
<p style="position:absolute;top:53px;left:84px;white-space:nowrap" class="ft120">Budnikova&#160;<i>et&#160;al.&#160;BMC&#160;Bioinformatics&#160;</i>2014,&#160;<b>15</b>:178</p>
<p style="position:absolute;top:53px;left:740px;white-space:nowrap" class="ft120">Page&#160;12&#160;of&#160;14</p>
<p style="position:absolute;top:68px;left:84px;white-space:nowrap" class="ft120">http://www.biomedcentral.com/1471-2105/15/178</p>
<p style="position:absolute;top:136px;left:84px;white-space:nowrap" class="ft125">solution&#160;when&#160;selecting&#160;for&#160;a&#160;worm&#160;that&#160;properly&#160;regen-<br/>erates&#160;head&#160;after&#160;removal&#160;of&#160;the&#160;anterior&#160;region&#160;of&#160;an<br/>intact&#160;worm.&#160;Similarly,&#160;the&#160;network&#160;in&#160;Figure&#160;<a href="e1aa43c837743d72c31dbd52d6a9caacs.html#11">6(</a>c)&#160;comes<br/>from&#160;a&#160;representative&#160;worm&#160;that&#160;was&#160;evolved&#160;to&#160;regener-<br/>ate&#160;a&#160;tail&#160;following&#160;removal&#160;of&#160;the&#160;posterior&#160;end.&#160;There&#160;are<br/>noticeable&#160;similarities&#160;between&#160;the&#160;two&#160;solutions.&#160;In&#160;both<br/>solutions,&#160;a&#160;direct&#160;connection&#160;was&#160;made&#160;between&#160;the&#160;cut<br/>response&#160;to&#160;regeneration&#160;of&#160;the&#160;missing&#160;region,&#160;head&#160;or<br/>tail,&#160;respectively.&#160;Although&#160;each&#160;search&#160;found&#160;a&#160;solution&#160;to<br/>the&#160;experiment&#160;at&#160;hand,&#160;the&#160;solutions&#160;were&#160;limited&#160;in&#160;their<br/>flexibility&#160;to&#160;respond&#160;to&#160;other&#160;permutations.&#160;As&#160;expected,<br/>neither&#160;network&#160;was&#160;capable&#160;of&#160;solving&#160;the&#160;reverse&#160;prob-<br/>lem&#160;as&#160;their&#160;was&#160;no&#160;selective&#160;pressure&#160;in&#160;the&#160;evolutionary<br/>search.&#160;Nonetheless,&#160;these&#160;results&#160;show&#160;that&#160;our&#160;evolu-<br/>tionary&#160;search&#160;process&#160;is&#160;capable&#160;of&#160;finding&#160;solutions&#160;using<br/>our&#160;connected&#160;component&#160;analysis&#160;to&#160;convert&#160;cell-based<br/>individuals&#160;into&#160;graphs,&#160;which&#160;are&#160;compared&#160;with&#160;the<br/>appropriate&#160;target&#160;extracted&#160;from&#160;the&#160;PlanformDB&#160;using<br/>the&#160;graph&#160;edit&#160;difference&#160;evaluation&#160;metric.</p>
<p style="position:absolute;top:476px;left:96px;white-space:nowrap" class="ft123">The&#160;inflexible&#160;network&#160;solutions&#160;emphasize&#160;the&#160;impor-</p>
<p style="position:absolute;top:494px;left:84px;white-space:nowrap" class="ft125">tance&#160;of&#160;searching&#160;for&#160;solutions&#160;using&#160;rigorous&#160;fitness&#160;cri-<br/>teria&#160;and&#160;why&#160;an&#160;automated&#160;approach&#160;is&#160;necessary.&#160;Future<br/>experiments&#160;will&#160;target&#160;networks&#160;that&#160;are&#160;capable&#160;of&#160;han-<br/>dling&#160;both&#160;anterior&#160;and&#160;posterior&#160;ablations.&#160;Simulation<br/>snapshots&#160;contain&#160;a&#160;detailed&#160;description&#160;of&#160;the&#160;current<br/>state&#160;of&#160;the&#160;simulation,&#160;including&#160;a&#160;list&#160;of&#160;all&#160;cells,&#160;their<br/>location,&#160;shape,&#160;genomes,&#160;metabolic&#160;equations,&#160;environ-<br/>mental&#160;conditions,&#160;and&#160;the&#160;concentration&#160;of&#160;all&#160;resources.<br/>Thus,&#160;snapshots&#160;provide&#160;much&#160;richer&#160;information&#160;about<br/>the&#160;cells&#160;and&#160;individuals&#160;than&#160;the&#160;graph&#160;formalism&#160;and<br/>will&#160;provide&#160;opportunities&#160;to&#160;develop&#160;additional&#160;fitness<br/>metrics&#160;to&#160;complement&#160;graph&#160;edit&#160;distance&#160;in&#160;the&#160;future.</p>
<p style="position:absolute;top:727px;left:84px;white-space:nowrap" class="ft125"><b>Discussion&#160;and&#160;conclusion<br/></b>One&#160;of&#160;the&#160;challenges&#160;in&#160;the&#160;biological&#160;sciences&#160;is&#160;the<br/>development&#160;of&#160;new&#160;methods&#160;for&#160;data&#160;visualization&#160;and<br/>integration&#160;to&#160;provide&#160;informative&#160;and&#160;predictive&#160;insight<br/>into&#160;the&#160;scientific&#160;process.&#160;Computer&#160;models&#160;hold&#160;great<br/>promise&#160;in&#160;this&#160;area,&#160;but&#160;often&#160;involve&#160;significant&#160;human<br/>interaction&#160;and&#160;time.&#160;In&#160;this&#160;study,&#160;we&#160;laid&#160;the&#160;foun-<br/>dation&#160;for&#160;a&#160;system&#160;of&#160;automated&#160;model&#160;discovery&#160;and<br/>development&#160;that&#160;incorporates&#160;shape-based&#160;experimen-<br/>tal&#160;data&#160;from&#160;a&#160;repository&#160;of&#160;documented&#160;experiments.<br/>Graphs&#160;are&#160;a&#160;powerful&#160;and&#160;convenient&#160;means&#160;of&#160;describ-<br/>ing&#160;morphological&#160;data.&#160;Using&#160;comparison&#160;methods,&#160;such<br/>as&#160;the&#160;graph&#160;difference&#160;evaluation,&#160;one&#160;can&#160;easily&#160;search<br/>such&#160;a&#160;database&#160;for&#160;results&#160;that&#160;are&#160;similar&#160;or&#160;identi-<br/>cal.&#160;We&#160;showed&#160;that&#160;the&#160;utility&#160;of&#160;the&#160;graph&#160;difference<br/>evaluation&#160;could&#160;be&#160;further&#160;extended&#160;as&#160;a&#160;fitness&#160;evalua-<br/>tion&#160;metric&#160;during&#160;evolutionary&#160;search.&#160;This&#160;method&#160;was<br/>combined&#160;with&#160;a&#160;cell-based&#160;modeling&#160;platform&#160;to&#160;model<br/>basic&#160;regeneration&#160;of&#160;the&#160;planarian&#160;flatworm.&#160;Agent-based<br/>models&#160;are&#160;particularly&#160;amenable&#160;to&#160;this&#160;approach&#160;as&#160;they<br/>are&#160;tractable&#160;to&#160;simulated&#160;experimental&#160;manipulations</p>
<p style="position:absolute;top:136px;left:456px;white-space:nowrap" class="ft125">combined&#160;with&#160;fully&#160;emergent&#160;outcomes.&#160;The&#160;ability&#160;to<br/>automate&#160;these&#160;behaviors&#160;fits&#160;nicely&#160;into&#160;an&#160;automated<br/>discovery&#160;system&#160;that&#160;can&#160;be&#160;driven&#160;by&#160;a&#160;genetic&#160;algo-<br/>rithm&#160;search&#160;engine.&#160;Furthermore,&#160;simulators&#160;that&#160;include<br/>robust&#160;visualization&#160;capabilities&#160;make&#160;it&#160;very&#160;convenient<br/>for&#160;the&#160;scientist&#160;to&#160;evaluate&#160;or&#160;experiment&#160;on&#160;a&#160;set&#160;of<br/>search&#160;results.</p>
<p style="position:absolute;top:261px;left:468px;white-space:nowrap" class="ft123">Planarian&#160;worms&#160;provide&#160;an&#160;excellent&#160;model&#160;system&#160;for</p>
<p style="position:absolute;top:279px;left:456px;white-space:nowrap" class="ft125">developing&#160;such&#160;an&#160;automated&#160;search&#160;process&#160;due&#160;to&#160;the<br/>plethora&#160;of&#160;experimental&#160;data&#160;in&#160;the&#160;literature,&#160;and&#160;now<br/>available&#160;in&#160;a&#160;curated&#160;database&#160;(PlanformDB).&#160;However,<br/>the&#160;principles&#160;inherent&#160;in&#160;this&#160;design&#160;are&#160;extensible&#160;to<br/>any&#160;system&#160;where&#160;shape&#160;is&#160;an&#160;integral&#160;component&#160;of&#160;the<br/>observable&#160;outcomes.&#160;That&#160;said,&#160;developing&#160;model&#160;discov-<br/>ery&#160;systems&#160;that&#160;automatically&#160;incorporate&#160;experimental<br/>data&#160;is&#160;a&#160;general&#160;and&#160;attainable&#160;goal&#160;that&#160;is&#160;not&#160;limited&#160;to<br/>systems&#160;dependent&#160;upon&#160;morphological&#160;data.</p>
<p style="position:absolute;top:440px;left:468px;white-space:nowrap" class="ft123">The&#160;challenge&#160;of&#160;describing&#160;phenotypic&#160;outcomes&#160;based</p>
<p style="position:absolute;top:458px;left:456px;white-space:nowrap" class="ft125">upon&#160;morphological&#160;characteristics&#160;is&#160;challenging&#160;for<br/>biological&#160;systems&#160;and&#160;cell-based&#160;computational&#160;models<br/>alike.&#160;We&#160;showed&#160;that&#160;converting&#160;cell-based&#160;simulation<br/>output&#160;into&#160;graphs&#160;can&#160;be&#160;achieved&#160;using&#160;a&#160;component<br/>gathering&#160;algorithm&#160;that&#160;identifies&#160;regions&#160;and&#160;their&#160;jux-<br/>taposition&#160;to&#160;each&#160;other&#160;and&#160;converts&#160;them&#160;into&#160;nodes<br/>joined&#160;by&#160;linkages.&#160;The&#160;resulting&#160;graphs&#160;can&#160;be&#160;stored&#160;to&#160;a<br/>database&#160;and&#160;easily&#160;reconstructed&#160;later&#160;and/or&#160;compared<br/>with&#160;other&#160;graphs&#160;using&#160;algorithms&#160;such&#160;as&#160;the&#160;graph<br/>edit&#160;distance.&#160;These&#160;comparisons&#160;and&#160;the&#160;resulting&#160;metric<br/>were&#160;incorporated&#160;into&#160;an&#160;evolutionary&#160;search&#160;where&#160;the<br/>genetic&#160;algorithm&#160;retrieved&#160;its&#160;target&#160;morphology&#160;from&#160;a<br/>data&#160;repository&#160;of&#160;experimental&#160;outcomes&#160;and&#160;used&#160;the<br/>graph&#160;edit&#160;distance&#160;as&#160;a&#160;fitness&#160;metric&#160;to&#160;drive&#160;develop-<br/>ment.&#160;Using&#160;a&#160;small&#160;population&#160;size&#160;of&#160;30&#160;individuals,<br/>our&#160;mutation&#160;and&#160;crossover&#160;frequencies&#160;were&#160;sufficient&#160;to<br/>generate&#160;a&#160;solution&#160;state&#160;in&#160;as&#160;few&#160;as&#160;19&#160;generations&#160;for&#160;the<br/>experiments&#160;shown&#160;in&#160;Figure&#160;<a href="e1aa43c837743d72c31dbd52d6a9caacs.html#11">6(a).</a></p>
<p style="position:absolute;top:781px;left:468px;white-space:nowrap" class="ft123">The&#160;method&#160;for&#160;converting&#160;a&#160;simulation&#160;snapshot&#160;to&#160;a</p>
<p style="position:absolute;top:799px;left:456px;white-space:nowrap" class="ft125">graph&#160;formulation&#160;works&#160;well,&#160;and&#160;the&#160;converted&#160;mor-<br/>phologies&#160;reflected&#160;shape&#160;and&#160;positions&#160;of&#160;body&#160;regions<br/>relative&#160;to&#160;each&#160;other&#160;in&#160;space.&#160;The&#160;graph-based&#160;fitness<br/>function&#160;thus&#160;accurately&#160;distinguishes&#160;the&#160;shapes&#160;of&#160;dif-<br/>ferent&#160;morphologies,&#160;but&#160;does&#160;have&#160;difficulty&#160;predicting<br/>which&#160;morphology&#160;will&#160;more&#160;likely&#160;regenerate&#160;into&#160;the<br/>target&#160;morphology.&#160;To&#160;improve&#160;the&#160;effectiveness&#160;of&#160;our<br/>graph-based&#160;fitness&#160;function,&#160;future&#160;work&#160;will&#160;seek&#160;to<br/>automate&#160;the&#160;optimization&#160;of&#160;the&#160;graph&#160;edit&#160;costs&#160;to&#160;not<br/>only&#160;reflect&#160;the&#160;shapes&#160;of&#160;individuals,&#160;but&#160;also&#160;to&#160;favor<br/>morphologies&#160;that&#160;are&#160;more&#160;likely&#160;to&#160;regenerate&#160;into&#160;the<br/>target&#160;individual.&#160;The&#160;optimization&#160;of&#160;graph&#160;edit&#160;costs<br/>needs&#160;to&#160;be&#160;automated&#160;since&#160;tweaking&#160;graph&#160;edit&#160;costs<br/>allows&#160;one&#160;to&#160;achieve&#160;a&#160;more&#160;accurate&#160;graph&#160;comparison<br/>for&#160;some&#160;cases,&#160;but&#160;there&#160;is&#160;no&#160;perfect&#160;parameter&#160;assign-<br/>ment&#160;that&#160;covers&#160;all&#160;cases,&#160;and&#160;so&#160;it&#160;should&#160;be&#160;chosen<br/>depending&#160;on&#160;the&#160;needs&#160;and&#160;design&#160;of&#160;the&#160;experiment.</p>
</div>
</body>
</html>
<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="" xml:lang="">
<head>
<title>e1aa43c837743d72c31dbd52d6a9caac-html.html</title>

<meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
 <br/>
<style type="text/css">
<!--
	p {margin: 0; padding: 0;}	.ft130{font-size:10px;font-family:Times;color:#000000;}
	.ft131{font-size:10px;font-family:Times;color:#000000;}
	.ft132{font-size:10px;font-family:Times;color:#000000;}
	.ft133{font-size:12px;font-family:Times;color:#000000;}
	.ft134{font-size:13px;font-family:Times;color:#000000;}
	.ft135{font-size:11px;font-family:Times;color:#000000;}
	.ft136{font-size:12px;font-family:Times;color:#000000;}
	.ft137{font-size:9px;font-family:Times;color:#000000;}
	.ft138{font-size:9px;font-family:Times;color:#000000;}
	.ft139{font-size:6px;font-family:Times;color:#000000;}
	.ft1310{font-size:6px;font-family:Times;color:#000000;}
	.ft1311{font-size:9px;font-family:Times;color:#000000;}
	.ft1312{font-size:12px;line-height:17px;font-family:Times;color:#000000;}
	.ft1313{font-size:13px;line-height:19px;font-family:Times;color:#000000;}
	.ft1314{font-size:11px;line-height:17px;font-family:Times;color:#000000;}
	.ft1315{font-size:9px;line-height:13px;font-family:Times;color:#000000;}
	.ft1316{font-size:9px;line-height:13px;font-family:Times;color:#000000;}
	.ft1317{font-size:9px;line-height:11px;font-family:Times;color:#000000;}
	.ft1318{font-size:9px;line-height:15px;font-family:Times;color:#000000;}
-->
</style>
</head>
<body bgcolor="#A0A0A0" vlink="blue" link="blue">
<div id="page13-div" style="position:relative;width:892px;height:1191px;">
<p style="position:absolute;top:53px;left:86px;white-space:nowrap" class="ft130">Budnikova&#160;<i>et&#160;al.&#160;BMC&#160;Bioinformatics&#160;</i>2014,&#160;<b>15</b>:178</p>
<p style="position:absolute;top:53px;left:742px;white-space:nowrap" class="ft130">Page&#160;13&#160;of&#160;14</p>
<p style="position:absolute;top:68px;left:86px;white-space:nowrap" class="ft130">http://www.biomedcentral.com/1471-2105/15/178</p>
<p style="position:absolute;top:136px;left:98px;white-space:nowrap" class="ft133">Future&#160;work&#160;will&#160;also&#160;develop&#160;additional&#160;morphological</p>
<p style="position:absolute;top:153px;left:86px;white-space:nowrap" class="ft1312">based&#160;fitness&#160;functions&#160;to&#160;act&#160;side&#160;by&#160;side&#160;with&#160;the&#160;graph<br/>edit&#160;distance&#160;fitness&#160;function.&#160;This&#160;will&#160;allow&#160;researchers<br/>to&#160;choose&#160;from&#160;among&#160;multiple&#160;morphological&#160;fitness<br/>operators,&#160;possibly&#160;combining&#160;the&#160;output&#160;of&#160;several&#160;eval-<br/>uators&#160;instead&#160;of&#160;assigning&#160;a&#160;fitness&#160;value&#160;based&#160;on&#160;one<br/>fitness&#160;function&#160;evaluation.</p>
<p style="position:absolute;top:279px;left:86px;white-space:nowrap" class="ft1312"><b>Methods<br/>Cell-based&#160;modeling&#160;platform<br/></b>Cellsim&#160;is&#160;an&#160;agent-based&#160;modeling&#160;platform&#160;whose&#160;prin-<br/>ciple&#160;agents&#160;are&#160;cellular&#160;in&#160;nature.&#160;These&#160;cells&#160;and&#160;their<br/>behaviors&#160;are&#160;rooted&#160;in&#160;biology&#160;as&#160;the&#160;environment,&#160;inter-<br/>actions,&#160;metabolism,&#160;and&#160;signaling&#160;networks&#160;are&#160;designed<br/>based&#160;upon&#160;biological&#160;primitives.&#160;Cells&#160;are&#160;capable&#160;of<br/>proliferating,&#160;growing,&#160;dividing,&#160;dying,&#160;and&#160;regulating<br/>metabolic&#160;and&#160;genetic&#160;networks&#160;in&#160;response&#160;to&#160;changes<br/>in&#160;their&#160;local&#160;environment,&#160;including&#160;cell&#160;interactions&#160;and<br/>signaling.&#160;As&#160;a&#160;result,&#160;cells&#160;have&#160;emergent&#160;properties&#160;as<br/>they&#160;are&#160;autonomous,&#160;evolvable,&#160;exhibit&#160;inheritance,&#160;and<br/>are&#160;contingent&#160;upon&#160;their&#160;neighbors.&#160;Another&#160;important<br/>feature&#160;of&#160;this&#160;system&#160;is&#160;its&#160;ability&#160;to&#160;be&#160;automated&#160;and<br/>manipulated&#160;using&#160;a&#160;genetic&#160;algorithm&#160;search&#160;engine&#160;<a href="e1aa43c837743d72c31dbd52d6a9caacs.html#14">[17].</a></p>
<p style="position:absolute;top:548px;left:98px;white-space:nowrap" class="ft133">The&#160;versatile&#160;genetic&#160;algorithm&#160;associated&#160;with&#160;Cell-</p>
<p style="position:absolute;top:566px;left:86px;white-space:nowrap" class="ft1312">sim&#160;was&#160;expanded&#160;as&#160;part&#160;of&#160;this&#160;work&#160;and&#160;includes<br/>many&#160;parameters&#160;to&#160;customize&#160;the&#160;common&#160;elements&#160;of<br/>a&#160;genetic&#160;algorithm,&#160;such&#160;as&#160;number&#160;of&#160;crossovers,&#160;muta-<br/>tion&#160;rates,&#160;selection&#160;criteria,&#160;and&#160;population&#160;size.</p>
<p style="position:absolute;top:656px;left:86px;white-space:nowrap" class="ft1312"><b>Graph&#160;edit&#160;distance&#160;algorithm<br/></b>Formally,&#160;to&#160;transform&#160;the&#160;morphology&#160;graph&#160;<i>g</i>1&#160;into<br/>graph&#160;<i>g</i>2,&#160;a&#160;sequence&#160;of&#160;operations&#160;(the&#160;edit&#160;path)&#160;must&#160;be<br/>performed&#160;<a href="e1aa43c837743d72c31dbd52d6a9caacs.html#14">[23,24].&#160;</a>The&#160;edit&#160;distance&#160;between&#160;two&#160;graphs<br/>is&#160;defined&#160;as&#160;the&#160;minimum&#160;cost&#160;edit&#160;path&#160;that&#160;transforms<br/>graph&#160;<i>g</i>1&#160;into&#160;graph&#160;<i>g</i>2&#160;as&#160;represented&#160;in&#160;Equation&#160;<a href="e1aa43c837743d72c31dbd52d6a9caacs.html#13">2.</a></p>
<p style="position:absolute;top:792px;left:108px;white-space:nowrap" class="ft136"><i>d(g</i>1,&#160;<i>g</i>2<i>)</i></p>
<p style="position:absolute;top:788px;left:162px;white-space:nowrap" class="ft133">=&#160;<i>min(e</i></p>
<p style="position:absolute;top:800px;left:211px;white-space:nowrap" class="ft139">1,..<i>ek&#160;)</i>â<i>P(g</i>1,<i>g</i>2<i>)</i></p>
<p style="position:absolute;top:776px;left:292px;white-space:nowrap" class="ft138"><i>k</i></p>
<p style="position:absolute;top:774px;left:285px;white-space:nowrap" class="ft133"></p>
<p style="position:absolute;top:812px;left:286px;white-space:nowrap" class="ft138"><i>i</i></p>
<p style="position:absolute;top:809px;left:290px;white-space:nowrap" class="ft137">=1</p>
<p style="position:absolute;top:792px;left:308px;white-space:nowrap" class="ft136"><i>c(ei)</i>,</p>
<p style="position:absolute;top:792px;left:420px;white-space:nowrap" class="ft133">(2)</p>
<p style="position:absolute;top:836px;left:86px;white-space:nowrap" class="ft1312">where&#160;<i>P(g</i>1,&#160;<i>g</i>2<i>)&#160;</i>is&#160;the&#160;set&#160;of&#160;edit&#160;paths&#160;that&#160;transform&#160;graph<br/><i>g</i>1&#160;into&#160;<i>g</i>2,&#160;<i>c&#160;</i>is&#160;the&#160;edit&#160;cost&#160;function&#160;and&#160;<i>ei&#160;</i>denotes&#160;an&#160;edit<br/>operation.&#160;Generally&#160;speaking,&#160;determining&#160;the&#160;graph&#160;edit<br/>distance&#160;requires&#160;that&#160;we&#160;examine&#160;the&#160;set&#160;of&#160;paths&#160;that<br/>transform&#160;<i>g</i>1&#160;to&#160;<i>g</i>2&#160;and&#160;calculate&#160;the&#160;path&#160;costs&#160;for&#160;each.<br/>This&#160;is&#160;non-trivial&#160;but&#160;can&#160;be&#160;achieved&#160;in&#160;an&#160;optimally<br/>efficient&#160;manner&#160;using&#160;the&#160;A*&#160;best-first&#160;search&#160;algorithm<br/><a href="e1aa43c837743d72c31dbd52d6a9caacs.html#14">[26].</a></p>
<p style="position:absolute;top:979px;left:98px;white-space:nowrap" class="ft133">The&#160;graph&#160;edit&#160;distance&#160;calculation&#160;has&#160;been&#160;adapted</p>
<p style="position:absolute;top:997px;left:86px;white-space:nowrap" class="ft1312">for&#160;comparing&#160;planaria&#160;graph&#160;representations.&#160;A&#160;list&#160;of<br/>edit&#160;operations&#160;and&#160;an&#160;example&#160;of&#160;the&#160;corresponding&#160;costs<br/>used&#160;in&#160;this&#160;paper&#160;are&#160;given&#160;in&#160;Table&#160;<a href="e1aa43c837743d72c31dbd52d6a9caacs.html#9">1.</a></p>
<p style="position:absolute;top:1051px;left:98px;white-space:nowrap" class="ft133">For&#160;simplicity,&#160;we&#160;are&#160;working&#160;with&#160;worms&#160;whose&#160;graph</p>
<p style="position:absolute;top:1069px;left:86px;white-space:nowrap" class="ft1312">representations&#160;are&#160;devoid&#160;of&#160;organs&#160;for&#160;this&#160;analysis,&#160;but<br/>will&#160;include&#160;them&#160;in&#160;later&#160;experiments.</p>
<p style="position:absolute;top:136px;left:470px;white-space:nowrap" class="ft133">The&#160;minimal&#160;path&#160;cost&#160;between&#160;two&#160;graphs&#160;<i>g</i>1&#160;and&#160;<i>g</i>2</p>
<p style="position:absolute;top:153px;left:458px;white-space:nowrap" class="ft1312">is&#160;found&#160;using&#160;the&#160;A*&#160;search&#160;algorithm.&#160;The&#160;possible&#160;edit<br/>paths&#160;can&#160;be&#160;viewed&#160;as&#160;forming&#160;a&#160;tree,&#160;where&#160;the&#160;edges<br/>of&#160;the&#160;tree&#160;are&#160;individual&#160;edit&#160;operations&#160;and&#160;the&#160;nodes<br/>are&#160;graphs.&#160;The&#160;inner&#160;nodes&#160;of&#160;the&#160;tree&#160;correspond&#160;to<br/>partially&#160;edited&#160;graphs,&#160;and&#160;the&#160;leaf&#160;nodes&#160;represent&#160;com-<br/>plete&#160;edit&#160;paths,&#160;all&#160;of&#160;which&#160;terminate&#160;at&#160;the&#160;target&#160;graph.<br/>The&#160;search&#160;for&#160;the&#160;minimal&#160;edit&#160;path&#160;starts&#160;with&#160;one&#160;of<br/>the&#160;graphs,&#160;say&#160;<i>g</i>1,&#160;as&#160;the&#160;root&#160;of&#160;the&#160;tree,&#160;and&#160;defines&#160;the<br/>possible&#160;branches&#160;from&#160;this&#160;node&#160;to&#160;be&#160;all&#160;possible&#160;single<br/>edit&#160;operations&#160;that&#160;could&#160;be&#160;applied&#160;to&#160;the&#160;original&#160;graph<br/><i>g</i>1.&#160;At&#160;each&#160;step&#160;of&#160;the&#160;A*&#160;search,&#160;the&#160;algorithm&#160;expands<br/>(explores)&#160;the&#160;branch&#160;of&#160;the&#160;tree&#160;that&#160;leads&#160;to&#160;a&#160;node&#160;on&#160;the<br/>search&#160;frontier&#160;that&#160;has&#160;the&#160;minimum&#160;estimated&#160;total&#160;path<br/>cost.&#160;The&#160;minimum&#160;estimated&#160;total&#160;path&#160;cost&#160;is&#160;derived<br/>from&#160;the&#160;sum&#160;of&#160;the&#160;cost&#160;of&#160;the&#160;edits&#160;required&#160;to&#160;reach&#160;the<br/>node&#160;being&#160;expanded&#160;from&#160;the&#160;root,&#160;plus&#160;an&#160;estimate&#160;of<br/>the&#160;total&#160;cost&#160;of&#160;the&#160;edits&#160;required&#160;to&#160;reach&#160;the&#160;goal&#160;state<br/>(graph&#160;<i>g</i>2)&#160;from&#160;the&#160;node&#160;being&#160;expanded.&#160;The&#160;algorithm<br/>terminates&#160;the&#160;first&#160;time&#160;it&#160;expands&#160;the&#160;goal&#160;state,&#160;as&#160;the<br/>path&#160;that&#160;it&#160;finds&#160;at&#160;this&#160;point&#160;is&#160;guaranteed&#160;to&#160;be&#160;optimal.</p>
<p style="position:absolute;top:536px;left:458px;white-space:nowrap" class="ft1316"><b>Competing&#160;interests<br/></b>Tim&#160;Andersen&#160;and&#160;Jeff&#160;Habig&#160;are&#160;former&#160;employees&#160;of&#160;Crowley&#160;Davis&#160;Research<br/>who&#160;developed&#160;the&#160;Cellsim&#160;platform,&#160;but&#160;are&#160;no&#160;longer&#160;associated&#160;with&#160;the<br/>company&#160;in&#160;any&#160;way&#160;and&#160;declare&#160;no&#160;financial&#160;interest&#160;or&#160;conflict&#160;of&#160;interest&#160;for<br/>themselves&#160;or&#160;any&#160;other&#160;authors.&#160;Crowley&#160;Davis&#160;Research&#160;is&#160;not&#160;directly<br/>involved&#160;in&#160;this&#160;project.</p>
<p style="position:absolute;top:630px;left:458px;white-space:nowrap" class="ft1316"><b>Authorsâ&#160;contributions<br/></b>Component&#160;gathering&#160;design,&#160;development,&#160;and&#160;analysis&#160;(MB,&#160;NC,&#160;and&#160;TA).<br/>Modeling,&#160;network&#160;design,&#160;and&#160;genetic&#160;algorithm&#160;searches&#160;(JH).&#160;Experimental<br/>design&#160;and&#160;interpretation&#160;(JH,&#160;TA,&#160;and&#160;ML).&#160;Graph-edit&#160;distance&#160;evaluation&#160;and<br/>live&#160;planarian&#160;experiments&#160;(DL).&#160;Manuscript&#160;preparation&#160;(MB,&#160;JH,&#160;and&#160;TA).&#160;All<br/>authors&#160;read&#160;and&#160;approved&#160;the&#160;final&#160;manuscript.</p>
<p style="position:absolute;top:724px;left:458px;white-space:nowrap" class="ft1316"><b>Acknowledgements<br/></b>We&#160;would&#160;like&#160;to&#160;thank&#160;past&#160;and&#160;present&#160;employees&#160;of&#160;Crowley&#160;Davis<br/>Research&#160;for&#160;their&#160;hard&#160;work&#160;on&#160;the&#160;Cellsim&#160;platform&#160;and&#160;for&#160;making&#160;it<br/>available&#160;for&#160;these&#160;studies.&#160;This&#160;work&#160;was&#160;made&#160;available&#160;through&#160;an&#160;NSF-CDI<br/>grant(EF-1124665&#160;and&#160;EF-1124651).</p>
<p style="position:absolute;top:805px;left:458px;white-space:nowrap" class="ft1316"><b>Author&#160;details<br/></b>1Department&#160;of&#160;Computer&#160;Science,&#160;Boise&#160;State&#160;University,&#160;1910&#160;University<br/>Drive,&#160;Boise,&#160;ID&#160;83725,&#160;USA.&#160;2Department&#160;of&#160;Biology&#160;and&#160;Tufts&#160;Center&#160;for<br/>Regeneration&#160;and&#160;Developmental&#160;Biology,&#160;Tufts&#160;University,&#160;200&#160;Boston&#160;Ave,<br/>Medford,&#160;MA&#160;02155,&#160;USA.</p>
<p style="position:absolute;top:886px;left:458px;white-space:nowrap" class="ft1316">Received:&#160;31&#160;August&#160;2013&#160;Accepted:&#160;30&#160;April&#160;2014<br/>Published:&#160;10&#160;June&#160;2014</p>
<p style="position:absolute;top:927px;left:458px;white-space:nowrap" class="ft1315"><b>References<br/></b>1.</p>
<p style="position:absolute;top:940px;left:480px;white-space:nowrap" class="ft1315">Pallas&#160;PS:&#160;<b>Spicilegia&#160;zoologica&#160;quibus&#160;novae&#160;imprimis&#160;et&#160;obscurae<br/>animalium&#160;species&#160;iconibus,&#160;descriptionibus&#160;atque&#160;commentariis<br/>illustrantur.&#160;</b><i>Berolini,&#160;Prostant,&#160;Apud&#160;Gottl&#160;</i>1774,&#160;<b>8</b>(4):23.</p>
<p style="position:absolute;top:981px;left:458px;white-space:nowrap" class="ft137">2.</p>
<p style="position:absolute;top:981px;left:480px;white-space:nowrap" class="ft1316">Aboobaker&#160;AA:&#160;<b>Planarian&#160;stem&#160;cells:&#160;a&#160;simple&#160;paradigm&#160;for<br/>regeneration.&#160;</b><i>Trends&#160;Cell&#160;Biol&#160;</i>2011,&#160;<b>21</b>(5):304â311.&#160;<a href="http://www.sciencedirect.com/science/article/pii/S0962892411000158">[http://www.<br/>sciencedirect.com/science/article/pii/S0962892411000158]</a></p>
<p style="position:absolute;top:1021px;left:458px;white-space:nowrap" class="ft137">3.</p>
<p style="position:absolute;top:1021px;left:480px;white-space:nowrap" class="ft1316">Sarnat&#160;HB,&#160;Netsky&#160;MG:&#160;<b>The&#160;brain&#160;of&#160;the&#160;planarian&#160;as&#160;the&#160;ancestor&#160;of<br/>the&#160;human&#160;brain.&#160;</b><i>Can&#160;J&#160;Neurol&#160;Sci&#160;</i>1985,&#160;<b>12</b>(4):296â302.&#160;<a href="http://view.ncbi.nlm.nih.gov/pubmed/4084864">[http://view.ncbi.<br/>nlm.nih.gov/pubmed/4084864]</a></p>
<p style="position:absolute;top:1062px;left:458px;white-space:nowrap" class="ft137">4.</p>
<p style="position:absolute;top:1062px;left:480px;white-space:nowrap" class="ft1316">Reddien&#160;PW,&#160;Alvarado&#160;AS:&#160;<b>Fundamentals&#160;of&#160;planarian&#160;regeneration.<br/></b><i>Annu&#160;Rev&#160;Cell&#160;Dev&#160;Biol&#160;</i>2004,&#160;<b>20:</b>725â757.&#160;<a href="http://www.annualreviews.org/doi/abs/10.1146/annurev.cellbio.20.010403.095114">[http://www.annualreviews.org/<br/>doi/abs/10.1146/annurev.cellbio.20.010403.095114]&#160;</a>[PMID15473858]</p>
</div>
</body>
</html>
<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="" xml:lang="">
<head>
<title>e1aa43c837743d72c31dbd52d6a9caac-html.html</title>

<meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
 <br/>
<style type="text/css">
<!--
	p {margin: 0; padding: 0;}	.ft140{font-size:10px;font-family:Times;color:#000000;}
	.ft141{font-size:10px;font-family:Times;color:#000000;}
	.ft142{font-size:10px;font-family:Times;color:#000000;}
	.ft143{font-size:9px;font-family:Times;color:#000000;}
	.ft144{font-size:9px;font-family:Times;color:#000000;}
	.ft145{font-size:9px;font-family:Times;color:#000000;}
	.ft146{font-size:8px;font-family:Times;color:#000000;}
	.ft147{font-size:8px;font-family:Times;color:#000000;}
	.ft148{font-size:8px;font-family:Times;color:#000000;}
	.ft149{font-size:11px;font-family:Times;color:#231f20;}
	.ft1410{font-size:8px;font-family:Times;color:#231f20;}
	.ft1411{font-size:8px;font-family:Times;color:#557ab1;}
	.ft1412{font-size:7px;font-family:Times;color:#231f20;}
	.ft1413{font-size:9px;line-height:13px;font-family:Times;color:#000000;}
	.ft1414{font-size:9px;line-height:13px;font-family:Times;color:#000000;}
	.ft1415{font-size:9px;line-height:13px;font-family:Times;color:#000000;}
	.ft1416{font-size:8px;line-height:12px;font-family:Times;color:#000000;}
	.ft1417{font-size:8px;line-height:12px;font-family:Times;color:#000000;}
	.ft1418{font-size:11px;line-height:16px;font-family:Times;color:#231f20;}
	.ft1419{font-size:7px;line-height:10px;font-family:Times;color:#231f20;}
-->
</style>
</head>
<body bgcolor="#A0A0A0" vlink="blue" link="blue">
<div id="page14-div" style="position:relative;width:892px;height:1191px;">
<p style="position:absolute;top:53px;left:84px;white-space:nowrap" class="ft140">Budnikova&#160;<i>et&#160;al.&#160;BMC&#160;Bioinformatics&#160;</i>2014,&#160;<b>15</b>:178</p>
<p style="position:absolute;top:53px;left:740px;white-space:nowrap" class="ft140">Page&#160;14&#160;of&#160;14</p>
<p style="position:absolute;top:68px;left:84px;white-space:nowrap" class="ft140">http://www.biomedcentral.com/1471-2105/15/178</p>
<p style="position:absolute;top:138px;left:84px;white-space:nowrap" class="ft143">5.</p>
<p style="position:absolute;top:138px;left:106px;white-space:nowrap" class="ft1413">Morgan&#160;TH:&#160;<b>Experimental&#160;studies&#160;of&#160;the&#160;regeneration&#160;of&#160;Planaria<br/>Maculata.&#160;</b><i>Arch&#160;Entwickelungsmech&#160;Org&#160;</i>1898,&#160;<b>7:</b>364â397.</p>
<p style="position:absolute;top:165px;left:84px;white-space:nowrap" class="ft143">6.</p>
<p style="position:absolute;top:165px;left:106px;white-space:nowrap" class="ft1414">Adell&#160;T,&#160;CebriÃ &#160;F,&#160;SalÃ³&#160;E:&#160;<b>Gradients&#160;in&#160;planarian&#160;regeneration&#160;and<br/>homeostasis.&#160;</b><i>Cold&#160;Spring&#160;Harb&#160;Perspect&#160;Biol&#160;</i>2010,&#160;<b>2:</b>.&#160;<a href="http://dx.doi.org/10.1101/cshperspect.a000505">[http://dx.doi.org/<br/>10.1101/cshperspect.a000505]</a></p>
<p style="position:absolute;top:206px;left:84px;white-space:nowrap" class="ft143">7.</p>
<p style="position:absolute;top:206px;left:106px;white-space:nowrap" class="ft1414">Oviedo&#160;NJ,&#160;Morokuma&#160;J,&#160;Walentek&#160;P,&#160;Kema&#160;IP,&#160;Gu&#160;MB,&#160;Ahn&#160;JM,&#160;Hwang&#160;JS,<br/>Gojobori&#160;T,&#160;Levin&#160;M:&#160;<b>Long-range&#160;neural&#160;and&#160;gap&#160;junction<br/>protein-mediated&#160;cues&#160;control&#160;polarity&#160;during&#160;planarian<br/>regeneration.&#160;</b><i>Dev&#160;Biol&#160;</i>2010,&#160;<b>339:</b>188â199.&#160;<a href="http://www.sciencedirect.com/science/article/pii/S001216060901402X">[http://www.sciencedirect.<br/>com/science/article/pii/S001216060901402X]</a></p>
<p style="position:absolute;top:273px;left:84px;white-space:nowrap" class="ft143">8.</p>
<p style="position:absolute;top:273px;left:106px;white-space:nowrap" class="ft1414">Beane&#160;WS,&#160;Morokuma&#160;J,&#160;Adams&#160;DS,&#160;Levin&#160;M:&#160;<b>A&#160;chemical&#160;genetics<br/>approach&#160;reveals&#160;H,K-ATPase-mediated&#160;membrane&#160;voltage&#160;is<br/>required&#160;for&#160;planarian&#160;head&#160;regeneration.&#160;</b><i>Chem&#160;Biol&#160;</i>2011,&#160;<b>18:</b>77â89.<br/><a href="http://www.sciencedirect.com/science/article/pii/S1074552110004473">[http://www.sciencedirect.com/science/article/pii/S1074552110004473]</a></p>
<p style="position:absolute;top:327px;left:84px;white-space:nowrap" class="ft143">9.</p>
<p style="position:absolute;top:327px;left:106px;white-space:nowrap" class="ft1414">Petersen&#160;CP,&#160;Reddien&#160;PW:&#160;<b>Smed-</b><i>Î²</i><b>catenin-1&#160;is&#160;required&#160;for<br/>anteroposterior&#160;blastema&#160;polarity&#160;in&#160;planarian&#160;regeneration.&#160;</b><i>Science<br/></i>2008,&#160;<b>319</b>(5861):327â330.&#160;<a href="http://www.sciencemag.org/content/319/5861/327.abstract">[http://www.sciencemag.org/content/319/<br/>5861/327.abstract]</a></p>
<p style="position:absolute;top:381px;left:84px;white-space:nowrap" class="ft143">10.&#160;Rink&#160;JC,&#160;Gurley&#160;KA,&#160;Elliott&#160;SA,&#160;Alvarado&#160;AS:&#160;<b>Planarian&#160;Hh&#160;signaling</b></p>
<p style="position:absolute;top:395px;left:106px;white-space:nowrap" class="ft1414"><b>regulates&#160;regeneration&#160;polarity&#160;and&#160;links&#160;Hh&#160;pathway&#160;evolution&#160;to<br/>cilia.&#160;</b><i>Science&#160;</i>2009,&#160;<b>326</b>(5958):1406â1410.&#160;<a href="http://www.sciencemag.org/content/326/5958/1406.abstract">[http://www.sciencemag.org/<br/>content/326/5958/1406.abstract]</a></p>
<p style="position:absolute;top:435px;left:84px;white-space:nowrap" class="ft143">11.&#160;Petersen&#160;CP,&#160;Reddien&#160;PW:&#160;<b>Polarized&#160;notum&#160;activation&#160;at&#160;wounds</b></p>
<p style="position:absolute;top:449px;left:106px;white-space:nowrap" class="ft1414"><b>inhibits&#160;wnt&#160;function&#160;to&#160;promote&#160;planarian&#160;head&#160;regeneration.<br/></b><i>Science&#160;</i>2011,&#160;<b>332</b>(6031):852â855.&#160;<a href="http://www.sciencemag.org/content/332/6031/852.abstract">[http://www.sciencemag.org/content/<br/>332/6031/852.abstract]</a></p>
<p style="position:absolute;top:489px;left:84px;white-space:nowrap" class="ft143">12.&#160;Umesono&#160;Y,&#160;Tasaki&#160;J,&#160;Nishimura&#160;Y,&#160;Hrouda&#160;M,&#160;Kawaguchi&#160;E,&#160;Yazawa&#160;S,</p>
<p style="position:absolute;top:503px;left:106px;white-space:nowrap" class="ft1415">Nishimura&#160;O,&#160;Hosoda&#160;K,&#160;Inoue&#160;T,&#160;Agata&#160;K:&#160;<b>The&#160;molecular&#160;logic&#160;for<br/>planarian&#160;regeneration&#160;along&#160;the&#160;anterior-posterior&#160;axis.&#160;</b><i>Nature<br/></i>2013.&#160;<a href="http://dx.doi.org/10.1038/nature12359">[http://dx.doi.org/10.1038/nature12359]</a></p>
<p style="position:absolute;top:543px;left:84px;white-space:nowrap" class="ft143">13.&#160;Sikes&#160;JM,&#160;Newmark&#160;PA:&#160;<b>Restoration&#160;of&#160;anterior&#160;regeneration&#160;in&#160;a</b></p>
<p style="position:absolute;top:557px;left:106px;white-space:nowrap" class="ft1414"><b>planarian&#160;with&#160;limited&#160;regenerative&#160;ability.&#160;</b><i>Nature&#160;</i>2013.<br/><a href="http://dx.doi.org/10.1038/nature12403">[http://dx.doi.org/10.1038/nature12403]</a></p>
<p style="position:absolute;top:584px;left:84px;white-space:nowrap" class="ft143">14.&#160;Lobo&#160;D,&#160;Beane&#160;WS,&#160;Levin&#160;M:&#160;<b>Modeling&#160;planarian&#160;regeneration:&#160;a</b></p>
<p style="position:absolute;top:597px;left:106px;white-space:nowrap" class="ft1414"><b>primer&#160;for&#160;reverse-engineering&#160;the&#160;worm.&#160;</b><i>PLoS&#160;Comput&#160;Biol&#160;</i>2012,&#160;<b>8</b>(4).<br/><a href="http://dx.doi.org/10.1371/journal.pcbi.1002481">[http://dx.doi.org/10.1371/journal.pcbi.1002481]</a></p>
<p style="position:absolute;top:624px;left:84px;white-space:nowrap" class="ft143">15.&#160;Lobo&#160;D,&#160;Malone&#160;TJ,&#160;Levin&#160;M:&#160;<b>Planform:&#160;an&#160;application&#160;and&#160;database&#160;of</b></p>
<p style="position:absolute;top:638px;left:106px;white-space:nowrap" class="ft1415"><b>graph-encoded&#160;planarian&#160;regenerative&#160;experiments.&#160;</b><i>Bioinformatics<br/></i>2013,&#160;<b>29</b>(8):1098â1100.</p>
<p style="position:absolute;top:665px;left:84px;white-space:nowrap" class="ft143">16.&#160;Lobo&#160;D,&#160;Malone&#160;TJ,&#160;Levin&#160;M:&#160;<b>Towards&#160;a&#160;bioinformatics&#160;of&#160;patterning:&#160;a</b></p>
<p style="position:absolute;top:678px;left:106px;white-space:nowrap" class="ft1413"><b>computational&#160;approach&#160;to&#160;understanding&#160;regulative<br/>morphogensis.&#160;</b><i>Biol&#160;Open&#160;</i>2013,&#160;<b>2</b>(2):156â169.</p>
<p style="position:absolute;top:705px;left:84px;white-space:nowrap" class="ft143">17.&#160;Andersen&#160;T,&#160;Newman&#160;R,&#160;Otter&#160;T:&#160;<b>Shape&#160;homeostasis&#160;in&#160;virtual</b></p>
<p style="position:absolute;top:719px;left:106px;white-space:nowrap" class="ft144"><b>embryos.&#160;</b><i>Artif&#160;Life&#160;</i>2009,&#160;<b>15</b>(2):161â183.</p>
<p style="position:absolute;top:732px;left:84px;white-space:nowrap" class="ft143">18.&#160;Grandstrand&#160;O:&#160;<i>Agent&#160;Based&#160;Evolutionary&#160;Search</i>.&#160;Berlin,&#160;Heidelberg:</p>
<p style="position:absolute;top:746px;left:106px;white-space:nowrap" class="ft1414">Springer-Verlag;&#160;2010.&#160;chap.&#160;Agent&#160;Based&#160;Evolutionary&#160;Approach:&#160;An<br/>Introduction.</p>
<p style="position:absolute;top:773px;left:84px;white-space:nowrap" class="ft143">19.&#160;Lobo&#160;D,&#160;Vico&#160;V,&#160;Dassow&#160;J:&#160;<b>Graph&#160;grammars&#160;with&#160;string-regulated</b></p>
<p style="position:absolute;top:786px;left:106px;white-space:nowrap" class="ft144"><b>rewriting.&#160;</b><i>Theor&#160;Comput&#160;Sci&#160;</i>2011,&#160;<b>412:</b>6101â6111.</p>
<p style="position:absolute;top:800px;left:84px;white-space:nowrap" class="ft143">20.&#160;Bandelt&#160;H,&#160;Chepoi&#160;V:&#160;<b>Metric&#160;graph&#160;tehory&#160;and&#160;geometry:&#160;a&#160;survey.</b></p>
<p style="position:absolute;top:813px;left:106px;white-space:nowrap" class="ft145"><i>Contemp&#160;Math&#160;</i>2008,&#160;<b>453:</b>49â86.</p>
<p style="position:absolute;top:827px;left:84px;white-space:nowrap" class="ft143">21.&#160;Mason&#160;O,&#160;Verwoerd&#160;M:&#160;<b>Graph&#160;theory&#160;and&#160;networks&#160;in&#160;biology.&#160;</b><i>IET&#160;Syst</i></p>
<p style="position:absolute;top:840px;left:106px;white-space:nowrap" class="ft145"><i>Biol&#160;</i>2007,&#160;<b>1:</b>89â119.</p>
<p style="position:absolute;top:854px;left:84px;white-space:nowrap" class="ft143">22.&#160;Conte&#160;D,&#160;Foggia&#160;P,&#160;Sansone&#160;C:&#160;<b>Thirty&#160;years&#160;of&#160;graph&#160;matching&#160;in</b></p>
<p style="position:absolute;top:867px;left:106px;white-space:nowrap" class="ft144"><b>pattern&#160;recognition.&#160;</b><i>Int&#160;J&#160;Pattern&#160;Recogn&#160;Artif&#160;Intell&#160;</i>2004,&#160;<b>18:</b>265â298.</p>
<p style="position:absolute;top:881px;left:84px;white-space:nowrap" class="ft143">23.&#160;Neuhaus&#160;M,&#160;Bunke&#160;H:&#160;<i>Bridging&#160;the&#160;Gap&#160;Between&#160;Graph&#160;Edit&#160;Distance&#160;and</i></p>
<p style="position:absolute;top:894px;left:106px;white-space:nowrap" class="ft145"><i>Kernel&#160;Machines</i>.&#160;Singapore:&#160;World&#160;Scientific;&#160;2007.</p>
<p style="position:absolute;top:908px;left:84px;white-space:nowrap" class="ft143">24.&#160;Sanfeliu&#160;A,&#160;Fu&#160;K:&#160;<b>A&#160;distance&#160;measure&#160;between&#160;attributed&#160;relational</b></p>
<p style="position:absolute;top:921px;left:106px;white-space:nowrap" class="ft1414"><b>graphs&#160;for&#160;pattern&#160;recognition.&#160;</b><i>IEEE&#160;Trans&#160;Syst&#160;Man&#160;Cybern&#160;</i>1983,<br/><b>13:</b>353â362.</p>
<p style="position:absolute;top:948px;left:84px;white-space:nowrap" class="ft143">25.&#160;Rosenfeld&#160;A,&#160;Pfaltz&#160;J:&#160;<b>Sequential&#160;operations&#160;in&#160;digital&#160;image</b></p>
<p style="position:absolute;top:962px;left:106px;white-space:nowrap" class="ft144"><b>processing.&#160;</b><i>J&#160;ACM&#160;</i>1966,&#160;<b>13:</b>471â494.</p>
<p style="position:absolute;top:975px;left:84px;white-space:nowrap" class="ft143">26.&#160;Hart&#160;P:&#160;<b>A&#160;formal&#160;basis&#160;for&#160;the&#160;heuristic&#160;determination&#160;of&#160;minimum</b></p>
<p style="position:absolute;top:989px;left:106px;white-space:nowrap" class="ft144"><b>cost&#160;paths.&#160;</b><i>IEEE&#160;Trans&#160;Syst&#160;Man&#160;Cybern&#160;</i>1968,&#160;<b>4:</b>100â107.</p>
<p style="position:absolute;top:1018px;left:92px;white-space:nowrap" class="ft1417">doi:10.1186/1471-2105-15-178<br/><b>Cite&#160;this&#160;article&#160;as:&#160;</b>Budnikova&#160;<i>et&#160;al.</i>:&#160;Design&#160;of&#160;a&#160;flexible&#160;component<br/>gathering&#160;algorithm&#160;for&#160;converting&#160;cell-based&#160;models&#160;to&#160;graph<br/>representations&#160;for&#160;use&#160;in&#160;evolutionary&#160;search.&#160;<i>BMC&#160;Bioinformatics<br/></i>2014&#160;<b>15</b>:178.</p>
<p style="position:absolute;top:893px;left:482px;white-space:nowrap" class="ft1418"><b>Submit your next manuscript to BioMed Central<br/>and take full advantage of:&#160;</b></p>
<p style="position:absolute;top:943px;left:481px;white-space:nowrap" class="ft1410"><b>â¢&#160;Convenient online submission</b></p>
<p style="position:absolute;top:962px;left:481px;white-space:nowrap" class="ft1410"><b>â¢&#160;Thorough peer review</b></p>
<p style="position:absolute;top:980px;left:481px;white-space:nowrap" class="ft1410"><b>â¢&#160;No space constraints or color ï¬gure charges</b></p>
<p style="position:absolute;top:999px;left:481px;white-space:nowrap" class="ft1410"><b>â¢&#160;Immediate publication on acceptance</b></p>
<p style="position:absolute;top:1018px;left:481px;white-space:nowrap" class="ft1410"><b>â¢&#160;Inclusion in PubMed, CAS, Scopus and Google Scholar</b></p>
<p style="position:absolute;top:1036px;left:481px;white-space:nowrap" class="ft1410"><b>â¢&#160;Research which is freely available for redistribution</b></p>
<p style="position:absolute;top:1069px;left:482px;white-space:nowrap" class="ft1419">Submit your manuscript at&#160;<br/>www.biomedcentral.com/submit</p>
</div>
</body>
</html>
